<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>ColabFit Dataset Page</title>

  <script type="application/ld+json">
    {
      "@context": "https://schema.org",
      "@type": "Project",
      "id": "https://materials.colabfit.org",
      "logo": "https://colabfit.org/images/colabfit-logo-600.png",
      "url": "https://materials.colabfit.org",
      "name": "ColabFit Exchange",
    "address": [{
    "@type": "PostalAddress",
    "streetAddress": "736 Broadway",
    "addressLocality": "New York",
    "addressRegion": "NY",
    "postalCode": "10003",
    "addressCountry": "US"
  }],
    "keywords": [
      "DDIP",
      "data-driven interatomic potentials",
      "machine learning",
      "materials science",
      "dataset database",
      "data repository",
      "data-driven materials science",
      "data-driven chemistry",
      "data-driven materials discovery",
      "data-driven design",
      "data-driven research",
      "data-driven innovation",
      "data-driven modeling",
      "data-driven simulation",
      "MLIP database",
      "MLIP repository",
      "ab initio",
      "first-principles",
      "density functional theory",
      "molecular dynamics simulations",
      "molecular dynamics trajectories",
      "DFT",
      "machine learning interatomic potentials",
      "MLIP",
      "deep learning interatomic potentials",
      "machine learning potentials"
    ],
    "description": "The ColabFit Exchange is an online resource for the discovery, exploration and submission of datasets for data-driven interatomic potential (DDIP) development for materials science and chemistry applications. ColabFit's goal is to increase the Findability, Accessibility, Interoperability, and Reusability (FAIR) of DDIP data by providing convenient access to well-curated and standardized first-principles and experimental datasets. Content on the ColabFit Exchange is open source and freely available."
  }
  </script>

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <!-- Bootstrap 5 CDN -->
  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet">
</head>
<body class="container my-5">

  <h1 class="mb-4">
    Currently 414 datasets available on <a href="https://colabfit.org" target="_blank">colabfit.org</a>:
  </h1>


</body>
</html>

  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_q4h7q8q0fnve_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Ag
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Ag from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Ag__Andolina-Saidi__DS_q4h7q8q0fnve_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Ag
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Ag
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,795
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 104,827
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_8y775we7um7w_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Al
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Al from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Al__Andolina-Saidi__DS_8y775we7um7w_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Al
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Al
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,572
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 88,139
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_iie3c31ar46x_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Au
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Au from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Au__Andolina-Saidi__DS_iie3c31ar46x_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Au
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Au
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,601
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 89,366
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_jt0lax9yd15r_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Co
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Co from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Co__Andolina-Saidi__DS_jt0lax9yd15r_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Co
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Co
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,356
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 67,320
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_dc3o40aou2le_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Cu
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Cu from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Cu__Andolina-Saidi__DS_dc3o40aou2le_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Cu
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Cu
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,366
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 96,568
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_90fnsmavv1am_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Ge
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Ge from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Ge__Andolina-Saidi__DS_90fnsmavv1am_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Ge
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Ge
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,895
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 195,270
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_gc1y80tpyylb_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-I
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of I from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-I__Andolina-Saidi__DS_gc1y80tpyylb_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-I
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : I
    <br/>
    <b>
     Number of Configurations
    </b>
    : 4,532
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 115,902
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_omnl1yy49sdh_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Kr
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Kr from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Kr__Andolina-Saidi__DS_omnl1yy49sdh_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Kr
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Kr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,975
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 97,920
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_wyo91w20wlgm_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Li
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Li from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Li__Andolina-Saidi__DS_wyo91w20wlgm_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Li
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Li
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,536
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 93,724
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_mevqyitwxukc_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Mg
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Mg from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Mg__Andolina-Saidi__DS_mevqyitwxukc_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Mg
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Mg
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,004
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 58,567
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_bjkftn0ug9r3_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Mo
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Mo from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Mo__Andolina-Saidi__DS_bjkftn0ug9r3_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Mo
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Mo
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,718
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 66,612
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_zbxayq0diq6l_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Nb
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Nb from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Nb__Andolina-Saidi__DS_zbxayq0diq6l_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Nb
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Nb
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,246
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 56,191
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_nue14kckbkdh_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Ni
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Ni from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Ni__Andolina-Saidi__DS_nue14kckbkdh_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Ni
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Ni
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,817
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 75,534
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_098x6q7kbeat_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Os
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Os from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Os__Andolina-Saidi__DS_098x6q7kbeat_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Os
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Os
    <br/>
    <b>
     Number of Configurations
    </b>
    : 4,779
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 117,968
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_k065jfggbq43_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Pb
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Pb from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Pb__Andolina-Saidi__DS_k065jfggbq43_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Pb
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Pb
    <br/>
    <b>
     Number of Configurations
    </b>
    : 5,350
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 119,252
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_g0sb0h7usqw7_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Pd
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Pd from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Pd__Andolina-Saidi__DS_g0sb0h7usqw7_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Pd
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Pd
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,478
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 140,196
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_0zgz34a90a6i_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Pt
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Pt from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Pt__Andolina-Saidi__DS_0zgz34a90a6i_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Pt
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Pt
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,609
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 62,152
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_gzbvdicu231p_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Re
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Re from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Re__Andolina-Saidi__DS_gzbvdicu231p_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Re
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Re
    <br/>
    <b>
     Number of Configurations
    </b>
    : 5,029
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 101,248
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_z90lfjg88fzo_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Sb
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Sb from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Sb__Andolina-Saidi__DS_z90lfjg88fzo_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Sb
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Sb
    <br/>
    <b>
     Number of Configurations
    </b>
    : 5,529
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 122,289
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_o3itca7mk80r_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Sr
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Sr from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Sr__Andolina-Saidi__DS_o3itca7mk80r_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Sr
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Sr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,155
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 49,426
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_ofpcyxez6xsc_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Ti
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Ti from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Ti__Andolina-Saidi__DS_ofpcyxez6xsc_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Ti
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Ti
    <br/>
    <b>
     Number of Configurations
    </b>
    : 5,665
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 153,659
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_xsad0btc0bsn_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Zn
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Zn from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Zn__Andolina-Saidi__DS_xsad0btc0bsn_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Zn
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Zn
    <br/>
    <b>
     Number of Configurations
    </b>
    : 4,052
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 107,039
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_h9w8v7289utq_0" target="_blank">
     23-Single-Element-DNPs_RSCDD_2023-Zr
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of Zr from Andolina &amp; Saidi, 2023. One of 23 minimalist, curated sets of DFT-calculated properties for individual elements for the purpose of providing input to machine learning of deep neural network potentials (DNPs). Each element set contains on average ~4000 structures with 27 atoms per structure. Configuration metadata includes Materials Project ID where available, as well as temperatures at which MD trajectories were calculated.These temperatures correspond to the melting temperature (MT) and 0.25*MT for elements with MT &lt; 2000K, and MT, 0.6*MT and 0.25*MT for elements with MT &gt; 2000K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Zr__Andolina-Saidi__DS_h9w8v7289utq_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_RSCDD_2023-Zr
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 4,730
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 81,165
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_whwcq2rzt2of_0" target="_blank">
     23-Single-Element-DNPs_all_trajectories
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The full trajectories from the VASP runs used to generate the 23-Single-Element-DNPs training sets. Configuration sets are available for each element.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    23-Single-Element-DNPs_all_trajectories__Andolina-Saidi__DS_whwcq2rzt2of_0
    <br/>
    <b>
     Name:
    </b>
    23-Single-Element-DNPs_all_trajectories
    <br/>
    <b>
     Authors:
    </b>
    Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, Au, Co, Cu, Ge, I, Kr, Li, Mg, Mo, Nb, Ni, Os, Pb, Pd, Pt, Re, Sb, Sr, Ti, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 108,644
    <br/>
    <b>
     Number of Elements
    </b>
    : 23
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,352,424
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/saidigroup/23-Single-Element-DNPs" target="_blank">
     https://github.com/saidigroup/23-Single-Element-DNPs
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D3DD00046J" target="_blank">
     https://doi.org/10.1039/D3DD00046J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_gaq0fec8i6ik_0" target="_blank">
     3BPA_isolated_atoms
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Reference C, H, O, and N atoms from 3BPA, used to showcase the performance of linear atomic cluster expansion (ACE) force fields in a machine learning model to predict the potential energy surfaces of organic molecules.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    3BPA_isolated_atoms__Kovács-Oord-Kucera-Allen-Cole-Ortner-Csányi__DS_gaq0fec8i6ik_0
    <br/>
    <b>
     Name:
    </b>
    3BPA_isolated_atoms
    <br/>
    <b>
     Authors:
    </b>
    Dávid Péter Kovács, Cas van der Oord, Jiri Kucera, Alice E. A. Allen, Daniel J. Cole, Christoph Ortner, Gábor Csányi
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 4
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 4
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.1c00647" target="_blank">
     https://doi.org/10.1021/acs.jctc.1c00647
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.1c00647" target="_blank">
     https://doi.org/10.1021/acs.jctc.1c00647
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_xaasqmrdv28s_0" target="_blank">
     3BPA_test_1200K
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test configurations with MD simulations performed at 1200K from 3BPA, used to showcase the performance of linear atomic cluster expansion (ACE) force fields in a machine learning model to predict the potential energy surfaces of organic molecules.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    3BPA_test_1200K__Kovács-Oord-Kucera-Allen-Cole-Ortner-Csányi__DS_xaasqmrdv28s_0
    <br/>
    <b>
     Name:
    </b>
    3BPA_test_1200K
    <br/>
    <b>
     Authors:
    </b>
    Dávid Péter Kovács, Cas van der Oord, Jiri Kucera, Alice E. A. Allen, Daniel J. Cole, Christoph Ortner, Gábor Csányi
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,139
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 57,753
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.1c00647" target="_blank">
     https://doi.org/10.1021/acs.jctc.1c00647
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.1c00647" target="_blank">
     https://doi.org/10.1021/acs.jctc.1c00647
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_i4hwummq4c7j_0" target="_blank">
     3BPA_test_300K
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test configurations with MD simulations performed at 300K from 3BPA, used to showcase the performance of linear atomic cluster expansion (ACE) force fields in a machine learning model to predict the potential energy surfaces of organic molecules.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    3BPA_test_300K__Kovács-Oord-Kucera-Allen-Cole-Ortner-Csányi__DS_i4hwummq4c7j_0
    <br/>
    <b>
     Name:
    </b>
    3BPA_test_300K
    <br/>
    <b>
     Authors:
    </b>
    Dávid Péter Kovács, Cas van der Oord, Jiri Kucera, Alice E. A. Allen, Daniel J. Cole, Christoph Ortner, Gábor Csányi
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,669
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 45,063
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.1c00647" target="_blank">
     https://doi.org/10.1021/acs.jctc.1c00647
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.1c00647" target="_blank">
     https://doi.org/10.1021/acs.jctc.1c00647
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_epqxcavda3kt_0" target="_blank">
     3BPA_test_600K
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test configurations with MD simulations performed at 600K from 3BPA, used to showcase the performance of linear atomic cluster expansion (ACE) force fields in a machine learning model to predict the potential energy surfaces of organic molecules.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    3BPA_test_600K__Kovács-Oord-Kucera-Allen-Cole-Ortner-Csányi__DS_epqxcavda3kt_0
    <br/>
    <b>
     Name:
    </b>
    3BPA_test_600K
    <br/>
    <b>
     Authors:
    </b>
    Dávid Péter Kovács, Cas van der Oord, Jiri Kucera, Alice E. A. Allen, Daniel J. Cole, Christoph Ortner, Gábor Csányi
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,138
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 57,726
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.1c00647" target="_blank">
     https://doi.org/10.1021/acs.jctc.1c00647
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.1c00647" target="_blank">
     https://doi.org/10.1021/acs.jctc.1c00647
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_8cg3pdvxt0pa_0" target="_blank">
     3BPA_test_dih_beta120
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test configurations with fixed value for dihedral beta in alpha-gamma  plane of 120 degreesfrom 3BPA dataset. Used to showcase the performance of linear atomic cluster expansion (ACE) force fields in a machine learning model to predict the potential energy surfaces of organic molecules.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    3BPA_test_dih_beta120__Kovács-Oord-Kucera-Allen-Cole-Ortner-Csányi__DS_8cg3pdvxt0pa_0
    <br/>
    <b>
     Name:
    </b>
    3BPA_test_dih_beta120
    <br/>
    <b>
     Authors:
    </b>
    Dávid Péter Kovács, Cas van der Oord, Jiri Kucera, Alice E. A. Allen, Daniel J. Cole, Christoph Ortner, Gábor Csányi
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,347
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 63,369
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.1c00647" target="_blank">
     https://doi.org/10.1021/acs.jctc.1c00647
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.1c00647" target="_blank">
     https://doi.org/10.1021/acs.jctc.1c00647
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_199ama4h9t7m_0" target="_blank">
     3BPA_test_dih_beta150
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test configurations with fixed value for dihedral beta in alpha-gamma  plane of 150 degreesfrom 3BPA dataset. Used to showcase the performance of linear atomic cluster expansion (ACE) force fields in a machine learning model to predict the potential energy surfaces of organic molecules.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    3BPA_test_dih_beta150__Kovács-Oord-Kucera-Allen-Cole-Ortner-Csányi__DS_199ama4h9t7m_0
    <br/>
    <b>
     Name:
    </b>
    3BPA_test_dih_beta150
    <br/>
    <b>
     Authors:
    </b>
    Dávid Péter Kovács, Cas van der Oord, Jiri Kucera, Alice E. A. Allen, Daniel J. Cole, Christoph Ortner, Gábor Csányi
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,350
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 63,450
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.1c00647" target="_blank">
     https://doi.org/10.1021/acs.jctc.1c00647
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.1c00647" target="_blank">
     https://doi.org/10.1021/acs.jctc.1c00647
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_ilijykl0t0fn_0" target="_blank">
     3BPA_test_dih_beta180
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test configurations with fixed value for dihedral beta in alpha-gamma  plane of 180 degreesfrom 3BPA dataset. Used to showcase the performance of linear atomic cluster expansion (ACE) force fields in a machine learning model to predict the potential energy surfaces of organic molecules.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    3BPA_test_dih_beta180__Kovács-Oord-Kucera-Allen-Cole-Ortner-Csányi__DS_ilijykl0t0fn_0
    <br/>
    <b>
     Name:
    </b>
    3BPA_test_dih_beta180
    <br/>
    <b>
     Authors:
    </b>
    Dávid Péter Kovács, Cas van der Oord, Jiri Kucera, Alice E. A. Allen, Daniel J. Cole, Christoph Ortner, Gábor Csányi
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,350
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 63,450
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.1c00647" target="_blank">
     https://doi.org/10.1021/acs.jctc.1c00647
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.1c00647" target="_blank">
     https://doi.org/10.1021/acs.jctc.1c00647
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_hu0btdblv8x6_0" target="_blank">
     3BPA_train_300K
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training configurations with MD simulations performed at 300K from 3BPA, used to showcase the performance of linear atomic cluster expansion (ACE) force fields in a machine learning model to predict the potential energy surfaces of organic molecules.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    3BPA_train_300K__Kovács-Oord-Kucera-Allen-Cole-Ortner-Csányi__DS_hu0btdblv8x6_0
    <br/>
    <b>
     Name:
    </b>
    3BPA_train_300K
    <br/>
    <b>
     Authors:
    </b>
    Dávid Péter Kovács, Cas van der Oord, Jiri Kucera, Alice E. A. Allen, Daniel J. Cole, Christoph Ortner, Gábor Csányi
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 500
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 13,500
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.1c00647" target="_blank">
     https://doi.org/10.1021/acs.jctc.1c00647
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.1c00647" target="_blank">
     https://doi.org/10.1021/acs.jctc.1c00647
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_v88913tvdxbr_0" target="_blank">
     3BPA_train_mixed
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training configurations with MD simulation performed at 300K, 600K and 1200K from 3BPA dataset, used to showcase the performance of linear atomic cluster expansion (ACE) force fields in a machine learning model to predict the potential energy surfaces of organic molecules.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    3BPA_train_mixed__Kovács-Oord-Kucera-Allen-Cole-Ortner-Csányi__DS_v88913tvdxbr_0
    <br/>
    <b>
     Name:
    </b>
    3BPA_train_mixed
    <br/>
    <b>
     Authors:
    </b>
    Dávid Péter Kovács, Cas van der Oord, Jiri Kucera, Alice E. A. Allen, Daniel J. Cole, Christoph Ortner, Gábor Csányi
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 500
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 13,500
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.1c00647" target="_blank">
     https://doi.org/10.1021/acs.jctc.1c00647
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.1c00647" target="_blank">
     https://doi.org/10.1021/acs.jctc.1c00647
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_0ady7a8a8n6p_0" target="_blank">
     ABC2D6-16_PRL_2018
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset used to train a machine learning model to calculate density functional theory-quality formation energies of all ~2 x 106 pristine ABC2D6 elpasolite crystals that can be made up from main-group elements (up to bismuth).
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    ABC2D6-16_PRL_2018__Faber-Lindmaa-Lilienfeld-Armiento__DS_0ady7a8a8n6p_0
    <br/>
    <b>
     Name:
    </b>
    ABC2D6-16_PRL_2018
    <br/>
    <b>
     Authors:
    </b>
    Felix Faber, Alexander Lindmaa, O. Anatole von Lilienfeld, Rickard Armiento
    <br/>
    <b>
     Elements
    </b>
    : Al, Ar, As, B, Ba, Be, Bi, Br, C, Ca, Cl, Cs, F, Ga, Ge, H, He, I, In, K, Kr, Li, Mg, N, Na, Ne, O, P, Pb, Rb, S, Sb, Se, Si, Sn, Sr, Te, Tl, Xe
    <br/>
    <b>
     Number of Configurations
    </b>
    : 21,882
    <br/>
    <b>
     Number of Elements
    </b>
    : 39
    <br/>
    <b>
     Number of Atoms
    </b>
    : 218,820
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://qmml.org/datasets.html" target="_blank">
     https://qmml.org/datasets.html
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevLett.117.135502" target="_blank">
     https://doi.org/10.1103/PhysRevLett.117.135502
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_0nnbymlcjota_0" target="_blank">
     AENET_amorphous_LiSi_JCP2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The amorphous LiSi data set comprises 45,169 atomic structures with compositions Li(x)Si (0.0≤x≤4.75) and the corresponding energies and interatomic forces, which were generated using an iterative approach based on an evolutionary algorithm and subsequent refinement, as described in detail in reference [15]. The data includes bulk, surface, and cluster structures with system sizes of up to 608 atoms. The energies and forces of the LiSi structures were obtained from DFT calculations using the Perdew-Burke-Ernzerhof [10] exchange-correlation functional and projector-augmented wave pseudopotentials [16], as implemented in the Vienna Ab-Initio Simulation Package (VASP) [17,18]. We employed a plane-wave basis set with an energy cutoff of 520 eV for the representation of the wavefunctions and a uniform gamma-centered k-point grid for the Brillouin zone integration, with a mesh density corresponding to a number of k points of at least 1000 divided by the number of atoms. The atomic positions and lattice parameters of all structures were optimized until residual forces were below 20 meV/Å. This dataset was also used for the construction of the ANN potential in Ref. [15] and [19]. [10] J. P. Perdew, K. Burke, and M. Ernzerhof, Phys. Rev. Lett. 77, 3865 (1996). [15] N. Artrith, A. Urban, G. Ceder, J. Chem. Phys. 148 (2018) 241711. [16] P. E. Blöchl, Phys. Rev. B 50, 17953–17979 (1994). [17] G. Kresse, J. Furthmüller, Phys. Rev. B 54, 11169–11186 (1996). [18] Kresse, J. Furthmüller, Comput. Mater. Sci. 6, 15–50 (1996). [19] N. Artrith, A. Urban, Y. Wang, G. Ceder, arXiv:1901.09272, https://arxiv.org/pdf/1901.09272.pdf
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    AENET_amorphous_LiSi_JCP2021__Chen-Morawietz-Markland-Artrith__DS_0nnbymlcjota_0
    <br/>
    <b>
     Name:
    </b>
    AENET_amorphous_LiSi_JCP2021
    <br/>
    <b>
     Authors:
    </b>
    Michael S. Chen, Tobias Morawietz, Thomas E. Markland, Nongnuch Artrith
    <br/>
    <b>
     Elements
    </b>
    : Li, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 44,652
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 5,741,142
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:dx-ct" target="_blank">
     https://doi.org/10.24435/materialscloud:dx-ct
    </a>
    <br/>
    <a href="http://doi.org/10.1063/5.0063880" target="_blank">
     http://doi.org/10.1063/5.0063880
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_v4bk24pq0had_0" target="_blank">
     AENET_liquid_water_dataset_JCP2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The water data set comprises energies and forces of 9,189 condensed-phase structures. The data was obtained in an iterative procedure described in detail in Ref. [4]. The final ANN potential was employed in Refs. [4,5] to analyze temperature-dependent Raman spectra of liquid water. The data set contains structures from four iterations: Initial structures (iteration 0) were obtained from classical and path integral AIMD simulations of bulk liquid water in a cubic box containing 64 water molecules at 300 K as reported in Ref. [6]. Distorted configurations with higher forces were added by randomly displacing the Cartesian coordinates of these configurations. Iteration 1 contains a set of 500 configurations from MD simulations with the fully flexible SPC/E flex water model [7] employing a 25 % increased water density (simulation box with 80 water molecules) and elevated temperatures (T = 500 K) in order to sample highly repulsive configurations. Structures in iteration 2 were obtained by classical MD simulations with preliminary ANN potentials at T = 300 K, 325 K, 350 K, and 370 K employing cubic boxes with 64 molecules and the corresponding experimental densities. The final iteration 3 data contains structures from preliminary ANN simulations with classical and quantum nuclei, respectively, at a wide range of temperatures (T = 258 K, 268 K, 280 K, 290 K, 300 K, 310 K, 320 K, 330 K, 340 K, 350 K, 360 K, and 370 K) using cubic boxes with 64 molecules and the corresponding experimental densities. Energies and atomic forces were calculated with the CP2K program [8,9] using the revPBE exchange-correlation functional [10,11] with D3 dispersion correction [12] following the setup reported in Ref. [4]. Atomic cores were represented using the dual-space Goedecker-Teter-Hutter pseudopotentials [13], Kohn-Sham orbitals were expanded in the TZV2P basis set within the GPW method [14], and the density was represented by an auxiliary plane-wave basis with a cutoff of 400 Ry. [1] A. Kokalj, J. Mol. Graphics Modell. 17, 176–179 (1999). [2] N. Artrith, A. Urban, Comput. Mater. Sci. 114, 135–150 (2016). [3] N. Artrith, A. Urban, G. Ceder, Phys. Rev. B 96, 014112 (2017). [4] T. Morawietz, O. Marsalek, S. R. Pattenaude, L. M. Streacker, D. Ben-Amotz, and T. E. Markland, J. Phys. Chem. Lett. 9, 851 (2018). [5] T. Morawietz, A. S. Urbina, P. K. Wise, X. Wu, W. Lu, D. Ben-Amotz, and T. E. Markland, J. Phys. Chem. Lett. 10, 6067 (2019). [6] Marsalek and T. E. Markland, J. Phys. Chem. Lett. 8, 1545 (2017). [7] X. B. Zhang, Q. L. Liu, and A. M. Zhu, Fluid Ph. Equilibria 262, 210(2007). [8] J. VandeVondele, M. Krack, F. Mohamed, M. Parrinello, T. Chassaing, and J. Hutter, Comput. Phys. Commun. 167, 103 (2005). [9] J. Hutter, M. Iannuzzi, F. Schiffmann, and J. VandeVondele, WIRES Comput. Mol. Sci. 4, 15 (2014). [10] J. P. Perdew, K. Burke, and M. Ernzerhof, Phys. Rev. Lett. 77, 3865 (1996). [11] Y. Zhang and W. Yang, Phys. Rev. Lett. 80, 890 (1998). [12] S. Grimme, J. Antony, S. Ehrlich, and H. Krieg, J. Chem. Phys. 132, 154104 (2010). [13] S. Goedecker, M. Teter, and J. Hutter, Phys. Rev. B 54, 1703 (1996). [14] B. G. Lippert, J. Hutter, and M. Parrinello, Mol. Phys. 92, 477 (1997).
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    AENET_liquid_water_dataset_JCP2021__Chen-Morawietz-Markland-Artrith__DS_v4bk24pq0had_0
    <br/>
    <b>
     Name:
    </b>
    AENET_liquid_water_dataset_JCP2021
    <br/>
    <b>
     Authors:
    </b>
    Michael S. Chen, Tobias Morawietz, Thomas E. Markland, Nongnuch Artrith
    <br/>
    <b>
     Elements
    </b>
    : H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 9,189
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,788,288
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:dx-ct" target="_blank">
     https://doi.org/10.24435/materialscloud:dx-ct
    </a>
    <br/>
    <a href="http://doi.org/10.1063/5.0063880" target="_blank">
     http://doi.org/10.1063/5.0063880
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_5lhmgnxhuia3_0" target="_blank">
     AFF_JCP_2022
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 145,000 configurations of alkane, aspirin, alpha-glucose and uracil, partly taken from the MD-17 dataset, used in training an 'Atomic Neural Net' model.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    AFF_JCP_2022__Li-Zhou-Sebastian-Wu-Gu__DS_5lhmgnxhuia3_0
    <br/>
    <b>
     Name:
    </b>
    AFF_JCP_2022
    <br/>
    <b>
     Authors:
    </b>
    Hao Li, Musen Zhou, Jessalyn Sebastian, Jianzhong Wu, Mengyang Gu
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 143,770
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,911,240
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/UncertaintyQuantification/AFF/tree/master" target="_blank">
     https://github.com/UncertaintyQuantification/AFF/tree/master
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0088017" target="_blank">
     https://doi.org/10.1063/5.0088017
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_p4evspy1ntcs_0" target="_blank">
     ANI-1
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    ANI-1 is a dataset of 20 million conformations with calculated non-equilibrium energy values. The conformations are based on a subset of the GDB-11 dataset, each molecule containing between 1 and 8 heavy atoms, with atomic species limited to C, N and O. Configuration sets are included for standard and high energy (defined as energies greater than 275 kcal*mol-1 higher than the lowest energy conformer) conformations, and, within these, number of heavy atoms per molecule.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    ANI-1__Smith-Isayev-Roitberg__DS_p4evspy1ntcs_0
    <br/>
    <b>
     Name:
    </b>
    ANI-1
    <br/>
    <b>
     Authors:
    </b>
    Justin S. Smith, Olexandr Isayev, Adrian E. Roitberg
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 24,416,306
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 392,606,016
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.c.3846712.v1" target="_blank">
     https://doi.org/10.6084/m9.figshare.c.3846712.v1
    </a>
    <br/>
    <a href="https://doi.org/10.1038/sdata.2017.193" target="_blank">
     https://doi.org/10.1038/sdata.2017.193
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_ko3rpzre7bea_0" target="_blank">
     ANI-1x
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    ANI-1x contains DFT calculations for approximately 5 million molecular conformations. From an initial training set, an active learning method was used to iteratively add conformations where insufficient diversity was detected. Additional conformations were sampled from existing databases of molecules, such as GDB-11 and ChEMBL. On each of these configurations, one of molecular dynamics sampling, normal mode sampling, dimer sampling, or torsion sampling was performed.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    ANI-1x__Smith-Zubatyuk-Nebgen-Lubbers-Barros-Roitberg-Isayev-Tretiak__DS_ko3rpzre7bea_0
    <br/>
    <b>
     Name:
    </b>
    ANI-1x
    <br/>
    <b>
     Authors:
    </b>
    Justin S. Smith, Roman Zubatyuk, Benjamin Nebgen, Nicholas Lubbers, Kipton Barros, Adrian E. Roitberg, Olexandr Isayev, Sergei Tretiak
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 4,956,005
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 75,700,481
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.c.4712477.v1" target="_blank">
     https://doi.org/10.6084/m9.figshare.c.4712477.v1
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-020-0473-z" target="_blank">
     https://doi.org/10.1038/s41597-020-0473-z
    </a>
    <br/>
    <a href="https://doi.org/10.1063/1.5023802" target="_blank">
     https://doi.org/10.1063/1.5023802
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-019-10827-4" target="_blank">
     https://doi.org/10.1038/s41467-019-10827-4
    </a>
    <br/>
    <a href="https://doi.org/10.1126/sciadv.aav6490" target="_blank">
     https://doi.org/10.1126/sciadv.aav6490
    </a>
    <br/>
    <a href="https://github.com/aiqm/ANI1x_datasets" target="_blank">
     https://github.com/aiqm/ANI1x_datasets
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_gjr8gi1tb8wh_0" target="_blank">
     ANI-2x-B973c-def2mTZVP
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    ANI-2x-B973c-def2mTZVP is a portion of the ANI-2x dataset, which includes DFT-calculated energies for structures from 2 to 63 atoms in size containing H, C, N, O, S, F, and Cl. This portion of ANI-2x was calculated in ORCA at the B973c level of theory using the def2m-TZVP basis set. Configuration sets are divided by number of atoms per structure. Force corrections and dipoles are recorded in the metadata.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    ANI-2x-B973c-def2mTZVP__Huddleston-Zubatyuk-Smith-Roitberg-Isayev-Pickering-Devereux-Barros__DS_gjr8gi1tb8wh_0
    <br/>
    <b>
     Name:
    </b>
    ANI-2x-B973c-def2mTZVP
    <br/>
    <b>
     Authors:
    </b>
    Kate Huddleston, Roman Zubatyuk, Justin Smith, Adrian Roitberg, Olexandr Isayev, Ignacio Pickering, Christian Devereux, Kipton Barros
    <br/>
    <b>
     Elements
    </b>
    : C, Cl, F, H, N, O, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 9,643,594
    <br/>
    <b>
     Number of Elements
    </b>
    : 7
    <br/>
    <b>
     Number of Atoms
    </b>
    : 146,656,635
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.10108942" target="_blank">
     https://doi.org/10.5281/zenodo.10108942
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.0c00121" target="_blank">
     https://doi.org/10.1021/acs.jctc.0c00121
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_yxeu6us2i6dh_0" target="_blank">
     ANI-2x-wB97MD3BJ-def2TZVPP
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    ANI-2x-wB97MD3BJ-def2TZVPP is a portion of the ANI-2x dataset, which includes DFT-calculated energies for structures from 2 to 63 atoms in size containing H, C, N, O, S, F, and Cl. This portion of ANI-2x was calculated in ORCA at the wB97M level of theory with D3 and BJ energy corrections, using the def2-TZVPP basis set. Configuration sets are divided by number of atoms per structure. Uncorrected SCF energy values and dipoles are recorded in the metadata.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    ANI-2x-wB97MD3BJ-def2TZVPP__Huddleston-Zubatyuk-Smith-Roitberg-Isayev-Pickering-Devereux-Barros__DS_yxeu6us2i6dh_0
    <br/>
    <b>
     Name:
    </b>
    ANI-2x-wB97MD3BJ-def2TZVPP
    <br/>
    <b>
     Authors:
    </b>
    Kate Huddleston, Roman Zubatyuk, Justin Smith, Adrian Roitberg, Olexandr Isayev, Ignacio Pickering, Christian Devereux, Kipton Barros
    <br/>
    <b>
     Elements
    </b>
    : C, Cl, F, H, N, O, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 9,650,572
    <br/>
    <b>
     Number of Elements
    </b>
    : 7
    <br/>
    <b>
     Number of Atoms
    </b>
    : 146,715,621
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.10108942" target="_blank">
     https://doi.org/10.5281/zenodo.10108942
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.0c00121" target="_blank">
     https://doi.org/10.1021/acs.jctc.0c00121
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_wxf8f3t3abul_0" target="_blank">
     ANI-2x-wB97MV-def2TZVPP
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    ANI-2x-wB97MV-def2TZVPP is a portion of the ANI-2x dataset, which includes DFT-calculated energies for structures from 2 to 63 atoms in size containing H, C, N, O, S, F, and Cl. This portion of ANI-2x was calculated at the WB97MV level of theory using the def2TZVPP basis set. Configuration sets are divided by number of atoms per structure.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    ANI-2x-wB97MV-def2TZVPP__Huddleston-Zubatyuk-Smith-Roitberg-Isayev-Pickering-Devereux-Barros__DS_wxf8f3t3abul_0
    <br/>
    <b>
     Name:
    </b>
    ANI-2x-wB97MV-def2TZVPP
    <br/>
    <b>
     Authors:
    </b>
    Kate Huddleston, Roman Zubatyuk, Justin Smith, Adrian Roitberg, Olexandr Isayev, Ignacio Pickering, Christian Devereux, Kipton Barros
    <br/>
    <b>
     Elements
    </b>
    : C, Cl, F, H, N, O, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 9,650,572
    <br/>
    <b>
     Number of Elements
    </b>
    : 7
    <br/>
    <b>
     Number of Atoms
    </b>
    : 146,715,621
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.10108942" target="_blank">
     https://doi.org/10.5281/zenodo.10108942
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.0c00121" target="_blank">
     https://doi.org/10.1021/acs.jctc.0c00121
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_rde4chkdmm18_0" target="_blank">
     ANI-2x-wB97X-631Gd
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    ANI-2x-wB97X-631Gd is a portion of the ANI-2x dataset, which includes DFT-calculated energies for structures from 2 to 63 atoms in size containing H, C, N, O, S, F, and Cl. This portion of ANI-2x was calculated in Gaussian 09 at the wB97X level of theory using the 6-31G(d) basis set. Configuration sets are divided by number of atoms per structure.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    ANI-2x-wB97X-631Gd__Huddleston-Zubatyuk-Smith-Roitberg-Isayev-Pickering-Devereux-Barros__DS_rde4chkdmm18_0
    <br/>
    <b>
     Name:
    </b>
    ANI-2x-wB97X-631Gd
    <br/>
    <b>
     Authors:
    </b>
    Kate Huddleston, Roman Zubatyuk, Justin Smith, Adrian Roitberg, Olexandr Isayev, Ignacio Pickering, Christian Devereux, Kipton Barros
    <br/>
    <b>
     Elements
    </b>
    : C, Cl, F, H, N, O, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 9,651,712
    <br/>
    <b>
     Number of Elements
    </b>
    : 7
    <br/>
    <b>
     Number of Atoms
    </b>
    : 146,736,809
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.10108942" target="_blank">
     https://doi.org/10.5281/zenodo.10108942
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.0c00121" target="_blank">
     https://doi.org/10.1021/acs.jctc.0c00121
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_bnnyjpaqb09i_0" target="_blank">
     ANI-Al_NC2021-test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 2800 configurations from a test dataset–one of a pair of train/test datasets of aluminum in crystal and melt phases, used for training and testing an ANI neural network model.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    ANI-Al_NC2021-test__Smith-Nebgen-Mathew-Chen-Lubbers-Burakovsky-Tretiak-Nam-Germann-Fensin-Barros__DS_bnnyjpaqb09i_0
    <br/>
    <b>
     Name:
    </b>
    ANI-Al_NC2021-test
    <br/>
    <b>
     Authors:
    </b>
    Justin S. Smith, Benjamin Nebgen, Nithin Mathew, Jie Chen, Nicholas Lubbers, Leonid Burakovsky, Sergei Tretiak, Hai Ah Nam, Timothy Germann, Saryu Fensin, Kipton Barros
    <br/>
    <b>
     Elements
    </b>
    : Al
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,872
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 371,645
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/atomistic-ml/ani-al" target="_blank">
     https://github.com/atomistic-ml/ani-al
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-021-21376-0" target="_blank">
     https://doi.org/10.1038/s41467-021-21376-0
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_tz8gsj3xi2g4_0" target="_blank">
     ANI-Al_NC2021-train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 2800 configurations from a train dataset–one of a pair of train/test datasets of aluminum in crystal and melt phases, used for training and testing an ANI neural network model.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    ANI-Al_NC2021-train__Smith-Nebgen-Mathew-Chen-Lubbers-Burakovsky-Tretiak-Nam-Germann-Fensin-Barros__DS_tz8gsj3xi2g4_0
    <br/>
    <b>
     Name:
    </b>
    ANI-Al_NC2021-train
    <br/>
    <b>
     Authors:
    </b>
    Justin S. Smith, Benjamin Nebgen, Nithin Mathew, Jie Chen, Nicholas Lubbers, Leonid Burakovsky, Sergei Tretiak, Hai Ah Nam, Timothy Germann, Saryu Fensin, Kipton Barros
    <br/>
    <b>
     Elements
    </b>
    : Al
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,864
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 375,121
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/atomistic-ml/ani-al" target="_blank">
     https://github.com/atomistic-ml/ani-al
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-021-21376-0" target="_blank">
     https://doi.org/10.1038/s41467-021-21376-0
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_6e5ljaqa3cfr_0" target="_blank">
     Ag-PBE_MSMSE_2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 7,600 configurations of Ag used as part of a training dataset for a DP-GEN-based ML model for a Ag-Au nanoalloy potential.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Ag-PBE_MSMSE_2021__Wang-Wang-Zhang-Xu-Wang__DS_6e5ljaqa3cfr_0
    <br/>
    <b>
     Name:
    </b>
    Ag-PBE_MSMSE_2021
    <br/>
    <b>
     Authors:
    </b>
    Yinan Wang, Xiaoyang Wang, Linfeng Zhang, Ben Xu, Han Wang
    <br/>
    <b>
     Elements
    </b>
    : Ag
    <br/>
    <b>
     Number of Configurations
    </b>
    : 7,608
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 152,318
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=Ag-PBE" target="_blank">
     https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=Ag-PBE
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2108.06232" target="_blank">
     https://doi.org/10.48550/arXiv.2108.06232
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_t0ec1irmeo59_0" target="_blank">
     AgAu-nanoalloy_MSMSE_2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 50,000 configurations of Au, Ag and AuAg used as part of a training dataset for a DP-GEN-based ML model for a Ag-Au nanoalloy potential.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    AgAu-nanoalloy_MSMSE_2021__Wang-Wang-Zhang-Xu-Wang__DS_t0ec1irmeo59_0
    <br/>
    <b>
     Name:
    </b>
    AgAu-nanoalloy_MSMSE_2021
    <br/>
    <b>
     Authors:
    </b>
    Yinan Wang, Xiaoyang Wang, Linfeng Zhang, Ben Xu, Han Wang
    <br/>
    <b>
     Elements
    </b>
    : Ag, Au
    <br/>
    <b>
     Number of Configurations
    </b>
    : 51,771
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,188,220
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=AgAu-nanoalloy" target="_blank">
     https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=AgAu-nanoalloy
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2108.06232" target="_blank">
     https://doi.org/10.48550/arXiv.2108.06232
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_y5m3hunroa7x_0" target="_blank">
     AgPd_NPJ_2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The dataset consists of energies, forces and virials for DFT-VASP-generated Ag-Pd systems. The data was used to fit an active learned dataset which was used to compare MTP- and SOAP-GAP-generated potentials.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    AgPd_NPJ_2021__Rosenbrock-Gubaev-Shapeev-Pártay-Bernstein-Csányi-Hart__DS_y5m3hunroa7x_0
    <br/>
    <b>
     Name:
    </b>
    AgPd_NPJ_2021
    <br/>
    <b>
     Authors:
    </b>
    Conrad W. Rosenbrock, Konstantin Gubaev, Alexander V. Shapeev, Livia B. Pártay, Noam Bernstein, Gábor Csányi, Gus L. W. Hart
    <br/>
    <b>
     Elements
    </b>
    : Ag, Pd
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,691
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 14,180
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/msg-byu/agpd" target="_blank">
     https://github.com/msg-byu/agpd
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-020-00477-2" target="_blank">
     https://doi.org/10.1038/s41524-020-00477-2
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_vmnudsz7kx0a_0" target="_blank">
     AlNiCu_AIP_2020
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset is formed from two parts: single-species datasets for Al, Ni, and Cu from the NOMAD Encyclopedia and multi-species datasets that include Al, Ni and Cu from NOMAD Archive. Duplicates have been removed from NOMAD Encyclopedia data. For the multi-species data, only the last configuration steps for each NOMAD Archive record were used because the last configuration typically cooresponds with a fully relaxed configuration. In this dataset, the NOMAD unique reference access IDs are retained along with a subset of their meta information that includes whether the supplied configuration is from a converged calculation as well as the Density Functional Theory (DFT) code, version, and type of DFT functionals with the total potential energies. This dataset consists of 39.1% Al, 30.7% Ni, and 30.2% Cu and has 27,987 atomic environments in 3337 structures.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    AlNiCu_AIP_2020__Onat-Ortner-Kermode__DS_vmnudsz7kx0a_0
    <br/>
    <b>
     Name:
    </b>
    AlNiCu_AIP_2020
    <br/>
    <b>
     Authors:
    </b>
    Berk Onat, Christoph Ortner, James R. Kermode
    <br/>
    <b>
     Elements
    </b>
    : Al, Cu, Ni
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,017
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 4,650
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/DescriptorZoo/sensitivity-dimensionality-results" target="_blank">
     https://github.com/DescriptorZoo/sensitivity-dimensionality-results
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0016005" target="_blank">
     https://doi.org/10.1063/5.0016005
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_dtjyh96dypuu_0" target="_blank">
     AlNiTi_CMS_2019
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset was generated using the following active learning scheme: 1) candidate structures were relaxed by a partially-trained MTP model, 2) structures for which the MTP had to perform extrapolation were passed to DFT to be re-computed, 3) the MTP was retrained, including the structures that were re-computed with DFT, 4) steps 1-3 were repeated until the MTP no longer extrapolated on any of the original candidate structures. The original candidate structures for this dataset included about 375,000 binary and ternary structures, enumerating all possible unit cells with different symmetries (BCC, FCC, and HCP) and different number of atoms.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    AlNiTi_CMS_2019__Gubaev-Podryabinkin-Hart-Shapeev__DS_dtjyh96dypuu_0
    <br/>
    <b>
     Name:
    </b>
    AlNiTi_CMS_2019
    <br/>
    <b>
     Authors:
    </b>
    Konstantin Gubaev, Evgeny V. Podryabinkin, Gus L.W. Hart, Alexander V. Shapeev
    <br/>
    <b>
     Elements
    </b>
    : Al, Ni, Ti
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,684
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 25,067
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://gitlab.com/kgubaev/accelerating-high-throughput-searches-for-new-alloys-with-active-learning-data" target="_blank">
     https://gitlab.com/kgubaev/accelerating-high-throughput-searches-for-new-alloys-with-active-learning-data
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.commatsci.2018.09.031" target="_blank">
     https://doi.org/10.1016/j.commatsci.2018.09.031
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_styimm9cal6v_0" target="_blank">
     Al_Cu_Mg_GSFE_JMPS2019
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset from "Stress-dependence of generalized stacking fault energies":DFT calculations of generalized stacking fault energies (GSFE) for Al, Cu, and Mg.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Al_Cu_Mg_GSFE_JMPS2019__Yin-Andric-Curtin__DS_styimm9cal6v_0
    <br/>
    <b>
     Name:
    </b>
    Al_Cu_Mg_GSFE_JMPS2019
    <br/>
    <b>
     Authors:
    </b>
    Binglun Yin, Predrag Andric, W. A. Curtin
    <br/>
    <b>
     Elements
    </b>
    : Al, Cu, Mg
    <br/>
    <b>
     Number of Configurations
    </b>
    : 273
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,264
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:2019.0089/v1" target="_blank">
     https://doi.org/10.24435/materialscloud:2019.0089/v1
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.jmps.2018.09.007" target="_blank">
     https://doi.org/10.1016/j.jmps.2018.09.007
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_xnio123pebli_0" target="_blank">
     Alexandria_geometry_optimization_paths_PBE_1D
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The Alexandria Materials Database contains theoretical crystal structures in 1D, 2D and 3D discovered by machine learning approaches using DFT with PBE, PBEsol and SCAN methods. This dataset represents the geometry optimization paths for 1D crystal structures from Alexandria calculated using PBE methods.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Alexandria_geometry_optimization_paths_PBE_1D__Schmidt-Hoffmann-Wang-Borlido-Carrico-Cerqueira-Botti-Marques__DS_xnio123pebli_0
    <br/>
    <b>
     Name:
    </b>
    Alexandria_geometry_optimization_paths_PBE_1D
    <br/>
    <b>
     Authors:
    </b>
    Jonathan Schmidt, Noah Hoffmann, Hai-Chen Wang, Pedro Borlido, Pedro J. M. A. Carriço, Tiago F. T. Cerqueira, Silvana Botti, Miguel A. L. Marques
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, Hf, Hg, Ho, I, In, Ir, K, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Te, Ti, Tl, Tm, V, W, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 614,833
    <br/>
    <b>
     Number of Elements
    </b>
    : 74
    <br/>
    <b>
     Number of Atoms
    </b>
    : 6,062,475
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://alexandria.icams.rub.de/" target="_blank">
     https://alexandria.icams.rub.de/
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_6pieq95jrqpn_0" target="_blank">
     Alexandria_geometry_optimization_paths_PBE_2D
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The Alexandria Materials Database contains theoretical crystal structures in 1D, 2D and 3D discovered by machine learning approaches using DFT with PBE, PBEsol and SCAN methods. This dataset represents the geometry optimization paths for 2D crystal structures from Alexandria calculated using PBE methods.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Alexandria_geometry_optimization_paths_PBE_2D__Schmidt-Hoffmann-Wang-Borlido-Carrico-Cerqueira-Botti-Marques__DS_6pieq95jrqpn_0
    <br/>
    <b>
     Name:
    </b>
    Alexandria_geometry_optimization_paths_PBE_2D
    <br/>
    <b>
     Authors:
    </b>
    Jonathan Schmidt, Noah Hoffmann, Hai-Chen Wang, Pedro Borlido, Pedro J. M. A. Carriço, Tiago F. T. Cerqueira, Silvana Botti, Miguel A. L. Marques
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, Hf, Hg, Ho, I, In, Ir, K, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 11,742,482
    <br/>
    <b>
     Number of Elements
    </b>
    : 84
    <br/>
    <b>
     Number of Atoms
    </b>
    : 118,265,549
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://alexandria.icams.rub.de/" target="_blank">
     https://alexandria.icams.rub.de/
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_s6gf4z2hcjqy_0" target="_blank">
     Alexandria_geometry_optimization_paths_PBE_3D
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The Alexandria Materials Database contains theoretical crystal structures in 1D, 2D and 3D discovered by machine learning approaches using DFT with PBE, PBEsol and SCAN methods. This dataset represents the geometry optimization paths for 3D crystal structures from Alexandria calculated using PBE methods.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Alexandria_geometry_optimization_paths_PBE_3D__Schmidt-Hoffmann-Wang-Borlido-Carrico-Cerqueira-Botti-Marques__DS_s6gf4z2hcjqy_0
    <br/>
    <b>
     Name:
    </b>
    Alexandria_geometry_optimization_paths_PBE_3D
    <br/>
    <b>
     Authors:
    </b>
    Jonathan Schmidt, Noah Hoffmann, Hai-Chen Wang, Pedro Borlido, Pedro J. M. A. Carriço, Tiago F. T. Cerqueira, Silvana Botti, Miguel A. L. Marques
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ne, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 106,825,218
    <br/>
    <b>
     Number of Elements
    </b>
    : 89
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,313,552,132
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://alexandria.icams.rub.de/" target="_blank">
     https://alexandria.icams.rub.de/
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_tydu7u0h0hhz_0" target="_blank">
     Au-PBE_MSMSE_2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 20,000 configurations of Au used as part of a  training dataset for a DP-GEN-based ML model for a Ag-Au nanoalloy  potential.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Au-PBE_MSMSE_2021__Wang-Wang-Zhang-Xu-Wang__DS_tydu7u0h0hhz_0
    <br/>
    <b>
     Name:
    </b>
    Au-PBE_MSMSE_2021
    <br/>
    <b>
     Authors:
    </b>
    Yinan Wang, Xiaoyang Wang, Linfeng Zhang, Ben Xu, Han Wang
    <br/>
    <b>
     Elements
    </b>
    : Au
    <br/>
    <b>
     Number of Configurations
    </b>
    : 19,434
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 310,792
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=Au-PBE" target="_blank">
     https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=Au-PBE
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2108.06232" target="_blank">
     https://doi.org/10.48550/arXiv.2108.06232
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_lifzo8zpa76m_0" target="_blank">
     BA10-18
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset (DFT-10B) contains structures of the 10 binary alloys AgCu, AlFe, AlMg, AlNi, AlTi, CoNi, CuFe, CuNi, FeV, and NbNi. Each alloy system includes all possible unit cells with 1-8 atoms for face-centered cubic (fcc) and body-centered cubic (bcc) crystal types, and all possible unit cells with 2-8 atoms for the hexagonal close-packed (hcp) crystal type. This results in 631 fcc, 631 bcc, and 333 hcp structures, yielding 1595 x 10 = 15,950 unrelaxed structures in total. Lattice parameters for each crystal structure were set according to Vegard's law. Total energies were computed using DFT with projector-augmented wave (PAW) potentials within the generalized gradient approximation (GGA) of Perdew, Burke, and Ernzerhof (PBE) as implemented in the Vienna Ab Initio Simulation Package (VASP). The k-point meshes for sampling the Brillouin zone were constructed using generalized regular grids.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    BA10-18__Nyshadham-Rupp-Bekker-Shapeev-Mueller-Rosenbrock-Csányi-Wingate-Hart__DS_lifzo8zpa76m_0
    <br/>
    <b>
     Name:
    </b>
    BA10-18
    <br/>
    <b>
     Authors:
    </b>
    Chandramouli Nyshadham, Matthias Rupp, Brayden Bekker, Alexander V. Shapeev, Tim Mueller, Conrad W. Rosenbrock, Gábor Csányi, David W. Wingate, Gus L. W. Hart
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, Co, Cu, Fe, Mg, Nb, Ni, Ti, V
    <br/>
    <b>
     Number of Configurations
    </b>
    : 15,920
    <br/>
    <b>
     Number of Elements
    </b>
    : 10
    <br/>
    <b>
     Number of Atoms
    </b>
    : 116,380
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://qmml.org/datasets.html" target="_blank">
     https://qmml.org/datasets.html
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-019-0189-9" target="_blank">
     https://doi.org/10.1038/s41524-019-0189-9
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_0xfqsnoawhoo_0" target="_blank">
     BOTnet_ACAC_2022_Dihedral_scan
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dihedral scan about one of the C-C bonds of the conjugated system. Acetylacetone dataset generated from a long molecular dynamics simulation at 300 K using a Langevin thermostat at the semi-empirical GFN2-xTB level of theory. Configurations were sampled at an interval of 1 ps and the resulting set of configurations were recomputed with density functional theory using the PBE exchange-correlation functional with D3 dispersion correction and def2-SVP basis set and VeryTightSCF convergence settings using the ORCA electronic structure package.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    BOTnet_ACAC_2022_Dihedral_scan__Batatia-Batzner-Kovács-Musaelian-Simm-Drautz-Ortner-Kozinsky-Csányi__DS_0xfqsnoawhoo_0
    <br/>
    <b>
     Name:
    </b>
    BOTnet_ACAC_2022_Dihedral_scan
    <br/>
    <b>
     Authors:
    </b>
    Ilyes Batatia, Simon Batzner, Dávid Péter Kovács, Albert Musaelian, Gregor N. C. Simm, Ralf Drautz, Christoph Ortner, Boris Kozinsky, Gábor Csányi
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 45
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 675
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/davkovacs/BOTNet-datasets" target="_blank">
     https://github.com/davkovacs/BOTNet-datasets
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2205.06643" target="_blank">
     https://doi.org/10.48550/arXiv.2205.06643
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_ztlvjmuwskom_0" target="_blank">
     BOTnet_ACAC_2022_H_transfer
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    NEB path of proton transfer reaction between the two forms of acetylacetone. Acetylacetone dataset generated from a long molecular dynamics simulation at 300 K using a Langevin thermostat at the semi-empirical GFN2-xTB level of theory. Configurations were sampled at an interval of 1 ps and the resulting set of configurations were recomputed with density functional theory using the PBE exchange-correlation functional with D3 dispersion correction and def2-SVP basis set and VeryTightSCF convergence settings using the ORCA electronic structure package.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    BOTnet_ACAC_2022_H_transfer__Batatia-Batzner-Kovács-Musaelian-Simm-Drautz-Ortner-Kozinsky-Csányi__DS_ztlvjmuwskom_0
    <br/>
    <b>
     Name:
    </b>
    BOTnet_ACAC_2022_H_transfer
    <br/>
    <b>
     Authors:
    </b>
    Ilyes Batatia, Simon Batzner, Dávid Péter Kovács, Albert Musaelian, Gregor N. C. Simm, Ralf Drautz, Christoph Ortner, Boris Kozinsky, Gábor Csányi
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 15
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 225
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/davkovacs/BOTNet-datasets" target="_blank">
     https://github.com/davkovacs/BOTNet-datasets
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2205.06643" target="_blank">
     https://doi.org/10.48550/arXiv.2205.06643
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_e22u3j5sbilx_0" target="_blank">
     BOTnet_ACAC_2022_isolated
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Energies of the isolated atoms evalauted at the reference DFT settings. Acetylacetone dataset generated from a long molecular dynamics simulation at 300 K using a Langevin thermostat at the semi-empirical GFN2-xTB level of theory. Configurations were sampled at an interval of 1 ps and the resulting set of configurations were recomputed with density functional theory using the PBE exchange-correlation functional with D3 dispersion correction and def2-SVP basis set and VeryTightSCF convergence settings using the ORCA electronic structure package.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    BOTnet_ACAC_2022_isolated__Batatia-Batzner-Kovács-Musaelian-Simm-Drautz-Ortner-Kozinsky-Csányi__DS_e22u3j5sbilx_0
    <br/>
    <b>
     Name:
    </b>
    BOTnet_ACAC_2022_isolated
    <br/>
    <b>
     Authors:
    </b>
    Ilyes Batatia, Simon Batzner, Dávid Péter Kovács, Albert Musaelian, Gregor N. C. Simm, Ralf Drautz, Christoph Ortner, Boris Kozinsky, Gábor Csányi
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/davkovacs/BOTNet-datasets" target="_blank">
     https://github.com/davkovacs/BOTNet-datasets
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2205.06643" target="_blank">
     https://doi.org/10.48550/arXiv.2205.06643
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_gt0oa9fngod7_0" target="_blank">
     BOTnet_ACAC_2022_test_300K_MD
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test set of decorrelated geometries sampled from 300 K xTB MD. Acetylacetone dataset generated from a long molecular dynamics simulation at 300 K using a Langevin thermostat at the semi-empirical GFN2-xTB level of theory. Configurations were sampled at an interval of 1 ps and the resulting set of configurations were recomputed with density functional theory using the PBE exchange-correlation functional with D3 dispersion correction and def2-SVP basis set and VeryTightSCF convergence settings using the ORCA electronic structure package.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    BOTnet_ACAC_2022_test_300K_MD__Batatia-Batzner-Kovács-Musaelian-Simm-Drautz-Ortner-Kozinsky-Csányi__DS_gt0oa9fngod7_0
    <br/>
    <b>
     Name:
    </b>
    BOTnet_ACAC_2022_test_300K_MD
    <br/>
    <b>
     Authors:
    </b>
    Ilyes Batatia, Simon Batzner, Dávid Péter Kovács, Albert Musaelian, Gregor N. C. Simm, Ralf Drautz, Christoph Ortner, Boris Kozinsky, Gábor Csányi
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 650
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 9,750
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/davkovacs/BOTNet-datasets" target="_blank">
     https://github.com/davkovacs/BOTNet-datasets
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2205.06643" target="_blank">
     https://doi.org/10.48550/arXiv.2205.06643
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_k0rexhnv58vn_0" target="_blank">
     BOTnet_ACAC_2022_test_600K_MD
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test set of decorrelated geometries sampled from 600 K xTB MD. Acetylacetone dataset generated from a long molecular dynamics simulation at 300 K using a Langevin thermostat at the semi-empirical GFN2-xTB level of theory. Configurations were sampled at an interval of 1 ps and the resulting set of configurations were recomputed with density functional theory using the PBE exchange-correlation functional with D3 dispersion correction and def2-SVP basis set and VeryTightSCF convergence settings using the ORCA electronic structure package.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    BOTnet_ACAC_2022_test_600K_MD__Batatia-Batzner-Kovács-Musaelian-Simm-Drautz-Ortner-Kozinsky-Csányi__DS_k0rexhnv58vn_0
    <br/>
    <b>
     Name:
    </b>
    BOTnet_ACAC_2022_test_600K_MD
    <br/>
    <b>
     Authors:
    </b>
    Ilyes Batatia, Simon Batzner, Dávid Péter Kovács, Albert Musaelian, Gregor N. C. Simm, Ralf Drautz, Christoph Ortner, Boris Kozinsky, Gábor Csányi
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 650
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 9,750
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/davkovacs/BOTNet-datasets" target="_blank">
     https://github.com/davkovacs/BOTNet-datasets
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2205.06643" target="_blank">
     https://doi.org/10.48550/arXiv.2205.06643
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_8gqwz5bw1pvq_0" target="_blank">
     BOTnet_ACAC_2022_train_300K_MD
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    500 decorrelated geometries sampled from 300 K xTB MD run. Acetylacetone dataset generated from a long molecular dynamics simulation at 300 K using a Langevin thermostat at the semi-empirical GFN2-xTB level of theory. Configurations were sampled at an interval of 1 ps and the resulting set of configurations were recomputed with density functional theory using the PBE exchange-correlation functional with D3 dispersion correction and def2-SVP basis set and VeryTightSCF convergence settings using the ORCA electronic structure package.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    BOTnet_ACAC_2022_train_300K_MD__Batatia-Batzner-Kovács-Musaelian-Simm-Drautz-Ortner-Kozinsky-Csányi__DS_8gqwz5bw1pvq_0
    <br/>
    <b>
     Name:
    </b>
    BOTnet_ACAC_2022_train_300K_MD
    <br/>
    <b>
     Authors:
    </b>
    Ilyes Batatia, Simon Batzner, Dávid Péter Kovács, Albert Musaelian, Gregor N. C. Simm, Ralf Drautz, Christoph Ortner, Boris Kozinsky, Gábor Csányi
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 500
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 7,500
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/davkovacs/BOTNet-datasets" target="_blank">
     https://github.com/davkovacs/BOTNet-datasets
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2205.06643" target="_blank">
     https://doi.org/10.48550/arXiv.2205.06643
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_632yputkfhpg_0" target="_blank">
     BOTnet_ACAC_2022_train_600K_MD
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    500 decorrelated geometries sampled from 600 K xTB MD run. Acetylacetone dataset generated from a long molecular dynamics simulation at 300 K using a Langevin thermostat at the semi-empirical GFN2-xTB level of theory. Configurations were sampled at an interval of 1 ps and the resulting set of configurations were recomputed with density functional theory using the PBE exchange-correlation functional with D3 dispersion correction and def2-SVP basis set and VeryTightSCF convergence settings using the ORCA electronic structure package.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    BOTnet_ACAC_2022_train_600K_MD__Batatia-Batzner-Kovács-Musaelian-Simm-Drautz-Ortner-Kozinsky-Csányi__DS_632yputkfhpg_0
    <br/>
    <b>
     Name:
    </b>
    BOTnet_ACAC_2022_train_600K_MD
    <br/>
    <b>
     Authors:
    </b>
    Ilyes Batatia, Simon Batzner, Dávid Péter Kovács, Albert Musaelian, Gregor N. C. Simm, Ralf Drautz, Christoph Ortner, Boris Kozinsky, Gábor Csányi
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 500
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 7,500
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/davkovacs/BOTNet-datasets" target="_blank">
     https://github.com/davkovacs/BOTNet-datasets
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2205.06643" target="_blank">
     https://doi.org/10.48550/arXiv.2205.06643
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_olmzgsz9hmxy_0" target="_blank">
     C7H10O2
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    6095 isomers of C7O2H10. Energetics were calculated at the G4MP2 level of theory.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    C7H10O2__Ramakrishnan-Dral-Rupp-Lilienfeld__DS_olmzgsz9hmxy_0
    <br/>
    <b>
     Name:
    </b>
    C7H10O2
    <br/>
    <b>
     Authors:
    </b>
    Raghunathan Ramakrishnan, Pavlo Dral, Matthias Rupp, O. Anatole von Lilienfeld
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 6,095
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 115,805
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.c.978904.v5" target="_blank">
     https://doi.org/10.6084/m9.figshare.c.978904.v5
    </a>
    <br/>
    <a href="https://doi.org/10.1038/sdata.2014.22" target="_blank">
     https://doi.org/10.1038/sdata.2014.22
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_l7inbtql4ea9_0" target="_blank">
     CA-9_BB_training
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Binning-binning configurations from CA-9 dataset used for training NNP_BB potential. CA-9 consists of configurations of carbon with curated subsets chosen to test the effects of intentionally choosing dissimilar configurations when training neural network potentials
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CA-9_BB_training__Hedman-Rothe-Johansson-Sandin-Larsson-Miyamoto__DS_l7inbtql4ea9_0
    <br/>
    <b>
     Name:
    </b>
    CA-9_BB_training
    <br/>
    <b>
     Authors:
    </b>
    Daniel Hedman, Tom Rothe, Gustav Johansson, Fredrik Sandin, J. Andreas Larsson, Yoshiyuki Miyamoto
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 20,012
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,054,055
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:6h-yj" target="_blank">
     https://doi.org/10.24435/materialscloud:6h-yj
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.cartre.2021.100027" target="_blank">
     https://doi.org/10.1016/j.cartre.2021.100027
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_8zft35i5sbgd_0" target="_blank">
     CA-9_BB_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Binning-binning configurations from CA-9 dataset used during validation step for NNP_BB potential. CA-9 consists of configurations of carbon with curated subsets chosen to test the effects of intentionally choosing dissimilar configurations when training neural network potentials
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CA-9_BB_validation__Hedman-Rothe-Johansson-Sandin-Larsson-Miyamoto__DS_8zft35i5sbgd_0
    <br/>
    <b>
     Name:
    </b>
    CA-9_BB_validation
    <br/>
    <b>
     Authors:
    </b>
    Daniel Hedman, Tom Rothe, Gustav Johansson, Fredrik Sandin, J. Andreas Larsson, Yoshiyuki Miyamoto
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 4,003
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 233,034
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:6h-yj" target="_blank">
     https://doi.org/10.24435/materialscloud:6h-yj
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.cartre.2021.100027" target="_blank">
     https://doi.org/10.1016/j.cartre.2021.100027
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_fw7m5d8b0fa9_0" target="_blank">
     CA-9_BR_training
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Binning-random configurations from CA-9 dataset used for training NNP_BR potential. CA-9 consists of configurations of carbon with curated subsets chosen to test the effects of intentionally choosing dissimilar configurations when training neural network potentials
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CA-9_BR_training__Hedman-Rothe-Johansson-Sandin-Larsson-Miyamoto__DS_fw7m5d8b0fa9_0
    <br/>
    <b>
     Name:
    </b>
    CA-9_BR_training
    <br/>
    <b>
     Authors:
    </b>
    Daniel Hedman, Tom Rothe, Gustav Johansson, Fredrik Sandin, J. Andreas Larsson, Yoshiyuki Miyamoto
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 20,013
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,072,779
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:6h-yj" target="_blank">
     https://doi.org/10.24435/materialscloud:6h-yj
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.cartre.2021.100027" target="_blank">
     https://doi.org/10.1016/j.cartre.2021.100027
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_fgakfcwhn9zc_0" target="_blank">
     CA-9_BR_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Binning-random configurations from CA-9 dataset used during validation step for NNP_BR potential. CA-9 consists of configurations of carbon with curated subsets chosen to test the effects of intentionally choosing dissimilar configurations when training neural network potentials
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CA-9_BR_validation__Hedman-Rothe-Johansson-Sandin-Larsson-Miyamoto__DS_fgakfcwhn9zc_0
    <br/>
    <b>
     Name:
    </b>
    CA-9_BR_validation
    <br/>
    <b>
     Authors:
    </b>
    Daniel Hedman, Tom Rothe, Gustav Johansson, Fredrik Sandin, J. Andreas Larsson, Yoshiyuki Miyamoto
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 4,002
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 214,310
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:6h-yj" target="_blank">
     https://doi.org/10.24435/materialscloud:6h-yj
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.cartre.2021.100027" target="_blank">
     https://doi.org/10.1016/j.cartre.2021.100027
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_tag5zubl21w8_0" target="_blank">
     CA-9_RR_training
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Random-random configurations from CA-9 dataset used for training NNP_RR potential. CA-9 consists of configurations of carbon with curated subsets chosen to test the effects of intentionally choosing dissimilar configurations when training neural network potentials
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CA-9_RR_training__Hedman-Rothe-Johansson-Sandin-Larsson-Miyamoto__DS_tag5zubl21w8_0
    <br/>
    <b>
     Name:
    </b>
    CA-9_RR_training
    <br/>
    <b>
     Authors:
    </b>
    Daniel Hedman, Tom Rothe, Gustav Johansson, Fredrik Sandin, J. Andreas Larsson, Yoshiyuki Miyamoto
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 20,013
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,100,042
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:6h-yj" target="_blank">
     https://doi.org/10.24435/materialscloud:6h-yj
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.cartre.2021.100027" target="_blank">
     https://doi.org/10.1016/j.cartre.2021.100027
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_mr2qaiqh2sx2_0" target="_blank">
     CA-9_RR_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Random-random configurations from CA-9 dataset used during validation step for NNP_RR potential. CA-9 consists of configurations of carbon with curated subsets chosen to test the effects of intentionally choosing dissimilar configurations when training neural network potentials
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CA-9_RR_validation__Hedman-Rothe-Johansson-Sandin-Larsson-Miyamoto__DS_mr2qaiqh2sx2_0
    <br/>
    <b>
     Name:
    </b>
    CA-9_RR_validation
    <br/>
    <b>
     Authors:
    </b>
    Daniel Hedman, Tom Rothe, Gustav Johansson, Fredrik Sandin, J. Andreas Larsson, Yoshiyuki Miyamoto
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 4,002
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 218,184
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:6h-yj" target="_blank">
     https://doi.org/10.24435/materialscloud:6h-yj
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.cartre.2021.100027" target="_blank">
     https://doi.org/10.1016/j.cartre.2021.100027
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_h0mshvvbxlai_0" target="_blank">
     CA-9_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test configurations from CA-9 dataset used to evaluate trained NNPs.CA-9 consists of configurations of carbon with curated subsets chosen to test the effects of intentionally choosing dissimilar configurations when training neural network potentials
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CA-9_test__Hedman-Rothe-Johansson-Sandin-Larsson-Miyamoto__DS_h0mshvvbxlai_0
    <br/>
    <b>
     Name:
    </b>
    CA-9_test
    <br/>
    <b>
     Authors:
    </b>
    Daniel Hedman, Tom Rothe, Gustav Johansson, Fredrik Sandin, J. Andreas Larsson, Yoshiyuki Miyamoto
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,727
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 206,302
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:6h-yj" target="_blank">
     https://doi.org/10.24435/materialscloud:6h-yj
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.cartre.2021.100027" target="_blank">
     https://doi.org/10.1016/j.cartre.2021.100027
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_9cdtww9pcjqd_0" target="_blank">
     CA-9_training
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations from CA-9 dataset used for training NNP_CA-9 potential. CA-9 consists of configurations of carbon with curated subsets chosen to test the effects of intentionally choosing dissimilar configurations when training neural network potentials
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CA-9_training__Hedman-Rothe-Johansson-Sandin-Larsson-Miyamoto__DS_9cdtww9pcjqd_0
    <br/>
    <b>
     Name:
    </b>
    CA-9_training
    <br/>
    <b>
     Authors:
    </b>
    Daniel Hedman, Tom Rothe, Gustav Johansson, Fredrik Sandin, J. Andreas Larsson, Yoshiyuki Miyamoto
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 40,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,195,399
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:6h-yj" target="_blank">
     https://doi.org/10.24435/materialscloud:6h-yj
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.cartre.2021.100027" target="_blank">
     https://doi.org/10.1016/j.cartre.2021.100027
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_07zjrtacjg3z_0" target="_blank">
     CA-9_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations from CA-9 dataset used during validation step for NNP_CA-9 potential. CA-9 consists of configurations of carbon with curated subsets chosen to test the effects of intentionally choosing dissimilar configurations when training neural network potentials
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CA-9_validation__Hedman-Rothe-Johansson-Sandin-Larsson-Miyamoto__DS_07zjrtacjg3z_0
    <br/>
    <b>
     Name:
    </b>
    CA-9_validation
    <br/>
    <b>
     Authors:
    </b>
    Daniel Hedman, Tom Rothe, Gustav Johansson, Fredrik Sandin, J. Andreas Larsson, Yoshiyuki Miyamoto
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 8,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 436,601
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:6h-yj" target="_blank">
     https://doi.org/10.24435/materialscloud:6h-yj
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.cartre.2021.100027" target="_blank">
     https://doi.org/10.1016/j.cartre.2021.100027
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_392q5d027yja_0" target="_blank">
     CGM-MLP_natcomm2023_Cr-C_deposition
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training simulations from CGM-MLP_natcomm2023 of carbon deposition on a Cr surface. This dataset was one of the datasets used in training during the process of producing an active learning dataset for the purposes of exploring substrate-catalyzed deposition on metal surfaces such as Cu(111), Cr(110), Ti(001), and oxygen-contaminated Cu(111) as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu, Cr and Ti surfaces.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CGM-MLP_natcomm2023_Cr-C_deposition__Zhang-Yi-Lai-Peng-Li__DS_392q5d027yja_0
    <br/>
    <b>
     Name:
    </b>
    CGM-MLP_natcomm2023_Cr-C_deposition
    <br/>
    <b>
     Authors:
    </b>
    Di Zhang, Peiyun Yi, Xinmin Lai, Linfa Peng, Hao Li
    <br/>
    <b>
     Elements
    </b>
    : C, Cr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,192
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 298,114
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/sjtudizhang/CGM-MLP" target="_blank">
     https://github.com/sjtudizhang/CGM-MLP
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-023-44525-z" target="_blank">
     https://doi.org/10.1038/s41467-023-44525-z
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_xho213jy5pf9_0" target="_blank">
     CGM-MLP_natcomm2023_Cu-C-O
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training simulations from CGM-MLP_natcomm2023 of carbon on an oxygen-contaminated Cu surface. This dataset was one of the datasets used in training during the process of producing an active learning dataset for the purposes of exploring substrate-catalyzed deposition on metal surfaces such as Cu(111), Cr(110), Ti(001), and oxygen-contaminated Cu(111) as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu, Cr and Ti surfaces.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CGM-MLP_natcomm2023_Cu-C-O__Zhang-Yi-Lai-Peng-Li__DS_xho213jy5pf9_0
    <br/>
    <b>
     Name:
    </b>
    CGM-MLP_natcomm2023_Cu-C-O
    <br/>
    <b>
     Authors:
    </b>
    Di Zhang, Peiyun Yi, Xinmin Lai, Linfa Peng, Hao Li
    <br/>
    <b>
     Elements
    </b>
    : C, Cu, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,717
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 387,151
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/sjtudizhang/CGM-MLP" target="_blank">
     https://github.com/sjtudizhang/CGM-MLP
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-023-44525-z" target="_blank">
     https://doi.org/10.1038/s41467-023-44525-z
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_xzo1tvni8bay_0" target="_blank">
     CGM-MLP_natcomm2023_Cu-C-O_deposition
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training simulations from CGM-MLP_natcomm2023 of carbon deposition on a Cu surface. This appears similar to CGM-MLP_natcomm2023_CU-C_deposition, as there are no O atoms present in this set. This dataset was one of the datasets used in training during the process of producing an active learning dataset for the purposes of exploring substrate-catalyzed deposition on metal surfaces such as Cu(111), Cr(110), Ti(001), and oxygen-contaminated Cu(111) as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu, Cr and Ti surfaces.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CGM-MLP_natcomm2023_Cu-C-O_deposition__Zhang-Yi-Lai-Peng-Li__DS_xzo1tvni8bay_0
    <br/>
    <b>
     Name:
    </b>
    CGM-MLP_natcomm2023_Cu-C-O_deposition
    <br/>
    <b>
     Authors:
    </b>
    Di Zhang, Peiyun Yi, Xinmin Lai, Linfa Peng, Hao Li
    <br/>
    <b>
     Elements
    </b>
    : C, Cu
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,694
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 326,328
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/sjtudizhang/CGM-MLP" target="_blank">
     https://github.com/sjtudizhang/CGM-MLP
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-023-44525-z" target="_blank">
     https://doi.org/10.1038/s41467-023-44525-z
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_vgy50b4qz4p7_0" target="_blank">
     CGM-MLP_natcomm2023_Cu-C_deposition
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training simulations from CGM-MLP_natcomm2023 of carbon deposition on a Cu surface. This dataset was one of the datasets used in training during the process of producing an active learning dataset for the purposes of exploring substrate-catalyzed deposition on metal surfaces such as Cu(111), Cr(110), Ti(001), and oxygen-contaminated Cu(111) as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu, Cr and Ti surfaces.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CGM-MLP_natcomm2023_Cu-C_deposition__Zhang-Yi-Lai-Peng-Li__DS_vgy50b4qz4p7_0
    <br/>
    <b>
     Name:
    </b>
    CGM-MLP_natcomm2023_Cu-C_deposition
    <br/>
    <b>
     Authors:
    </b>
    Di Zhang, Peiyun Yi, Xinmin Lai, Linfa Peng, Hao Li
    <br/>
    <b>
     Elements
    </b>
    : C, Cu
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,177
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 204,591
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/sjtudizhang/CGM-MLP" target="_blank">
     https://github.com/sjtudizhang/CGM-MLP
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-023-44525-z" target="_blank">
     https://doi.org/10.1038/s41467-023-44525-z
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_mo7q9ajg1lw8_0" target="_blank">
     CGM-MLP_natcomm2023_Cu-C_metal_surface
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training simulations from CGM-MLP_natcomm2023 of carbon on a Cu metal surface. This dataset was one of the datasets used in training during the process of producing an active learning dataset for the purposes of exploring substrate-catalyzed deposition on metal surfaces such as Cu(111), Cr(110), Ti(001), and oxygen-contaminated Cu(111) as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu, Cr and Ti surfaces.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CGM-MLP_natcomm2023_Cu-C_metal_surface__Zhang-Yi-Lai-Peng-Li__DS_mo7q9ajg1lw8_0
    <br/>
    <b>
     Name:
    </b>
    CGM-MLP_natcomm2023_Cu-C_metal_surface
    <br/>
    <b>
     Authors:
    </b>
    Di Zhang, Peiyun Yi, Xinmin Lai, Linfa Peng, Hao Li
    <br/>
    <b>
     Elements
    </b>
    : C, Cu
    <br/>
    <b>
     Number of Configurations
    </b>
    : 520
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 122,294
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/sjtudizhang/CGM-MLP" target="_blank">
     https://github.com/sjtudizhang/CGM-MLP
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-023-44525-z" target="_blank">
     https://doi.org/10.1038/s41467-023-44525-z
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_qslbmhe4bcgg_0" target="_blank">
     CGM-MLP_natcomm2023_GAP_20
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Carbon_GAP_20 dataset from CGM-MLP_natcomm2023. This dataset was one of the datasets used in training during the process of producing an active learning dataset for the purposes of exploring substrate-catalyzed deposition on metal surfaces such as Cu(111), Cr(110), Ti(001), and oxygen-contaminated Cu(111) as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu, Cr and Ti surfaces.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CGM-MLP_natcomm2023_GAP_20__Zhang-Yi-Lai-Peng-Li__DS_qslbmhe4bcgg_0
    <br/>
    <b>
     Name:
    </b>
    CGM-MLP_natcomm2023_GAP_20
    <br/>
    <b>
     Authors:
    </b>
    Di Zhang, Peiyun Yi, Xinmin Lai, Linfa Peng, Hao Li
    <br/>
    <b>
     Elements
    </b>
    : C, Cu
    <br/>
    <b>
     Number of Configurations
    </b>
    : 6,178
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 400,485
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/sjtudizhang/CGM-MLP" target="_blank">
     https://github.com/sjtudizhang/CGM-MLP
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-023-44525-z" target="_blank">
     https://doi.org/10.1038/s41467-023-44525-z
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_8frq3jczb0dt_0" target="_blank">
     CGM-MLP_natcomm2023_Ti-C_deposition
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training simulations from CGM-MLP_natcomm2023 of carbon deposition on a Ti surface. This dataset was one of the datasets used in training during the process of producing an active learning dataset for the purposes of exploring substrate-catalyzed deposition on metal surfaces such as Cu(111), Cr(110), Ti(001), and oxygen-contaminated Cu(111) as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu, Cr and Ti surfaces.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CGM-MLP_natcomm2023_Ti-C_deposition__Zhang-Yi-Lai-Peng-Li__DS_8frq3jczb0dt_0
    <br/>
    <b>
     Name:
    </b>
    CGM-MLP_natcomm2023_Ti-C_deposition
    <br/>
    <b>
     Authors:
    </b>
    Di Zhang, Peiyun Yi, Xinmin Lai, Linfa Peng, Hao Li
    <br/>
    <b>
     Elements
    </b>
    : C, Ti
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,309
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 259,636
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/sjtudizhang/CGM-MLP" target="_blank">
     https://github.com/sjtudizhang/CGM-MLP
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-023-44525-z" target="_blank">
     https://doi.org/10.1038/s41467-023-44525-z
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_wt0m3lxcdy52_0" target="_blank">
     CGM-MLP_natcomm2023_screening_amorphous_carbon_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    493 structures available from the GAP-20 database, excluding any structures present in the training set. This dataset was one of the datasets used in testing screening parameters during the process of producing an active learning dataset for Cu-C interactions for the purposes of exploring substrate-catalyzed deposition as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu(111) surface.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CGM-MLP_natcomm2023_screening_amorphous_carbon_test__Zhang-Yi-Lai-Peng-Li__DS_wt0m3lxcdy52_0
    <br/>
    <b>
     Name:
    </b>
    CGM-MLP_natcomm2023_screening_amorphous_carbon_test
    <br/>
    <b>
     Authors:
    </b>
    Di Zhang, Peiyun Yi, Xinmin Lai, Linfa Peng, Hao Li
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 494
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 32,279
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/sjtudizhang/CGM-MLP" target="_blank">
     https://github.com/sjtudizhang/CGM-MLP
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-023-44525-z" target="_blank">
     https://doi.org/10.1038/s41467-023-44525-z
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_taxwl7jo4f8a_0" target="_blank">
     CGM-MLP_natcomm2023_screening_amorphous_carbon_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    2558 structures selected from the GAP-20 database. This dataset was one of the datasets used in testing screening parameters during the process of producing an active learning dataset for Cu-C interactions for the purposes of exploring substrate-catalyzed deposition as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu(111) surface.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CGM-MLP_natcomm2023_screening_amorphous_carbon_train__Zhang-Yi-Lai-Peng-Li__DS_taxwl7jo4f8a_0
    <br/>
    <b>
     Name:
    </b>
    CGM-MLP_natcomm2023_screening_amorphous_carbon_train
    <br/>
    <b>
     Authors:
    </b>
    Di Zhang, Peiyun Yi, Xinmin Lai, Linfa Peng, Hao Li
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,559
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 168,191
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/sjtudizhang/CGM-MLP" target="_blank">
     https://github.com/sjtudizhang/CGM-MLP
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-023-44525-z" target="_blank">
     https://doi.org/10.1038/s41467-023-44525-z
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_gkumjkncy8ft_0" target="_blank">
     CGM-MLP_natcomm2023_screening_carbon-cluster@Cu_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    192 structures were uniformly selected from the AIMD simulation, excluding any structures that are part of the training set. This dataset was one of the datasets used in testing screening parameters during the process of producing an active learning dataset for Cu-C interactions for the purposes of exploring substrate-catalyzed deposition as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu(111) surface.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CGM-MLP_natcomm2023_screening_carbon-cluster@Cu_test__Zhang-Yi-Lai-Peng-Li__DS_gkumjkncy8ft_0
    <br/>
    <b>
     Name:
    </b>
    CGM-MLP_natcomm2023_screening_carbon-cluster@Cu_test
    <br/>
    <b>
     Authors:
    </b>
    Di Zhang, Peiyun Yi, Xinmin Lai, Linfa Peng, Hao Li
    <br/>
    <b>
     Elements
    </b>
    : C, Cu
    <br/>
    <b>
     Number of Configurations
    </b>
    : 193
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 38,004
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/sjtudizhang/CGM-MLP" target="_blank">
     https://github.com/sjtudizhang/CGM-MLP
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-023-44525-z" target="_blank">
     https://doi.org/10.1038/s41467-023-44525-z
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_6sjhg1f8fv5j_0" target="_blank">
     CGM-MLP_natcomm2023_screening_carbon-cluster@Cu_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    588 structures selected from the AIMD simulation of the Cu(111) slab, including both the C1-C18 clusters on the Cu(111) slab. This dataset was one of the datasets used in testing screening parameters during the process of producing an active learning dataset for Cu-C interactions for the purposes of exploring substrate-catalyzed deposition as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu(111) surface.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CGM-MLP_natcomm2023_screening_carbon-cluster@Cu_train__Zhang-Yi-Lai-Peng-Li__DS_6sjhg1f8fv5j_0
    <br/>
    <b>
     Name:
    </b>
    CGM-MLP_natcomm2023_screening_carbon-cluster@Cu_train
    <br/>
    <b>
     Authors:
    </b>
    Di Zhang, Peiyun Yi, Xinmin Lai, Linfa Peng, Hao Li
    <br/>
    <b>
     Elements
    </b>
    : C, Cu
    <br/>
    <b>
     Number of Configurations
    </b>
    : 588
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 115,460
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/sjtudizhang/CGM-MLP" target="_blank">
     https://github.com/sjtudizhang/CGM-MLP
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-023-44525-z" target="_blank">
     https://doi.org/10.1038/s41467-023-44525-z
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_5r9dvov8nq7c_0" target="_blank">
     CGM-MLP_natcomm2023_screening_deposited-carbon@Cu_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    468 structures uniformly selected from the MD/tfMC simulation, excluding any structures that are part of the training set. This dataset was one of the datasets used in testing screening parameters during the process of producing an active learning dataset for Cu-C interactions for the purposes of exploring substrate-catalyzed deposition as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu(111) surface.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CGM-MLP_natcomm2023_screening_deposited-carbon@Cu_test__Zhang-Yi-Lai-Peng-Li__DS_5r9dvov8nq7c_0
    <br/>
    <b>
     Name:
    </b>
    CGM-MLP_natcomm2023_screening_deposited-carbon@Cu_test
    <br/>
    <b>
     Authors:
    </b>
    Di Zhang, Peiyun Yi, Xinmin Lai, Linfa Peng, Hao Li
    <br/>
    <b>
     Elements
    </b>
    : C, Cu
    <br/>
    <b>
     Number of Configurations
    </b>
    : 469
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 156,312
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/sjtudizhang/CGM-MLP" target="_blank">
     https://github.com/sjtudizhang/CGM-MLP
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-023-44525-z" target="_blank">
     https://doi.org/10.1038/s41467-023-44525-z
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_j6w4ru800ukq_0" target="_blank">
     CGM-MLP_natcomm2023_screening_deposited-carbon@Cu_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    1090 structures uniformly selected from the MD/tfMC simulation during the training process of CGM-MLPs. This dataset was one of the datasets used in testing screening parameters during the process of producing an active learning dataset for Cu-C interactions for the purposes of exploring substrate-catalyzed deposition as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu(111) surface.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CGM-MLP_natcomm2023_screening_deposited-carbon@Cu_train__Zhang-Yi-Lai-Peng-Li__DS_j6w4ru800ukq_0
    <br/>
    <b>
     Name:
    </b>
    CGM-MLP_natcomm2023_screening_deposited-carbon@Cu_train
    <br/>
    <b>
     Authors:
    </b>
    Di Zhang, Peiyun Yi, Xinmin Lai, Linfa Peng, Hao Li
    <br/>
    <b>
     Elements
    </b>
    : C, Cu
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,091
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 362,898
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/sjtudizhang/CGM-MLP" target="_blank">
     https://github.com/sjtudizhang/CGM-MLP
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-023-44525-z" target="_blank">
     https://doi.org/10.1038/s41467-023-44525-z
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_jasbxoigo7r4_0" target="_blank">
     CGM-MLP_natcomm2023_screening_graphite_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    40 graphite structures with different lattice constants ranging from 2.0 to 3.2 Å, with a 0.03 Å increment. This dataset was one of the datasets used in testing screening parameters during the process of producing an active learning dataset for Cu-C interactions for the purposes of exploring substrate-catalyzed deposition as a means of controllable synthesis of carbon nanomaterials. The combined dataset includes structures from the Carbon_GAP_20 dataset and additional configurations of carbon clusters on a Cu(111) surface.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CGM-MLP_natcomm2023_screening_graphite_train__Zhang-Yi-Lai-Peng-Li__DS_jasbxoigo7r4_0
    <br/>
    <b>
     Name:
    </b>
    CGM-MLP_natcomm2023_screening_graphite_train
    <br/>
    <b>
     Authors:
    </b>
    Di Zhang, Peiyun Yi, Xinmin Lai, Linfa Peng, Hao Li
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 41
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,968
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/sjtudizhang/CGM-MLP" target="_blank">
     https://github.com/sjtudizhang/CGM-MLP
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-023-44525-z" target="_blank">
     https://doi.org/10.1038/s41467-023-44525-z
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_3td9plyix4ib_0" target="_blank">
     CHON_JCP_2020
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset of molecular structures was extracted, using the NOMAD API, from all available structures in the NOMAD Archive that only include C, H, O, and N. This dataset consists of 50.42% H, 30.41% C, 10.36% N, and 8.81% O and includes 96 804 atomic environments in 5217 structures.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CHON_JCP_2020__Onat-Ortner-Kermode__DS_3td9plyix4ib_0
    <br/>
    <b>
     Name:
    </b>
    CHON_JCP_2020
    <br/>
    <b>
     Authors:
    </b>
    Berk Onat, Christoph Ortner, James R. Kermode
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 5,216
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 96,736
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/DescriptorZoo/sensitivity-dimensionality-results/tree/master/datasets" target="_blank">
     https://github.com/DescriptorZoo/sensitivity-dimensionality-results/tree/master/datasets
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0016005" target="_blank">
     https://doi.org/10.1063/5.0016005
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_l9f0rjjqfd67_0" target="_blank">
     COHInPt_schaaf_2023
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training and simulation data from machine learning force field model applied to steps of the hydrogenation of carbon dioxide to methanol over an indium oxide catalyst, with and without platinum doping.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    COHInPt_schaaf_2023__Schaaf-Fako-De-Schafer-Csanyi__DS_l9f0rjjqfd67_0
    <br/>
    <b>
     Name:
    </b>
    COHInPt_schaaf_2023
    <br/>
    <b>
     Authors:
    </b>
    Lars Schaaf, Edvin Fako, Sandip De, Ansgar Schafer, Gabor Csanyi
    <br/>
    <b>
     Elements
    </b>
    : C, H, In, O, Pt
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,994
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 163,746
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.8268726" target="_blank">
     https://doi.org/10.5281/zenodo.8268726
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2301.09931" target="_blank">
     https://doi.org/10.48550/arXiv.2301.09931
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_anwiyxoek66t_0" target="_blank">
     COLL_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test set from COLL. Consists of configurations taken from molecular collisions of different small organic molecules. Energies and forces for 140,000 random snapshots taken from these trajectories were recomputed with density functional theory (DFT). These calculations were performed with the revPBE functional and def2-TZVP basis, including D3 dispersion corrections
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    COLL_test__Gasteiger-Giri-Margraf-Günnemann__DS_anwiyxoek66t_0
    <br/>
    <b>
     Name:
    </b>
    COLL_test
    <br/>
    <b>
     Authors:
    </b>
    Johannes Gasteiger, Shankari Giri, Johannes T. Margraf, Stephan Günnemann
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 9,480
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 97,886
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.13289165.v1" target="_blank">
     https://doi.org/10.6084/m9.figshare.13289165.v1
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2011.14115" target="_blank">
     https://doi.org/10.48550/arXiv.2011.14115
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_cifhpgzw3ckj_0" target="_blank">
     COLL_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training set from COLL. Consists of configurations taken from molecular collisions of different small organic molecules. Energies and forces for 140,000 random snapshots taken from these trajectories were recomputed with density functional theory (DFT). These calculations were performed with the revPBE functional and def2-TZVP basis, including D3 dispersion corrections
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    COLL_train__Gasteiger-Giri-Margraf-Günnemann__DS_cifhpgzw3ckj_0
    <br/>
    <b>
     Name:
    </b>
    COLL_train
    <br/>
    <b>
     Authors:
    </b>
    Johannes Gasteiger, Shankari Giri, Johannes T. Margraf, Stephan Günnemann
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 120,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,225,350
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.13289165.v1" target="_blank">
     https://doi.org/10.6084/m9.figshare.13289165.v1
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2011.14115" target="_blank">
     https://doi.org/10.48550/arXiv.2011.14115
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_1y25o4zvyfm0_0" target="_blank">
     COLL_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Validation set from COLL. Consists of configurations taken from molecular collisions of different small organic molecules. Energies and forces for 140,000 random snapshots taken from these trajectories were recomputed with density functional theory (DFT). These calculations were performed with the revPBE functional and def2-TZVP basis, including D3 dispersion corrections
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    COLL_validation__Gasteiger-Giri-Margraf-Günnemann__DS_1y25o4zvyfm0_0
    <br/>
    <b>
     Name:
    </b>
    COLL_validation
    <br/>
    <b>
     Authors:
    </b>
    Johannes Gasteiger, Shankari Giri, Johannes T. Margraf, Stephan Günnemann
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 10,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 101,847
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.13289165.v1" target="_blank">
     https://doi.org/10.6084/m9.figshare.13289165.v1
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2011.14115" target="_blank">
     https://doi.org/10.48550/arXiv.2011.14115
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_0dbwoxq96gga_0" target="_blank">
     COMP6v2-B973c-def2mTZVP
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    COMP6v2-B973c-def2mTZVP is the portion of COMP6v2 calculated at the B973c/def2mTZVP level of theory. COmprehensive Machine-learning Potential (COMP6) Benchmark Suite version 2.0 is an extension of the COMP6 benchmark found in the following repository: https://github.com/isayev/COMP6. COMP6v2 is a data set of density functional properties for molecules containing H, C, N, O, S, F, and Cl. It is available at the following levels of theory: wB97X/631Gd (data used to train model in the ANI-2x paper); wB97MD3BJ/def2TZVPP; wB97MV/def2TZVPP; B973c/def2mTZVP. The 6 subsets from COMP6 (ANI-MD, DrugBank, GDB07to09, GDB10to13 Tripeptides, and s66x8) are contained in each of the COMP6v2 datasets corresponding to the above levels of theory.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    COMP6v2-B973c-def2mTZVP__Huddleston-Zubatyuk-Smith-Roitberg-Isayev-Pickering-Devereux-Barros__DS_0dbwoxq96gga_0
    <br/>
    <b>
     Name:
    </b>
    COMP6v2-B973c-def2mTZVP
    <br/>
    <b>
     Authors:
    </b>
    Kate Huddleston, Roman Zubatyuk, Justin Smith, Adrian Roitberg, Olexandr Isayev, Ignacio Pickering, Christian Devereux, Kipton Barros
    <br/>
    <b>
     Elements
    </b>
    : C, Cl, F, H, N, O, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 156,330
    <br/>
    <b>
     Number of Elements
    </b>
    : 7
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,786,071
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.10126157" target="_blank">
     https://doi.org/10.5281/zenodo.10126157
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.0c00121" target="_blank">
     https://doi.org/10.1021/acs.jctc.0c00121
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_mznjdz4oqv11_0" target="_blank">
     COMP6v2-wB97MD3BJ-def2TZVPP
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    COMP6v2-wB97MD3BJ-def2TZVPP is the portion of COMP6v2 calculated at the wB97MD3BJ/def2TZVPP level of theory. COmprehensive Machine-learning Potential (COMP6) Benchmark Suite version 2.0 is an extension of the COMP6 benchmark found in the following repository: https://github.com/isayev/COMP6. COMP6v2 is a data set of density functional properties for molecules containing H, C, N, O, S, F, and Cl. It is available at the following levels of theory: wB97X/631Gd (data used to train model in the ANI-2x paper); wB97MD3BJ/def2TZVPP; wB97MV/def2TZVPP; B973c/def2mTZVP. The 6 subsets from COMP6 (ANI-MD, DrugBank, GDB07to09, GDB10to13 Tripeptides, and s66x8) are contained in each of the COMP6v2 datasets corresponding to the above levels of theory.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    COMP6v2-wB97MD3BJ-def2TZVPP__Huddleston-Zubatyuk-Smith-Roitberg-Isayev-Pickering-Devereux-Barros__DS_mznjdz4oqv11_0
    <br/>
    <b>
     Name:
    </b>
    COMP6v2-wB97MD3BJ-def2TZVPP
    <br/>
    <b>
     Authors:
    </b>
    Kate Huddleston, Roman Zubatyuk, Justin Smith, Adrian Roitberg, Olexandr Isayev, Ignacio Pickering, Christian Devereux, Kipton Barros
    <br/>
    <b>
     Elements
    </b>
    : C, Cl, F, H, N, O, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 156,353
    <br/>
    <b>
     Number of Elements
    </b>
    : 7
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,787,055
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.10126157" target="_blank">
     https://doi.org/10.5281/zenodo.10126157
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.0c00121" target="_blank">
     https://doi.org/10.1021/acs.jctc.0c00121
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_m6etjqhlu40m_0" target="_blank">
     COMP6v2-wB97MV-def2TZVPP
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    COMP6v2-wB97MV-def2TZVPP is the portion of COMP6v2 calculated at the wB97MV/def2TZVPP level of theory. COmprehensive Machine-learning Potential (COMP6) Benchmark Suite version 2.0 is an extension of the COMP6 benchmark found in the following repository: https://github.com/isayev/COMP6. COMP6v2 is a data set of density functional properties for molecules containing H, C, N, O, S, F, and Cl. It is available at the following levels of theory: wB97X/631Gd (data used to train model in the ANI-2x paper); wB97MD3BJ/def2TZVPP; wB97MV/def2TZVPP; B973c/def2mTZVP. The 6 subsets from COMP6 (ANI-MD, DrugBank, GDB07to09, GDB10to13 Tripeptides, and s66x8) are contained in each of the COMP6v2 datasets corresponding to the above levels of theory.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    COMP6v2-wB97MV-def2TZVPP__Huddleston-Zubatyuk-Smith-Roitberg-Isayev-Pickering-Devereux-Barros__DS_m6etjqhlu40m_0
    <br/>
    <b>
     Name:
    </b>
    COMP6v2-wB97MV-def2TZVPP
    <br/>
    <b>
     Authors:
    </b>
    Kate Huddleston, Roman Zubatyuk, Justin Smith, Adrian Roitberg, Olexandr Isayev, Ignacio Pickering, Christian Devereux, Kipton Barros
    <br/>
    <b>
     Elements
    </b>
    : C, Cl, F, H, N, O, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 156,369
    <br/>
    <b>
     Number of Elements
    </b>
    : 7
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,787,406
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.10126157" target="_blank">
     https://doi.org/10.5281/zenodo.10126157
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.0c00121" target="_blank">
     https://doi.org/10.1021/acs.jctc.0c00121
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_15m7etw1om8a_0" target="_blank">
     COMP6v2-wB97X-631Gd
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    COMP6v2-wB97X-631Gd is the portion of COMP6v2 calculated at the wB97X/631Gd level of theory. COmprehensive Machine-learning Potential (COMP6) Benchmark Suite version 2.0 is an extension of the COMP6 benchmark found in the following repository: https://github.com/isayev/COMP6. COMP6v2 is a data set of density functional properties for molecules containing H, C, N, O, S, F, and Cl. It is available at the following levels of theory: wB97X/631Gd (data used to train model in the ANI-2x paper); wB97MD3BJ/def2TZVPP; wB97MV/def2TZVPP; B973c/def2mTZVP. The 6 subsets from COMP6 (ANI-MD, DrugBank, GDB07to09, GDB10to13 Tripeptides, and s66x8) are contained in each of the COMP6v2 datasets corresponding to the above levels of theory.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    COMP6v2-wB97X-631Gd__Huddleston-Zubatyuk-Smith-Roitberg-Isayev-Pickering-Devereux-Barros__DS_15m7etw1om8a_0
    <br/>
    <b>
     Name:
    </b>
    COMP6v2-wB97X-631Gd
    <br/>
    <b>
     Authors:
    </b>
    Kate Huddleston, Roman Zubatyuk, Justin Smith, Adrian Roitberg, Olexandr Isayev, Ignacio Pickering, Christian Devereux, Kipton Barros
    <br/>
    <b>
     Elements
    </b>
    : C, Cl, F, H, N, O, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 157,728
    <br/>
    <b>
     Number of Elements
    </b>
    : 7
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,897,978
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.10126157" target="_blank">
     https://doi.org/10.5281/zenodo.10126157
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.0c00121" target="_blank">
     https://doi.org/10.1021/acs.jctc.0c00121
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_mma0w04wa4cs_0" target="_blank">
     C_Gardner_2022
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 115,000 configurations of carbon with 200 atoms, with simulated melt, quench, reheat, then annealing at the noted temperature. Includes a variety of carbon structures.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    C_Gardner_2022__Gardner-Beaulieu-Deringer__DS_mma0w04wa4cs_0
    <br/>
    <b>
     Name:
    </b>
    C_Gardner_2022
    <br/>
    <b>
     Authors:
    </b>
    John L. A. Gardner, Zoé Faure Beaulieu, Volker L. Deringer
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 115,206
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 23,041,200
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/jla-gardner/carbon-data" target="_blank">
     https://github.com/jla-gardner/carbon-data
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2211.16443" target="_blank">
     https://doi.org/10.48550/arXiv.2211.16443
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_qph0akhjv9kv_0" target="_blank">
     C_NPJ2020
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The dataset consists of energies and forces for monolayer graphene, bilayer graphene, graphite, and diamond in various states, including strained static structures and configurations drawn from ab initio MD trajectories. A total number of 4788 configurations was generated from DFT calculations using the Vienna Ab initio Simulation Package (VASP). The energies and forces are stored in the extended XYZ format. One file for each configuration.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    C_NPJ2020__Wen-Tadmor__DS_qph0akhjv9kv_0
    <br/>
    <b>
     Name:
    </b>
    C_NPJ2020
    <br/>
    <b>
     Authors:
    </b>
    Mingjian Wen, Ellad B. Tadmor
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 4,776
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 228,852
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.12649811.v1" target="_blank">
     https://doi.org/10.6084/m9.figshare.12649811.v1
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-020-00390-8" target="_blank">
     https://doi.org/10.1038/s41524-020-00390-8
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_on6fusdw5n8q_0" target="_blank">
     Carbon_GAP_JCP_2020
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    GAP-20 describes the properties of the bulk crystalline and amorphous phases, crystal surfaces, and defect structures with an accuracy approaching that of direct ab initio simulation, but at a significantly reduced cost. The final potential is fitted to reference data computed using the optB88-vdW density functional theory (DFT) functional.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Carbon_GAP_JCP_2020__Rowe-Deringer-Gasparotto-Csányi-Michaelides__DS_on6fusdw5n8q_0
    <br/>
    <b>
     Name:
    </b>
    Carbon_GAP_JCP_2020
    <br/>
    <b>
     Authors:
    </b>
    Patrick Rowe, Volker L. Deringer, Piero Gasparotto, Gábor Csányi, Angelos Michaelides
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 17,227
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,312,457
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://www.repository.cam.ac.uk/handle/1810/307452" target="_blank">
     https://www.repository.cam.ac.uk/handle/1810/307452
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0005084" target="_blank">
     https://doi.org/10.1063/5.0005084
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_sz9me9r61581_0" target="_blank">
     Carbon_GAP_JCP_2020_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training data generated for GAP-20. GAP-20 describes the properties of the bulk crystalline and amorphous phases, crystal surfaces, and defect structures with an accuracy approaching that of direct ab initio simulation, but at a significantly reduced cost. The final potential is fitted to reference data computed using the optB88-vdW density functional theory (DFT) functional.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Carbon_GAP_JCP_2020_train__Rowe-Deringer-Gasparotto-Csányi-Michaelides__DS_sz9me9r61581_0
    <br/>
    <b>
     Name:
    </b>
    Carbon_GAP_JCP_2020_train
    <br/>
    <b>
     Authors:
    </b>
    Patrick Rowe, Volker L. Deringer, Piero Gasparotto, Gábor Csányi, Angelos Michaelides
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 6,088
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 400,275
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://www.repository.cam.ac.uk/handle/1810/307452" target="_blank">
     https://www.repository.cam.ac.uk/handle/1810/307452
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0005084" target="_blank">
     https://doi.org/10.1063/5.0005084
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_zn8tf2kmgrdm_0" target="_blank">
     Carbon_allotrope_multilayer_graphene_graphite_PRB2019
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The dataset consists of energies and forces for pristine and defected monolayer graphene, bilayer graphene, and
graphite in various states. The configurations in the dataset are generated in two ways: (1) crystals with distortions
(compression and stretching of the simulation cell together with random perturbations of atoms), and (2) configura-
tions drawn from ab initio molecular dynamics (AIMD) trajectories at 300, 900, and 1500 K.
For monolayer graphene, the configurations include:
* pristine
- In-plane compressed and stretched monolayers
- AIMD trajectories
* defected
- Configurations from the minimization of a monolayer with a single vacancy
- AIMD trajectories of monolayers with a single vacancy
For bilayer graphene, the configurations include:
* pristine
- AB-stacked bilayers with compression and stretching in the basal plane
- Bilayers with different translational registry (e.g. AA, AB, and SP) at various layer separations
- Twisted bilayers with different twisting angles at various layer separations
- AIMD trajectories of twisted bilayers and bilayers in AB and AA stackings
* defected
- Configurations from the minimization of a bilayer with a single vacancy in each layer
- AIMD trajectories of a bilayer with a single vacancy in one layer and the other layer pristine
- AIMD trajectories of a bilayer with a single vacancy in each layer; Initial configuration without interlayer
bonds
- AIMD trajectories of a bilayer with a single vacancy in each layer; Initial configuration with interlayer
bonds formed
For graphite, the configurations include:
* pristine
- Graphite with compression and stretching in the basal plane
- Graphite with compression and stretching along the c-axis
- AIMD trajectories
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Carbon_allotrope_multilayer_graphene_graphite_PRB2019__Wen-Tadmor__DS_zn8tf2kmgrdm_0
    <br/>
    <b>
     Name:
    </b>
    Carbon_allotrope_multilayer_graphene_graphite_PRB2019
    <br/>
    <b>
     Authors:
    </b>
    Mingjian Wen, Ellad B. Tadmor
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 14,179
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 656,204
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://journals.aps.org/prb/supplemental/10.1103/PhysRevB.100.195419/dataset.tar" target="_blank">
     https://journals.aps.org/prb/supplemental/10.1103/PhysRevB.100.195419/dataset.tar
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevB.100.195419" target="_blank">
     https://doi.org/10.1103/PhysRevB.100.195419
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_r2r8k3fyb6ny_0" target="_blank">
     Carolina_Materials
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Carolina Materials contains structures used to train several machine learning models for the efficient generation of hypothetical inorganic materials. The database is built using structures from OQMD, Materials Project and ICSD, as well as ML generated structures validated by DFT.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Carolina_Materials__Zhao-Al-Fahdi-Hu-Siriwardane-Song-Nasiri-Hu__DS_r2r8k3fyb6ny_0
    <br/>
    <b>
     Name:
    </b>
    Carolina_Materials
    <br/>
    <b>
     Authors:
    </b>
    Yong Zhao, Mohammed Al-Fahdi, Ming Hu, Edirisuriya M. D. Siriwardane, Yuqi Song, Alireza Nasiri, Jianjun Hu
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, F, Fe, Ga, Ge, H, Hf, Hg, I, In, Ir, K, Li, Mg, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Po, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 214,280
    <br/>
    <b>
     Number of Elements
    </b>
    : 64
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,168,502
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://zenodo.org/records/8381476" target="_blank">
     https://zenodo.org/records/8381476
    </a>
    <br/>
    <a href="https://doi.org/10.1002/advs.202100566" target="_blank">
     https://doi.org/10.1002/advs.202100566
    </a>
    <br/>
    <a href="http://www.carolinamatdb.org/" target="_blank">
     http://www.carolinamatdb.org/
    </a>
    <br/>
    <a href="https://github.com/IntelLabs/matsciml" target="_blank">
     https://github.com/IntelLabs/matsciml
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_y6bp7td4dle0_0" target="_blank">
     Cationic_phenoxyimine_complexes_of_yttrium
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset contains DFT calculations that were carried out in conjunction with experimental investigation of a cationic phenoxyimine yttrium complex as an isoprene polimerization catalyst. Calculations were performed using the Gaussian 09 D.01 suite of programs.Electronic structure calculations were performed at the DFT level using the B3PW91 functional. The Stuttgart-Cologne small-core quasi-relativistic pseudopotential ECP28MWB and its available basis set including up to the g function were used to describe yttrium. Similarly, silicon and phosphorus were represented by a Stuttgart-Dresden-Bonn pseudopotential along with the related basis set augmented by a d function of polarization (αd(P) = 0.387 and αd(Si) = 0.284). Other atoms were described by a polarized all-electron triple-ζ 6-311G(d,p) basis set. Bulk solvent effect of toluene or THF was simulated using the SMD continuum model. The Grimme empirical correction with the original D3 damping function was used to include the dispersion correction as a single-point calculation. Transition-state optimization was followed by frequency calculations to characterize the stationary point. Intrinsic reaction coordinate calculations were performed to confirm the connectivity of the transition states. Gibbs energies were estimated within the harmonic oscillator approximation and estimated at 298 K and 1 atm.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Cationic_phenoxyimine_complexes_of_yttrium__Oswald-Verrieux-Breuil-Olivier-Bourbigou-Thuilliez-Vaultier-Taoufik-Perrin-Boisson__DS_y6bp7td4dle0_0
    <br/>
    <b>
     Name:
    </b>
    Cationic_phenoxyimine_complexes_of_yttrium
    <br/>
    <b>
     Authors:
    </b>
    Alexis D. Oswald, Ludmilla Verrieux, Pierre-Alain R. Breuil, Hélène Olivier-Bourbigou, Julien Thuilliez, Florent Vaultier, Mostafa Taoufik, Lionel Perrin, Christophe Boisson
    <br/>
    <b>
     Elements
    </b>
    : Al, B, C, F, H, N, O, Si, Y
    <br/>
    <b>
     Number of Configurations
    </b>
    : 109
    <br/>
    <b>
     Number of Elements
    </b>
    : 9
    <br/>
    <b>
     Number of Atoms
    </b>
    : 9,074
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1021/acs.organomet.2c00238.s001" target="_blank">
     https://doi.org/10.1021/acs.organomet.2c00238.s001
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.organomet.2c00238" target="_blank">
     https://doi.org/10.1021/acs.organomet.2c00238
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_e08dhew7z0r6_0" target="_blank">
     Chig-AIMD_random_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test configurations from the 'random' split of Chig-AIMD. This dataset covers the conformational space of chignolin with DFT-level precision. We sequentially applied replica exchange molecular dynamics (REMD), conventional MD, and ab initio MD (AIMD) simulations on a 10 amino acid protein, Chignolin, and finally collected 2 million biomolecule structures with quantum level energy and force records.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Chig-AIMD_random_test__Wang-He-Li-Shao-Liu__DS_e08dhew7z0r6_0
    <br/>
    <b>
     Name:
    </b>
    Chig-AIMD_random_test
    <br/>
    <b>
     Authors:
    </b>
    Tong Wang, Xinheng He, Mingyu Li, Bin Shao, Tie-Yan Liu
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 199,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 33,034,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1038/s41597-023-02465-9" target="_blank">
     https://doi.org/10.1038/s41597-023-02465-9
    </a>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.22786730.v4" target="_blank">
     https://doi.org/10.6084/m9.figshare.22786730.v4
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_dsikv10na4f8_0" target="_blank">
     Chig-AIMD_random_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training configurations from the 'random' split of Chig-AIMD. This dataset covers the conformational space of chignolin with DFT-level precision. We sequentially applied replica exchange molecular dynamics (REMD), conventional MD, and ab initio MD (AIMD) simulations on a 10 amino acid protein, Chignolin, and finally collected 2 million biomolecule structures with quantum level energy and force records.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Chig-AIMD_random_train__Wang-He-Li-Shao-Liu__DS_dsikv10na4f8_0
    <br/>
    <b>
     Name:
    </b>
    Chig-AIMD_random_train
    <br/>
    <b>
     Authors:
    </b>
    Tong Wang, Xinheng He, Mingyu Li, Bin Shao, Tie-Yan Liu
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,592,800
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 264,404,800
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1038/s41597-023-02465-9" target="_blank">
     https://doi.org/10.1038/s41597-023-02465-9
    </a>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.22786730.v4" target="_blank">
     https://doi.org/10.6084/m9.figshare.22786730.v4
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_u0ggh1iie0uc_0" target="_blank">
     Chig-AIMD_random_val
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Validation configurations from the 'random' split of Chig-AIMD. This dataset covers the conformational space of chignolin with DFT-level precision. We sequentially applied replica exchange molecular dynamics (REMD), conventional MD, and ab initio MD (AIMD) simulations on a 10 amino acid protein, Chignolin, and finally collected 2 million biomolecule structures with quantum level energy and force records.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Chig-AIMD_random_val__Wang-He-Li-Shao-Liu__DS_u0ggh1iie0uc_0
    <br/>
    <b>
     Name:
    </b>
    Chig-AIMD_random_val
    <br/>
    <b>
     Authors:
    </b>
    Tong Wang, Xinheng He, Mingyu Li, Bin Shao, Tie-Yan Liu
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 199,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 33,034,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1038/s41597-023-02465-9" target="_blank">
     https://doi.org/10.1038/s41597-023-02465-9
    </a>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.22786730.v4" target="_blank">
     https://doi.org/10.6084/m9.figshare.22786730.v4
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_zmzkyvep8l55_0" target="_blank">
     Chig-AIMD_scaffold_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test configurations from the 'scaffold' split of Chig-AIMD. This dataset covers the conformational space of chignolin with DFT-level precision. We sequentially applied replica exchange molecular dynamics (REMD), conventional MD, and ab initio MD (AIMD) simulations on a 10 amino acid protein, Chignolin, and finally collected 2 million biomolecule structures with quantum level energy and force records.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Chig-AIMD_scaffold_test__Wang-He-Li-Shao-Liu__DS_zmzkyvep8l55_0
    <br/>
    <b>
     Name:
    </b>
    Chig-AIMD_scaffold_test
    <br/>
    <b>
     Authors:
    </b>
    Tong Wang, Xinheng He, Mingyu Li, Bin Shao, Tie-Yan Liu
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 199,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 33,034,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1038/s41597-023-02465-9" target="_blank">
     https://doi.org/10.1038/s41597-023-02465-9
    </a>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.22786730.v4" target="_blank">
     https://doi.org/10.6084/m9.figshare.22786730.v4
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_7puixss6qd61_0" target="_blank">
     Chig-AIMD_scaffold_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training configurations from the 'scaffold' split of Chig-AIMD. This dataset covers the conformational space of chignolin with DFT-level precision. We sequentially applied replica exchange molecular dynamics (REMD), conventional MD, and ab initio MD (AIMD) simulations on a 10 amino acid protein, Chignolin, and finally collected 2 million biomolecule structures with quantum level energy and force records.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Chig-AIMD_scaffold_train__Wang-He-Li-Shao-Liu__DS_7puixss6qd61_0
    <br/>
    <b>
     Name:
    </b>
    Chig-AIMD_scaffold_train
    <br/>
    <b>
     Authors:
    </b>
    Tong Wang, Xinheng He, Mingyu Li, Bin Shao, Tie-Yan Liu
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,592,800
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 264,404,800
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1038/s41597-023-02465-9" target="_blank">
     https://doi.org/10.1038/s41597-023-02465-9
    </a>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.22786730.v4" target="_blank">
     https://doi.org/10.6084/m9.figshare.22786730.v4
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_mzz13lim5qfi_0" target="_blank">
     Chig-AIMD_scaffold_val
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Validation configurations from the 'scaffold' split of Chig-AIMD. This dataset covers the conformational space of chignolin with DFT-level precision. We sequentially applied replica exchange molecular dynamics (REMD), conventional MD, and ab initio MD (AIMD) simulations on a 10 amino acid protein, Chignolin, and finally collected 2 million biomolecule structures with quantum level energy and force records.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Chig-AIMD_scaffold_val__Wang-He-Li-Shao-Liu__DS_mzz13lim5qfi_0
    <br/>
    <b>
     Name:
    </b>
    Chig-AIMD_scaffold_val
    <br/>
    <b>
     Authors:
    </b>
    Tong Wang, Xinheng He, Mingyu Li, Bin Shao, Tie-Yan Liu
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 199,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 33,034,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1038/s41597-023-02465-9" target="_blank">
     https://doi.org/10.1038/s41597-023-02465-9
    </a>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.22786730.v4" target="_blank">
     https://doi.org/10.6084/m9.figshare.22786730.v4
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_3a3auhtmkv45_0" target="_blank">
     Co-Co_coupling_at_liquid_water-Cu(100)_interfaces_JC2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset contains data from eight AIMD simulations run in VASP to study electrochemical *CO-*CO coupling -- coupling of two *CO molecules -- at the liquid water-Cu(100) interface.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Co-Co_coupling_at_liquid_water-Cu(100)_interfaces_JC2021__Kristoffersen-Chan__DS_3a3auhtmkv45_0
    <br/>
    <b>
     Name:
    </b>
    Co-Co_coupling_at_liquid_water-Cu(100)_interfaces_JC2021
    <br/>
    <b>
     Authors:
    </b>
    Henrik H. Kristoffersen, Karen Chan
    <br/>
    <b>
     Elements
    </b>
    : C, Cs, Cu, H, Li, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,671,203
    <br/>
    <b>
     Number of Elements
    </b>
    : 6
    <br/>
    <b>
     Number of Atoms
    </b>
    : 226,264,514
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:p9-q7" target="_blank">
     https://doi.org/10.24435/materialscloud:p9-q7
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.jcat.2021.02.023" target="_blank">
     https://doi.org/10.1016/j.jcat.2021.02.023
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_4w9gf8dxzskh_0" target="_blank">
     CoCrFeNiPd_MRL2020
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The dataset for "Origin of high strength in the CoCrFeNiPd high-entropy alloy", containing DFT-calculated values of the high-entropy alloy CoCrFeNiPd, created to explore the reasons behind experimental findings of the increased strength CoCrFeNiPd in comparison to CoCrFeNi.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CoCrFeNiPd_MRL2020__Yin-Curtin__DS_4w9gf8dxzskh_0
    <br/>
    <b>
     Name:
    </b>
    CoCrFeNiPd_MRL2020
    <br/>
    <b>
     Authors:
    </b>
    Binglun Yin, W. A. Curtin
    <br/>
    <b>
     Elements
    </b>
    : Co, Cr, Fe, Ni, Pd
    <br/>
    <b>
     Number of Configurations
    </b>
    : 116
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 8,552
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:2020.0045/v1" target="_blank">
     https://doi.org/10.24435/materialscloud:2020.0045/v1
    </a>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:2020.0045/v1" target="_blank">
     https://doi.org/10.24435/materialscloud:2020.0045/v1
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_sn623uhg2d1b_0" target="_blank">
     CoNbV_CMS2019
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset was generated using the following active learning scheme: 1) candidate structures were relaxed by a partially-trained MTP model, 2) structures for which the MTP had to perform extrapolation were passed to DFT to be re-computed, 3) the MTP was retrained, including the structures that were re-computed with DFT, 4) steps 1-3 were repeated until the MTP no longer extrapolated on any of the original candidate structures. The original candidate structures for this dataset included about 27,000 configurations that were bcc-like and close-packed (fcc, hcp, etc.) with 8 or fewer atoms in the unit cell and different concentrations of Co, Nb, and V.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CoNbV_CMS2019__Gubaev-Podryabinkin-Hart-Shapeev__DS_sn623uhg2d1b_0
    <br/>
    <b>
     Name:
    </b>
    CoNbV_CMS2019
    <br/>
    <b>
     Authors:
    </b>
    Konstantin Gubaev, Evgeny V. Podryabinkin, Gus L.W. Hart, Alexander V. Shapeev
    <br/>
    <b>
     Elements
    </b>
    : Co, Nb, V
    <br/>
    <b>
     Number of Configurations
    </b>
    : 383
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,812
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://gitlab.com/kgubaev/accelerating-high-throughput-searches-for-new-alloys-with-active-learning-data" target="_blank">
     https://gitlab.com/kgubaev/accelerating-high-throughput-searches-for-new-alloys-with-active-learning-data
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.commatsci.2018.09.031" target="_blank">
     https://doi.org/10.1016/j.commatsci.2018.09.031
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_uwqpfuuj6utw_0" target="_blank">
     Co_dimer_JPCA_2022
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset contains dimer molecules of Co(II) with potential energy calculations for structures with ferromagnetic and antiferromagnetic spin configurations. Calculations were carried out in Gaussian 16 with the PBE exchange-correlation functional and 6-31+G* basis set. All molecules contain the same atomic core region, consisting of the tetrahedral and octahedral Co centers and the three PO2R2 bridging ligands. The ligand exchange provides a broad range of exchange energies (ΔEJ), from +50 to -200 meV, with 80% of the ligands yielding ΔEJ &lt; 10 meV.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Co_dimer_JPCA_2022__Ren-Fonseca-Perry-Cheng-Zhang-Hennig__DS_uwqpfuuj6utw_0
    <br/>
    <b>
     Name:
    </b>
    Co_dimer_JPCA_2022
    <br/>
    <b>
     Authors:
    </b>
    Sijin Ren, Eric Fonseca, William Perry, Hai-Ping Cheng, Xiao-Guang Zhang, Richard Hennig
    <br/>
    <b>
     Elements
    </b>
    : C, Cl, Co, H, N, O, P, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,162
    <br/>
    <b>
     Number of Elements
    </b>
    : 8
    <br/>
    <b>
     Number of Atoms
    </b>
    : 188,438
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:pe-zv" target="_blank">
     https://doi.org/10.24435/materialscloud:pe-zv
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpca.1c08950" target="_blank">
     https://doi.org/10.1021/acs.jpca.1c08950
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_doo9ltj3vsnu_0" target="_blank">
     Co_dimer_JPCA_2022_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training data only from the Co_dimer_JPCA_2022 dataset. This dataset contains dimer molecules of Co(II) with potential energy calculations for structures with ferromagnetic and antiferromagnetic spin configurations. Calculations were carried out in Gaussian 16 with the PBE exchange-correlation functional and 6-31+G* basis set. All molecules contain the same atomic core region, consisting of the tetrahedral and octahedral Co centers and the three PO2R2 bridging ligands. The ligand exchange provides a broad range of exchange energies (ΔEJ), from +50 to -200 meV, with 80% of the ligands yielding ΔEJ &lt; 10 meV.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Co_dimer_JPCA_2022_train__Ren-Fonseca-Perry-Cheng-Zhang-Hennig__DS_doo9ltj3vsnu_0
    <br/>
    <b>
     Name:
    </b>
    Co_dimer_JPCA_2022_train
    <br/>
    <b>
     Authors:
    </b>
    Sijin Ren, Eric Fonseca, William Perry, Hai-Ping Cheng, Xiao-Guang Zhang, Richard Hennig
    <br/>
    <b>
     Elements
    </b>
    : C, Cl, Co, H, N, O, P, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,798
    <br/>
    <b>
     Number of Elements
    </b>
    : 8
    <br/>
    <b>
     Number of Atoms
    </b>
    : 154,882
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:pe-zv" target="_blank">
     https://doi.org/10.24435/materialscloud:pe-zv
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpca.1c08950" target="_blank">
     https://doi.org/10.1021/acs.jpca.1c08950
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_xs942mj0c3dx_0" target="_blank">
     ComBat
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    DFT-optimized geometries and properties for Li-S electrolytes. These make up the Computational Database for Li-S Batteries (ComBat), calculated using Gaussian 16 at the B3LYP/6-31+G* level of theory.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    ComBat__Atwi-Bliss-Makeev-Rajput__DS_xs942mj0c3dx_0
    <br/>
    <b>
     Name:
    </b>
    ComBat
    <br/>
    <b>
     Authors:
    </b>
    Rasha Atwi, Matthew Bliss, Maxim Makeev, Nav Nidhi Rajput
    <br/>
    <b>
     Elements
    </b>
    : C, F, H, Li, N, O, P, S, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 230
    <br/>
    <b>
     Number of Elements
    </b>
    : 9
    <br/>
    <b>
     Number of Atoms
    </b>
    : 5,662
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/rashatwi/combat/" target="_blank">
     https://github.com/rashatwi/combat/
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41598-022-20009-w" target="_blank">
     https://doi.org/10.1038/s41598-022-20009-w
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_z2mkok0egrm8_0" target="_blank">
     CrCoNi_Cao_2022
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training dataset that captures chemical short-range order in equiatomic CrCoNi medium-entropy alloy published with our work Quantifying chemical short-range order in metallic alloys (description provided by authors)
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CrCoNi_Cao_2022__Cao-Sheriff-Freitas__DS_z2mkok0egrm8_0
    <br/>
    <b>
     Name:
    </b>
    CrCoNi_Cao_2022
    <br/>
    <b>
     Authors:
    </b>
    Yifan Cao, Killian Sheriff, Rodrigo Freitas
    <br/>
    <b>
     Elements
    </b>
    : Co, Cr, Ni
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,257
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 108,684
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/yifan-henry-cao/MachineLearningPotential/blob/main/Training_datasets/Training_Cao_20220823.cfg" target="_blank">
     https://github.com/yifan-henry-cao/MachineLearningPotential/blob/main/Training_datasets/Training_Cao_20220823.cfg
    </a>
    <br/>
    <a href="https://arxiv.org/abs/2311.01545" target="_blank">
     https://arxiv.org/abs/2311.01545
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_0ry6z1j8mi8c_0" target="_blank">
     CuPd_CMS2019
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset was generated using the following active learning scheme: 1) candidate structures were relaxed by a partially-trained MTP model, 2) structures for which the MTP had to perform extrapolation were passed to DFT to be re-computed, 3) the MTP was retrained, including the structures that were re-computed with DFT, 4) steps 1-3 were repeated until the MTP no longer extrapolated on any of the original candidate structures. The original candidate structures for this dataset included 40,000 unrelaxed configurations with BCC, FCC, and HCP lattices.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    CuPd_CMS2019__Gubaev-Podryabinkin-Hart-Shapeev__DS_0ry6z1j8mi8c_0
    <br/>
    <b>
     Name:
    </b>
    CuPd_CMS2019
    <br/>
    <b>
     Authors:
    </b>
    Konstantin Gubaev, Evgeny V. Podryabinkin, Gus L.W. Hart, Alexander V. Shapeev
    <br/>
    <b>
     Elements
    </b>
    : Cu, Pd
    <br/>
    <b>
     Number of Configurations
    </b>
    : 522
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,450
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://gitlab.com/kgubaev/accelerating-high-throughput-searches-for-new-alloys-with-active-learning-data" target="_blank">
     https://gitlab.com/kgubaev/accelerating-high-throughput-searches-for-new-alloys-with-active-learning-data
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.commatsci.2018.09.031" target="_blank">
     https://doi.org/10.1016/j.commatsci.2018.09.031
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_39v8vn61tlzl_0" target="_blank">
     Cu_FHI-aims_NPJCM_2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 46,000 configurations of copper, including small and bulk structures, surfaces, interfaces, point defects, and randomly modified variants. Also includes structures with displaced or missing atoms.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Cu_FHI-aims_NPJCM_2021__Lysogorskiy-Oord-Bochkarev-Menon-Rinaldi-Hammerschmidt-Mrovec-Thompson-Csányi-Ortner-Drautz__DS_39v8vn61tlzl_0
    <br/>
    <b>
     Name:
    </b>
    Cu_FHI-aims_NPJCM_2021
    <br/>
    <b>
     Authors:
    </b>
    Yury Lysogorskiy, Cas van der Oord, Anton Bochkarev, Sarath Menon, Matteo Rinaldi, Thomas Hammerschmidt, Matous Mrovec, Aidan Thompson, Gábor Csányi, Christoph Ortner, Ralf Drautz
    <br/>
    <b>
     Elements
    </b>
    : Cu
    <br/>
    <b>
     Number of Configurations
    </b>
    : 46,583
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 308,679
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.4734035" target="_blank">
     https://doi.org/10.5281/zenodo.4734035
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-021-00559-9" target="_blank">
     https://doi.org/10.1038/s41524-021-00559-9
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_eyo7scxvrfq0_0" target="_blank">
     DAS_MLIP_CoSb_MgSb
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 850 configurations of CoSb3 and Mg3Sb2 generated using a dual adaptive sampling (DAS) method for use with machine learning of interatomic potentials (MLIP).
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DAS_MLIP_CoSb_MgSb__Yang-Zhu-Dong-Wu-Yang-Zhang__DS_eyo7scxvrfq0_0
    <br/>
    <b>
     Name:
    </b>
    DAS_MLIP_CoSb_MgSb
    <br/>
    <b>
     Authors:
    </b>
    Hongliang Yang, Yifan Zhu, Erting Dong, Yabei Wu, Jiong Yang, Wenqing Zhang
    <br/>
    <b>
     Elements
    </b>
    : Mg, Sb
    <br/>
    <b>
     Number of Configurations
    </b>
    : 846
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 247,744
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevB.104.094310" target="_blank">
     https://doi.org/10.1103/PhysRevB.104.094310
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevB.104.094310" target="_blank">
     https://doi.org/10.1103/PhysRevB.104.094310
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_ech19sh1koj4_0" target="_blank">
     DFT_polymorphs_PNAS_2022_PBE0_MBD_benzene_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Benzene test PBE0-MBD dataset from "Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2π A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DFT_polymorphs_PNAS_2022_PBE0_MBD_benzene_test__Kapil-Engel__DS_ech19sh1koj4_0
    <br/>
    <b>
     Name:
    </b>
    DFT_polymorphs_PNAS_2022_PBE0_MBD_benzene_test
    <br/>
    <b>
     Authors:
    </b>
    Venkat Kapil, Edgar A. Engel
    <br/>
    <b>
     Elements
    </b>
    : C, H
    <br/>
    <b>
     Number of Configurations
    </b>
    : 200
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 5,760
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:vp-jf" target="_blank">
     https://doi.org/10.24435/materialscloud:vp-jf
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2111769119" target="_blank">
     https://doi.org/10.1073/pnas.2111769119
    </a>
    <br/>
    <a href="https://github.com/venkatkapil24/data_molecular_fluctuations" target="_blank">
     https://github.com/venkatkapil24/data_molecular_fluctuations
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_ce4zytu7ph7e_0" target="_blank">
     DFT_polymorphs_PNAS_2022_PBE0_MBD_benzene_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Benzene training PBE0-MBD dataset from "Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2π A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DFT_polymorphs_PNAS_2022_PBE0_MBD_benzene_train__Kapil-Engel__DS_ce4zytu7ph7e_0
    <br/>
    <b>
     Name:
    </b>
    DFT_polymorphs_PNAS_2022_PBE0_MBD_benzene_train
    <br/>
    <b>
     Authors:
    </b>
    Venkat Kapil, Edgar A. Engel
    <br/>
    <b>
     Elements
    </b>
    : C, H
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,800
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 49,536
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:vp-jf" target="_blank">
     https://doi.org/10.24435/materialscloud:vp-jf
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2111769119" target="_blank">
     https://doi.org/10.1073/pnas.2111769119
    </a>
    <br/>
    <a href="https://github.com/venkatkapil24/data_molecular_fluctuations" target="_blank">
     https://github.com/venkatkapil24/data_molecular_fluctuations
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_xaq6jwha0a7o_0" target="_blank">
     DFT_polymorphs_PNAS_2022_PBE0_MBD_benzene_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Benzene validation PBE0-MBD dataset from "Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2π A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DFT_polymorphs_PNAS_2022_PBE0_MBD_benzene_validation__Kapil-Engel__DS_xaq6jwha0a7o_0
    <br/>
    <b>
     Name:
    </b>
    DFT_polymorphs_PNAS_2022_PBE0_MBD_benzene_validation
    <br/>
    <b>
     Authors:
    </b>
    Venkat Kapil, Edgar A. Engel
    <br/>
    <b>
     Elements
    </b>
    : C, H
    <br/>
    <b>
     Number of Configurations
    </b>
    : 200
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 6,072
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:vp-jf" target="_blank">
     https://doi.org/10.24435/materialscloud:vp-jf
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2111769119" target="_blank">
     https://doi.org/10.1073/pnas.2111769119
    </a>
    <br/>
    <a href="https://github.com/venkatkapil24/data_molecular_fluctuations" target="_blank">
     https://github.com/venkatkapil24/data_molecular_fluctuations
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_h3jymhr74qv4_0" target="_blank">
     DFT_polymorphs_PNAS_2022_PBE0_MBD_glycine_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Glycine test PBE0-MBD dataset from "Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2π A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DFT_polymorphs_PNAS_2022_PBE0_MBD_glycine_test__Kapil-Engel__DS_h3jymhr74qv4_0
    <br/>
    <b>
     Name:
    </b>
    DFT_polymorphs_PNAS_2022_PBE0_MBD_glycine_test
    <br/>
    <b>
     Authors:
    </b>
    Venkat Kapil, Edgar A. Engel
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 200
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 6,880
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:vp-jf" target="_blank">
     https://doi.org/10.24435/materialscloud:vp-jf
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2111769119" target="_blank">
     https://doi.org/10.1073/pnas.2111769119
    </a>
    <br/>
    <a href="https://github.com/venkatkapil24/data_molecular_fluctuations" target="_blank">
     https://github.com/venkatkapil24/data_molecular_fluctuations
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_b1n228x610dj_0" target="_blank">
     DFT_polymorphs_PNAS_2022_PBE0_MBD_glycine_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Glycine training PBE0-MBD dataset from "Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2π A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DFT_polymorphs_PNAS_2022_PBE0_MBD_glycine_train__Kapil-Engel__DS_b1n228x610dj_0
    <br/>
    <b>
     Name:
    </b>
    DFT_polymorphs_PNAS_2022_PBE0_MBD_glycine_train
    <br/>
    <b>
     Authors:
    </b>
    Venkat Kapil, Edgar A. Engel
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,594
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 109,940
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:vp-jf" target="_blank">
     https://doi.org/10.24435/materialscloud:vp-jf
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2111769119" target="_blank">
     https://doi.org/10.1073/pnas.2111769119
    </a>
    <br/>
    <a href="https://github.com/venkatkapil24/data_molecular_fluctuations" target="_blank">
     https://github.com/venkatkapil24/data_molecular_fluctuations
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_n1nilf7ralhq_0" target="_blank">
     DFT_polymorphs_PNAS_2022_PBE0_MBD_glycine_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Glycine validation PBE0-MBD dataset from "Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2π A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DFT_polymorphs_PNAS_2022_PBE0_MBD_glycine_validation__Kapil-Engel__DS_n1nilf7ralhq_0
    <br/>
    <b>
     Name:
    </b>
    DFT_polymorphs_PNAS_2022_PBE0_MBD_glycine_validation
    <br/>
    <b>
     Authors:
    </b>
    Venkat Kapil, Edgar A. Engel
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 200
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 7,120
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:vp-jf" target="_blank">
     https://doi.org/10.24435/materialscloud:vp-jf
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2111769119" target="_blank">
     https://doi.org/10.1073/pnas.2111769119
    </a>
    <br/>
    <a href="https://github.com/venkatkapil24/data_molecular_fluctuations" target="_blank">
     https://github.com/venkatkapil24/data_molecular_fluctuations
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_vbjvssevrff5_0" target="_blank">
     DFT_polymorphs_PNAS_2022_PBE0_MBD_succinic_acid_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Succinic acid test PBE0-MBD dataset from "Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2π A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DFT_polymorphs_PNAS_2022_PBE0_MBD_succinic_acid_test__Kapil-Engel__DS_vbjvssevrff5_0
    <br/>
    <b>
     Name:
    </b>
    DFT_polymorphs_PNAS_2022_PBE0_MBD_succinic_acid_test
    <br/>
    <b>
     Authors:
    </b>
    Venkat Kapil, Edgar A. Engel
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 200
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 5,600
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:vp-jf" target="_blank">
     https://doi.org/10.24435/materialscloud:vp-jf
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2111769119" target="_blank">
     https://doi.org/10.1073/pnas.2111769119
    </a>
    <br/>
    <a href="https://github.com/venkatkapil24/data_molecular_fluctuations" target="_blank">
     https://github.com/venkatkapil24/data_molecular_fluctuations
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_pqmtyp5bxd46_0" target="_blank">
     DFT_polymorphs_PNAS_2022_PBE0_MBD_succinic_acid_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Succinic acid training PBE0-MBD dataset from "Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2π A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DFT_polymorphs_PNAS_2022_PBE0_MBD_succinic_acid_train__Kapil-Engel__DS_pqmtyp5bxd46_0
    <br/>
    <b>
     Name:
    </b>
    DFT_polymorphs_PNAS_2022_PBE0_MBD_succinic_acid_train
    <br/>
    <b>
     Authors:
    </b>
    Venkat Kapil, Edgar A. Engel
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,800
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 50,400
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:vp-jf" target="_blank">
     https://doi.org/10.24435/materialscloud:vp-jf
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2111769119" target="_blank">
     https://doi.org/10.1073/pnas.2111769119
    </a>
    <br/>
    <a href="https://github.com/venkatkapil24/data_molecular_fluctuations" target="_blank">
     https://github.com/venkatkapil24/data_molecular_fluctuations
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_w18131j6r4oo_0" target="_blank">
     DFT_polymorphs_PNAS_2022_PBE0_MBD_succinic_acid_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Succinic acid validation PBE0-MBD dataset from "Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2π A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DFT_polymorphs_PNAS_2022_PBE0_MBD_succinic_acid_validation__Kapil-Engel__DS_w18131j6r4oo_0
    <br/>
    <b>
     Name:
    </b>
    DFT_polymorphs_PNAS_2022_PBE0_MBD_succinic_acid_validation
    <br/>
    <b>
     Authors:
    </b>
    Venkat Kapil, Edgar A. Engel
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 200
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 5,600
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:vp-jf" target="_blank">
     https://doi.org/10.24435/materialscloud:vp-jf
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2111769119" target="_blank">
     https://doi.org/10.1073/pnas.2111769119
    </a>
    <br/>
    <a href="https://github.com/venkatkapil24/data_molecular_fluctuations" target="_blank">
     https://github.com/venkatkapil24/data_molecular_fluctuations
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_23820czfaq24_0" target="_blank">
     DFT_polymorphs_PNAS_2022_PBE_TS_benzene_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Benzene test PBE-TS dataset from "Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2π A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DFT_polymorphs_PNAS_2022_PBE_TS_benzene_test__Kapil-Engel__DS_23820czfaq24_0
    <br/>
    <b>
     Name:
    </b>
    DFT_polymorphs_PNAS_2022_PBE_TS_benzene_test
    <br/>
    <b>
     Authors:
    </b>
    Venkat Kapil, Edgar A. Engel
    <br/>
    <b>
     Elements
    </b>
    : C, H
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 29,736
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:vp-jf" target="_blank">
     https://doi.org/10.24435/materialscloud:vp-jf
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2111769119" target="_blank">
     https://doi.org/10.1073/pnas.2111769119
    </a>
    <br/>
    <a href="https://github.com/venkatkapil24/data_molecular_fluctuations" target="_blank">
     https://github.com/venkatkapil24/data_molecular_fluctuations
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_3qi25f3sxkwr_0" target="_blank">
     DFT_polymorphs_PNAS_2022_PBE_TS_benzene_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Benzene training PBE-TS dataset from "Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2π A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DFT_polymorphs_PNAS_2022_PBE_TS_benzene_train__Kapil-Engel__DS_3qi25f3sxkwr_0
    <br/>
    <b>
     Name:
    </b>
    DFT_polymorphs_PNAS_2022_PBE_TS_benzene_train
    <br/>
    <b>
     Authors:
    </b>
    Venkat Kapil, Edgar A. Engel
    <br/>
    <b>
     Elements
    </b>
    : C, H
    <br/>
    <b>
     Number of Configurations
    </b>
    : 55,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,602,048
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:vp-jf" target="_blank">
     https://doi.org/10.24435/materialscloud:vp-jf
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2111769119" target="_blank">
     https://doi.org/10.1073/pnas.2111769119
    </a>
    <br/>
    <a href="https://github.com/venkatkapil24/data_molecular_fluctuations" target="_blank">
     https://github.com/venkatkapil24/data_molecular_fluctuations
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_8ou78o18mvfz_0" target="_blank">
     DFT_polymorphs_PNAS_2022_PBE_TS_benzene_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Benzene validation PBE-TS dataset from "Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2π A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DFT_polymorphs_PNAS_2022_PBE_TS_benzene_validation__Kapil-Engel__DS_8ou78o18mvfz_0
    <br/>
    <b>
     Name:
    </b>
    DFT_polymorphs_PNAS_2022_PBE_TS_benzene_validation
    <br/>
    <b>
     Authors:
    </b>
    Venkat Kapil, Edgar A. Engel
    <br/>
    <b>
     Elements
    </b>
    : C, H
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 29,712
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:vp-jf" target="_blank">
     https://doi.org/10.24435/materialscloud:vp-jf
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2111769119" target="_blank">
     https://doi.org/10.1073/pnas.2111769119
    </a>
    <br/>
    <a href="https://github.com/venkatkapil24/data_molecular_fluctuations" target="_blank">
     https://github.com/venkatkapil24/data_molecular_fluctuations
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_qx4h8j4luk70_0" target="_blank">
     DFT_polymorphs_PNAS_2022_PBE_TS_glycine_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Glycine test PBE-TS dataset from "Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2π A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DFT_polymorphs_PNAS_2022_PBE_TS_glycine_test__Kapil-Engel__DS_qx4h8j4luk70_0
    <br/>
    <b>
     Name:
    </b>
    DFT_polymorphs_PNAS_2022_PBE_TS_glycine_test
    <br/>
    <b>
     Authors:
    </b>
    Venkat Kapil, Edgar A. Engel
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 500
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 17,710
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:vp-jf" target="_blank">
     https://doi.org/10.24435/materialscloud:vp-jf
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2111769119" target="_blank">
     https://doi.org/10.1073/pnas.2111769119
    </a>
    <br/>
    <a href="https://github.com/venkatkapil24/data_molecular_fluctuations" target="_blank">
     https://github.com/venkatkapil24/data_molecular_fluctuations
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_op9kvcm7ui6l_0" target="_blank">
     DFT_polymorphs_PNAS_2022_PBE_TS_glycine_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Glycine training PBE-TS dataset from "Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2π A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DFT_polymorphs_PNAS_2022_PBE_TS_glycine_train__Kapil-Engel__DS_op9kvcm7ui6l_0
    <br/>
    <b>
     Name:
    </b>
    DFT_polymorphs_PNAS_2022_PBE_TS_glycine_train
    <br/>
    <b>
     Authors:
    </b>
    Venkat Kapil, Edgar A. Engel
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 29,070
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 952,650
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:vp-jf" target="_blank">
     https://doi.org/10.24435/materialscloud:vp-jf
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2111769119" target="_blank">
     https://doi.org/10.1073/pnas.2111769119
    </a>
    <br/>
    <a href="https://github.com/venkatkapil24/data_molecular_fluctuations" target="_blank">
     https://github.com/venkatkapil24/data_molecular_fluctuations
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_052bb0zufjaz_0" target="_blank">
     DFT_polymorphs_PNAS_2022_PBE_TS_glycine_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Glycine validation PBE-TS dataset from "Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2π A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DFT_polymorphs_PNAS_2022_PBE_TS_glycine_validation__Kapil-Engel__DS_052bb0zufjaz_0
    <br/>
    <b>
     Name:
    </b>
    DFT_polymorphs_PNAS_2022_PBE_TS_glycine_validation
    <br/>
    <b>
     Authors:
    </b>
    Venkat Kapil, Edgar A. Engel
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 500
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 17,800
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:vp-jf" target="_blank">
     https://doi.org/10.24435/materialscloud:vp-jf
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2111769119" target="_blank">
     https://doi.org/10.1073/pnas.2111769119
    </a>
    <br/>
    <a href="https://github.com/venkatkapil24/data_molecular_fluctuations" target="_blank">
     https://github.com/venkatkapil24/data_molecular_fluctuations
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_qmsg3kejmsy2_0" target="_blank">
     DFT_polymorphs_PNAS_2022_PBE_TS_succinic_acid_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Succinic acid test PBE-TS dataset from "Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2π A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DFT_polymorphs_PNAS_2022_PBE_TS_succinic_acid_test__Kapil-Engel__DS_qmsg3kejmsy2_0
    <br/>
    <b>
     Name:
    </b>
    DFT_polymorphs_PNAS_2022_PBE_TS_succinic_acid_test
    <br/>
    <b>
     Authors:
    </b>
    Venkat Kapil, Edgar A. Engel
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 500
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 14,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:vp-jf" target="_blank">
     https://doi.org/10.24435/materialscloud:vp-jf
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2111769119" target="_blank">
     https://doi.org/10.1073/pnas.2111769119
    </a>
    <br/>
    <a href="https://github.com/venkatkapil24/data_molecular_fluctuations" target="_blank">
     https://github.com/venkatkapil24/data_molecular_fluctuations
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_0u0k9ghrlkpx_0" target="_blank">
     DFT_polymorphs_PNAS_2022_PBE_TS_succinic_acid_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Succinic acid training PBE-TS dataset from "Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2π A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DFT_polymorphs_PNAS_2022_PBE_TS_succinic_acid_train__Kapil-Engel__DS_0u0k9ghrlkpx_0
    <br/>
    <b>
     Name:
    </b>
    DFT_polymorphs_PNAS_2022_PBE_TS_succinic_acid_train
    <br/>
    <b>
     Authors:
    </b>
    Venkat Kapil, Edgar A. Engel
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 29,212
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 817,936
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:vp-jf" target="_blank">
     https://doi.org/10.24435/materialscloud:vp-jf
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2111769119" target="_blank">
     https://doi.org/10.1073/pnas.2111769119
    </a>
    <br/>
    <a href="https://github.com/venkatkapil24/data_molecular_fluctuations" target="_blank">
     https://github.com/venkatkapil24/data_molecular_fluctuations
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_nny0vpj1dy9s_0" target="_blank">
     DFT_polymorphs_PNAS_2022_PBE_TS_succinic_acid_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Succinic acid validation PBE-TS dataset from "Semi-local and hybrid functional DFT data for thermalised snapshots of polymorphs of benzene, succinic acid, and glycine". DFT reference energies and forces were calculated using Quantum Espresso v6.3. The calculations were performed with the semi-local PBE xc functional, Tkatchenko-Scheffler dispersion correction, optimised norm-conserving Vanderbilt pseudopotentials, a Monkhorst-Pack k-point grid with a maximum spacing of 0.06 x 2π A^-1, and a plane-wave energy cut-off of 100 Ry for the wavefunction.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DFT_polymorphs_PNAS_2022_PBE_TS_succinic_acid_validation__Kapil-Engel__DS_nny0vpj1dy9s_0
    <br/>
    <b>
     Name:
    </b>
    DFT_polymorphs_PNAS_2022_PBE_TS_succinic_acid_validation
    <br/>
    <b>
     Authors:
    </b>
    Venkat Kapil, Edgar A. Engel
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 500
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 14,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:vp-jf" target="_blank">
     https://doi.org/10.24435/materialscloud:vp-jf
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2111769119" target="_blank">
     https://doi.org/10.1073/pnas.2111769119
    </a>
    <br/>
    <a href="https://github.com/venkatkapil24/data_molecular_fluctuations" target="_blank">
     https://github.com/venkatkapil24/data_molecular_fluctuations
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_101uk70asqhq_0" target="_blank">
     DP-GEN_Cu
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 15,000 configurations of copper used to demonstrate the DP-GEN data generator for PES machine learning models.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DP-GEN_Cu__Zhang-Wang-Chen-Zeng-Zhang-Wang-E__DS_101uk70asqhq_0
    <br/>
    <b>
     Name:
    </b>
    DP-GEN_Cu
    <br/>
    <b>
     Authors:
    </b>
    Yuzhi Zhang, Haidi Wang, Weijie Chen, Jinzhe Zeng, Linfeng Zhang, Han Wang, Weinan E
    <br/>
    <b>
     Elements
    </b>
    : Cu
    <br/>
    <b>
     Number of Configurations
    </b>
    : 15,286
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 297,597
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=Cu-dpgen" target="_blank">
     https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=Cu-dpgen
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.cpc.2020.107206" target="_blank">
     https://doi.org/10.1016/j.cpc.2020.107206
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_k85kj1kiekip_0" target="_blank">
     DeePMD_SE
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    127,000 configurations from a dataset used to benchmark and train a modified DeePMD model called DeepPot-SE, or Deep Potential - Smooth Edition
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    DeePMD_SE__Zhang-Han-Wang-Saidi-Car-E__DS_k85kj1kiekip_0
    <br/>
    <b>
     Name:
    </b>
    DeePMD_SE
    <br/>
    <b>
     Authors:
    </b>
    Linfeng Zhang, Jiequn Han, Han Wang, Wissam A. Saidi, Roberto Car, Weinan E
    <br/>
    <b>
     Elements
    </b>
    : Al, C, Co, Cr, Cu, Fe, Ge, H, Mn, Mo, N, Ni, O, Pt, S, Si, Ti
    <br/>
    <b>
     Number of Configurations
    </b>
    : 127,112
    <br/>
    <b>
     Number of Elements
    </b>
    : 17
    <br/>
    <b>
     Number of Atoms
    </b>
    : 26,278,380
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=deepmd-se-dataset" target="_blank">
     https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=deepmd-se-dataset
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.1805.09003" target="_blank">
     https://doi.org/10.48550/arXiv.1805.09003
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_5h3810yhu4wj_0" target="_blank">
     Fe_nanoparticles_PRB_2023
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This iron nanoparticles database contains dimers; trimers; bcc, fcc, hexagonal close-packed (hcp), simple cubic, and diamond crystalline structures. A wide range of cell parameters, as well as rattled structures, bcc-fcc and bcc-hcp transitional structures, surface slabs cleaved from relaxed bulk structures, nanoparticles and liquid configurations are included. The energy, forces and virials for the atomic structures were computed at the DFT level of theory using VASP with the PBE functional and standard PAW pseudopotentials for Fe (with 8 valence electrons, 4s^23d^6). The kinetic energy cutoff for plane waves was set to 400 eV and the energy threshold for convergence was 10-7 eV. All the DFT calculations were carried out with spin polarization.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Fe_nanoparticles_PRB_2023__Jana-Caro__DS_5h3810yhu4wj_0
    <br/>
    <b>
     Name:
    </b>
    Fe_nanoparticles_PRB_2023
    <br/>
    <b>
     Authors:
    </b>
    Richard Jana, Miguel A. Caro
    <br/>
    <b>
     Elements
    </b>
    : Fe
    <br/>
    <b>
     Number of Configurations
    </b>
    : 198
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 20,097
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7632315" target="_blank">
     https://doi.org/10.5281/zenodo.7632315
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevB.107.245421" target="_blank">
     https://doi.org/10.1103/PhysRevB.107.245421
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_ej5u0tuycoph_0" target="_blank">
     FitSNAP_Fe_NPJ_2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    About 2,500 configurations of alpha-Fe used in the training and testing of a ML model with the goal of building magneto-elastic machine-learning interatomic potentials for large-scale spin-lattice dynamics simulations.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    FitSNAP_Fe_NPJ_2021__Nikolov-Wood-Cangi-Maillet-Marinica-Thompson-Desjarlais-Tranchida__DS_ej5u0tuycoph_0
    <br/>
    <b>
     Name:
    </b>
    FitSNAP_Fe_NPJ_2021
    <br/>
    <b>
     Authors:
    </b>
    Svetoslav Nikolov, Mitchell A. Wood, Attila Cangi, Jean-Bernard Maillet, Mihai-Cosmin Marinica, Aidan P. Thompson, Michael P. Desjarlais, Julien Tranchida
    <br/>
    <b>
     Elements
    </b>
    : Fe
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,517
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 61,526
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/FitSNAP" target="_blank">
     https://github.com/FitSNAP
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-021-00617-2" target="_blank">
     https://doi.org/10.1038/s41524-021-00617-2
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_tku3ae1rtxiy_0" target="_blank">
     Forces_are_not_enough
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 300,000 benchmarking configurations derived partly from the MD-17 and LiPS datasets, partly from original simulated water and alanine dipeptide configurations.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Forces_are_not_enough__Fu-Wu-Wang-Xie-Keten-Gomez-Bombarelli-Jaakkola__DS_tku3ae1rtxiy_0
    <br/>
    <b>
     Name:
    </b>
    Forces_are_not_enough
    <br/>
    <b>
     Authors:
    </b>
    Xiang Fu, Zhenghao Wu, Wujie Wang, Tian Xie, Sinan Keten, Rafael Gomez-Bombarelli, Tommi Jaakkola
    <br/>
    <b>
     Elements
    </b>
    : C, H, Li, N, O, P, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 295,001
    <br/>
    <b>
     Number of Elements
    </b>
    : 7
    <br/>
    <b>
     Number of Atoms
    </b>
    : 23,735,083
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7196767" target="_blank">
     https://doi.org/10.5281/zenodo.7196767
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2210.07237" target="_blank">
     https://doi.org/10.48550/arXiv.2210.07237
    </a>
    <br/>
    <a href="https://github.com/kyonofx/MDsim/" target="_blank">
     https://github.com/kyonofx/MDsim/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_2s5grfmdez1q_0" target="_blank">
     GDB_9_nature_2014
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    133,855 configurations of stable small organic molecules composed of CHONF. A subset of GDB-17, with calculations of energies, dipole moment, polarizability and enthalpy. Calculations performed at B3LYP/6-31G(2df,p) level of theory.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    GDB_9_nature_2014__Ramakrishnan-Dral-Rupp-Lilienfeld__DS_2s5grfmdez1q_0
    <br/>
    <b>
     Name:
    </b>
    GDB_9_nature_2014
    <br/>
    <b>
     Authors:
    </b>
    Raghunathan Ramakrishnan, Pavlo O. Dral, Matthias Rupp, O. Anatole von Lilienfeld
    <br/>
    <b>
     Elements
    </b>
    : C, F, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 133,885
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,407,753
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.c.978904.v5" target="_blank">
     https://doi.org/10.6084/m9.figshare.c.978904.v5
    </a>
    <br/>
    <a href="https://doi.org/10.1038/sdata.2014.22" target="_blank">
     https://doi.org/10.1038/sdata.2014.22
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_0x8ozlt9g3y5_0" target="_blank">
     GFN-xTB_JCIM_2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    10,000 configurations of organosilicon compounds with energies predicted by an improved GFN-xTB Hamiltonian parameterization, using revPBE.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    GFN-xTB_JCIM_2021__Komissarov-Verstraelen__DS_0x8ozlt9g3y5_0
    <br/>
    <b>
     Name:
    </b>
    GFN-xTB_JCIM_2021
    <br/>
    <b>
     Authors:
    </b>
    Leonid Komissarov, Toon Verstraelen
    <br/>
    <b>
     Elements
    </b>
    : Br, C, Cl, F, H, N, O, P, S, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 157,367
    <br/>
    <b>
     Number of Elements
    </b>
    : 10
    <br/>
    <b>
     Number of Atoms
    </b>
    : 4,022,149
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:14-4m" target="_blank">
     https://doi.org/10.24435/materialscloud:14-4m
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jcim.1c01170" target="_blank">
     https://doi.org/10.1021/acs.jcim.1c01170
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_efcq7a0h6z5w_0" target="_blank">
     GST_GAP_22_extended
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The extended training dataset for GST_GAP_22, calculated using the PBEsol functional. New configurations, simulated under external electric fields, were labelled with DFT and added to the original reference database GST-GAP-22 contains configurations of phase-change materials on the quasi-binary GeTe-Sb2Te3 (GST) line of chemical compositions. Data was used for training a machine learning interatomic potential to simulate a range of germanium-antimony-tellurium compositions under realistic device conditions.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    GST_GAP_22_extended__Zhou-Zhang-Ma-Deringer__DS_efcq7a0h6z5w_0
    <br/>
    <b>
     Name:
    </b>
    GST_GAP_22_extended
    <br/>
    <b>
     Authors:
    </b>
    Yuxing Zhou, Wei Zhang, Evan Ma, Volker L. Deringer
    <br/>
    <b>
     Elements
    </b>
    : Ge, Sb, Te
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,916
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 399,247
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.8208202" target="_blank">
     https://doi.org/10.5281/zenodo.8208202
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41928-023-01030-x" target="_blank">
     https://doi.org/10.1038/s41928-023-01030-x
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_r3hav37ufnmb_0" target="_blank">
     GST_GAP_22_main
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The main training dataset for GST_GAP_22, calculated using the PBEsol functional. GST-GAP-22 contains configurations of phase-change materials on the quasi-binary GeTe-Sb2Te3 (GST) line of chemical compositions. Data was used for training a machine learning interatomic potential to simulate a range of germanium-antimony-tellurium compositions under realistic device conditions.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    GST_GAP_22_main__Zhou-Zhang-Ma-Deringer__DS_r3hav37ufnmb_0
    <br/>
    <b>
     Name:
    </b>
    GST_GAP_22_main
    <br/>
    <b>
     Authors:
    </b>
    Yuxing Zhou, Wei Zhang, Evan Ma, Volker L. Deringer
    <br/>
    <b>
     Elements
    </b>
    : Ge, Sb, Te
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,692
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 341,068
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.8208202" target="_blank">
     https://doi.org/10.5281/zenodo.8208202
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41928-023-01030-x" target="_blank">
     https://doi.org/10.1038/s41928-023-01030-x
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_jy3ylaf48xg3_0" target="_blank">
     GST_GAP_22_refitted
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The training dataset for GST_GAP_22, recalculated using the PBE functional. GST-GAP-22 contains configurations of phase-change materials on the quasi-binary GeTe-Sb2Te3 (GST) line of chemical compositions. Data was used for training a machine learning interatomic potential to simulate a range of germanium-antimony-tellurium compositions under realistic device conditions.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    GST_GAP_22_refitted__Zhou-Zhang-Ma-Deringer__DS_jy3ylaf48xg3_0
    <br/>
    <b>
     Name:
    </b>
    GST_GAP_22_refitted
    <br/>
    <b>
     Authors:
    </b>
    Yuxing Zhou, Wei Zhang, Evan Ma, Volker L. Deringer
    <br/>
    <b>
     Elements
    </b>
    : Ge, Sb, Te
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,692
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 341,004
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.8208202" target="_blank">
     https://doi.org/10.5281/zenodo.8208202
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41928-023-01030-x" target="_blank">
     https://doi.org/10.1038/s41928-023-01030-x
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_in2572v33lon_0" target="_blank">
     HDNNP_H2O
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 28,000 configurations split into 4 datasets, each using a different functional, used in the training of a high-dimensional neural network potential (HDNNP).
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    HDNNP_H2O__Morawietz-Behler__DS_in2572v33lon_0
    <br/>
    <b>
     Name:
    </b>
    HDNNP_H2O
    <br/>
    <b>
     Authors:
    </b>
    Tobias Morawietz, Jörg Behler
    <br/>
    <b>
     Elements
    </b>
    : H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 28,678
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,327,628
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.2634097" target="_blank">
     https://doi.org/10.5281/zenodo.2634097
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.1602375113" target="_blank">
     https://doi.org/10.1073/pnas.1602375113
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevLett.98.146401" target="_blank">
     https://doi.org/10.1103/PhysRevLett.98.146401
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_d5zht3ykr8hi_0" target="_blank">
     HEA25S_high_entropy_alloys
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset from "Surface segregation in high-entropy alloys from alchemical machine learning: dataset HEA25S". Includes 10000 bulk HEA structures (Dataset O), 2640 HEA surface slabs (Dataset A), together with 1000 bulk and 1000 surface slabs snapshots from the molecular dynamics (MD) runs (Datasets B and C), and 500 MD snapshots of the 25 elements Cantor-style alloy surface slabs. These splits, along with their respective train, test, and validation splits, are included as configuration sets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    HEA25S_high_entropy_alloys__Mazitov-Springer-Lopanitsyna-Fraux-De-Ceriotti__DS_d5zht3ykr8hi_0
    <br/>
    <b>
     Name:
    </b>
    HEA25S_high_entropy_alloys
    <br/>
    <b>
     Authors:
    </b>
    Arslan Mazitov, Maximilian A. Springer, Nataliya Lopanitsyna, Guillaume Fraux, Sandip De, Michele Ceriotti
    <br/>
    <b>
     Elements
    </b>
    : Ag, Au, Co, Cr, Cu, Fe, Hf, Ir, Lu, Mn, Mo, Nb, Ni, Pd, Pt, Rh, Ru, Sc, Ta, Ti, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 15,004
    <br/>
    <b>
     Number of Elements
    </b>
    : 25
    <br/>
    <b>
     Number of Atoms
    </b>
    : 633,387
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:ps-20" target="_blank">
     https://doi.org/10.24435/materialscloud:ps-20
    </a>
    <br/>
    <a href="http://doi.org/10.48550/arXiv.2310.07604" target="_blank">
     http://doi.org/10.48550/arXiv.2310.07604
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_hvx1spm3gmia_0" target="_blank">
     HEA25_high_entropy_transition-metal_alloys
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset from "Modeling high-entropy transition-metal alloys with alchemical compression". Includes 25,000 structures utilized for fitting the aforementioned potential, with a focus on 25 d-block transition metals, excluding Tc, Cd, Re, Os and Hg. Each configuration includes a "class" field, indicating the crystal class of the structure. The class represents the following: 1: perfect crystals; 3-8 elements per structure, 2: shuffled positions (standard deviation 0.2\AA ); 3-8 elements per structure, 3: shuffled positions (standard deviation 0.5\AA ); 3-8 elements per structure, 4: shuffled positions (standard deviation 0.2\AA ); 3-25 elements per structure. Configuration sets include divisions into fcc and bcc crystals, further split by class as described above.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    HEA25_high_entropy_transition-metal_alloys__Lopanitsyna-Fraux-Springer-De-Ceriotti__DS_hvx1spm3gmia_0
    <br/>
    <b>
     Name:
    </b>
    HEA25_high_entropy_transition-metal_alloys
    <br/>
    <b>
     Authors:
    </b>
    Nataliya Lopanitsyna, Guillaume Fraux, Maximilian A. Springer, Sandip De, Michele Ceriotti
    <br/>
    <b>
     Elements
    </b>
    : Ag, Au, Co, Cr, Cu, Fe, Hf, Ir, Lu, Mn, Mo, Nb, Ni, Pd, Pt, Rh, Ru, Sc, Ta, Ti, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 25,627
    <br/>
    <b>
     Number of Elements
    </b>
    : 25
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,063,680
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:73-yn" target="_blank">
     https://doi.org/10.24435/materialscloud:73-yn
    </a>
    <br/>
    <a href="http://doi.org/10.48550/arXiv.2212.13254" target="_blank">
     http://doi.org/10.48550/arXiv.2212.13254
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_cpgyq72fs7uk_0" target="_blank">
     HME21_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The test set from HME21. The  high-temperature multi-element 2021 (HME21) dataset comprises approximately 25,000 configurations, including 37 elements, used in the training of a universal NNP called PreFerential Potential (PFP). The dataset specifically contains disordered and unstable structures, and structures that include irregular substitutions, as well as varied temperature and density.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    HME21_test__Takamoto-Shinagawa-Motoki-Nakago-Li-Kurata-Watanabe-Yayama-Iriguchi-Asano-Onodera-Ishii-Kudo-Ono-Sawada-Ishitani-Ong-Yamaguchi-Kataoka-Hayashi-Charoenphakdee-Ibuka__DS_cpgyq72fs7uk_0
    <br/>
    <b>
     Name:
    </b>
    HME21_test
    <br/>
    <b>
     Authors:
    </b>
    So Takamoto, Chikashi Shinagawa, Daisuke Motoki, Kosuke Nakago, Wenwen Li, Iori Kurata, Taku Watanabe, Yoshihiro Yayama, Hiroki Iriguchi, Yusuke Asano, Tasuku Onodera, Takafumi Ishii, Takao Kudo, Hideki Ono, Ryohto Sawada, Ryuichiro Ishitani, Marc Ong, Taiki Yamaguchi, Toshiki Kataoka, Akihide Hayashi, Nontawat Charoenphakdee, Takeshi Ibuka
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, Au, Ba, C, Ca, Cl, Co, Cr, Cu, F, Fe, H, In, Ir, K, Li, Mg, Mn, Mo, N, Na, Ni, O, P, Pb, Pd, Pt, Rh, Ru, S, Sc, Si, Sn, Ti, V, Zn
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,495
    <br/>
    <b>
     Number of Elements
    </b>
    : 37
    <br/>
    <b>
     Number of Atoms
    </b>
    : 69,572
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.19658538.v2" target="_blank">
     https://doi.org/10.6084/m9.figshare.19658538.v2
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-022-30687-9" target="_blank">
     https://doi.org/10.1038/s41467-022-30687-9
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_jhfis7syauhm_0" target="_blank">
     HME21_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The training set from HME21. The  high-temperature multi-element 2021 (HME21) dataset comprises approximately 25,000 configurations, including 37 elements, used in the training of a universal NNP called PreFerential Potential (PFP). The dataset specifically contains disordered and unstable structures, and structures that include irregular substitutions, as well as varied temperature and density.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    HME21_train__Takamoto-Shinagawa-Motoki-Nakago-Li-Kurata-Watanabe-Yayama-Iriguchi-Asano-Onodera-Ishii-Kudo-Ono-Sawada-Ishitani-Ong-Yamaguchi-Kataoka-Hayashi-Charoenphakdee-Ibuka__DS_jhfis7syauhm_0
    <br/>
    <b>
     Name:
    </b>
    HME21_train
    <br/>
    <b>
     Authors:
    </b>
    So Takamoto, Chikashi Shinagawa, Daisuke Motoki, Kosuke Nakago, Wenwen Li, Iori Kurata, Taku Watanabe, Yoshihiro Yayama, Hiroki Iriguchi, Yusuke Asano, Tasuku Onodera, Takafumi Ishii, Takao Kudo, Hideki Ono, Ryohto Sawada, Ryuichiro Ishitani, Marc Ong, Taiki Yamaguchi, Toshiki Kataoka, Akihide Hayashi, Nontawat Charoenphakdee, Takeshi Ibuka
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, Au, Ba, C, Ca, Cl, Co, Cr, Cu, F, Fe, H, In, Ir, K, Li, Mg, Mn, Mo, N, Na, Ni, O, P, Pb, Pd, Pt, Rh, Ru, S, Sc, Si, Sn, Ti, V, Zn
    <br/>
    <b>
     Number of Configurations
    </b>
    : 19,956
    <br/>
    <b>
     Number of Elements
    </b>
    : 37
    <br/>
    <b>
     Number of Atoms
    </b>
    : 555,050
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.19658538.v2" target="_blank">
     https://doi.org/10.6084/m9.figshare.19658538.v2
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-022-30687-9" target="_blank">
     https://doi.org/10.1038/s41467-022-30687-9
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_vf8hkwxhibio_0" target="_blank">
     HME21_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The validation set from HME21. The  high-temperature multi-element 2021 (HME21) dataset comprises approximately 25,000 configurations, including 37 elements, used in the training of a universal NNP called PreFerential Potential (PFP). The dataset specifically contains disordered and unstable structures, and structures that include irregular substitutions, as well as varied temperature and density.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    HME21_validation__Takamoto-Shinagawa-Motoki-Nakago-Li-Kurata-Watanabe-Yayama-Iriguchi-Asano-Onodera-Ishii-Kudo-Ono-Sawada-Ishitani-Ong-Yamaguchi-Kataoka-Hayashi-Charoenphakdee-Ibuka__DS_vf8hkwxhibio_0
    <br/>
    <b>
     Name:
    </b>
    HME21_validation
    <br/>
    <b>
     Authors:
    </b>
    So Takamoto, Chikashi Shinagawa, Daisuke Motoki, Kosuke Nakago, Wenwen Li, Iori Kurata, Taku Watanabe, Yoshihiro Yayama, Hiroki Iriguchi, Yusuke Asano, Tasuku Onodera, Takafumi Ishii, Takao Kudo, Hideki Ono, Ryohto Sawada, Ryuichiro Ishitani, Marc Ong, Taiki Yamaguchi, Toshiki Kataoka, Akihide Hayashi, Nontawat Charoenphakdee, Takeshi Ibuka
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, Au, Ba, C, Ca, Cl, Co, Cr, Cu, F, Fe, H, In, Ir, K, Li, Mg, Mn, Mo, N, Na, Ni, O, P, Pb, Pd, Pt, Rh, Ru, S, Sc, Si, Sn, Ti, V, Zn
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,498
    <br/>
    <b>
     Number of Elements
    </b>
    : 37
    <br/>
    <b>
     Number of Atoms
    </b>
    : 69,420
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.19658538.v2" target="_blank">
     https://doi.org/10.6084/m9.figshare.19658538.v2
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-022-30687-9" target="_blank">
     https://doi.org/10.1038/s41467-022-30687-9
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_1i4q8uyams32_0" target="_blank">
     HO_LiMoNiTi_NPJCM_2020_LiMoNiTi_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training configurations of Li8Mo2Ni7Ti7O32 from HO_LiMoNiTi_NPJCM_2020 used in the training of an ANN, whereby total energy is extrapolated by a Taylor expansion as a means of reducing computational costs.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    HO_LiMoNiTi_NPJCM_2020_LiMoNiTi_train__Cooper-Kästner-Urban-Artrith__DS_1i4q8uyams32_0
    <br/>
    <b>
     Name:
    </b>
    HO_LiMoNiTi_NPJCM_2020_LiMoNiTi_train
    <br/>
    <b>
     Authors:
    </b>
    April M. Cooper, Johannes Kästner, Alexander Urban, Nongnuch Artrith
    <br/>
    <b>
     Elements
    </b>
    : Li, Mo, Ni, O, Ti
    <br/>
    <b>
     Number of Configurations
    </b>
    : 824
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 46,144
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:2020.0037/v1" target="_blank">
     https://doi.org/10.24435/materialscloud:2020.0037/v1
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-020-0323-8" target="_blank">
     https://doi.org/10.1038/s41524-020-0323-8
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_fgjil336jos6_0" target="_blank">
     HO_LiMoNiTi_NPJCM_2020_LiMoNiTi_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Validation configurations of Li8Mo2Ni7Ti7O32 from HO_LiMoNiTi_NPJCM_2020 used in the training of an ANN, whereby total energy is extrapolated by a Taylor expansion as a means of reducing computational costs.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    HO_LiMoNiTi_NPJCM_2020_LiMoNiTi_validation__Cooper-Kästner-Urban-Artrith__DS_fgjil336jos6_0
    <br/>
    <b>
     Name:
    </b>
    HO_LiMoNiTi_NPJCM_2020_LiMoNiTi_validation
    <br/>
    <b>
     Authors:
    </b>
    April M. Cooper, Johannes Kästner, Alexander Urban, Nongnuch Artrith
    <br/>
    <b>
     Elements
    </b>
    : Li, Mo, Ni, O, Ti
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,792
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 100,352
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:2020.0037/v1" target="_blank">
     https://doi.org/10.24435/materialscloud:2020.0037/v1
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-020-0323-8" target="_blank">
     https://doi.org/10.1038/s41524-020-0323-8
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_jjiywlyfpnme_0" target="_blank">
     HO_LiMoNiTi_NPJCM_2020_bulk_water_train_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training and testing configurations of bulk water from HO_LiMoNiTi_NPJCM_2020 used in the training of an ANN, whereby total energy is extrapolated by a Taylor expansion as a means of reducing computational costs.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    HO_LiMoNiTi_NPJCM_2020_bulk_water_train_test__Cooper-Kästner-Urban-Artrith__DS_jjiywlyfpnme_0
    <br/>
    <b>
     Name:
    </b>
    HO_LiMoNiTi_NPJCM_2020_bulk_water_train_test
    <br/>
    <b>
     Authors:
    </b>
    April M. Cooper, Johannes Kästner, Alexander Urban, Nongnuch Artrith
    <br/>
    <b>
     Elements
    </b>
    : H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 700
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 134,400
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:2020.0037/v1" target="_blank">
     https://doi.org/10.24435/materialscloud:2020.0037/v1
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-020-0323-8" target="_blank">
     https://doi.org/10.1038/s41524-020-0323-8
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_w3whk1nr1bld_0" target="_blank">
     HO_LiMoNiTi_NPJCM_2020_bulk_water_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Validation configurations of bulk water from HO_LiMoNiTi_NPJCM_2020 used in the training of an ANN, whereby total energy is extrapolated by a Taylor expansion as a means of reducing computational costs.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    HO_LiMoNiTi_NPJCM_2020_bulk_water_validation__Cooper-Kästner-Urban-Artrith__DS_w3whk1nr1bld_0
    <br/>
    <b>
     Name:
    </b>
    HO_LiMoNiTi_NPJCM_2020_bulk_water_validation
    <br/>
    <b>
     Authors:
    </b>
    April M. Cooper, Johannes Kästner, Alexander Urban, Nongnuch Artrith
    <br/>
    <b>
     Elements
    </b>
    : H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,112
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 405,504
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:2020.0037/v1" target="_blank">
     https://doi.org/10.24435/materialscloud:2020.0037/v1
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-020-0323-8" target="_blank">
     https://doi.org/10.1038/s41524-020-0323-8
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_or3nu4t64mvk_0" target="_blank">
     HO_LiMoNiTi_NPJCM_2020_water_clusters
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of water clusters from HO_LiMoNiTi_NPJCM_2020 used in the training of an ANN, whereby total energy is extrapolated by a Taylor expansion as a means of reducing computational costs.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    HO_LiMoNiTi_NPJCM_2020_water_clusters__Cooper-Kästner-Urban-Artrith__DS_or3nu4t64mvk_0
    <br/>
    <b>
     Name:
    </b>
    HO_LiMoNiTi_NPJCM_2020_water_clusters
    <br/>
    <b>
     Authors:
    </b>
    April M. Cooper, Johannes Kästner, Alexander Urban, Nongnuch Artrith
    <br/>
    <b>
     Elements
    </b>
    : H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,848
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 33,264
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:2020.0037/v1" target="_blank">
     https://doi.org/10.24435/materialscloud:2020.0037/v1
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-020-0323-8" target="_blank">
     https://doi.org/10.1038/s41524-020-0323-8
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_o73heom14iew_0" target="_blank">
     HO_PNAS_2019
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    1590 configurations of H2O/water with total energy and forces calculated using a hybrid approach at DFT/revPBE0-D3 level of theory.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    HO_PNAS_2019__Cheng-Engel-Behler-Dellago-Ceriotti__DS_o73heom14iew_0
    <br/>
    <b>
     Name:
    </b>
    HO_PNAS_2019
    <br/>
    <b>
     Authors:
    </b>
    Bingqing Cheng, Edgar A. Engel, Jörg Behler, Christoph Dellago, Michele Ceriotti
    <br/>
    <b>
     Elements
    </b>
    : H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,588
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 304,896
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://archive.materialscloud.org/record/2018.0020/v1" target="_blank">
     https://archive.materialscloud.org/record/2018.0020/v1
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.1815117116" target="_blank">
     https://doi.org/10.1073/pnas.1815117116
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_fhvr3y1alb97_0" target="_blank">
     HPt_NC_2022
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A training dataset of 90,000 configurations with interaction properties between H2 and Pt(111) surfaces.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    HPt_NC_2022__Vandermause-Xie-Lim-Owen-Kozinsky__DS_fhvr3y1alb97_0
    <br/>
    <b>
     Name:
    </b>
    HPt_NC_2022
    <br/>
    <b>
     Authors:
    </b>
    Jonathan Vandermause, Yu Xie, Jin Soo Lim, Cameron J. Owen, Boris Kozinsky
    <br/>
    <b>
     Elements
    </b>
    : H, Pt
    <br/>
    <b>
     Number of Configurations
    </b>
    : 90,740
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 5,706,023
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:r0-84" target="_blank">
     https://doi.org/10.24435/materialscloud:r0-84
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-022-32294-0" target="_blank">
     https://doi.org/10.1038/s41467-022-32294-0
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_lvixye0ynk1o_0" target="_blank">
     H_nature_2022
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Over 300,000 configurations in an expanded dataset of 19 hydrogen combustion reaction channels. Intrinsic reaction coordinate calculations (IRC) are combined with ab initio simulations (AIMD) and normal mode displacement (NM) calculations.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    H_nature_2022__Guan-Das-Stein-Heidar-Zadeh-Bertels-Liu-Haghighatlari-Li-Zhang-Hao-Leven-Head-Gordon-Head-Gordon__DS_lvixye0ynk1o_0
    <br/>
    <b>
     Name:
    </b>
    H_nature_2022
    <br/>
    <b>
     Authors:
    </b>
    Xingyi Guan, Akshaya Das, Christopher J. Stein, Farnaz Heidar-Zadeh, Luke Bertels, Meili Liu, Mojtaba Haghighatlari, Jie Li, Oufan Zhang, Hongxia Hao, Itai Leven, Martin Head-Gordon, Teresa Head-Gordon
    <br/>
    <b>
     Elements
    </b>
    : H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 350,121
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,513,654
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.19601689.v3" target="_blank">
     https://doi.org/10.6084/m9.figshare.19601689.v3
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-022-01330-5" target="_blank">
     https://doi.org/10.1038/s41597-022-01330-5
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_qgi1kbtdzwmm_0" target="_blank">
     HfO2_DPGEN_PRB_2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 28,500 configurations of hafnia (HfO2) used in the training of a DP model for the prediction of properties of various hafnia polymorphs, including transition barriers between different phases.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    HfO2_DPGEN_PRB_2021__Wu-Zhang-Zhang-Liu__DS_qgi1kbtdzwmm_0
    <br/>
    <b>
     Name:
    </b>
    HfO2_DPGEN_PRB_2021
    <br/>
    <b>
     Authors:
    </b>
    Jing Wu, Yuzhi Zhang, Linfeng Zhang, Shi Liu
    <br/>
    <b>
     Elements
    </b>
    : Hf, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 28,564
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,741,376
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=HfO2-dpgen" target="_blank">
     https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=HfO2-dpgen
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevB.103.024108" target="_blank">
     https://doi.org/10.1103/PhysRevB.103.024108
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_u40eq96ge40x_0" target="_blank">
     HfO2_NPJ_2020
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    6000 configurations of liquid and amorphous HfO2 generated for use with an active learning ML model.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    HfO2_NPJ_2020__Sivaraman-Krishnamoorthy-Baur-Holm-Stan-Csányi-Benmore-Vázquez-Mayagoitia__DS_u40eq96ge40x_0
    <br/>
    <b>
     Name:
    </b>
    HfO2_NPJ_2020
    <br/>
    <b>
     Authors:
    </b>
    Ganesh Sivaraman, Anand Narayanan Krishnamoorthy, Matthias Baur, Christian Holm, Marius Stan, Gábor Csányi, Chris Benmore, Álvaro Vázquez-Mayagoitia
    <br/>
    <b>
     Elements
    </b>
    : Hf, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 6,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 576,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/argonne-lcf/active-learning-md" target="_blank">
     https://github.com/argonne-lcf/active-learning-md
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-020-00367-7" target="_blank">
     https://doi.org/10.1038/s41524-020-00367-7
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_r1sl5dye35kg_0" target="_blank">
     Hydrogen-induced_insulating_state_SmNiO3
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A dataset of DFT-calculated energies created to investigate the effect of hydrogen doping on the crystal structure and the electronic state in SmNiO3.Configuration sets include sets for apically and side-bonded hydrogen atoms for 1-9 hydrogen atoms.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Hydrogen-induced_insulating_state_SmNiO3__Yamauchi-Hamada__DS_r1sl5dye35kg_0
    <br/>
    <b>
     Name:
    </b>
    Hydrogen-induced_insulating_state_SmNiO3
    <br/>
    <b>
     Authors:
    </b>
    Kunihiko Yamauchi, Ikutaro Hamada
    <br/>
    <b>
     Elements
    </b>
    : H, Ni, O, Sm
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,318
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 156,419
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:4w-qm" target="_blank">
     https://doi.org/10.24435/materialscloud:4w-qm
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2210.07656" target="_blank">
     https://doi.org/10.48550/arXiv.2210.07656
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_ngsaypfb9rnj_0" target="_blank">
     ISO17_NC_2017
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    129 molecules of composition C7O2H10 from the QM9 dataset with 5000 conformational geometries apiece. Molecular dynamics data was simulated using the Fritz-Haber Institute ab initio simulation software.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    ISO17_NC_2017__Vandermause-Xie-Lim-Owen-Kozinsky__DS_ngsaypfb9rnj_0
    <br/>
    <b>
     Name:
    </b>
    ISO17_NC_2017
    <br/>
    <b>
     Authors:
    </b>
    Jonathan Vandermause, Yu Xie, Jin Soo Lim, Cameron J. Owen, Boris Kozinsky
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 640,855
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 12,176,245
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://quantum-machine.org/datasets/" target="_blank">
     http://quantum-machine.org/datasets/
    </a>
    <br/>
    <a href="https://proceedings.neurips.cc/paper/2017/hash/303ed4c69846ab36c2904d3ba8573050-Abstract.html" target="_blank">
     https://proceedings.neurips.cc/paper/2017/hash/303ed4c69846ab36c2904d3ba8573050-Abstract.html
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-019-12875-2" target="_blank">
     https://doi.org/10.1038/s41467-019-12875-2
    </a>
    <br/>
    <a href="https://doi.org/10.1038/ncomms13890" target="_blank">
     https://doi.org/10.1038/ncomms13890
    </a>
    <br/>
    <a href="https://doi.org/10.1038/sdata.2014.22" target="_blank">
     https://doi.org/10.1038/sdata.2014.22
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_v4e6t79tt5xh_0" target="_blank">
     In2Se3_2D_DPGEN
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 11,500 configurations of In2Se3, including monolayer (20-atom slab) and bulk (30-atom supercell) models.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    In2Se3_2D_DPGEN__Wu-Bai-Huang-Ma-Liu-Liu__DS_v4e6t79tt5xh_0
    <br/>
    <b>
     Name:
    </b>
    In2Se3_2D_DPGEN
    <br/>
    <b>
     Authors:
    </b>
    Jing Wu, Liyi Bai, Jiawei Huang, Liyang Ma, Jian Liu, Shi Liu
    <br/>
    <b>
     Elements
    </b>
    : In, Se
    <br/>
    <b>
     Number of Configurations
    </b>
    : 11,523
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 248,510
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=In2Se3-2D-dpgen" target="_blank">
     https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=In2Se3-2D-dpgen
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevB.104.174107" target="_blank">
     https://doi.org/10.1103/PhysRevB.104.174107
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_03ph25s4yepy_0" target="_blank">
     InP_JPCA2020
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This data set was used to generate a multi-element linear SNAP potential for InP, as published in Cusentino, M. A. et. al, J. Chem. Phys. (2020). Intended to produce an interatomic potential for indium phosphide capable of capturing high-energy defects that result from radiation damage cascades.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    InP_JPCA2020__Cusentino-Wood-Thompson__DS_03ph25s4yepy_0
    <br/>
    <b>
     Name:
    </b>
    InP_JPCA2020
    <br/>
    <b>
     Authors:
    </b>
    Mary Alice Cusentino, Mitchell A. Wood, Aidan P. Thompson
    <br/>
    <b>
     Elements
    </b>
    : In, P
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,802
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 106,761
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/FitSNAP/FitSNAP/tree/master/examples/InP_JPCA2020" target="_blank">
     https://github.com/FitSNAP/FitSNAP/tree/master/examples/InP_JPCA2020
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpca.0c02450" target="_blank">
     https://doi.org/10.1021/acs.jpca.0c02450
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_dbgckv1il6v7_0" target="_blank">
     JARVIS-Polymer-Genome
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS-Polymer-Genome dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the Polymer Genome dataset, as created for the linked publication (Huan, T., Mannodi-Kanakkithodi, A., Kim, C. et al.). Structures were curated from existing sources and the original authors' works, removing redundant, identical structures before calculations, and removing redundant datapoints after calculations were performed. Band gap energies were calculated using two different DFT functionals: rPW86 and HSE06; atomization energy was calculated using rPW86. JARVIS is a set of tools and collected datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS-Polymer-Genome__Huan-Mannodi-Kanakkithodi-Kim-Sharma-Pilania-Ramprasad__DS_dbgckv1il6v7_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS-Polymer-Genome
    <br/>
    <b>
     Authors:
    </b>
    Tran Doan Huan, Arun Mannodi-Kanakkithodi, Chiho Kim, Vinit Sharma, Ghanshyam Pilania, Rampi Ramprasad
    <br/>
    <b>
     Elements
    </b>
    : Al, C, Ca, Cd, Cl, F, H, Hf, Mg, N, O, Pb, S, Sn, Ti, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,073
    <br/>
    <b>
     Number of Elements
    </b>
    : 17
    <br/>
    <b>
     Number of Atoms
    </b>
    : 34,441
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://ndownloader.figshare.com/files/26809907" target="_blank">
     https://ndownloader.figshare.com/files/26809907
    </a>
    <br/>
    <a href="https://doi.org/10.1038/sdata.2016.12" target="_blank">
     https://doi.org/10.1038/sdata.2016.12
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_tat5i46x3hkr_0" target="_blank">
     JARVIS-QM9-DGL
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS-QM9-DGL dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the QM9 dataset, originally created as part of the datasets at quantum-machine.org, as implemented with the Deep Graph Library (DGL) Python package. Units for r2 (electronic spatial extent) are a0^2; for alpha (isotropic polarizability), a0^3; for mu (dipole moment), D; for Cv (heat capacity), cal/mol K. Units for all other properties are eV. JARVIS is a set of tools and collected datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS-QM9-DGL__Ramakrishnan-Dral-Rupp-Lilienfeld__DS_tat5i46x3hkr_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS-QM9-DGL
    <br/>
    <b>
     Authors:
    </b>
    Raghunathan Ramakrishnan, Pavlo O. Dral, Matthias Rupp, O. Anatole von Lilienfeld
    <br/>
    <b>
     Elements
    </b>
    : C, F, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 130,831
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,358,210
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://ndownloader.figshare.com/files/28541196" target="_blank">
     https://ndownloader.figshare.com/files/28541196
    </a>
    <br/>
    <a href="https://doi.org/10.1038/sdata.2014.22" target="_blank">
     https://doi.org/10.1038/sdata.2014.22
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_hdv6si8yu2mv_0" target="_blank">
     JARVIS_2DMatPedia
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS-2DMatPedia dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This subset contains configurations with 2D materials from the 2DMatPedia database, generated through two methods: a top-down exfoliation approach, using structures of bulk materials from the Materials Project database; and a bottom-up approach, replacing each element in a 2D material with another from the same group (according to column number). JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_2DMatPedia__Zhou-Shen-Costa-Persson-Huck-Lu-Ma-Chen-Tang-Feng__DS_hdv6si8yu2mv_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_2DMatPedia
    <br/>
    <b>
     Authors:
    </b>
    Jun Zhou, Lei Shen, Miguel Dias Costa, Kristin A. Persson, Shyue Ping Ong", "Patrick Huck, Yunhao Lu, Xiaoyang Ma, Yiming Chen, Hanmei Tang, Yuan Ping Feng
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, Hf, Hg, Ho, I, In, Ir, K, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 6,351
    <br/>
    <b>
     Number of Elements
    </b>
    : 83
    <br/>
    <b>
     Number of Atoms
    </b>
    : 66,295
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://ndownloader.figshare.com/files/26789006" target="_blank">
     https://ndownloader.figshare.com/files/26789006
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-019-0097-3" target="_blank">
     https://doi.org/10.1038/s41597-019-0097-3
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_z3s0qui5vg5c_0" target="_blank">
     JARVIS_AGRA_CHO
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_AGRA_CHO dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This dataset contains data from the CO2 reduction reaction (CO2RR) dataset from Chen et al., as used in the automated graph representation algorithm (AGRA) training dataset: a collection of DFT training data for training a graph representation method to extract the local chemical environment of metallic surface adsorption sites. JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_AGRA_CHO__Chen-Gariepy-Chen-Yao-Anand-Liu-Feugmo-Tamblyn-Singh__DS_z3s0qui5vg5c_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_AGRA_CHO
    <br/>
    <b>
     Authors:
    </b>
    Zhi Wen Chen, Zachary Gariepy, Lixin Chen, Xue Yao, Abu Anand, Szu-Jia Liu, Conrard Giresse Tetsassi Feugmo, Isaac Tamblyn, Chandra Veer Singh
    <br/>
    <b>
     Elements
    </b>
    : C, Co, Cu, Fe, H, Mo, Ni, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 216
    <br/>
    <b>
     Number of Elements
    </b>
    : 8
    <br/>
    <b>
     Number of Atoms
    </b>
    : 14,472
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://figshare.com/ndownloader/files/41923284" target="_blank">
     https://figshare.com/ndownloader/files/41923284
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.2c03675" target="_blank">
     https://doi.org/10.1021/acscatal.2c03675
    </a>
    <br/>
    <a href="https://github.com/Feugmo-Group/AGRA" target="_blank">
     https://github.com/Feugmo-Group/AGRA
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0140487" target="_blank">
     https://doi.org/10.1063/5.0140487
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_1ynp8v7d3er5_0" target="_blank">
     JARVIS_AGRA_CO
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_AGRA_CO dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This dataset contains data from the CO2 reduction reaction (CO2RR) dataset from Chen et al., as used in the automated graph representation algorithm (AGRA) training dataset: a collection of DFT training data for training a graph representation method to extract the local chemical environment of metallic surface adsorption sites. JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_AGRA_CO__Chen-Gariepy-Chen-Yao-Anand-Liu-Feugmo-Tamblyn-Singh__DS_1ynp8v7d3er5_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_AGRA_CO
    <br/>
    <b>
     Authors:
    </b>
    Zhi Wen Chen, Zachary Gariepy, Lixin Chen, Xue Yao, Abu Anand, Szu-Jia Liu, Conrard Giresse Tetsassi Feugmo, Isaac Tamblyn, Chandra Veer Singh
    <br/>
    <b>
     Elements
    </b>
    : C, Co, Cu, Fe, Mo, Ni, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 194
    <br/>
    <b>
     Number of Elements
    </b>
    : 7
    <br/>
    <b>
     Number of Atoms
    </b>
    : 12,804
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://figshare.com/ndownloader/files/41923284" target="_blank">
     https://figshare.com/ndownloader/files/41923284
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.2c03675" target="_blank">
     https://doi.org/10.1021/acscatal.2c03675
    </a>
    <br/>
    <a href="https://github.com/Feugmo-Group/AGRA" target="_blank">
     https://github.com/Feugmo-Group/AGRA
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0140487" target="_blank">
     https://doi.org/10.1063/5.0140487
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_bggn2p6qw8zv_0" target="_blank">
     JARVIS_AGRA_COOH
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_AGRA_COOH dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This dataset contains data from the CO2 reduction reaction (CO2RR) dataset from Chen et al., as used in the automated graph representation algorithm (AGRA) training dataset: a collection of DFT training data for training a graph representation method to extract the local chemical environment of metallic surface adsorption sites. JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_AGRA_COOH__Chen-Gariepy-Chen-Yao-Anand-Liu-Feugmo-Tamblyn-Singh__DS_bggn2p6qw8zv_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_AGRA_COOH
    <br/>
    <b>
     Authors:
    </b>
    Zhi Wen Chen, Zachary Gariepy, Lixin Chen, Xue Yao, Abu Anand, Szu-Jia Liu, Conrard Giresse Tetsassi Feugmo, Isaac Tamblyn, Chandra Veer Singh
    <br/>
    <b>
     Elements
    </b>
    : C, Co, Cu, Fe, H, Mo, Ni, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 280
    <br/>
    <b>
     Number of Elements
    </b>
    : 8
    <br/>
    <b>
     Number of Atoms
    </b>
    : 19,040
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://figshare.com/ndownloader/files/41923284" target="_blank">
     https://figshare.com/ndownloader/files/41923284
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.2c03675" target="_blank">
     https://doi.org/10.1021/acscatal.2c03675
    </a>
    <br/>
    <a href="https://github.com/Feugmo-Group/AGRA" target="_blank">
     https://github.com/Feugmo-Group/AGRA
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0140487" target="_blank">
     https://doi.org/10.1063/5.0140487
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_ikx8oaixaoz0_0" target="_blank">
     JARVIS_AGRA_O
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_AGRA_O dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This dataset contains data from the training set for the oxygen reduction reaction (ORR) dataset from Batchelor et al., as used in the automated graph representation algorithm (AGRA) training dataset: a collection of DFT training data for training a graph representation method to extract the local chemical environment of metallic surface adsorption sites.  Bulk calculations were performed with k-point = 8 x 8 x 4. Training adsorption energies were calculated on slabs, k-point = 4 x 4 x 1, while testing energies used k-point = 3 x 3 x 1. JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_AGRA_O__Batchelor-Pedersen-Winther-Castelli-Jacobsen-Rossmeisl__DS_ikx8oaixaoz0_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_AGRA_O
    <br/>
    <b>
     Authors:
    </b>
    Thomas A.A. Batchelor, Jack K. Pedersen, Simon H. Winther, Ivano E. Castelli, Karsten W. Jacobsen, Jan Rossmeisl
    <br/>
    <b>
     Elements
    </b>
    : Ir, O, Pd, Pt, Rh, Ru
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 6
    <br/>
    <b>
     Number of Atoms
    </b>
    : 17,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://figshare.com/ndownloader/files/41923284" target="_blank">
     https://figshare.com/ndownloader/files/41923284
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.joule.2018.12.015" target="_blank">
     https://doi.org/10.1016/j.joule.2018.12.015
    </a>
    <br/>
    <a href="https://github.com/Feugmo-Group/AGRA" target="_blank">
     https://github.com/Feugmo-Group/AGRA
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0140487" target="_blank">
     https://doi.org/10.1063/5.0140487
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_osavzsrz5mgf_0" target="_blank">
     JARVIS_AGRA_OH
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_AGRA_OH dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This dataset contains data from the training set for the oxygen reduction reaction (ORR) dataset from Batchelor et al., as used in the automated graph representation algorithm (AGRA) training dataset: a collection of DFT training data for training a graph representation method to extract the local chemical environment of metallic surface adsorption sites.  Bulk calculations were performed with k-point = 8 x 8 x 4. Training adsorption energies were calculated on slabs, k-point = 4 x 4 x 1, while testing energies used k-point = 3 x 3 x 1. JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_AGRA_OH__Batchelor-Pedersen-Winther-Castelli-Jacobsen-Rossmeisl__DS_osavzsrz5mgf_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_AGRA_OH
    <br/>
    <b>
     Authors:
    </b>
    Thomas A.A. Batchelor, Jack K. Pedersen, Simon H. Winther, Ivano E. Castelli, Karsten W. Jacobsen, Jan Rossmeisl
    <br/>
    <b>
     Elements
    </b>
    : H, Ir, O, Pd, Pt, Rh, Ru
    <br/>
    <b>
     Number of Configurations
    </b>
    : 877
    <br/>
    <b>
     Number of Elements
    </b>
    : 7
    <br/>
    <b>
     Number of Atoms
    </b>
    : 15,786
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://figshare.com/ndownloader/files/41923284" target="_blank">
     https://figshare.com/ndownloader/files/41923284
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.joule.2018.12.015" target="_blank">
     https://doi.org/10.1016/j.joule.2018.12.015
    </a>
    <br/>
    <a href="https://github.com/Feugmo-Group/AGRA" target="_blank">
     https://github.com/Feugmo-Group/AGRA
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0140487" target="_blank">
     https://doi.org/10.1063/5.0140487
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_720rbshv96l1_0" target="_blank">
     JARVIS_ALIGNN_FF
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_ALIGNN_FF dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset is a subset of the JARVIS DFT dataset, filtered to contain just the first, last, middle, maximum energy and minimum energy structures. Additionally, calculation run snapshots are filtered for uniqueness, and the dataset contains only perfect structures. DFT energies, stresses and forces in this dataset were used to train an atomisitic line graph neural network (ALIGNN)-based FF model. JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_ALIGNN_FF__Choudhary-DeCost-Major-Butler-Thiyagalingam-Tavazza__DS_720rbshv96l1_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_ALIGNN_FF
    <br/>
    <b>
     Authors:
    </b>
    Kamal Choudhary, Brian DeCost, Lily Major, Keith Butler, Jeyan Thiyagalingam, Francesca Tavazza
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ne, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 306,411
    <br/>
    <b>
     Number of Elements
    </b>
    : 89
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,193,703
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://ndownloader.figshare.com/files/38522315" target="_blank">
     https://ndownloader.figshare.com/files/38522315
    </a>
    <br/>
    <a href="https://doi.org/10.1039/D2DD00096B" target="_blank">
     https://doi.org/10.1039/D2DD00096B
    </a>
    <br/>
    <a href="https://github.com/usnistgov/alignn" target="_blank">
     https://github.com/usnistgov/alignn
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_8hgxhsfkcfa7_0" target="_blank">
     JARVIS_C2DB
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS-C2DB dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This subset contains configurations from the Computational 2D Database (C2DB), which contains a variety of properties for 2-dimensional materials across more than 30 differentcrystal structures. JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_C2DB__Haastrup-Strange-Pandey-Deilmann-Schmidt-Hinsche-Gjerding-Torelli-Larsen-Riis-Jensen-Gath-Jacobsen-Mortensen-Olsen-Thygesen__DS_8hgxhsfkcfa7_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_C2DB
    <br/>
    <b>
     Authors:
    </b>
    Sten Haastrup, Mikkel Strange, Mohnish Pandey, Thorsten Deilmann, Per S Schmidt, Nicki F Hinsche, Morten N Gjerding, Daniele Torelli, Peter M Larsen, Anders C Riis-Jensen, Jakob Gath, Karsten W Jacobsen, Jens Jørgen Mortensen, Thomas Olsen, Kristian S Thygesen
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Ba, Bi, Br, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, F, Fe, Ga, Ge, H, Hf, Hg, I, In, Ir, K, Li, Mg, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,520
    <br/>
    <b>
     Number of Elements
    </b>
    : 61
    <br/>
    <b>
     Number of Atoms
    </b>
    : 17,990
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://ndownloader.figshare.com/files/28682010" target="_blank">
     https://ndownloader.figshare.com/files/28682010
    </a>
    <br/>
    <a href="https://doi.org/10.1088/2053-1583/aacfc1" target="_blank">
     https://doi.org/10.1088/2053-1583/aacfc1
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_np1noc8iip47_0" target="_blank">
     JARVIS_CFID_3D_8_18_2022
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_CFID_3D_8_18_2022 dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains configurations of 3D materials. JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_CFID_3D_8_18_2022__Choudhary-Garrity-Reid-DeCost-Biacchi-Walker-Trautt-Hattrick-Simpers-Kusne-Centrone-Davydov-Jiang-Pachter-Cheon-Reed-Agrawal-Qian-Sharma-Zhuang-Kalinin-Sumpter-Pilania-Acar-Mandal-Haule-Vanderbilt-Rabe-Tavazza__DS_np1noc8iip47_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_CFID_3D_8_18_2022
    <br/>
    <b>
     Authors:
    </b>
    Kamal Choudhary, Kevin F. Garrity, Andrew C. E. Reid, Brian DeCost, Adam J. Biacchi, Angela R. Hight Walker, Zachary Trautt, Jason Hattrick-Simpers, A. Gilad Kusne, Andrea Centrone, Albert Davydov, Jie Jiang, Ruth Pachter, Gowoon Cheon, Evan Reed, Ankit Agrawal, Xiaofeng Qian, Vinit Sharma, Houlong Zhuang, Sergei V. Kalinin, Bobby G. Sumpter, Ghanshyam Pilania, Pinar Acar, Subhasish Mandal, Kristjan Haule, David Vanderbilt, Karin Rabe, Francesca Tavazza
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ne, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 55,631
    <br/>
    <b>
     Number of Elements
    </b>
    : 89
    <br/>
    <b>
     Number of Atoms
    </b>
    : 561,738
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.6815699" target="_blank">
     https://doi.org/10.6084/m9.figshare.6815699
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-020-00440-1" target="_blank">
     https://doi.org/10.1038/s41524-020-00440-1
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_u8strp7hm0cy_0" target="_blank">
     JARVIS_CFID_OQMD
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_CFID_OQMD dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the Open Quantum Materials Database (OQMD), created to hold information about the electronic structure and stability of organic materials for the purpose of aiding in materials discovery. Calculations were performed at the DFT level of theory, using the PAW-PBE functional implemented by VASP. This dataset also includes classical force-field inspired descriptors (CFID) for each configuration. JARVIS is a set of tools and collected datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_CFID_OQMD__Kirklin-Saal-Meredig-Thompson-Doak-Aykol-Rühl-Wolverton__DS_u8strp7hm0cy_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_CFID_OQMD
    <br/>
    <b>
     Authors:
    </b>
    Scott Kirklin, James E Saal, Bryce Meredig, Alex Thompson, Jeff W Doak, Muratahan Aykol, Stephan Rühl, Chris Wolverton
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ne, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 459,991
    <br/>
    <b>
     Number of Elements
    </b>
    : 89
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,366,255
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://ndownloader.figshare.com/files/24981170" target="_blank">
     https://ndownloader.figshare.com/files/24981170
    </a>
    <br/>
    <a href="https://doi.org/10.1038/npjcompumats.2015.10" target="_blank">
     https://doi.org/10.1038/npjcompumats.2015.10
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_4ml1yrigmar0_0" target="_blank">
     JARVIS_DFT_2D_3_12_2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The DFT-2D-3-12-2021 dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains configurations of 2D materials. JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_DFT_2D_3_12_2021__Choudhary-Garrity-Reid-DeCost-Biacchi-Walker-Trautt-Hattrick-Simpers-Kusne-Centrone-Davydov-Jiang-Pachter-Cheon-Reed-Agrawal-Qian-Sharma-Zhuang-Kalinin-Sumpter-Pilania-Acar-Mandal-Haule-Vanderbilt-Rabe-Tavazza__DS_4ml1yrigmar0_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_DFT_2D_3_12_2021
    <br/>
    <b>
     Authors:
    </b>
    Kamal Choudhary, Kevin F. Garrity, Andrew C. E. Reid, Brian DeCost, Adam J. Biacchi, Angela R. Hight Walker, Zachary Trautt, Jason Hattrick-Simpers, A. Gilad Kusne, Andrea Centrone, Albert Davydov, Jie Jiang, Ruth Pachter, Gowoon Cheon, Evan Reed, Ankit Agrawal, Xiaofeng Qian, Vinit Sharma, Houlong Zhuang, Sergei V. Kalinin, Bobby G. Sumpter, Ghanshyam Pilania, Pinar Acar, Subhasish Mandal, Kristjan Haule, David Vanderbilt, Karin Rabe, Francesca Tavazza
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cu, Dy, Er, F, Fe, Ga, Ge, H, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,074
    <br/>
    <b>
     Number of Elements
    </b>
    : 81
    <br/>
    <b>
     Number of Atoms
    </b>
    : 7,903
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://ndownloader.figshare.com/files/26808917" target="_blank">
     https://ndownloader.figshare.com/files/26808917
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-020-00440-1" target="_blank">
     https://doi.org/10.1038/s41524-020-00440-1
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_t02h9eraswtv_0" target="_blank">
     JARVIS_DFT_3D_12_12_2022
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The DFT_3D_12_12_2022 dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains configurations of 3D materials. JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_DFT_3D_12_12_2022__Choudhary-Garrity-Reid-DeCost-Biacchi-Walker-Trautt-Hattrick-Simpers-Kusne-Centrone-Davydov-Jiang-Pachter-Cheon-Reed-Agrawal-Qian-Sharma-Zhuang-Kalinin-Sumpter-Pilania-Acar-Mandal-Haule-Vanderbilt-Rabe-Tavazza__DS_t02h9eraswtv_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_DFT_3D_12_12_2022
    <br/>
    <b>
     Authors:
    </b>
    Kamal Choudhary, Kevin F. Garrity, Andrew C. E. Reid, Brian DeCost, Adam J. Biacchi, Angela R. Hight Walker, Zachary Trautt, Jason Hattrick-Simpers, A. Gilad Kusne, Andrea Centrone, Albert Davydov, Jie Jiang, Ruth Pachter, Gowoon Cheon, Evan Reed, Ankit Agrawal, Xiaofeng Qian, Vinit Sharma, Houlong Zhuang, Sergei V. Kalinin, Bobby G. Sumpter, Ghanshyam Pilania, Pinar Acar, Subhasish Mandal, Kristjan Haule, David Vanderbilt, Karin Rabe, Francesca Tavazza
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ne, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 75,909
    <br/>
    <b>
     Number of Elements
    </b>
    : 89
    <br/>
    <b>
     Number of Atoms
    </b>
    : 785,250
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.6815699" target="_blank">
     https://doi.org/10.6084/m9.figshare.6815699
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-020-00440-1" target="_blank">
     https://doi.org/10.1038/s41524-020-00440-1
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_soziqjohq6hm_0" target="_blank">
     JARVIS_DFT_3D_8_18_2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_DFT_3D_8_18_2021 dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains configurations of 3D materials. JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_DFT_3D_8_18_2021__Choudhary-Garrity-Reid-DeCost-Biacchi-Walker-Trautt-Hattrick-Simpers-Kusne-Centrone-Davydov-Jiang-Pachter-Cheon-Reed-Agrawal-Qian-Sharma-Zhuang-Kalinin-Sumpter-Pilania-Acar-Mandal-Haule-Vanderbilt-Rabe-Tavazza__DS_soziqjohq6hm_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_DFT_3D_8_18_2021
    <br/>
    <b>
     Authors:
    </b>
    Kamal Choudhary, Kevin F. Garrity, Andrew C. E. Reid, Brian DeCost, Adam J. Biacchi, Angela R. Hight Walker, Zachary Trautt, Jason Hattrick-Simpers, A. Gilad Kusne, Andrea Centrone, Albert Davydov, Jie Jiang, Ruth Pachter, Gowoon Cheon, Evan Reed, Ankit Agrawal, Xiaofeng Qian, Vinit Sharma, Houlong Zhuang, Sergei V. Kalinin, Bobby G. Sumpter, Ghanshyam Pilania, Pinar Acar, Subhasish Mandal, Kristjan Haule, David Vanderbilt, Karin Rabe, Francesca Tavazza
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ne, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 55,631
    <br/>
    <b>
     Number of Elements
    </b>
    : 89
    <br/>
    <b>
     Number of Atoms
    </b>
    : 561,741
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.6815699" target="_blank">
     https://doi.org/10.6084/m9.figshare.6815699
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-020-00440-1" target="_blank">
     https://doi.org/10.1038/s41524-020-00440-1
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_7jji22dy5hix_0" target="_blank">
     JARVIS_EPC_2D
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_EPC_2D dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains configurations sourced from the JARVIS-DFT-2D dataset, rerelaxed with Quantum ESPRESSO. JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_EPC_2D__Wines-Choudhary-Biacchi-Garrity-Tavazza__DS_7jji22dy5hix_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_EPC_2D
    <br/>
    <b>
     Authors:
    </b>
    Daniel Wines, Kamal Choudhary, Adam J. Biacchi, evin F. Garrity, Francesca Tavazza
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cl, Co, Cr, Cu, F, Fe, Ga, Ge, H, Hf, I, In, Ir, K, La, Li, Mg, Mo, N, Na, Nb, Ni, O, P, Pb, Pd, Pt, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 161
    <br/>
    <b>
     Number of Elements
    </b>
    : 55
    <br/>
    <b>
     Number of Atoms
    </b>
    : 788
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://figshare.com/ndownloader/files/38950433" target="_blank">
     https://figshare.com/ndownloader/files/38950433
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.nanolett.2c04420" target="_blank">
     https://doi.org/10.1021/acs.nanolett.2c04420
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_9yr94hhj1k33_0" target="_blank">
     JARVIS_MEGNet
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS-MEGNet dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This subset contains configurations with 3D materials properties from the 2018 version of Materials Project, as used in the training of the MEGNet ML model. JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_MEGNet__Chen-Ye-Zuo-Zheng-Ong__DS_9yr94hhj1k33_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_MEGNet
    <br/>
    <b>
     Authors:
    </b>
    Chi Chen, Weike Ye, Yunxing Zuo, Chen Zheng, Shyue Ping Ong
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ne, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 69,234
    <br/>
    <b>
     Number of Elements
    </b>
    : 89
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,070,948
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://ndownloader.figshare.com/files/26724977" target="_blank">
     https://ndownloader.figshare.com/files/26724977
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.chemmater.9b01294" target="_blank">
     https://doi.org/10.1021/acs.chemmater.9b01294
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_5ar73nonq6l1_0" target="_blank">
     JARVIS_MEGNet2
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS-MEGNet2 dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This subset contains 133K materials with formation energy from the Materials Project, as used in the training of the MEGNet ML model. JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_MEGNet2__Chen-Ye-Zuo-Zheng-Ong__DS_5ar73nonq6l1_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_MEGNet2
    <br/>
    <b>
     Authors:
    </b>
    Chi Chen, Weike Ye, Yunxing Zuo, Chen Zheng, Shyue Ping Ong
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ne, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 133,420
    <br/>
    <b>
     Number of Elements
    </b>
    : 89
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,880,497
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://ndownloader.figshare.com/files/28332741" target="_blank">
     https://ndownloader.figshare.com/files/28332741
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.chemmater.9b01294" target="_blank">
     https://doi.org/10.1021/acs.chemmater.9b01294
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_gpsibs9f47k4_0" target="_blank">
     JARVIS_Materials_Project_2020
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_Materials_Project_2020 dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains 127,000 configurations of 3D materials from the Materials Project database. JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_Materials_Project_2020__Jain-Ong-Hautier-Chen-Richards-Dacek-Cholia-Gunter-Skinner-Ceder-Persson__DS_gpsibs9f47k4_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_Materials_Project_2020
    <br/>
    <b>
     Authors:
    </b>
    Anubhav Jain, Shyue Ping Ong, Geoffroy Hautier, Wei Chen, William Davidson Richards, Stephen Dacek, Shreyas Cholia, Dan Gunter, David Skinner, Gerbrand Ceder, Kristin A. Persson
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ne, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 126,335
    <br/>
    <b>
     Number of Elements
    </b>
    : 89
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,725,727
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://ndownloader.figshare.com/files/26791259" target="_blank">
     https://ndownloader.figshare.com/files/26791259
    </a>
    <br/>
    <a href="https://doi.org/10.1063/1.4812323" target="_blank">
     https://doi.org/10.1063/1.4812323
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_1s8172fnm2ct_0" target="_blank">
     JARVIS_Materials_Project_84K
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_Materials_Project_84K dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains 84,000 configurations of 3D materials from the Materials Project database. JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_Materials_Project_84K__Jain-Ong-Hautier-Chen-Richards-Dacek-Cholia-Gunter-Skinner-Ceder-Persson__DS_1s8172fnm2ct_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_Materials_Project_84K
    <br/>
    <b>
     Authors:
    </b>
    Anubhav Jain, Shyue Ping Ong, Geoffroy Hautier, Wei Chen, William Davidson Richards, Stephen Dacek, Shreyas Cholia, Dan Gunter, David Skinner, Gerbrand Ceder, Kristin A. Persson
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ne, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 83,426
    <br/>
    <b>
     Number of Elements
    </b>
    : 89
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,339,932
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://ndownloader.figshare.com/files/24979850" target="_blank">
     https://ndownloader.figshare.com/files/24979850
    </a>
    <br/>
    <a href="https://doi.org/10.1063/1.4812323" target="_blank">
     https://doi.org/10.1063/1.4812323
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_hrt0twm503tr_0" target="_blank">
     JARVIS_OMDB
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_OMDB dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the Organic Materials Database (OMDB): a dataset of 12,500 crystal materials for the purpose of training models for the prediction of properties for complex and lattice-periodic organic crystals with large numbers of atoms per unit cell. Dataset covers 69 space groups, 65 elements; averages 82 atoms per unit cell. This dataset also includes classical force-field inspired descriptors (CFID) for each configuration. JARVIS is a set of tools and collected datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_OMDB__Olsthoorn-Geilhufe-Borysov-Balatsky__DS_hrt0twm503tr_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_OMDB
    <br/>
    <b>
     Authors:
    </b>
    Bart Olsthoorn, R. Matthias Geilhufe, Stanislav S. Borysov, Alexander V. Balatsky
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, F, Fe, Ga, Ge, H, Hf, Hg, I, In, Ir, K, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Te, Ti, Tl, U, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 12,497
    <br/>
    <b>
     Number of Elements
    </b>
    : 65
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,061,362
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://ndownloader.figshare.com/files/28501761" target="_blank">
     https://ndownloader.figshare.com/files/28501761
    </a>
    <br/>
    <a href="https://doi.org/10.1002/qute.201900023" target="_blank">
     https://doi.org/10.1002/qute.201900023
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_untxw8nljf92_0" target="_blank">
     JARVIS_OQMD_no_CFID
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_OQMD_no_CFID dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the Open Quantum Materials Database (OQMD), created to hold information about the electronic structure and stability of organic materials for the purpose of aiding in materials discovery. Calculations were performed at the DFT level of theory, using the PAW-PBE functional implemented by VASP. JARVIS is a set of tools and collected datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_OQMD_no_CFID__Kirklin-Saal-Meredig-Thompson-Doak-Aykol-Rühl-Wolverton__DS_untxw8nljf92_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_OQMD_no_CFID
    <br/>
    <b>
     Authors:
    </b>
    Scott Kirklin, James E Saal, Bryce Meredig, Alex Thompson, Jeff W Doak, Muratahan Aykol, Stephan Rühl, Chris Wolverton
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ne, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 811,782
    <br/>
    <b>
     Number of Elements
    </b>
    : 89
    <br/>
    <b>
     Number of Atoms
    </b>
    : 5,017,701
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://ndownloader.figshare.com/files/26790182" target="_blank">
     https://ndownloader.figshare.com/files/26790182
    </a>
    <br/>
    <a href="https://doi.org/10.1038/npjcompumats.2015.10" target="_blank">
     https://doi.org/10.1038/npjcompumats.2015.10
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_armtbsouma25_0" target="_blank">
     JARVIS_Open_Catalyst_100K
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_Open_Catalyst_100K dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains configurations from the 100K training, rest validation and test dataset from the Open Catalyst Project (OCP). JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_Open_Catalyst_100K__Chanussot-Das-Goyal-Lavril-Shuaibi-Riviere-Tran-Heras-Domingo-Ho-Hu-Palizhati-Sriram-Wood-Yoon-Parikh-Zitnick-Ulissi__DS_armtbsouma25_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_Open_Catalyst_100K
    <br/>
    <b>
     Authors:
    </b>
    Lowik Chanussot, Abhishek Das, Siddharth Goyal, Thibaut Lavril, Muhammed Shuaibi, Morgane Riviere, Kevin Tran, Javier Heras-Domingo, Caleb Ho, Weihua Hu, Aini Palizhati, Anuroop Sriram, Brandon Wood, Junwoong Yoon, Devi Parikh, C. Lawrence Zitnick, Zachary Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Bi, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 124,943
    <br/>
    <b>
     Number of Elements
    </b>
    : 56
    <br/>
    <b>
     Number of Atoms
    </b>
    : 9,720,870
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://figshare.com/ndownloader/files/40902845" target="_blank">
     https://figshare.com/ndownloader/files/40902845
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.0c04525" target="_blank">
     https://doi.org/10.1021/acscatal.0c04525
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
    <a href="https://github.com/Open-Catalyst-Project/ocp" target="_blank">
     https://github.com/Open-Catalyst-Project/ocp
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_6dcul0b17524_0" target="_blank">
     JARVIS_Open_Catalyst_10K
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_Open_Catalyst_10K dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains configurations from the 10K training, rest validation and test dataset from the Open Catalyst Project (OCP). JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_Open_Catalyst_10K__Chanussot-Das-Goyal-Lavril-Shuaibi-Riviere-Tran-Heras-Domingo-Ho-Hu-Palizhati-Sriram-Wood-Yoon-Parikh-Zitnick-Ulissi__DS_6dcul0b17524_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_Open_Catalyst_10K
    <br/>
    <b>
     Authors:
    </b>
    Lowik Chanussot, Abhishek Das, Siddharth Goyal, Thibaut Lavril, Muhammed Shuaibi, Morgane Riviere, Kevin Tran, Javier Heras-Domingo, Caleb Ho, Weihua Hu, Aini Palizhati, Anuroop Sriram, Brandon Wood, Junwoong Yoon, Devi Parikh, C. Lawrence Zitnick, Zachary Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Bi, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 34,943
    <br/>
    <b>
     Number of Elements
    </b>
    : 56
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,720,316
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://figshare.com/ndownloader/files/40566122" target="_blank">
     https://figshare.com/ndownloader/files/40566122
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.0c04525" target="_blank">
     https://doi.org/10.1021/acscatal.0c04525
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
    <a href="https://github.com/Open-Catalyst-Project/ocp" target="_blank">
     https://github.com/Open-Catalyst-Project/ocp
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_3u17g7ukggi5_0" target="_blank">
     JARVIS_Open_Catalyst_All
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_Open_Catalyst_All dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains configurations from the Open Catalyst Project (OCP) 460328 training, rest validation and test dataset. JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_Open_Catalyst_All__Chanussot-Das-Goyal-Lavril-Shuaibi-Riviere-Tran-Heras-Domingo-Ho-Hu-Palizhati-Sriram-Wood-Yoon-Parikh-Zitnick-Ulissi__DS_3u17g7ukggi5_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_Open_Catalyst_All
    <br/>
    <b>
     Authors:
    </b>
    Lowik Chanussot, Abhishek Das, Siddharth Goyal, Thibaut Lavril, Muhammed Shuaibi, Morgane Riviere, Kevin Tran, Javier Heras-Domingo, Caleb Ho, Weihua Hu, Aini Palizhati, Anuroop Sriram, Brandon Wood, Junwoong Yoon, Devi Parikh, C. Lawrence Zitnick, Zachary Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Bi, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 485,269
    <br/>
    <b>
     Number of Elements
    </b>
    : 56
    <br/>
    <b>
     Number of Atoms
    </b>
    : 37,728,919
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://figshare.com/ndownloader/files/40902845" target="_blank">
     https://figshare.com/ndownloader/files/40902845
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.0c04525" target="_blank">
     https://doi.org/10.1021/acscatal.0c04525
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
    <a href="https://github.com/Open-Catalyst-Project/ocp" target="_blank">
     https://github.com/Open-Catalyst-Project/ocp
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_e471qdt7c6db_0" target="_blank">
     JARVIS_QE_TB
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The QE-TB dataset is part of the joint automated repository for various integrated simulations (JARVIS) DFT database. This subset contains configurations generated in Quantum ESPRESSO. JARVIS is a set of tools and datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_QE_TB__Garrity-Choudhary__DS_e471qdt7c6db_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_QE_TB
    <br/>
    <b>
     Authors:
    </b>
    Kevin F. Garrity, Kamal Choudhary
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, F, Fe, Ga, Ge, H, Hf, Hg, I, In, Ir, K, La, Li, Mg, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 829,576
    <br/>
    <b>
     Number of Elements
    </b>
    : 64
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,578,920
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://ndownloader.figshare.com/files/29070555" target="_blank">
     https://ndownloader.figshare.com/files/29070555
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevMaterials.7.044603" target="_blank">
     https://doi.org/10.1103/PhysRevMaterials.7.044603
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_jz1q9juw7ycj_0" target="_blank">
     JARVIS_QM9_STD_JCTC
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_QM9_STD_JCTC dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the QM9 dataset, originally created as part of the datasets at quantum-machine.org. Units for r2 (electronic spatial extent) are a^2; for alpha (isotropic polarizability), a^3; for mu (dipole moment), D; for Cv (heat capacity), cal/mol K. Units for all other properties are eV. JARVIS is a set of tools and collected datasets built to meet current materials design challenges.For the first iteration of DFT calculations, Gaussian 09's default electronic and geometry thresholds have been used for all molecules. For those molecules which failed to reach SCF convergence ultrafine grids have been invoked within a second iteration for evaluating the XC energy contributions. Within a third iteration on the remaining unconverged molecules, we identified those which had relaxed to saddle points, and further tightened the SCF criteria using the keyword scf(maxcycle=200, verytight). All those molecules which still featured imaginary frequencies entered the fourth iteration using keywords, opt(calcfc, maxstep=5, maxcycles=1000). calcfc constructs a Hessian in the first step of the geometry relaxation for eigenvector following. Within the fifth and final iteration, all molecules which still failed to reach convergence, have subsequently been converged using opt(calcall, maxstep=1, maxcycles=1000)
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_QM9_STD_JCTC__Ramakrishnan-Dral-Rupp-Lilienfeld__DS_jz1q9juw7ycj_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_QM9_STD_JCTC
    <br/>
    <b>
     Authors:
    </b>
    Raghunathan Ramakrishnan, Pavlo O. Dral, Matthias Rupp, O. Anatole von Lilienfeld
    <br/>
    <b>
     Elements
    </b>
    : C, F, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 130,829
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,359,192
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://ndownloader.figshare.com/files/28715319" target="_blank">
     https://ndownloader.figshare.com/files/28715319
    </a>
    <br/>
    <a href="https://doi.org/10.1038/sdata.2014.22" target="_blank">
     https://doi.org/10.1038/sdata.2014.22
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_221svb9fxfk7_0" target="_blank">
     JARVIS_QMOF
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_QMOF dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the Quantum Metal-Organic Frameworks (QMOF) dataset, comprising quantum-chemical properties for &gt;14,000 experimentally synthesized MOFs. QMOF contains "DFT-ready" data: filtered to remove omitted, overlapping, unbonded or deleted atoms, along with other kinds of problematic structures commented on in the literature. Data were generated via high-throughput DFT workflow, at the PBE-D3(BJ) level of theory using VASP software. JARVIS is a set of tools and collected datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_QMOF__Rosen-Iyer-Ray-Yao-Aspuru-Guzik-Gagliardi-Notestein-Snurr__DS_221svb9fxfk7_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_QMOF
    <br/>
    <b>
     Authors:
    </b>
    Andrew S. Rosen, Shaelyn M. Iyer, Debmalya Ray, Zhenpeng Yao, Alán Aspuru-Guzik, Laura Gagliardi, Justin M. Notestein, Randall Q. Snurr
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, Hf, Hg, Ho, I, In, Ir, K, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, P, Pb, Pd, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 20,425
    <br/>
    <b>
     Number of Elements
    </b>
    : 79
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,321,633
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://figshare.com/ndownloader/files/30972640" target="_blank">
     https://figshare.com/ndownloader/files/30972640
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.matt.2021.02.015" target="_blank">
     https://doi.org/10.1016/j.matt.2021.02.015
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_1nbddfnjxbjc_0" target="_blank">
     JARVIS_SNUMAT
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_SNUMAT dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains band gap data for &gt;10,000 materials, computed using a hybrid functional and considering the stable magnetic ordering. Structure relaxation and band edges are obtained using the PBE XC functional; band gap energy is subsequently obtained using the HSE06 hybrid functional. Optical and fundamental band gap energies are included. Some gap energies are recalculated by including spin-orbit coupling. These are noted in the band gap metadata as "SOC=true". JARVIS is a set of tools and collected datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_SNUMAT__Kim-Lee-Hong-Yoon-An-Lee-Jeong-Yoo-Kang-Youn-Han__DS_1nbddfnjxbjc_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_SNUMAT
    <br/>
    <b>
     Authors:
    </b>
    Sangtae Kim, Miso Lee, Changho Hong, Youngchae Yoon, Hyungmin An, Dongheon Lee, Wonseok Jeong, Dongsun Yoo, Youngho Kang, Yong Youn, Seungwu Han
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, F, Fe, Ga, Ge, H, He, Hf, Hg, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Ne, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Th, Ti, Tl, V, W, Xe, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 10,481
    <br/>
    <b>
     Number of Elements
    </b>
    : 73
    <br/>
    <b>
     Number of Atoms
    </b>
    : 216,749
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://ndownloader.figshare.com/files/38521736" target="_blank">
     https://ndownloader.figshare.com/files/38521736
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-020-00723-8" target="_blank">
     https://doi.org/10.1038/s41597-020-00723-8
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_ffsgl0wufoft_0" target="_blank">
     JARVIS_TinNet_N
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_TinNet dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the TinNet-N dataset: a collection assembled to train a machine learning model for the purposes of assisting catalyst design by predicting chemical reactivity of transition-metal surfaces. The adsorption systems contained in this dataset consist of {100}-terminated Pt-based bimetallic surfaces doped with a third element. JARVIS is a set of tools and collected datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_TinNet_N__Wang-Pillai-Wang-Achenie-Xin__DS_ffsgl0wufoft_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_TinNet_N
    <br/>
    <b>
     Authors:
    </b>
    Shih-Han Wang, Hemanth Somarajan Pillai, Siwen Wang, Luke E. K. Achenie, Hongliang Xin
    <br/>
    <b>
     Elements
    </b>
    : Ag, Au, Cd, Co, Cr, Cu, Fe, H, Hf, Ir, Mn, Mo, N, Nb, Ni, O, Os, Pd, Pt, Re, Rh, Ru, Sc, Tc, V, W, Zn
    <br/>
    <b>
     Number of Configurations
    </b>
    : 329
    <br/>
    <b>
     Number of Elements
    </b>
    : 27
    <br/>
    <b>
     Number of Atoms
    </b>
    : 6,251
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://figshare.com/ndownloader/files/40934285" target="_blank">
     https://figshare.com/ndownloader/files/40934285
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-021-25639-8" target="_blank">
     https://doi.org/10.1038/s41467-021-25639-8
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
    <a href="https://github.com/hlxin/tinnet/tree/master" target="_blank">
     https://github.com/hlxin/tinnet/tree/master
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_ok9dbnj53zih_0" target="_blank">
     JARVIS_TinNet_O
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_TinNet dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the TinNet-O dataset: a collection assembled to train a machine learning model for the purposes of assisting catalyst design by predicting chemical reactivity of transition-metal surfaces. The adsorption systems contained in this dataset consist of {111}-terminated metal surfaces. JARVIS is a set of tools and collected datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_TinNet_O__Wang-Pillai-Wang-Achenie-Xin__DS_ok9dbnj53zih_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_TinNet_O
    <br/>
    <b>
     Authors:
    </b>
    Shih-Han Wang, Hemanth Somarajan Pillai, Siwen Wang, Luke E. K. Achenie, Hongliang Xin
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, Au, Bi, Cd, Co, Cr, Cu, Fe, Ga, Hf, In, Ir, La, Mn, Mo, Nb, Ni, O, Os, Pb, Pd, Pt, Re, Rh, Ru, Sc, Sn, Ta, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 747
    <br/>
    <b>
     Number of Elements
    </b>
    : 36
    <br/>
    <b>
     Number of Atoms
    </b>
    : 12,699
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://figshare.com/ndownloader/files/40934285" target="_blank">
     https://figshare.com/ndownloader/files/40934285
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-021-25639-8" target="_blank">
     https://doi.org/10.1038/s41467-021-25639-8
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
    <a href="https://github.com/hlxin/tinnet/tree/master" target="_blank">
     https://github.com/hlxin/tinnet/tree/master
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_mk2e641eop9w_0" target="_blank">
     JARVIS_TinNet_OH
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_TinNet dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the TinNet-OH dataset: a collection assembled to train a machine learning model for the purposes of assisting catalyst design by predicting chemical reactivity of transition-metal surfaces. The adsorption systems contained in this dataset consist of {111}-terminated metal surfaces. JARVIS is a set of tools and collected datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_TinNet_OH__Wang-Pillai-Wang-Achenie-Xin__DS_mk2e641eop9w_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_TinNet_OH
    <br/>
    <b>
     Authors:
    </b>
    Shih-Han Wang, Hemanth Somarajan Pillai, Siwen Wang, Luke E. K. Achenie, Hongliang Xin
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, Au, Bi, Cd, Co, Cr, Cu, Fe, Ga, H, Hf, In, Ir, La, Mn, Mo, Nb, Ni, O, Os, Pb, Pd, Pt, Re, Rh, Ru, Sc, Sn, Ta, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 748
    <br/>
    <b>
     Number of Elements
    </b>
    : 37
    <br/>
    <b>
     Number of Atoms
    </b>
    : 13,464
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://figshare.com/ndownloader/files/40934285" target="_blank">
     https://figshare.com/ndownloader/files/40934285
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-021-25639-8" target="_blank">
     https://doi.org/10.1038/s41467-021-25639-8
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
    <a href="https://github.com/hlxin/tinnet/tree/master" target="_blank">
     https://github.com/hlxin/tinnet/tree/master
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_g9sra4y8efji_0" target="_blank">
     JARVIS_mlearn
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The JARVIS_mlearn dataset is part of the joint automated repository for various integrated simulations (JARVIS) database. This dataset contains configurations from the Organic Materials Database (OMDB): a dataset of 12,500 crystal materials for the purpose of training models for the prediction of properties for complex and lattice-periodic organic crystals with large numbers of atoms per unit cell. Dataset covers 69 space groups, 65 elements; averages 82 atoms per unit cell. This dataset also includes classical force-field inspired descriptors (CFID) for each configuration. JARVIS is a set of tools and collected datasets built to meet current materials design challenges.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    JARVIS_mlearn__Zuo-Chen-Li-Deng-Chen-Behler-Csányi-Shapeev-Thompson-Wood-Ong__DS_g9sra4y8efji_0
    <br/>
    <b>
     Name:
    </b>
    JARVIS_mlearn
    <br/>
    <b>
     Authors:
    </b>
    Yunxing Zuo, Chi Chen, Xiangguo Li, Zhi Deng, Yiming Chen, Jörg Behler, Gábor Csányi, Alexander V. Shapeev, Aidan P. Thompson, Mitchell A. Wood, Shyue Ping Ong
    <br/>
    <b>
     Elements
    </b>
    : Cu, Ge, Li, Mo, Ni, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,566
    <br/>
    <b>
     Number of Elements
    </b>
    : 6
    <br/>
    <b>
     Number of Atoms
    </b>
    : 115,742
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://figshare.com/ndownloader/files/40424156" target="_blank">
     https://figshare.com/ndownloader/files/40424156
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpca.9b08723" target="_blank">
     https://doi.org/10.1021/acs.jpca.9b08723
    </a>
    <br/>
    <a href="https://jarvis.nist.gov/" target="_blank">
     https://jarvis.nist.gov/
    </a>
    <br/>
    <a href="https://github.com/materialsvirtuallab/mlearn" target="_blank">
     https://github.com/materialsvirtuallab/mlearn
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_4lev0d7cs1yl_0" target="_blank">
     LiGePS_SSE_PBE
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 6,500 configurations of Li10GeP2S12, based on crystal structures from the Materials Project database, material ID mp-696129. One of two LiGePS datasets from this source. The other uses the PBEsol functional, rather than the PBE functional.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    LiGePS_SSE_PBE__Huang-Zhang-Wang-Zhao-Cheng-E__DS_4lev0d7cs1yl_0
    <br/>
    <b>
     Name:
    </b>
    LiGePS_SSE_PBE
    <br/>
    <b>
     Authors:
    </b>
    Jianxing Huang, Linfeng Zhang, Han Wang, Jinbao Zhao, Jun Cheng, Weinan E
    <br/>
    <b>
     Elements
    </b>
    : Ge, Li, P, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 6,550
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,479,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=LiGePS-SSE-PBE" target="_blank">
     https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=LiGePS-SSE-PBE
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0041849" target="_blank">
     https://doi.org/10.1063/5.0041849
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_dhtzh0y2108p_0" target="_blank">
     LiGePS_SSE_PBEsol
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 2,800 configurations of Li10GeP2S12, based on crystal structures from the Materials Project database, material ID mp-696129. One of two LiGePS datasets from this source. The other uses the PBE functional, rather than the PBEsol functional.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    LiGePS_SSE_PBEsol__Huang-Zhang-Wang-Zhao-Cheng-E__DS_dhtzh0y2108p_0
    <br/>
    <b>
     Name:
    </b>
    LiGePS_SSE_PBEsol
    <br/>
    <b>
     Authors:
    </b>
    Jianxing Huang, Linfeng Zhang, Han Wang, Jinbao Zhao, Jun Cheng, Weinan E
    <br/>
    <b>
     Elements
    </b>
    : Ge, Li, P, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,835
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 504,350
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=LiGePS-SSE-PBEsol" target="_blank">
     https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=LiGePS-SSE-PBEsol
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0041849" target="_blank">
     https://doi.org/10.1063/5.0041849
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_9p3sip4yhiju_0" target="_blank">
     LiSiPS_SSE_PBE
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 9,100 configurations of Li10SiP2S12, based on crystal structures from the Materials Project database, material ID mp-696129. One of two LiSiPS datasets from this source. The other uses the PBEsol functional, rather than the PBE functional.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    LiSiPS_SSE_PBE__Huang-Zhang-Wang-Zhao-Cheng-E__DS_9p3sip4yhiju_0
    <br/>
    <b>
     Name:
    </b>
    LiSiPS_SSE_PBE
    <br/>
    <b>
     Authors:
    </b>
    Jianxing Huang, Linfeng Zhang, Han Wang, Jinbao Zhao, Jun Cheng, Weinan E
    <br/>
    <b>
     Elements
    </b>
    : Li, P, S, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 9,169
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,101,550
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=LiSiPS-SSE-PBE" target="_blank">
     https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=LiSiPS-SSE-PBE
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0041849" target="_blank">
     https://doi.org/10.1063/5.0041849
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_th9w66lyxspv_0" target="_blank">
     LiSiPS_SSE_PBEsol
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 2,300 configurations of Li10SiP2S12, based on crystal structures from the Materials Project database, material ID mp-696129. One of two LiSiPS datasets from this source. The other uses the PBE functional, rather than the PBEsol functional.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    LiSiPS_SSE_PBEsol__Huang-Zhang-Wang-Zhao-Cheng-E__DS_th9w66lyxspv_0
    <br/>
    <b>
     Name:
    </b>
    LiSiPS_SSE_PBEsol
    <br/>
    <b>
     Authors:
    </b>
    Jianxing Huang, Linfeng Zhang, Han Wang, Jinbao Zhao, Jun Cheng, Weinan E
    <br/>
    <b>
     Elements
    </b>
    : Li, P, S, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,357
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 313,150
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=LiSiPS-SSE-PBEsol" target="_blank">
     https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=LiSiPS-SSE-PBEsol
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0041849" target="_blank">
     https://doi.org/10.1063/5.0041849
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_0pv4lpmx2ov3_0" target="_blank">
     LiTiO_Science_2020
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset contains configurations of lithium titanate from the publication Kinetic Pathways of ionic transport in fast-charging lithium titanate. In order to understand the origin of various EELS (electron energy-loss spectroscopy) spectra features, EELS spectra were simulated using the Vienna Ab initio Simulation (VASP) package. For a specific Li in a given configuration, this is done by calculating the DOS and integrated DOS considering a Li core-hole on the position of the specific Li and calculating the EELS based on the DOS. The minimum energy paths (MEP) and migration energy of Li were calculated in various compositions, including Li4Ti5O12 with an additional Li carrier, Li5Ti5O12 with an additional Li carrier, and Li7Ti5O12 with a Li vacancy carrier.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    LiTiO_Science_2020__Chen-Seo__DS_0pv4lpmx2ov3_0
    <br/>
    <b>
     Name:
    </b>
    LiTiO_Science_2020
    <br/>
    <b>
     Authors:
    </b>
    Tina Chen, Dong-hwa Seo
    <br/>
    <b>
     Elements
    </b>
    : Be, Li, O, Ti
    <br/>
    <b>
     Number of Configurations
    </b>
    : 849
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 150,105
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:2020.0006/v1" target="_blank">
     https://doi.org/10.24435/materialscloud:2020.0006/v1
    </a>
    <br/>
    <a href="https://doi.org/10.1126/science.aax3520" target="_blank">
     https://doi.org/10.1126/science.aax3520
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_eqt5dbhsmm68_0" target="_blank">
     MD22_AT_AT
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset containing MD trajectories of AT-AT DNA base pairs from the MD22 benchmark set. {DESC}
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    MD22_AT_AT__Chmiela-Vassilev-Galindo-Unke-Kabylda-Sauceda-Tkatchenko-Müller__DS_eqt5dbhsmm68_0
    <br/>
    <b>
     Name:
    </b>
    MD22_AT_AT
    <br/>
    <b>
     Authors:
    </b>
    Stefan Chmiela, Valentin Vassilev-Galindo, Oliver T. Unke, Adil Kabylda, Huziel E. Sauceda, Alexandre Tkatchenko, Klaus-Robert Müller
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 20,001
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,200,060
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://sgdml.org/" target="_blank">
     http://sgdml.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1126/sciadv.adf0873" target="_blank">
     https://doi.org/10.1126/sciadv.adf0873
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_rx1ei5q0x9gy_0" target="_blank">
     MD22_AT_AT_CG_CG
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset containing MD trajectories of AT-AT-CG-CG DNA base pairs from the MD22 benchmark set. MD22 represents a collection of datasets in a benchmark that can be considered an updated version of the MD17 benchmark datasets, including more challenges with respect to system size, flexibility and degree of non-locality. The datasets in MD22 include MD trajectories of the protein Ac-Ala3-NHMe; the lipid DHA (docosahexaenoic acid); the carbohydrate stachyose; nucleic acids AT-AT and AT-AT-CG-CG; and the buckyball catcher and double-walled nanotube supramolecules. Each of these is included here in a separate dataset, as represented on sgdml.org. Calculations were performed using FHI-aims and i-Pi software at the DFT-PBE+MBD level of theory. Trajectories were sampled at temperatures between 400-500 K at 1 fs resolution.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    MD22_AT_AT_CG_CG__Chmiela-Vassilev-Galindo-Unke-Kabylda-Sauceda-Tkatchenko-Müller__DS_rx1ei5q0x9gy_0
    <br/>
    <b>
     Name:
    </b>
    MD22_AT_AT_CG_CG
    <br/>
    <b>
     Authors:
    </b>
    Stefan Chmiela, Valentin Vassilev-Galindo, Oliver T. Unke, Adil Kabylda, Huziel E. Sauceda, Alexandre Tkatchenko, Klaus-Robert Müller
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 10,153
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,198,054
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://sgdml.org/" target="_blank">
     http://sgdml.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1126/sciadv.adf0873" target="_blank">
     https://doi.org/10.1126/sciadv.adf0873
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_l6awe3jnivnm_0" target="_blank">
     MD22_Ac_Ala3_NHMe
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset containing MD trajectories of the 42-atom tetrapeptide Ac-Ala3-NHMe from the MD22 benchmark set. MD22 represents a collection of datasets in a benchmark that can be considered an updated version of the MD17 benchmark datasets, including more challenges with respect to system size, flexibility and degree of non-locality. The datasets in MD22 include MD trajectories of the protein Ac-Ala3-NHMe; the lipid DHA (docosahexaenoic acid); the carbohydrate stachyose; nucleic acids AT-AT and AT-AT-CG-CG; and the buckyball catcher and double-walled nanotube supramolecules. Each of these is included here in a separate dataset, as represented on sgdml.org. Calculations were performed using FHI-aims and i-Pi software at the DFT-PBE+MBD level of theory. Trajectories were sampled at temperatures between 400-500 K at 1 fs resolution.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    MD22_Ac_Ala3_NHMe__Chmiela-Vassilev-Galindo-Unke-Kabylda-Sauceda-Tkatchenko-Müller__DS_l6awe3jnivnm_0
    <br/>
    <b>
     Name:
    </b>
    MD22_Ac_Ala3_NHMe
    <br/>
    <b>
     Authors:
    </b>
    Stefan Chmiela, Valentin Vassilev-Galindo, Oliver T. Unke, Adil Kabylda, Huziel E. Sauceda, Alexandre Tkatchenko, Klaus-Robert Müller
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 85,109
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,574,578
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://sgdml.org/" target="_blank">
     http://sgdml.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1126/sciadv.adf0873" target="_blank">
     https://doi.org/10.1126/sciadv.adf0873
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_xwmthngerf71_0" target="_blank">
     MD22_DHA
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset containing MD trajectories of DHA (docosahexaenoic acid) from the MD22 benchmark set. MD22 represents a collection of datasets in a benchmark that can be considered an updated version of the MD17 benchmark datasets, including more challenges with respect to system size, flexibility and degree of non-locality. The datasets in MD22 include MD trajectories of the protein Ac-Ala3-NHMe; the lipid DHA (docosahexaenoic acid); the carbohydrate stachyose; nucleic acids AT-AT and AT-AT-CG-CG; and the buckyball catcher and double-walled nanotube supramolecules. Each of these is included here in a separate dataset, as represented on sgdml.org. Calculations were performed using FHI-aims and i-Pi software at the DFT-PBE+MBD level of theory. Trajectories were sampled at temperatures between 400-500 K at 1 fs resolution.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    MD22_DHA__Chmiela-Vassilev-Galindo-Unke-Kabylda-Sauceda-Tkatchenko-Müller__DS_xwmthngerf71_0
    <br/>
    <b>
     Name:
    </b>
    MD22_DHA
    <br/>
    <b>
     Authors:
    </b>
    Stefan Chmiela, Valentin Vassilev-Galindo, Oliver T. Unke, Adil Kabylda, Huziel E. Sauceda, Alexandre Tkatchenko, Klaus-Robert Müller
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 69,753
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,906,168
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://sgdml.org/" target="_blank">
     http://sgdml.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1126/sciadv.adf0873" target="_blank">
     https://doi.org/10.1126/sciadv.adf0873
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_7g8q68fszjvs_0" target="_blank">
     MD22_buckyball_catcher
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset containing MD trajectories of the buckyball-catcher supramolecule from the MD22 benchmark set. MD22 represents a collection of datasets in a benchmark that can be considered an updated version of the MD17 benchmark datasets, including more challenges with respect to system size, flexibility and degree of non-locality. The datasets in MD22 include MD trajectories of the protein Ac-Ala3-NHMe; the lipid DHA (docosahexaenoic acid); the carbohydrate stachyose; nucleic acids AT-AT and AT-AT-CG-CG; and the buckyball catcher and double-walled nanotube supramolecules. Each of these is included here in a separate dataset, as represented on sgdml.org. Calculations were performed using FHI-aims and i-Pi software at the DFT-PBE+MBD level of theory. Trajectories were sampled at temperatures between 400-500 K at 1 fs resolution.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    MD22_buckyball_catcher__Chmiela-Vassilev-Galindo-Unke-Kabylda-Sauceda-Tkatchenko-Müller__DS_7g8q68fszjvs_0
    <br/>
    <b>
     Name:
    </b>
    MD22_buckyball_catcher
    <br/>
    <b>
     Authors:
    </b>
    Stefan Chmiela, Valentin Vassilev-Galindo, Oliver T. Unke, Adil Kabylda, Huziel E. Sauceda, Alexandre Tkatchenko, Klaus-Robert Müller
    <br/>
    <b>
     Elements
    </b>
    : C, H
    <br/>
    <b>
     Number of Configurations
    </b>
    : 6,102
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 903,096
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://sgdml.org/" target="_blank">
     http://sgdml.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1126/sciadv.adf0873" target="_blank">
     https://doi.org/10.1126/sciadv.adf0873
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_d92bnafzwynr_0" target="_blank">
     MD22_double_walled_nanotube
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset containing MD trajectories of the double-walled nanotube supramolecule from the MD22 benchmark set. MD22 represents a collection of datasets in a benchmark that can be considered an updated version of the MD17 benchmark datasets, including more challenges with respect to system size, flexibility and degree of non-locality. The datasets in MD22 include MD trajectories of the protein Ac-Ala3-NHMe; the lipid DHA (docosahexaenoic acid); the carbohydrate stachyose; nucleic acids AT-AT and AT-AT-CG-CG; and the buckyball catcher and double-walled nanotube supramolecules. Each of these is included here in a separate dataset, as represented on sgdml.org. Calculations were performed using FHI-aims and i-Pi software at the DFT-PBE+MBD level of theory. Trajectories were sampled at temperatures between 400-500 K at 1 fs resolution.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    MD22_double_walled_nanotube__Chmiela-Vassilev-Galindo-Unke-Kabylda-Sauceda-Tkatchenko-Müller__DS_d92bnafzwynr_0
    <br/>
    <b>
     Name:
    </b>
    MD22_double_walled_nanotube
    <br/>
    <b>
     Authors:
    </b>
    Stefan Chmiela, Valentin Vassilev-Galindo, Oliver T. Unke, Adil Kabylda, Huziel E. Sauceda, Alexandre Tkatchenko, Klaus-Robert Müller
    <br/>
    <b>
     Elements
    </b>
    : C, H
    <br/>
    <b>
     Number of Configurations
    </b>
    : 5,032
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,861,840
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://sgdml.org/" target="_blank">
     http://sgdml.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1126/sciadv.adf0873" target="_blank">
     https://doi.org/10.1126/sciadv.adf0873
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_87ojug1k96ef_0" target="_blank">
     MD22_stachyose
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset containing MD trajectories of the tetrasaccharide stachyose from the MD22 benchmark set. MD22 represents a collection of datasets in a benchmark that can be considered an updated version of the MD17 benchmark datasets, including more challenges with respect to system size, flexibility and degree of non-locality. The datasets in MD22 include MD trajectories of the protein Ac-Ala3-NHMe; the lipid DHA (docosahexaenoic acid); the carbohydrate stachyose; nucleic acids AT-AT and AT-AT-CG-CG; and the buckyball catcher and double-walled nanotube supramolecules. Each of these is included here in a separate dataset, as represented on sgdml.org. Calculations were performed using FHI-aims and i-Pi software at the DFT-PBE+MBD level of theory. Trajectories were sampled at temperatures between 400-500 K at 1 fs resolution.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    MD22_stachyose__Chmiela-Vassilev-Galindo-Unke-Kabylda-Sauceda-Tkatchenko-Müller__DS_87ojug1k96ef_0
    <br/>
    <b>
     Name:
    </b>
    MD22_stachyose
    <br/>
    <b>
     Authors:
    </b>
    Stefan Chmiela, Valentin Vassilev-Galindo, Oliver T. Unke, Adil Kabylda, Huziel E. Sauceda, Alexandre Tkatchenko, Klaus-Robert Müller
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 27,272
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,372,664
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://sgdml.org/" target="_blank">
     http://sgdml.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1126/sciadv.adf0873" target="_blank">
     https://doi.org/10.1126/sciadv.adf0873
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_rsao7xrpu9ig_0" target="_blank">
     MISPR
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Example dataset for MISPR (Materials Informatics for Structure-Property Relationships) materials science simulation software, with DFT-calculated configuration properties for three different MISPR workflows: nuclear magnetic resonance (NMR) chemical shifts, electrostatic partial charges (ESP) and bond dissociation energies (BDE).
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    MISPR__Atwi-Bliss-Makeev-Rajput__DS_rsao7xrpu9ig_0
    <br/>
    <b>
     Name:
    </b>
    MISPR
    <br/>
    <b>
     Authors:
    </b>
    Rasha Atwi, Matthew Bliss, Maxim Makeev, Nav Nidhi Rajput
    <br/>
    <b>
     Elements
    </b>
    : C, Cl, F, H, N, O, P, S, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 503
    <br/>
    <b>
     Number of Elements
    </b>
    : 9
    <br/>
    <b>
     Number of Atoms
    </b>
    : 8,996
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1038/s41598-022-20009-w" target="_blank">
     https://doi.org/10.1038/s41598-022-20009-w
    </a>
    <br/>
    <a href="https://github.com/rashatwi/mispr-dataset" target="_blank">
     https://github.com/rashatwi/mispr-dataset
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_326i4urabisb_0" target="_blank">
     MTPu_2023
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A comprehensive database generated using density functional theory simulations, encompassing a wide range of crystal structures, point defects, extended defects, and disordered structure.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    MTPu_2023__Zongo-Sun-Ouellet-Plamondon-Beland__DS_326i4urabisb_0
    <br/>
    <b>
     Name:
    </b>
    MTPu_2023
    <br/>
    <b>
     Authors:
    </b>
    Karim Zongo, Hao Sun, Claudiane Ouellet-Plamondon, Laurent Karim Beland
    <br/>
    <b>
     Elements
    </b>
    : O, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,062
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 71,595
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://gitlab.com/Kazongogit/MTPu" target="_blank">
     https://gitlab.com/Kazongogit/MTPu
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2311.15170" target="_blank">
     https://doi.org/10.48550/arXiv.2311.15170
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_5drebe4tktiu_0" target="_blank">
     Matbench_mp_e_form
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Matbench v0.1 test dataset for predicting DFT formation energy from structure. Adapted from Materials Project database. Entries having formation energy more than 2.5eV and those containing noble gases are removed. Retrieved April 2, 2019. For benchmarking w/ nested cross validation, the order of the dataset must be identical to the retrieved data; refer to the Automatminer/Matbench publication for more details.Matbench is an automated leaderboard for benchmarking state of the art ML algorithms predicting a diverse range of solid materials' properties. It is hosted and maintained by the Materials Project.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Matbench_mp_e_form__Dunn-Wang-Ganose-Dopp-Jain__DS_5drebe4tktiu_0
    <br/>
    <b>
     Name:
    </b>
    Matbench_mp_e_form
    <br/>
    <b>
     Authors:
    </b>
    Alexander Dunn, Qi Wang, Alex Ganose, Daniel Dopp, Anubhav Jain
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, Hf, Hg, Ho, I, In, Ir, K, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 132,752
    <br/>
    <b>
     Number of Elements
    </b>
    : 84
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,869,658
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://matbench.materialsproject.org/" target="_blank">
     https://matbench.materialsproject.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-020-00406-3" target="_blank">
     https://doi.org/10.1038/s41524-020-00406-3
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.commatsci.2014.10.037" target="_blank">
     https://doi.org/10.1016/j.commatsci.2014.10.037
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_6lq2f26dluql_0" target="_blank">
     Matbench_mp_gap
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The Matbench_mp_gap dataset is a Matbench v0.1 test dataset for predicting DFT PBE band gap from structure, adapted from the Materials Project database. Entries having a formation energy (or energy above the convex hull) greater than 150meV and those containing noble gases have been removed. Retrieved April 2, 2019. Refer to the Automatminer/Matbench publication for more details. This dataset contains band gap as calculated by PBE DFT from the Materials Project, in eV. Matbench is an automated leaderboard for benchmarking state of the art ML algorithms predicting a diverse range of solid materials' properties. It is hosted and maintained by the Materials Project.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Matbench_mp_gap__Dunn-Wang-Ganose-Dopp-Jain__DS_6lq2f26dluql_0
    <br/>
    <b>
     Name:
    </b>
    Matbench_mp_gap
    <br/>
    <b>
     Authors:
    </b>
    Alexander Dunn, Qi Wang, Alex Ganose, Daniel Dopp, Anubhav Jain
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, Hf, Hg, Ho, I, In, Ir, K, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 106,113
    <br/>
    <b>
     Number of Elements
    </b>
    : 84
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,184,984
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://matbench.materialsproject.org/" target="_blank">
     https://matbench.materialsproject.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1039/C2EE22341D" target="_blank">
     https://doi.org/10.1039/C2EE22341D
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_sinisy47w8ff_0" target="_blank">
     Matbench_perovskites
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The Matbench_perovskites dataset is a Matbench v0.1 test dataset for predicting formation energy from crystal structure. Adapted from an original dataset generated by Castelli et al. Refer to the Automatminer/Matbench publication for more details. This dataset contains the energy of formation of the entire 5-atom perovskite cell in eV as calculated by RPBE GGA-DFT. Note the reference state for oxygen was computed from oxygen's chemical potential in water vapor, not as oxygen molecules, to reflect the application for which these perovskites were studied. Matbench is an automated leaderboard for benchmarking state of the art ML algorithms predicting a diverse range of solid materials' properties. It is hosted and maintained by the Materials Project.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Matbench_perovskites__Dunn-Wang-Ganose-Dopp-Jain__DS_sinisy47w8ff_0
    <br/>
    <b>
     Name:
    </b>
    Matbench_perovskites
    <br/>
    <b>
     Authors:
    </b>
    Alexander Dunn, Qi Wang, Alex Ganose, Daniel Dopp, Anubhav Jain
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Ba, Be, Bi, Ca, Cd, Co, Cr, Cs, Cu, F, Fe, Ga, Ge, Hf, Hg, In, Ir, K, La, Li, Mg, Mn, Mo, N, Na, Nb, Ni, O, Os, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Si, Sn, Sr, Ta, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 18,928
    <br/>
    <b>
     Number of Elements
    </b>
    : 56
    <br/>
    <b>
     Number of Atoms
    </b>
    : 94,640
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://matbench.materialsproject.org/" target="_blank">
     https://matbench.materialsproject.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1039/C2EE22341D" target="_blank">
     https://doi.org/10.1039/C2EE22341D
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_pv1f3dlo5dsc_0" target="_blank">
     Materials_Project
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations from the Materials Project database: an online resource with the goal of computing properties of all inorganic materials.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Materials_Project__Jain-Ong-Hautier-Chen-Richards-Dacek-Cholia-Gunter-Skinner-Ceder-Persson__DS_pv1f3dlo5dsc_0
    <br/>
    <b>
     Name:
    </b>
    Materials_Project
    <br/>
    <b>
     Authors:
    </b>
    Anubhav Jain, Shyue Ping Ong, Geoffroy Hautier, Wei Chen, William Davidson Richards, Stephen Dacek, Shreyas Cholia, Dan Gunter, David Skinner, Gerbrand Ceder, Kristin A. Persson
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ne, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 6,342,530
    <br/>
    <b>
     Number of Elements
    </b>
    : 89
    <br/>
    <b>
     Number of Atoms
    </b>
    : 200,016,161
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://materialsproject.org" target="_blank">
     https://materialsproject.org
    </a>
    <br/>
    <a href="https://doi.org/10.1063/1.4812323" target="_blank">
     https://doi.org/10.1063/1.4812323
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_caktb6z8yiy7_0" target="_blank">
     Mg_edmonds_2022
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    16748 configurations of magnesium with gathered energy, stress and forces at the DFT level of theory.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Mg_edmonds_2022__Poul__DS_caktb6z8yiy7_0
    <br/>
    <b>
     Name:
    </b>
    Mg_edmonds_2022
    <br/>
    <b>
     Authors:
    </b>
    Marvin Poul
    <br/>
    <b>
     Elements
    </b>
    : Mg
    <br/>
    <b>
     Number of Configurations
    </b>
    : 16,874
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 78,617
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.17617/3.A3MB7Z" target="_blank">
     https://doi.org/10.17617/3.A3MB7Z
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevB.107.104103" target="_blank">
     https://doi.org/10.1103/PhysRevB.107.104103
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_0shp3qrqk9k9_0" target="_blank">
     MoNbTaVW_PRB2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset was originally designed to fit a GAP model for the Mo-Nb-Ta-V-W quinary system that was used to study segregation and defects in the body-centered-cubic refractory high-entropy alloy MoNbTaVW.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    MoNbTaVW_PRB2021__Byggmästar-Nordlund-Djurabekova__DS_0shp3qrqk9k9_0
    <br/>
    <b>
     Name:
    </b>
    MoNbTaVW_PRB2021
    <br/>
    <b>
     Authors:
    </b>
    Jesper Byggmästar, Kai Nordlund, Flyura Djurabekova
    <br/>
    <b>
     Elements
    </b>
    : Mo, Nb, Ta, V, W
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,329
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 127,913
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.23729/1b845398-5291-4447-b417-1345acdd2eae" target="_blank">
     https://doi.org/10.23729/1b845398-5291-4447-b417-1345acdd2eae
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevB.104.104101" target="_blank">
     https://doi.org/10.1103/PhysRevB.104.104101
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_5aeg7va6k305_0" target="_blank">
     Mo_PRM2019
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset was designed to enable machine learning of Mo elastic, thermal, and defect properties, as well as surface energetics, melting, and the structure of the liquid phase. The dataset was constructed by starting with the dataset from J. Byggmästar et al., Phys. Rev. B 100, 144105 (2019), then rescaling all of the configurations to the correct lattice spacing and adding in gamma surface configurations.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Mo_PRM2019__Byggmästar-Nordlund-Djurabekova__DS_5aeg7va6k305_0
    <br/>
    <b>
     Name:
    </b>
    Mo_PRM2019
    <br/>
    <b>
     Authors:
    </b>
    Jesper Byggmästar, Kai Nordlund, Flyura Djurabekova
    <br/>
    <b>
     Elements
    </b>
    : Mo
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,785
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 45,667
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://gitlab.com/acclab/gap-data/-/tree/master/Mo" target="_blank">
     https://gitlab.com/acclab/gap-data/-/tree/master/Mo
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevMaterials.4.093802" target="_blank">
     https://doi.org/10.1103/PhysRevMaterials.4.093802
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_oqqwzogut1on_0" target="_blank">
     NDSC_TUT_2022
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    500 configurations of Mg2 for MD prediction using a model fitted on Al, W, Mg and Si.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    NDSC_TUT_2022__Allen-Bartok__DS_oqqwzogut1on_0
    <br/>
    <b>
     Name:
    </b>
    NDSC_TUT_2022
    <br/>
    <b>
     Authors:
    </b>
    Connor Allen, Albert P. Bartok
    <br/>
    <b>
     Elements
    </b>
    : Mg
    <br/>
    <b>
     Number of Configurations
    </b>
    : 500
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/ConnorSA/ndsc_tut" target="_blank">
     https://github.com/ConnorSA/ndsc_tut
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2207.11828" target="_blank">
     https://doi.org/10.48550/arXiv.2207.11828
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_0j2smy6relq0_0" target="_blank">
     NENCI-2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    NENCI-2021 is a database of approximately 8000 benchmark Non-Equilibirum Non-Covalent Interaction (NENCI) energies performed on molecular dimers;intermolecular complexes of biological and chemical relevance with a particular emphasis on close intermolecular contacts. Based on dimersfrom the S101 database.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    NENCI-2021__Sparrow-Ernst-Joo-Lao-Jr__DS_0j2smy6relq0_0
    <br/>
    <b>
     Name:
    </b>
    NENCI-2021
    <br/>
    <b>
     Authors:
    </b>
    Zachary M. Sparrow, Brian G. Ernst, Paul T. Joo, Ka Un Lao, Robert A. DiStasio, Jr
    <br/>
    <b>
     Elements
    </b>
    : Br, C, Cl, F, H, Li, N, Na, O, P, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 7,763
    <br/>
    <b>
     Number of Elements
    </b>
    : 11
    <br/>
    <b>
     Number of Atoms
    </b>
    : 129,402
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://pubs.aip.org/jcp/article-supplement/199609/zip/184303_1_supplements/" target="_blank">
     https://pubs.aip.org/jcp/article-supplement/199609/zip/184303_1_supplements/
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0068862" target="_blank">
     https://doi.org/10.1063/5.0068862
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_yg0yzoiwoux8_0" target="_blank">
     NEP_PRB_2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 7,000 distinct configurations of 2D-silicene, silicon, and PbTe. Silicon data used from http://dx.doi.org/10.1103/PhysRevX.8.041048. Dataset includes predicted force, potential energy and virial values.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    NEP_PRB_2021__Fan__DS_yg0yzoiwoux8_0
    <br/>
    <b>
     Name:
    </b>
    NEP_PRB_2021
    <br/>
    <b>
     Authors:
    </b>
    Zheyong Fan
    <br/>
    <b>
     Elements
    </b>
    : Pb, Si, Te
    <br/>
    <b>
     Number of Configurations
    </b>
    : 7,426
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 611,808
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.5109599" target="_blank">
     https://doi.org/10.5281/zenodo.5109599
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevB.104.104309" target="_blank">
     https://doi.org/10.1103/PhysRevB.104.104309
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_mc8p14cpn2ea_0" target="_blank">
     NEP_qHPF_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The test set of a train and test set pair.The combined datasets comprise approximately 275 configurations of monolayer quasi-hexagonal-phase fullerene (qHPF) membrane used to train and test an NEP model.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    NEP_qHPF_test__Ying__DS_mc8p14cpn2ea_0
    <br/>
    <b>
     Name:
    </b>
    NEP_qHPF_test
    <br/>
    <b>
     Authors:
    </b>
    Penghua Ying
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 39
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 4,680
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7018572" target="_blank">
     https://doi.org/10.5281/zenodo.7018572
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.eml.2022.101929" target="_blank">
     https://doi.org/10.1016/j.eml.2022.101929
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_umliq1qh50gy_0" target="_blank">
     NEP_qHPF_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The train set of a train and test set pair.The combined datasets comprise approximately 275 configurations of monolayer quasi-hexagonal-phase fullerene (qHPF) membrane used to train and test an NEP model.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    NEP_qHPF_train__Ying__DS_umliq1qh50gy_0
    <br/>
    <b>
     Name:
    </b>
    NEP_qHPF_train
    <br/>
    <b>
     Authors:
    </b>
    Penghua Ying
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 238
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 28,560
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7018572" target="_blank">
     https://doi.org/10.5281/zenodo.7018572
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.eml.2022.101929" target="_blank">
     https://doi.org/10.1016/j.eml.2022.101929
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_of1clsgf4nab_0" target="_blank">
     NMD-18
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    3,000 Al-Ga-In sesquioxides with energies and band gaps. Relaxed and Vegard's Law geometries with formation energy and band gaps at DFT-PBE level of theory of (Alx-Gay-Inz)2O3 oxides, x+y+z=1. Contains all structures from the NOMAD 2018 Kaggle challenge training and leaderboard data. The formation energy and bandgap energy were computed by using the PBE exchange-correlation DFT functional with the all-electron electronic structure code FHI-aims with tight setting.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    NMD-18__Sutton-Ghiringhelli-Yamamoto-Lysogorskiy-Blumenthal-Hammerschmidt-Golebiowski-Liu-Ziletti-Scheffler__DS_of1clsgf4nab_0
    <br/>
    <b>
     Name:
    </b>
    NMD-18
    <br/>
    <b>
     Authors:
    </b>
    Christopher Sutton, Luca M. Ghiringhelli, Takenori Yamamoto, Yury Lysogorskiy, Lars Blumenthal, Thomas Hammerschmidt, Jacek R. Golebiowski, Xiangyue Liu, Angelo Ziletti, Matthias Scheffler
    <br/>
    <b>
     Elements
    </b>
    : Al, Ga, In, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 185,070
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://qmml.org/datasets.html" target="_blank">
     https://qmml.org/datasets.html
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-019-0239-3" target="_blank">
     https://doi.org/10.1038/s41524-019-0239-3
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_nb8hcpibz1dt_0" target="_blank">
     NNIP_FeH_PRM_2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 20,000 configurations from a dataset of alpha-iron and hydrogen. Properties include forces and potential energy, calculated using VASP at the DFT level using the GGA-PBE functional.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    NNIP_FeH_PRM_2021__Meng-Du-Shinzato-Mori-Yu-Matsubara-Ishikawa-Ogata__DS_nb8hcpibz1dt_0
    <br/>
    <b>
     Name:
    </b>
    NNIP_FeH_PRM_2021
    <br/>
    <b>
     Authors:
    </b>
    Fan-Shun Meng, Jun-Ping Du, Shuhei Shinzato, Hideki Mori, Peijun Yu, Kazuki Matsubara, Nobuyuki Ishikawa, Shigenobu Ogata
    <br/>
    <b>
     Elements
    </b>
    : Fe, H
    <br/>
    <b>
     Number of Configurations
    </b>
    : 20,920
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,870,008
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/mengfsou/NNIP-FeH" target="_blank">
     https://github.com/mengfsou/NNIP-FeH
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevMaterials.5.113606" target="_blank">
     https://doi.org/10.1103/PhysRevMaterials.5.113606
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_3qumwe2j8lib_0" target="_blank">
     NNP-Ga2O3
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    9,200 configurations of beta-Ga2O3, including two configuration sets. One contains DFT data for 8400 configurations simulated between temperatures of 50K - 600K. The second contains configurations with 0K simulation temperature.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    NNP-Ga2O3__Li-Liu-Rohskopf-Gordiz-Henry-Lee-Luo__DS_3qumwe2j8lib_0
    <br/>
    <b>
     Name:
    </b>
    NNP-Ga2O3
    <br/>
    <b>
     Authors:
    </b>
    Ruiyang Li, Zeyu Liu, Andrew Rohskopf, Kiarash Gordiz, Asegun Henry, Eungkyu Lee, Tengfei Luo
    <br/>
    <b>
     Elements
    </b>
    : Ga, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 9,200
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,944,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/RuiyangLi6/NNP_Ga2O3" target="_blank">
     https://github.com/RuiyangLi6/NNP_Ga2O3
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0025051" target="_blank">
     https://doi.org/10.1063/5.0025051
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_0gyf3srv7xh7_0" target="_blank">
     NVNMD_GeTe
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 5,000 configurations of GeTe used in training of a non-von Neumann multiplication-less DNN model.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    NVNMD_GeTe__Mo-Li-Zhao-Zhang-Shi-Li-Liu__DS_0gyf3srv7xh7_0
    <br/>
    <b>
     Name:
    </b>
    NVNMD_GeTe
    <br/>
    <b>
     Authors:
    </b>
    Pinghui Mo, Chang Li, Dan Zhao, Yujia Zhang, Mengchao Shi, Junhua Li, Jie Liu
    <br/>
    <b>
     Elements
    </b>
    : Ge, Te
    <br/>
    <b>
     Number of Configurations
    </b>
    : 5,025
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 321,600
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/LiuGroupHNU/nvnmd" target="_blank">
     https://github.com/LiuGroupHNU/nvnmd
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-022-00773-z" target="_blank">
     https://doi.org/10.1038/s41524-022-00773-z
    </a>
    <br/>
    <a href="https://doi.org/10.1109/LED.2020.2964779" target="_blank">
     https://doi.org/10.1109/LED.2020.2964779
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_p74559sdjy1q_0" target="_blank">
     N_O_F_columns_non-bonded_vdW_potential_JCP2023
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset contains structures of materials from the N (15th), O (16th) and F (16th) columns of the periodic table used for generating a 2-body non-bonded vdW potential.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    N_O_F_columns_non-bonded_vdW_potential_JCP2023__Geng-Zybin-Naserifar-III__DS_p74559sdjy1q_0
    <br/>
    <b>
     Name:
    </b>
    N_O_F_columns_non-bonded_vdW_potential_JCP2023
    <br/>
    <b>
     Authors:
    </b>
    Peng Geng, Sergey Zybin, Saber Naserifar, William A. Goddard, III
    <br/>
    <b>
     Elements
    </b>
    : As, At, Bi, O, P, Po, S, Sb, Se, Te
    <br/>
    <b>
     Number of Configurations
    </b>
    : 262
    <br/>
    <b>
     Number of Elements
    </b>
    : 10
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,494
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1063/5.0174188" target="_blank">
     https://doi.org/10.1063/5.0174188
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0174188" target="_blank">
     https://doi.org/10.1063/5.0174188
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_tbpx3pawtlgt_0" target="_blank">
     Nb_PRM2019
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset was designed to enable machine-learning of Nb elastic, thermal, and defect properties, as well as surface energetics, melting, and the structure of the liquid phase. The dataset was constructed by starting with the dataset from J. Byggmästar et al., Phys. Rev. B 100, 144105 (2019), then rescaling all of the configurations to the correct lattice spacing and adding in gamma surface configurations.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Nb_PRM2019__Byggmästar-Nordlund-Djurabekova__DS_tbpx3pawtlgt_0
    <br/>
    <b>
     Name:
    </b>
    Nb_PRM2019
    <br/>
    <b>
     Authors:
    </b>
    Jesper Byggmästar, Kai Nordlund, Flyura Djurabekova
    <br/>
    <b>
     Elements
    </b>
    : Nb
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,787
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 45,641
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://gitlab.com/acclab/gap-data/-/tree/master/" target="_blank">
     https://gitlab.com/acclab/gap-data/-/tree/master/
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevMaterials.4.093802" target="_blank">
     https://doi.org/10.1103/PhysRevMaterials.4.093802
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_4pbhjtu62o2d_0" target="_blank">
     NequIP_NC_2022
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 57,000 configurations from the evaluation datasets for NequIP graph neural network model for interatomic potentials. Trajectories have been taken from LIPS, LIPO glass melt-quench simulation, and formate decomposition on Cu datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    NequIP_NC_2022__Batzner-Musaelian-Sun-Geiger-Mailoa-Kornbluth-Molinari-Smidt-Kozinsky__DS_4pbhjtu62o2d_0
    <br/>
    <b>
     Name:
    </b>
    NequIP_NC_2022
    <br/>
    <b>
     Authors:
    </b>
    Simon Batzner, Albert Musaelian, Lixin Sun, Mario Geiger, Jonathan P. Mailoa, Mordechai Kornbluth, Nicola Molinari, Tess E. Smidt, Boris Kozinsky
    <br/>
    <b>
     Elements
    </b>
    : C, Cu, H, Li, O, P, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 56,856
    <br/>
    <b>
     Number of Elements
    </b>
    : 7
    <br/>
    <b>
     Number of Atoms
    </b>
    : 7,631,075
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:s0-5n" target="_blank">
     https://doi.org/10.24435/materialscloud:s0-5n
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-022-29939-5" target="_blank">
     https://doi.org/10.1038/s41467-022-29939-5
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_5tegg4uvaixz_0" target="_blank">
     NiCoCr_NC2020
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The face-centered cubic medium-entropy alloy NiCoCr has received considerable attention for its good mechanical properties, uncertain stacking fault energy, etc, some of which have been attributed to chemical short-range order (SRO). Here, we examine the yield strength and misfit volumes of NiCoCr to determine whether SRO has measurably influenced mechanical properties. Polycrystalline strengths show no systematic trend with different processing conditions. Measured misfit volumes in NiCoCr are consistent with those in random binaries. Yield strength prediction of a random NiCoCr alloy matches well with experiments. Finally, we show that standard spin-polarized density functional theory (DFT) calculations of misfit volumes are not accurate for NiCoCr. This implies that DFT may be inaccurate for other subtle structural quantities such as atom-atom bond distance so that caution is required in drawing conclusions about NiCoCr based on DFT. These findings all lead to the conclusion that, under typical processing conditions, SRO in NiCoCr is either negligible or has no systematic measurable effect on strength.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    NiCoCr_NC2020__Yin-Curtin__DS_5tegg4uvaixz_0
    <br/>
    <b>
     Name:
    </b>
    NiCoCr_NC2020
    <br/>
    <b>
     Authors:
    </b>
    Binglun Yin, William Curtin
    <br/>
    <b>
     Elements
    </b>
    : Co, Cr, Ni
    <br/>
    <b>
     Number of Configurations
    </b>
    : 428
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 40,624
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:s4-g3" target="_blank">
     https://doi.org/10.24435/materialscloud:s4-g3
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-020-16083-1" target="_blank">
     https://doi.org/10.1038/s41467-020-16083-1
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_dgop3abwq9ya_0" target="_blank">
     OC20_IS2RES_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset contains all frames from the trajectories for the training configurations in the OC20 initial structure to relaxed energy (IS2RE) and initial structure to relaxed structure (IS2RS) tasks of Open Catalyst 2020 (OC20). Dataset corresponds to the "All IS2RE/S training" data split under the "Relaxation Trajectories" section of the Open Catalyst Project page.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC20_IS2RES_train__Chanussot-Das-Goyal-Lavril-Shuaibi-Riviere-Tran-Heras-Domingo-Ho-Hu-Palizhati-Sriram-Wood-Yoon-Parikh-Zitnick-Ulissi__DS_dgop3abwq9ya_0
    <br/>
    <b>
     Name:
    </b>
    OC20_IS2RES_train
    <br/>
    <b>
     Authors:
    </b>
    Lowik Chanussot, Abhishek Das, Siddharth Goyal, Thibaut Lavril, Muhammed Shuaibi, Morgane Riviere, Kevin Tran, Javier Heras-Domingo, Caleb Ho, Weihua Hu, Aini Palizhati, Anuroop Sriram, Brandon Wood, Junwoong Yoon, Devi Parikh, C. Lawrence Zitnick, Zachary Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Bi, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 92,899,663
    <br/>
    <b>
     Number of Elements
    </b>
    : 56
    <br/>
    <b>
     Number of Atoms
    </b>
    : 7,522,707,627
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/oc20.html" target="_blank">
     https://fair-chem.github.io/core/datasets/oc20.html
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.0c04525" target="_blank">
     https://doi.org/10.1021/acscatal.0c04525
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_paumiiad6ttu_0" target="_blank">
     OC20_IS2RES_val_id
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    OC20_IS2RES_val_id is the in-domain validation set for the OC20 Initial Structure to Relaxed Structure (IS2RS) and Initial Structure to Relaxed Energy (IS2RE) tasks. Features include energy, atomic forces and data from the OC20 mappings file, including adsorbate id, materials project bulk id and miller index.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC20_IS2RES_val_id__Chanussot-Das-Goyal-Lavril-Shuaibi-Riviere-Tran-Heras-Domingo-Ho-Hu-Palizhati-Sriram-Wood-Yoon-Parikh-Zitnick-Ulissi__DS_paumiiad6ttu_0
    <br/>
    <b>
     Name:
    </b>
    OC20_IS2RES_val_id
    <br/>
    <b>
     Authors:
    </b>
    Lowik Chanussot, Abhishek Das, Siddharth Goyal, Thibaut Lavril, Muhammed Shuaibi, Morgane Riviere, Kevin Tran, Javier Heras-Domingo, Caleb Ho, Weihua Hu, Aini Palizhati, Anuroop Sriram, Brandon Wood, Junwoong Yoon, Devi Parikh, C. Lawrence Zitnick, Zachary Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Bi, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 5,026,511
    <br/>
    <b>
     Number of Elements
    </b>
    : 56
    <br/>
    <b>
     Number of Atoms
    </b>
    : 406,702,097
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/oc20.html" target="_blank">
     https://fair-chem.github.io/core/datasets/oc20.html
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.0c04525" target="_blank">
     https://doi.org/10.1021/acscatal.0c04525
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_71806e6y8u2m_0" target="_blank">
     OC20_IS2RES_val_ood_ads
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    OC20_IS2RES_ood_ads is the out-of-domain validation set for the OC20 Initial Structure to Relaxed Structure (IS2RS) and Initial Structure to Relaxed Energy (IS2RE) tasks with unseen adsorbates. Features include energy, atomic forces and data from the OC20 mappings file, including adsorbate id, materials project bulk id and miller index.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC20_IS2RES_val_ood_ads__Chanussot-Das-Goyal-Lavril-Shuaibi-Riviere-Tran-Heras-Domingo-Ho-Hu-Palizhati-Sriram-Wood-Yoon-Parikh-Zitnick-Ulissi__DS_71806e6y8u2m_0
    <br/>
    <b>
     Name:
    </b>
    OC20_IS2RES_val_ood_ads
    <br/>
    <b>
     Authors:
    </b>
    Lowik Chanussot, Abhishek Das, Siddharth Goyal, Thibaut Lavril, Muhammed Shuaibi, Morgane Riviere, Kevin Tran, Javier Heras-Domingo, Caleb Ho, Weihua Hu, Aini Palizhati, Anuroop Sriram, Brandon Wood, Junwoong Yoon, Devi Parikh, C. Lawrence Zitnick, Zachary Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Bi, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 4,883,196
    <br/>
    <b>
     Number of Elements
    </b>
    : 56
    <br/>
    <b>
     Number of Atoms
    </b>
    : 390,308,139
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/oc20.html" target="_blank">
     https://fair-chem.github.io/core/datasets/oc20.html
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.0c04525" target="_blank">
     https://doi.org/10.1021/acscatal.0c04525
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_u9jm7oxyusjc_0" target="_blank">
     OC20_IS2RES_val_ood_both
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    OC20_IS2RES_ood_ads is the out-of-domain validation set for the OC20 Initial Structure to Relaxed Structure (IS2RS) and Initial Structure to Relaxed Energy (IS2RE) tasks with unseen adsorbates. Features include energy, atomic forces and data from the OC20 mappings file, including adsorbate id, materials project bulk id and miller index.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC20_IS2RES_val_ood_both__Chanussot-Das-Goyal-Lavril-Shuaibi-Riviere-Tran-Heras-Domingo-Ho-Hu-Palizhati-Sriram-Wood-Yoon-Parikh-Zitnick-Ulissi__DS_u9jm7oxyusjc_0
    <br/>
    <b>
     Name:
    </b>
    OC20_IS2RES_val_ood_both
    <br/>
    <b>
     Authors:
    </b>
    Lowik Chanussot, Abhishek Das, Siddharth Goyal, Thibaut Lavril, Muhammed Shuaibi, Morgane Riviere, Kevin Tran, Javier Heras-Domingo, Caleb Ho, Weihua Hu, Aini Palizhati, Anuroop Sriram, Brandon Wood, Junwoong Yoon, Devi Parikh, C. Lawrence Zitnick, Zachary Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, Bi, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,665,193
    <br/>
    <b>
     Number of Elements
    </b>
    : 55
    <br/>
    <b>
     Number of Atoms
    </b>
    : 308,297,930
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/oc20.html" target="_blank">
     https://fair-chem.github.io/core/datasets/oc20.html
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.0c04525" target="_blank">
     https://doi.org/10.1021/acscatal.0c04525
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_ava2xqam2xfi_0" target="_blank">
     OC20_IS2RES_val_ood_cat
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    OC20_IS2RES_val_ood_cat is the out-of-domain validation set for the OC20 Initial Structure to Relaxed Structure (IS2RS) and Initial Structure to Relaxed Energy (IS2RE) tasks with unseen catalyst composition. Features include energy, atomic forces and data from the OC20 mappings file, including adsorbate id, materials project bulk id and miller index.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC20_IS2RES_val_ood_cat__Chanussot-Das-Goyal-Lavril-Shuaibi-Riviere-Tran-Heras-Domingo-Ho-Hu-Palizhati-Sriram-Wood-Yoon-Parikh-Zitnick-Ulissi__DS_ava2xqam2xfi_0
    <br/>
    <b>
     Name:
    </b>
    OC20_IS2RES_val_ood_cat
    <br/>
    <b>
     Authors:
    </b>
    Lowik Chanussot, Abhishek Das, Siddharth Goyal, Thibaut Lavril, Muhammed Shuaibi, Morgane Riviere, Kevin Tran, Javier Heras-Domingo, Caleb Ho, Weihua Hu, Aini Palizhati, Anuroop Sriram, Brandon Wood, Junwoong Yoon, Devi Parikh, C. Lawrence Zitnick, Zachary Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, Bi, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 5,151,015
    <br/>
    <b>
     Number of Elements
    </b>
    : 55
    <br/>
    <b>
     Number of Atoms
    </b>
    : 411,767,380
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/oc20.html" target="_blank">
     https://fair-chem.github.io/core/datasets/oc20.html
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.0c04525" target="_blank">
     https://doi.org/10.1021/acscatal.0c04525
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_zdy2xz6y88nl_0" target="_blank">
     OC20_S2EF_train_200K
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    OC20_S2EF_train_200K is the 200K training split of the OC20 Structure to Energy and Forces (S2EF) task.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC20_S2EF_train_200K__Chanussot-Das-Goyal-Lavril-Shuaibi-Riviere-Tran-Heras-Domingo-Ho-Hu-Palizhati-Sriram-Wood-Yoon-Parikh-Zitnick-Ulissi__DS_zdy2xz6y88nl_0
    <br/>
    <b>
     Name:
    </b>
    OC20_S2EF_train_200K
    <br/>
    <b>
     Authors:
    </b>
    Lowik Chanussot, Abhishek Das, Siddharth Goyal, Thibaut Lavril, Muhammed Shuaibi, Morgane Riviere, Kevin Tran, Javier Heras-Domingo, Caleb Ho, Weihua Hu, Aini Palizhati, Anuroop Sriram, Brandon Wood, Junwoong Yoon, Devi Parikh, C. Lawrence Zitnick, Zachary Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Bi, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 200,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 56
    <br/>
    <b>
     Number of Atoms
    </b>
    : 14,631,937
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/oc20.html" target="_blank">
     https://fair-chem.github.io/core/datasets/oc20.html
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.0c04525" target="_blank">
     https://doi.org/10.1021/acscatal.0c04525
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_otx1qc9f3pm4_0" target="_blank">
     OC20_S2EF_train_20M
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    OC20_S2EF_train_20M is the 20 million structure training subset of the OC20 Structure to Energy and Forces dataset. Features include potential energy, free energy and atomic forces. Data from the OC20 mappings file, including adsorbate id, materials project bulk id, miller index, shift and others.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC20_S2EF_train_20M__Chanussot-Das-Goyal-Lavril-Shuaibi-Riviere-Tran-Heras-Domingo-Ho-Hu-Palizhati-Sriram-Wood-Yoon-Parikh-Zitnick-Ulissi__DS_otx1qc9f3pm4_0
    <br/>
    <b>
     Name:
    </b>
    OC20_S2EF_train_20M
    <br/>
    <b>
     Authors:
    </b>
    Lowik Chanussot, Abhishek Das, Siddharth Goyal, Thibaut Lavril, Muhammed Shuaibi, Morgane Riviere, Kevin Tran, Javier Heras-Domingo, Caleb Ho, Weihua Hu, Aini Palizhati, Anuroop Sriram, Brandon Wood, Junwoong Yoon, Devi Parikh, C. Lawrence Zitnick, Zachary Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Bi, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 20,000,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 56
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,465,265,878
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/oc20.html" target="_blank">
     https://fair-chem.github.io/core/datasets/oc20.html
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.0c04525" target="_blank">
     https://doi.org/10.1021/acscatal.0c04525
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_7qi6dh0ig7sd_0" target="_blank">
     OC20_S2EF_train_2M
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    OC20_S2EF_train_2M is the 2 million structure training subset of the OC20 Structure to Energy and Forces dataset. Features include potential energy, free energy and atomic forces. Data from the OC20 mappings file, including adsorbate id, materials project bulk id, miller index, shift and others.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC20_S2EF_train_2M__Chanussot-Das-Goyal-Lavril-Shuaibi-Riviere-Tran-Heras-Domingo-Ho-Hu-Palizhati-Sriram-Wood-Yoon-Parikh-Zitnick-Ulissi__DS_7qi6dh0ig7sd_0
    <br/>
    <b>
     Name:
    </b>
    OC20_S2EF_train_2M
    <br/>
    <b>
     Authors:
    </b>
    Lowik Chanussot, Abhishek Das, Siddharth Goyal, Thibaut Lavril, Muhammed Shuaibi, Morgane Riviere, Kevin Tran, Javier Heras-Domingo, Caleb Ho, Weihua Hu, Aini Palizhati, Anuroop Sriram, Brandon Wood, Junwoong Yoon, Devi Parikh, C. Lawrence Zitnick, Zachary Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Bi, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,000,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 56
    <br/>
    <b>
     Number of Atoms
    </b>
    : 146,496,199
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/oc20.html" target="_blank">
     https://fair-chem.github.io/core/datasets/oc20.html
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.0c04525" target="_blank">
     https://doi.org/10.1021/acscatal.0c04525
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_jyuwhl30jklq_0" target="_blank">
     OC20_S2EF_train_all
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    OC20_S2EF_train_all is the ~63 million structure full training set of the OC20 Structure to Energy and Forces (S2EF) dataset. Features include energy, atomic forces and data from the OC20 mappings file, including adsorbate id, materials project bulk id and miller index.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC20_S2EF_train_all__Chanussot-Das-Goyal-Lavril-Shuaibi-Riviere-Tran-Heras-Domingo-Ho-Hu-Palizhati-Sriram-Wood-Yoon-Parikh-Zitnick-Ulissi__DS_jyuwhl30jklq_0
    <br/>
    <b>
     Name:
    </b>
    OC20_S2EF_train_all
    <br/>
    <b>
     Authors:
    </b>
    Lowik Chanussot, Abhishek Das, Siddharth Goyal, Thibaut Lavril, Muhammed Shuaibi, Morgane Riviere, Kevin Tran, Javier Heras-Domingo, Caleb Ho, Weihua Hu, Aini Palizhati, Anuroop Sriram, Brandon Wood, Junwoong Yoon, Devi Parikh, C. Lawrence Zitnick, Zachary Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Bi, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 133,934,018
    <br/>
    <b>
     Number of Elements
    </b>
    : 56
    <br/>
    <b>
     Number of Atoms
    </b>
    : 9,810,895,377
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/oc20.html" target="_blank">
     https://fair-chem.github.io/core/datasets/oc20.html
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.0c04525" target="_blank">
     https://doi.org/10.1021/acscatal.0c04525
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_wv9zv6egp9vk_0" target="_blank">
     OC20_S2EF_val_id
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    OC20_S2EF_val_id is the ~1-million structure in-domain validation set of the OC20 Structure to Energy and Forces (S2EF) dataset. Features include energy, atomic forces and data from the OC20 mappings file, including adsorbate id, materials project bulk id and miller index.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC20_S2EF_val_id__Chanussot-Das-Goyal-Lavril-Shuaibi-Riviere-Tran-Heras-Domingo-Ho-Hu-Palizhati-Sriram-Wood-Yoon-Parikh-Zitnick-Ulissi__DS_wv9zv6egp9vk_0
    <br/>
    <b>
     Name:
    </b>
    OC20_S2EF_val_id
    <br/>
    <b>
     Authors:
    </b>
    Lowik Chanussot, Abhishek Das, Siddharth Goyal, Thibaut Lavril, Muhammed Shuaibi, Morgane Riviere, Kevin Tran, Javier Heras-Domingo, Caleb Ho, Weihua Hu, Aini Palizhati, Anuroop Sriram, Brandon Wood, Junwoong Yoon, Devi Parikh, C. Lawrence Zitnick, Zachary Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Bi, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 999,866
    <br/>
    <b>
     Number of Elements
    </b>
    : 56
    <br/>
    <b>
     Number of Atoms
    </b>
    : 73,147,343
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/oc20.html" target="_blank">
     https://fair-chem.github.io/core/datasets/oc20.html
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.0c04525" target="_blank">
     https://doi.org/10.1021/acscatal.0c04525
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_cgdsc3gxoamu_0" target="_blank">
     OC20_S2EF_val_ood_ads
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    OC20_S2EF_val_ood_ads is the out-of-domain validation set of the OC20 Structure to Energy and Forces (S2EF) dataset featuring unseen adsorbate. Features include energy, atomic forces and data from the OC20 mappings file, including adsorbate id, materials project bulk id and miller index.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC20_S2EF_val_ood_ads__Chanussot-Das-Goyal-Lavril-Shuaibi-Riviere-Tran-Heras-Domingo-Ho-Hu-Palizhati-Sriram-Wood-Yoon-Parikh-Zitnick-Ulissi__DS_cgdsc3gxoamu_0
    <br/>
    <b>
     Name:
    </b>
    OC20_S2EF_val_ood_ads
    <br/>
    <b>
     Authors:
    </b>
    Lowik Chanussot, Abhishek Das, Siddharth Goyal, Thibaut Lavril, Muhammed Shuaibi, Morgane Riviere, Kevin Tran, Javier Heras-Domingo, Caleb Ho, Weihua Hu, Aini Palizhati, Anuroop Sriram, Brandon Wood, Junwoong Yoon, Devi Parikh, C. Lawrence Zitnick, Zachary Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, B, Bi, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 999,838
    <br/>
    <b>
     Number of Elements
    </b>
    : 56
    <br/>
    <b>
     Number of Atoms
    </b>
    : 72,858,155
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/oc20.html" target="_blank">
     https://fair-chem.github.io/core/datasets/oc20.html
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.0c04525" target="_blank">
     https://doi.org/10.1021/acscatal.0c04525
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_889euoe7akyy_0" target="_blank">
     OC20_S2EF_val_ood_both
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    OC20_S2EF_val_ood_both is the out-of-domain validation set of the OC20 Structure to Energy and Forces (S2EF) dataset featuring both unseen catalyst composition and unseen adsorbate. Features include energy, atomic forces and data from the OC20 mappings file, including adsorbate id, materials project bulk id and miller index.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC20_S2EF_val_ood_both__Chanussot-Das-Goyal-Lavril-Shuaibi-Riviere-Tran-Heras-Domingo-Ho-Hu-Palizhati-Sriram-Wood-Yoon-Parikh-Zitnick-Ulissi__DS_889euoe7akyy_0
    <br/>
    <b>
     Name:
    </b>
    OC20_S2EF_val_ood_both
    <br/>
    <b>
     Authors:
    </b>
    Lowik Chanussot, Abhishek Das, Siddharth Goyal, Thibaut Lavril, Muhammed Shuaibi, Morgane Riviere, Kevin Tran, Javier Heras-Domingo, Caleb Ho, Weihua Hu, Aini Palizhati, Anuroop Sriram, Brandon Wood, Junwoong Yoon, Devi Parikh, C. Lawrence Zitnick, Zachary Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, Bi, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 999,944
    <br/>
    <b>
     Number of Elements
    </b>
    : 55
    <br/>
    <b>
     Number of Atoms
    </b>
    : 84,604,635
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/oc20.html" target="_blank">
     https://fair-chem.github.io/core/datasets/oc20.html
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.0c04525" target="_blank">
     https://doi.org/10.1021/acscatal.0c04525
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_wmgdq06mzdys_0" target="_blank">
     OC20_S2EF_val_ood_cat
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    OC20_S2EF_val_ood_cat is the out-of-domain validation set of the OC20 Structure to Energy and Forces (S2EF) dataset featuring unseen catalyst composition. Features include energy, atomic forces and data from the OC20 mappings file, including adsorbate id, materials project bulk id and miller index.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC20_S2EF_val_ood_cat__Chanussot-Das-Goyal-Lavril-Shuaibi-Riviere-Tran-Heras-Domingo-Ho-Hu-Palizhati-Sriram-Wood-Yoon-Parikh-Zitnick-Ulissi__DS_wmgdq06mzdys_0
    <br/>
    <b>
     Name:
    </b>
    OC20_S2EF_val_ood_cat
    <br/>
    <b>
     Authors:
    </b>
    Lowik Chanussot, Abhishek Das, Siddharth Goyal, Thibaut Lavril, Muhammed Shuaibi, Morgane Riviere, Kevin Tran, Javier Heras-Domingo, Caleb Ho, Weihua Hu, Aini Palizhati, Anuroop Sriram, Brandon Wood, Junwoong Yoon, Devi Parikh, C. Lawrence Zitnick, Zachary Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, Bi, C, Ca, Cd, Cl, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Mn, Mo, N, Na, Nb, Ni, O, Os, P, Pb, Pd, Pt, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sn, Sr, Ta, Tc, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 999,809
    <br/>
    <b>
     Number of Elements
    </b>
    : 55
    <br/>
    <b>
     Number of Atoms
    </b>
    : 74,059,718
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/oc20.html" target="_blank">
     https://fair-chem.github.io/core/datasets/oc20.html
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.0c04525" target="_blank">
     https://doi.org/10.1021/acscatal.0c04525
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_3h39eqiv9urv_0" target="_blank">
     OC22-IS2RE-Train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training configurations for the initial structure to relaxed total energy (IS2RE) task of OC22. Open Catalyst 2022 (OC22) is a database of training trajectories for predicting catalytic reactions on oxide surfaces meant to complement OC20, which did not contain oxide surfaces.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC22-IS2RE-Train__Tran-Lan-Shuaibi-Wood-Goyal-Das-Heras-Domingo-Kolluru-Rizvi-Shoghi-Sriram-Therrien-Abed-Voznyy-Sargent-Ulissi-Zitnick__DS_3h39eqiv9urv_0
    <br/>
    <b>
     Name:
    </b>
    OC22-IS2RE-Train
    <br/>
    <b>
     Authors:
    </b>
    Richard Tran, Janice Lan, Muhammed Shuaibi, Brandon M. Wood, Siddharth Goyal, Abhishek Das, Javier Heras-Domingo, Adeesh Kolluru, Ammar Rizvi, Nima Shoghi, Anuroop Sriram, Felix Therrien, Jehad Abed, Oleksandr Voznyy, Edward H. Sargent, Zachary Ulissi, C. Lawrence Zitnick
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, Ba, Be, Bi, C, Ca, Cd, Ce, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Ni, O, Os, Pb, Pd, Pt, Rb, Re, Rh, Ru, Sb, Sc, Se, Si, Sn, Sr, Ta, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 7,878,688
    <br/>
    <b>
     Number of Elements
    </b>
    : 57
    <br/>
    <b>
     Number of Atoms
    </b>
    : 634,845,874
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/Open-Catalyst-Project/ocp/blob/main/DATASET.md#open-catalyst-2022-oc22" target="_blank">
     https://github.com/Open-Catalyst-Project/ocp/blob/main/DATASET.md#open-catalyst-2022-oc22
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.2c05426" target="_blank">
     https://doi.org/10.1021/acscatal.2c05426
    </a>
    <br/>
    <a href="https://opencatalystproject.org/" target="_blank">
     https://opencatalystproject.org/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_4eb78xs9suoo_0" target="_blank">
     OC22-IS2RE-Validation-in-domain
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    In-domain validation configurations for the initial structure to relaxed total energy (IS2RE) task of OC22. Open Catalyst 2022 (OC22) is a database of training trajectories for predicting catalytic reactions on oxide surfaces meant to complement OC20, which did not contain oxide surfaces.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC22-IS2RE-Validation-in-domain__Tran-Lan-Shuaibi-Wood-Goyal-Das-Heras-Domingo-Kolluru-Rizvi-Shoghi-Sriram-Therrien-Abed-Voznyy-Sargent-Ulissi-Zitnick__DS_4eb78xs9suoo_0
    <br/>
    <b>
     Name:
    </b>
    OC22-IS2RE-Validation-in-domain
    <br/>
    <b>
     Authors:
    </b>
    Richard Tran, Janice Lan, Muhammed Shuaibi, Brandon M. Wood, Siddharth Goyal, Abhishek Das, Javier Heras-Domingo, Adeesh Kolluru, Ammar Rizvi, Nima Shoghi, Anuroop Sriram, Felix Therrien, Jehad Abed, Oleksandr Voznyy, Edward H. Sargent, Zachary Ulissi, C. Lawrence Zitnick
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, Ba, Be, Bi, C, Ca, Cd, Ce, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Ni, O, Os, Pb, Pd, Pt, Rb, Re, Rh, Ru, Sb, Sc, Se, Si, Sn, Sr, Ta, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 442,843
    <br/>
    <b>
     Number of Elements
    </b>
    : 57
    <br/>
    <b>
     Number of Atoms
    </b>
    : 35,275,211
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/Open-Catalyst-Project/ocp/blob/main/DATASET.md#open-catalyst-2022-oc22" target="_blank">
     https://github.com/Open-Catalyst-Project/ocp/blob/main/DATASET.md#open-catalyst-2022-oc22
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.2c05426" target="_blank">
     https://doi.org/10.1021/acscatal.2c05426
    </a>
    <br/>
    <a href="https://opencatalystproject.org/" target="_blank">
     https://opencatalystproject.org/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_ikbqbyd3dw25_0" target="_blank">
     OC22-IS2RE-Validation-out-of-domain
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Out-of-domain validation configurations for the initial structure to relaxed total energy (IS2RE) task of OC22. Open Catalyst 2022 (OC22) is a database of training trajectories for predicting catalytic reactions on oxide surfaces meant to complement OC20, which did not contain oxide surfaces.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC22-IS2RE-Validation-out-of-domain__Tran-Lan-Shuaibi-Wood-Goyal-Das-Heras-Domingo-Kolluru-Rizvi-Shoghi-Sriram-Therrien-Abed-Voznyy-Sargent-Ulissi-Zitnick__DS_ikbqbyd3dw25_0
    <br/>
    <b>
     Name:
    </b>
    OC22-IS2RE-Validation-out-of-domain
    <br/>
    <b>
     Authors:
    </b>
    Richard Tran, Janice Lan, Muhammed Shuaibi, Brandon M. Wood, Siddharth Goyal, Abhishek Das, Javier Heras-Domingo, Adeesh Kolluru, Ammar Rizvi, Nima Shoghi, Anuroop Sriram, Felix Therrien, Jehad Abed, Oleksandr Voznyy, Edward H. Sargent, Zachary Ulissi, C. Lawrence Zitnick
    <br/>
    <b>
     Elements
    </b>
    : Au, Ba, Be, Bi, C, Ca, Cd, Ce, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Ni, O, Os, Pb, Pd, Pt, Rb, Re, Rh, Ru, Sb, Sc, Se, Si, Sn, Sr, Ta, Ti, Tl, V, W, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 521,827
    <br/>
    <b>
     Number of Elements
    </b>
    : 52
    <br/>
    <b>
     Number of Atoms
    </b>
    : 42,219,955
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/Open-Catalyst-Project/ocp/blob/main/DATASET.md#open-catalyst-2022-oc22" target="_blank">
     https://github.com/Open-Catalyst-Project/ocp/blob/main/DATASET.md#open-catalyst-2022-oc22
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.2c05426" target="_blank">
     https://doi.org/10.1021/acscatal.2c05426
    </a>
    <br/>
    <a href="https://opencatalystproject.org/" target="_blank">
     https://opencatalystproject.org/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_jgaid7espcoc_0" target="_blank">
     OC22-S2EF-Train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training configurations for the structure to total energy and forces task (S2EF) of OC22. Open Catalyst 2022 (OC22) is a database of training trajectories for predicting catalytic reactions on oxide surfaces meant to complement OC20, which did not contain oxide surfaces.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC22-S2EF-Train__Tran-Lan-Shuaibi-Wood-Goyal-Das-Heras-Domingo-Kolluru-Rizvi-Shoghi-Sriram-Therrien-Abed-Voznyy-Sargent-Ulissi-Zitnick__DS_jgaid7espcoc_0
    <br/>
    <b>
     Name:
    </b>
    OC22-S2EF-Train
    <br/>
    <b>
     Authors:
    </b>
    Richard Tran, Janice Lan, Muhammed Shuaibi, Brandon M. Wood, Siddharth Goyal, Abhishek Das, Javier Heras-Domingo, Adeesh Kolluru, Ammar Rizvi, Nima Shoghi, Anuroop Sriram, Felix Therrien, Jehad Abed, Oleksandr Voznyy, Edward H. Sargent, Zachary Ulissi, C. Lawrence Zitnick
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, Ba, Be, Bi, C, Ca, Cd, Ce, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Ni, O, Os, Pb, Pd, Pt, Rb, Re, Rh, Ru, Sb, Sc, Se, Si, Sn, Sr, Ta, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 8,389,365
    <br/>
    <b>
     Number of Elements
    </b>
    : 57
    <br/>
    <b>
     Number of Atoms
    </b>
    : 669,615,870
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/Open-Catalyst-Project/ocp/blob/main/DATASET.md#open-catalyst-2022-oc22" target="_blank">
     https://github.com/Open-Catalyst-Project/ocp/blob/main/DATASET.md#open-catalyst-2022-oc22
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.2c05426" target="_blank">
     https://doi.org/10.1021/acscatal.2c05426
    </a>
    <br/>
    <a href="https://opencatalystproject.org/" target="_blank">
     https://opencatalystproject.org/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_f25kywgtdhks_0" target="_blank">
     OC22-S2EF-Validation-in-domain
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    In-domain validation configurations for the structure to total energy and forces (S2EF) task of OC22. Open Catalyst 2022 (OC22) is a database of training trajectories for predicting catalytic reactions on oxide surfaces meant to complement OC20, which did not contain oxide surfaces.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC22-S2EF-Validation-in-domain__Tran-Lan-Shuaibi-Wood-Goyal-Das-Heras-Domingo-Kolluru-Rizvi-Shoghi-Sriram-Therrien-Abed-Voznyy-Sargent-Ulissi-Zitnick__DS_f25kywgtdhks_0
    <br/>
    <b>
     Name:
    </b>
    OC22-S2EF-Validation-in-domain
    <br/>
    <b>
     Authors:
    </b>
    Richard Tran, Janice Lan, Muhammed Shuaibi, Brandon M. Wood, Siddharth Goyal, Abhishek Das, Javier Heras-Domingo, Adeesh Kolluru, Ammar Rizvi, Nima Shoghi, Anuroop Sriram, Felix Therrien, Jehad Abed, Oleksandr Voznyy, Edward H. Sargent, Zachary Ulissi, C. Lawrence Zitnick
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, As, Au, Ba, Be, Bi, C, Ca, Cd, Ce, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Ni, O, Os, Pb, Pd, Pt, Rb, Re, Rh, Ru, Sb, Sc, Se, Si, Sn, Sr, Ta, Te, Ti, Tl, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 407,195
    <br/>
    <b>
     Number of Elements
    </b>
    : 57
    <br/>
    <b>
     Number of Atoms
    </b>
    : 31,919,751
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/Open-Catalyst-Project/ocp/blob/main/DATASET.md#open-catalyst-2022-oc22" target="_blank">
     https://github.com/Open-Catalyst-Project/ocp/blob/main/DATASET.md#open-catalyst-2022-oc22
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.2c05426" target="_blank">
     https://doi.org/10.1021/acscatal.2c05426
    </a>
    <br/>
    <a href="https://opencatalystproject.org/" target="_blank">
     https://opencatalystproject.org/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_y8u933mhadjf_0" target="_blank">
     OC22-S2EF-Validation-out-of-domain
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Out-of-domain validation configurations for the structure to total energy and forces (S2EF) task of OC22. Open Catalyst 2022 (OC22) is a database of training trajectories for predicting catalytic reactions on oxide surfaces meant to complement OC20, which did not contain oxide surfaces.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OC22-S2EF-Validation-out-of-domain__Tran-Lan-Shuaibi-Wood-Goyal-Das-Heras-Domingo-Kolluru-Rizvi-Shoghi-Sriram-Therrien-Abed-Voznyy-Sargent-Ulissi-Zitnick__DS_y8u933mhadjf_0
    <br/>
    <b>
     Name:
    </b>
    OC22-S2EF-Validation-out-of-domain
    <br/>
    <b>
     Authors:
    </b>
    Richard Tran, Janice Lan, Muhammed Shuaibi, Brandon M. Wood, Siddharth Goyal, Abhishek Das, Javier Heras-Domingo, Adeesh Kolluru, Ammar Rizvi, Nima Shoghi, Anuroop Sriram, Felix Therrien, Jehad Abed, Oleksandr Voznyy, Edward H. Sargent, Zachary Ulissi, C. Lawrence Zitnick
    <br/>
    <b>
     Elements
    </b>
    : Au, Ba, Be, Bi, C, Ca, Cd, Ce, Co, Cr, Cs, Cu, Fe, Ga, Ge, H, Hf, Hg, In, Ir, K, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Ni, O, Os, Pb, Pd, Pt, Rb, Re, Rh, Ru, Sb, Sc, Se, Si, Sn, Sr, Ta, Ti, Tl, V, W, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 459,594
    <br/>
    <b>
     Number of Elements
    </b>
    : 52
    <br/>
    <b>
     Number of Atoms
    </b>
    : 36,999,141
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/Open-Catalyst-Project/ocp/blob/main/DATASET.md#open-catalyst-2022-oc22" target="_blank">
     https://github.com/Open-Catalyst-Project/ocp/blob/main/DATASET.md#open-catalyst-2022-oc22
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acscatal.2c05426" target="_blank">
     https://doi.org/10.1021/acscatal.2c05426
    </a>
    <br/>
    <a href="https://opencatalystproject.org/" target="_blank">
     https://opencatalystproject.org/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_jqrkc9e7cgmh_0" target="_blank">
     OMat24_train_aimd_from_PBE_1000_npt
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The aimd-from-PBE-1000-npt training split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in subdatasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_train_aimd_from_PBE_1000_npt__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_jqrkc9e7cgmh_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_train_aimd_from_PBE_1000_npt
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ne, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 21,269,486
    <br/>
    <b>
     Number of Elements
    </b>
    : 89
    <br/>
    <b>
     Number of Atoms
    </b>
    : 179,930,890
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_w0ra2olwcy29_0" target="_blank">
     OMat24_train_aimd_from_PBE_1000_nvt
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The aimd-from-PBE-1000-nvt training split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in sub-datasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_train_aimd_from_PBE_1000_nvt__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_w0ra2olwcy29_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_train_aimd_from_PBE_1000_nvt
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 20,256,650
    <br/>
    <b>
     Number of Elements
    </b>
    : 86
    <br/>
    <b>
     Number of Atoms
    </b>
    : 169,879,539
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_6xvvh8yl7rfd_0" target="_blank">
     OMat24_train_aimd_from_PBE_3000_npt
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The aimd-from-PBE-3000-npt training split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in sub-datasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_train_aimd_from_PBE_3000_npt__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_6xvvh8yl7rfd_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_train_aimd_from_PBE_3000_npt
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ne, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 6,076,290
    <br/>
    <b>
     Number of Elements
    </b>
    : 89
    <br/>
    <b>
     Number of Atoms
    </b>
    : 411,540,573
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_fi9ozets1s5f_0" target="_blank">
     OMat24_train_aimd_from_PBE_3000_nvt
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The aimd-from-PBE-3000-nvt training split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in sub-datasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_train_aimd_from_PBE_3000_nvt__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_fi9ozets1s5f_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_train_aimd_from_PBE_3000_nvt
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 7,839,846
    <br/>
    <b>
     Number of Elements
    </b>
    : 86
    <br/>
    <b>
     Number of Atoms
    </b>
    : 530,963,613
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_opcbnu5xs3pe_0" target="_blank">
     OMat24_train_rattled_1000
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The rattled-1000 training split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in sub-datasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_train_rattled_1000__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_opcbnu5xs3pe_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_train_rattled_1000
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ne, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 11,388,475
    <br/>
    <b>
     Number of Elements
    </b>
    : 89
    <br/>
    <b>
     Number of Atoms
    </b>
    : 161,511,768
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_6nu6i240an78_0" target="_blank">
     OMat24_train_rattled_1000_subsampled
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The rattled-1000-subsampled training split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in subdatasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_train_rattled_1000_subsampled__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_6nu6i240an78_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_train_rattled_1000_subsampled
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ne, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,879,731
    <br/>
    <b>
     Number of Elements
    </b>
    : 89
    <br/>
    <b>
     Number of Atoms
    </b>
    : 55,648,760
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_jcodhqs0atqm_0" target="_blank">
     OMat24_train_rattled_300
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The rattled-300 training split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in subdatasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_train_rattled_300__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_jcodhqs0atqm_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_train_rattled_300
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 6,319,086
    <br/>
    <b>
     Number of Elements
    </b>
    : 88
    <br/>
    <b>
     Number of Atoms
    </b>
    : 89,791,956
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_t8iygwyagbaj_0" target="_blank">
     OMat24_train_rattled_300_subsampled
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The rattled-300-subsampled training split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in subdatasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_train_rattled_300_subsampled__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_t8iygwyagbaj_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_train_rattled_300_subsampled
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ne, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,463,993
    <br/>
    <b>
     Number of Elements
    </b>
    : 88
    <br/>
    <b>
     Number of Atoms
    </b>
    : 49,674,369
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_weq0x6qxqbau_0" target="_blank">
     OMat24_train_rattled_500
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The rattled-500 training split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in subdatasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_train_rattled_500__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_weq0x6qxqbau_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_train_rattled_500
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 6,922,152
    <br/>
    <b>
     Number of Elements
    </b>
    : 88
    <br/>
    <b>
     Number of Atoms
    </b>
    : 98,860,290
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_z0545ougitpp_0" target="_blank">
     OMat24_train_rattled_500_subsampled
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The rattled-500-subsampled training split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in subdatasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_train_rattled_500_subsampled__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_z0545ougitpp_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_train_rattled_500_subsampled
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ne, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,975,399
    <br/>
    <b>
     Number of Elements
    </b>
    : 89
    <br/>
    <b>
     Number of Atoms
    </b>
    : 56,846,329
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_96l9f8z75uof_0" target="_blank">
     OMat24_train_rattled_relax
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The rattled-relax training split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in subdatasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_train_rattled_relax__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_96l9f8z75uof_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_train_rattled_relax
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 9,433,298
    <br/>
    <b>
     Number of Elements
    </b>
    : 87
    <br/>
    <b>
     Number of Atoms
    </b>
    : 78,952,123
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_qrkr2xiw1wtp_0" target="_blank">
     OMat24_validation_aimd-from-PBE-1000-npt
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The val_aimd-from-PBE-1000-npt validation split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in subdatasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_validation_aimd-from-PBE-1000-npt__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_qrkr2xiw1wtp_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_validation_aimd-from-PBE-1000-npt
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 202,758
    <br/>
    <b>
     Number of Elements
    </b>
    : 85
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,710,254
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_sng40qq19dak_0" target="_blank">
     OMat24_validation_aimd-from-PBE-1000-nvt
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The val_aimd-from-PBE-1000-nvt validation split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in subdatasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_validation_aimd-from-PBE-1000-nvt__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_sng40qq19dak_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_validation_aimd-from-PBE-1000-nvt
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, Hf, Hg, Ho, I, In, Ir, K, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 195,575
    <br/>
    <b>
     Number of Elements
    </b>
    : 85
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,643,554
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_krd83tt1g6wd_0" target="_blank">
     OMat24_validation_aimd-from-PBE-3000-npt
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The val_aimd-from-PBE-3000-npt validation split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in subdatasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_validation_aimd-from-PBE-3000-npt__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_krd83tt1g6wd_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_validation_aimd-from-PBE-3000-npt
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 59,516
    <br/>
    <b>
     Number of Elements
    </b>
    : 85
    <br/>
    <b>
     Number of Atoms
    </b>
    : 4,036,396
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_4vdrw3cfi4s7_0" target="_blank">
     OMat24_validation_aimd-from-PBE-3000-nvt
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The val_aimd-from-PBE-3000-nvt validation split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in subdatasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_validation_aimd-from-PBE-3000-nvt__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_4vdrw3cfi4s7_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_validation_aimd-from-PBE-3000-nvt
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, Hf, Hg, Ho, I, In, Ir, K, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 76,478
    <br/>
    <b>
     Number of Elements
    </b>
    : 84
    <br/>
    <b>
     Number of Atoms
    </b>
    : 5,186,115
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_8k237jm1rd3s_0" target="_blank">
     OMat24_validation_rattled_1000
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The rattled-1000 validation split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in subdatasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_validation_rattled_1000__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_8k237jm1rd3s_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_validation_rattled_1000
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, Ar, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 117,004
    <br/>
    <b>
     Number of Elements
    </b>
    : 86
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,657,765
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_c4s38mdirjf7_0" target="_blank">
     OMat24_validation_rattled_1000_subsampled
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The rattled-1000-subsampled validation split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in subdatasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_validation_rattled_1000_subsampled__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_c4s38mdirjf7_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_validation_rattled_1000_subsampled
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 38,271
    <br/>
    <b>
     Number of Elements
    </b>
    : 87
    <br/>
    <b>
     Number of Atoms
    </b>
    : 549,832
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_mm4npn96qxo1_0" target="_blank">
     OMat24_validation_rattled_300
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The rattled-300 validation split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in subdatasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_validation_rattled_300__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_mm4npn96qxo1_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_validation_rattled_300
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, Hf, Hg, Ho, I, In, Ir, K, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 62,451
    <br/>
    <b>
     Number of Elements
    </b>
    : 84
    <br/>
    <b>
     Number of Atoms
    </b>
    : 883,431
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_vxd0mrt5it19_0" target="_blank">
     OMat24_validation_rattled_300_subsampled
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The rattled-300-subsampled validation split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in subdatasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_validation_rattled_300_subsampled__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_vxd0mrt5it19_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_validation_rattled_300_subsampled
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 34,244
    <br/>
    <b>
     Number of Elements
    </b>
    : 85
    <br/>
    <b>
     Number of Atoms
    </b>
    : 490,880
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_zduxfk2oohzc_0" target="_blank">
     OMat24_validation_rattled_500
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The rattled-500 validation split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in subdatasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_validation_rattled_500__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_zduxfk2oohzc_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_validation_rattled_500
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, Hf, Hg, Ho, I, In, Ir, K, Kr, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 68,830
    <br/>
    <b>
     Number of Elements
    </b>
    : 85
    <br/>
    <b>
     Number of Atoms
    </b>
    : 985,338
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_h7gnyidyqcxe_0" target="_blank">
     OMat24_validation_rattled_500_subsampled
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The rattled-500-subsampled validation split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in subdatasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_validation_rattled_500_subsampled__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_h7gnyidyqcxe_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_validation_rattled_500_subsampled
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, Hf, Hg, Ho, I, In, Ir, K, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 39,464
    <br/>
    <b>
     Number of Elements
    </b>
    : 85
    <br/>
    <b>
     Number of Atoms
    </b>
    : 564,068
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_abagltajle7q_0" target="_blank">
     OMat24_validation_rattled_relax
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The rattled-relax validation split of OMat24 (Open Materials 2024). OMat24 is a large-scale open dataset of density functional theory (DFT) calculations. The dataset is available in subdatasets and subsampled sub-datasets based on the structure generation strategy used. There are two main splits in OMat24: train and validation, each divided into the aforementioned subsampling and sub-datasets.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OMat24_validation_rattled_relax__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_abagltajle7q_0
    <br/>
    <b>
     Name:
    </b>
    OMat24_validation_rattled_relax
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, Hf, Hg, Ho, I, In, Ir, K, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 91,043
    <br/>
    <b>
     Number of Elements
    </b>
    : 84
    <br/>
    <b>
     Number of Atoms
    </b>
    : 764,266
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_5obi9nxcgmof_0" target="_blank">
     OrbNet_Denali
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    All DFT single-point calculations for the OrbNet Denali training set were carried out in Entos Qcore version 0.8.17 at the ωB97X-D3/def2-TZVP level of theory using in-core density fitting with the neese=4 DFT integration grid.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    OrbNet_Denali__Christensen-Sirumalla-Qiao-OConnor-Smith-Ding-Bygrave-Anandkumar-Welborn-Manby-III__DS_5obi9nxcgmof_0
    <br/>
    <b>
     Name:
    </b>
    OrbNet_Denali
    <br/>
    <b>
     Authors:
    </b>
    Anders S. Christensen, Sai Krishna Sirumalla, Zhuoran Qiao, Michael B. OConnor, Daniel G. A. Smith, Feizhi Ding, Peter J. Bygrave, Animashree Anandkumar, Matthew Welborn, Frederick R. Manby, Thomas F. Miller III
    <br/>
    <b>
     Elements
    </b>
    : B, Br, C, Ca, Cl, F, H, I, K, Li, Mg, N, Na, O, P, S, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,338,215
    <br/>
    <b>
     Number of Elements
    </b>
    : 17
    <br/>
    <b>
     Number of Atoms
    </b>
    : 104,958,650
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.14883867.v2" target="_blank">
     https://doi.org/10.6084/m9.figshare.14883867.v2
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0061990" target="_blank">
     https://doi.org/10.1063/5.0061990
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_cgjdk1e2txjy_0" target="_blank">
     PWMLFF_feature_comparison_NPJ2023
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Partial dataset for "Accuracy evaluation of different machine learning force field features". The included data is limited to that hosted directly on the repository at the related GitHub link. From publication abstract: Predicting energies and forces using machine learning force field (MLFF) depends on accurate descriptions (features) of chemical environment. Despite the numerous features proposed, there is a lack of controlled comparison among them for their universality and accuracy. In this work, we compared several commonly used feature types for their ability to describe physical systems. These different feature types include cosine feature, Gaussian feature, moment tensor potential (MTP) feature, spectral neighbor analysis potential feature, simplified smooth deep potential with Chebyshev polynomials feature and Gaussian polynomials feature, and atomic cluster expansion feature. We evaluated the training root mean square error (RMSE) for the atomic group energy, total energy, and force using linear regression model regarding to the density functional theory results. We applied these MLFF models to an amorphous sulfur system and carbon systems, and the fitting results show that MTP feature can yield the smallest RMSE results compared with other feature types for either sulfur system or carbon system in the disordered atomic configurations. Moreover, as an extending test of other systems, the MTP feature combined with linear regression model can also reproduce similar quantities along the ab initio molecular dynamics trajectory as represented by Cu systems. Our results are helpful in selecting the proper features for the MLFF development.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    PWMLFF_feature_comparison_NPJ2023__Han-Li-Liu-Li-Wang__DS_cgjdk1e2txjy_0
    <br/>
    <b>
     Name:
    </b>
    PWMLFF_feature_comparison_NPJ2023
    <br/>
    <b>
     Authors:
    </b>
    Ting Han, Jie Li, Liping Liu, Fengyu Li, Lin-Wang Wang
    <br/>
    <b>
     Elements
    </b>
    : C, H, Mg, Ni, O, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 17,255
    <br/>
    <b>
     Number of Elements
    </b>
    : 6
    <br/>
    <b>
     Number of Atoms
    </b>
    : 918,240
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/LonxunQuantum/PWMLFF_library/tree/main" target="_blank">
     https://github.com/LonxunQuantum/PWMLFF_library/tree/main
    </a>
    <br/>
    <a href="https://www.doi.org/10.1088/1367-2630/acf2bb" target="_blank">
     https://www.doi.org/10.1088/1367-2630/acf2bb
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_cm3cr5kgqomw_0" target="_blank">
     Paramagnetic_lanthanide_compounds
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset is composed of fully-deuterated Gd(III) analogue d-[GdL] in a variety of solvent materials, including MeOH, D2O and d6-DMSO.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Paramagnetic_lanthanide_compounds__Alnami-Kragskow-Staab-Skelton-Chilton__DS_cm3cr5kgqomw_0
    <br/>
    <b>
     Name:
    </b>
    Paramagnetic_lanthanide_compounds
    <br/>
    <b>
     Authors:
    </b>
    Barak Alnami, Jon G. C. Kragskow, Jakob K. Staab, Jonathan M. Skelton, Nicholas F. Chilton
    <br/>
    <b>
     Elements
    </b>
    : C, Gd, H, N, O, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 41,748
    <br/>
    <b>
     Number of Elements
    </b>
    : 6
    <br/>
    <b>
     Number of Atoms
    </b>
    : 28,419,876
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1021/jacs.3c01342" target="_blank">
     https://doi.org/10.1021/jacs.3c01342
    </a>
    <br/>
    <a href="https://doi.org/10.48420/22015322.v1" target="_blank">
     https://doi.org/10.48420/22015322.v1
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_l3xhsrlgu8tq_0" target="_blank">
     PtNi_alloy_NPJ2022
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    DFT dataset consisting of 6828 resampled Pt-Ni alloys used for training an NNP. The energy and forces of each structure in the resampled database are calculated using DFT. All reference DFT calculations for the training set of 6828 Pt-Ni alloy structures have been performed using the Vienna Ab initio Simulation Package (VASP) with the spin-polarized revised Perdew-Burke-Ernzerhof (rPBE) exchange-correlation functional.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    PtNi_alloy_NPJ2022__Han-Barcaro-Fortunelli-Lysgaard-Vegge-Hansen__DS_l3xhsrlgu8tq_0
    <br/>
    <b>
     Name:
    </b>
    PtNi_alloy_NPJ2022
    <br/>
    <b>
     Authors:
    </b>
    Shuang Han, Giovanni Barcaro, Alessandro Fortunelli, Steen Lysgaard, Tejs Vegge, Heine Anton Hansen
    <br/>
    <b>
     Elements
    </b>
    : Ni, Pt
    <br/>
    <b>
     Number of Configurations
    </b>
    : 6,828
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,074,161
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://zenodo.org/record/5645281#.Y2CPkeTMJEa" target="_blank">
     https://zenodo.org/record/5645281#.Y2CPkeTMJEa
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-022-00807-6" target="_blank">
     https://doi.org/10.1038/s41524-022-00807-6
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_2qq05tdbyovn_0" target="_blank">
     QM-22
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Includes CHON molecules of 4-15 atoms, developed in counterpoint to the MD17 dataset, run at higher total energies (above 500 K) and with a broader configuration space.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    QM-22__Bowman-Qu-Conte-Nandi-Houston-Yu__DS_2qq05tdbyovn_0
    <br/>
    <b>
     Name:
    </b>
    QM-22
    <br/>
    <b>
     Authors:
    </b>
    Joel M. Bowman, Chen Qu, Riccardo Conte, Apurba Nandi, Paul L. Houston, Qi Yu
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 6,762
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 101,430
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/jmbowma/QM-22" target="_blank">
     https://github.com/jmbowma/QM-22
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0089200" target="_blank">
     https://doi.org/10.1063/5.0089200
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_3a5xucj4yqa8_0" target="_blank">
     QM7b_AlphaML
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Energy, computed with LR-CCSD, hybrid DFT (B3LYP &amp; SCAN0) for 7211 molecules in QM7b and 52 molecules in AlphaML showcase database.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    QM7b_AlphaML__Yang-Lao-Wilkins-Grisafi-Ceriotti-Jr__DS_3a5xucj4yqa8_0
    <br/>
    <b>
     Name:
    </b>
    QM7b_AlphaML
    <br/>
    <b>
     Authors:
    </b>
    Yang Yang, Ka Un Lao, David M. Wilkins, Andrea Grisafi, Michele Ceriotti, Robert A. DiStasio Jr
    <br/>
    <b>
     Elements
    </b>
    : C, Cl, H, N, O, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 29,033
    <br/>
    <b>
     Number of Elements
    </b>
    : 6
    <br/>
    <b>
     Number of Atoms
    </b>
    : 408,865
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:2019.0002/v3" target="_blank">
     https://doi.org/10.24435/materialscloud:2019.0002/v3
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-019-0157-8" target="_blank">
     https://doi.org/10.1038/s41597-019-0157-8
    </a>
    <br/>
    <a href="http://doi.org/10.1073/pnas.1816132116" target="_blank">
     http://doi.org/10.1073/pnas.1816132116
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_d5ug96vla5xy_0" target="_blank">
     QM9x
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset containing DFT calculations of energy and forces for all configurations in the QM9 dataset, recalculated with the ωB97X functional and 6-31G(d) basis set. Recalculating the energy and forces causes a slight shift of the potential energy surface, which results in forces acting on most configurations in the dataset. The data was generated by running Nudged Elastic Band (NEB) calculations with DFT on 10k reactions while saving intermediate calculations. QM9x is used as a benchmarking and comparison dataset for the dataset Transition1x.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    QM9x__Schreiner-Bhowmik-Vegge-Busk-Winther__DS_d5ug96vla5xy_0
    <br/>
    <b>
     Name:
    </b>
    QM9x
    <br/>
    <b>
     Authors:
    </b>
    Mathias Schreiner, Arghya Bhowmik, Tejs Vegge, Jonas Busk, Ole Winther
    <br/>
    <b>
     Elements
    </b>
    : C, F, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 133,885
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,407,753
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.20449701.v2" target="_blank">
     https://doi.org/10.6084/m9.figshare.20449701.v2
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-022-01870-w" target="_blank">
     https://doi.org/10.1038/s41597-022-01870-w
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_02cqe6a0bobu_0" target="_blank">
     QM_hamiltonian_nature_2019
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    ~100,000 configurations of water, ethanol, malondialdehyde and uracil gathered at the PBE/def2-SVP level of theory using ORCA.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    QM_hamiltonian_nature_2019__Schütt-Gastegger-Tkatchenko-Müller-Maurer__DS_02cqe6a0bobu_0
    <br/>
    <b>
     Name:
    </b>
    QM_hamiltonian_nature_2019
    <br/>
    <b>
     Authors:
    </b>
    Kristof T. Schütt, Michael Gastegger, Alexandre Tkatchenko, Klaus-Robert Müller, Reinhard J. Maurer
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 91,977
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 887,799
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://quantum-machine.org/datasets/" target="_blank">
     http://quantum-machine.org/datasets/
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-019-12875-2" target="_blank">
     https://doi.org/10.1038/s41467-019-12875-2
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_s818rozdb6x6_0" target="_blank">
     REANN_CO2_Ni100
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 9,850 configurations of CO2 with a movable Ni(100) surface.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    REANN_CO2_Ni100__Zhang-Xia-Jiang__DS_s818rozdb6x6_0
    <br/>
    <b>
     Name:
    </b>
    REANN_CO2_Ni100
    <br/>
    <b>
     Authors:
    </b>
    Yaolong Zhang, Junfan Xia, Bin Jiang
    <br/>
    <b>
     Elements
    </b>
    : C, Ni, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 9,850
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 384,150
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/zhangylch/REANN" target="_blank">
     https://github.com/zhangylch/REANN
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpclett.9b00085" target="_blank">
     https://doi.org/10.1021/acs.jpclett.9b00085
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_dixgsms1jm98_0" target="_blank">
     SAIT_semiconductors_ACS_2023_HfO_out-of-domain
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Out-of-domain configurations from the SAIT_semiconductors_ACS_2023_HfO dataset. This dataset contains HfO configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    SAIT_semiconductors_ACS_2023_HfO_out-of-domain__Kim-Na-Kim-Cho-Kang-Lee-Choi-Kim-Lee-Kim__DS_dixgsms1jm98_0
    <br/>
    <b>
     Name:
    </b>
    SAIT_semiconductors_ACS_2023_HfO_out-of-domain
    <br/>
    <b>
     Authors:
    </b>
    Geonu Kim, Byunggook Na, Gunhee Kim, Hyuntae Cho, Seung-Jin Kang, Hee Sun Lee, Saerom Choi, Heejae Kim, Seungwon Lee, Yongdeok Kim
    <br/>
    <b>
     Elements
    </b>
    : Hf, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 6,996
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 671,616
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/SAITPublic/MLFF-Framework" target="_blank">
     https://github.com/SAITPublic/MLFF-Framework
    </a>
    <br/>
    <a href="https://openreview.net/forum?id=hr9Bd1A9Un" target="_blank">
     https://openreview.net/forum?id=hr9Bd1A9Un
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_ekrypue10aay_0" target="_blank">
     SAIT_semiconductors_ACS_2023_HfO_raw
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Structures from the SAIT_semiconductors_ACS_2023_HfO dataset, separated into crystal, out-of-domain, and random (generated by randomly distributing 32 Hf and 64 O atoms within the unit cells of the HfO2 crystals) configuration sets. This dataset contains HfO configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    SAIT_semiconductors_ACS_2023_HfO_raw__Kim-Na-Kim-Cho-Kang-Lee-Choi-Kim-Lee-Kim__DS_ekrypue10aay_0
    <br/>
    <b>
     Name:
    </b>
    SAIT_semiconductors_ACS_2023_HfO_raw
    <br/>
    <b>
     Authors:
    </b>
    Geonu Kim, Byunggook Na, Gunhee Kim, Hyuntae Cho, Seung-Jin Kang, Hee Sun Lee, Saerom Choi, Heejae Kim, Seungwon Lee, Yongdeok Kim
    <br/>
    <b>
     Elements
    </b>
    : Hf, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 192,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 18,431,808
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/SAITPublic/MLFF-Framework" target="_blank">
     https://github.com/SAITPublic/MLFF-Framework
    </a>
    <br/>
    <a href="https://openreview.net/forum?id=hr9Bd1A9Un" target="_blank">
     https://openreview.net/forum?id=hr9Bd1A9Un
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_dxzkbbocjb0y_0" target="_blank">
     SAIT_semiconductors_ACS_2023_HfO_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test configurations from the SAIT_semiconductors_ACS_2023_HfO dataset. This dataset contains HfO configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    SAIT_semiconductors_ACS_2023_HfO_test__Kim-Na-Kim-Cho-Kang-Lee-Choi-Kim-Lee-Kim__DS_dxzkbbocjb0y_0
    <br/>
    <b>
     Name:
    </b>
    SAIT_semiconductors_ACS_2023_HfO_test
    <br/>
    <b>
     Authors:
    </b>
    Geonu Kim, Byunggook Na, Gunhee Kim, Hyuntae Cho, Seung-Jin Kang, Hee Sun Lee, Saerom Choi, Heejae Kim, Seungwon Lee, Yongdeok Kim
    <br/>
    <b>
     Elements
    </b>
    : Hf, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,510
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 336,960
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/SAITPublic/MLFF-Framework" target="_blank">
     https://github.com/SAITPublic/MLFF-Framework
    </a>
    <br/>
    <a href="https://openreview.net/forum?id=hr9Bd1A9Un" target="_blank">
     https://openreview.net/forum?id=hr9Bd1A9Un
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_ppk44zeithjj_0" target="_blank">
     SAIT_semiconductors_ACS_2023_HfO_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training configurations from the SAIT_semiconductors_ACS_2023_HfO dataset. This dataset contains HfO configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    SAIT_semiconductors_ACS_2023_HfO_train__Kim-Na-Kim-Cho-Kang-Lee-Choi-Kim-Lee-Kim__DS_ppk44zeithjj_0
    <br/>
    <b>
     Name:
    </b>
    SAIT_semiconductors_ACS_2023_HfO_train
    <br/>
    <b>
     Authors:
    </b>
    Geonu Kim, Byunggook Na, Gunhee Kim, Hyuntae Cho, Seung-Jin Kang, Hee Sun Lee, Saerom Choi, Heejae Kim, Seungwon Lee, Yongdeok Kim
    <br/>
    <b>
     Elements
    </b>
    : Hf, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 27,960
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,684,160
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/SAITPublic/MLFF-Framework" target="_blank">
     https://github.com/SAITPublic/MLFF-Framework
    </a>
    <br/>
    <a href="https://openreview.net/forum?id=hr9Bd1A9Un" target="_blank">
     https://openreview.net/forum?id=hr9Bd1A9Un
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_89w2iq8fw7qi_0" target="_blank">
     SAIT_semiconductors_ACS_2023_HfO_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Validation configurations from the SAIT_semiconductors_ACS_2023_HfO dataset. This dataset contains HfO configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    SAIT_semiconductors_ACS_2023_HfO_validation__Kim-Na-Kim-Cho-Kang-Lee-Choi-Kim-Lee-Kim__DS_89w2iq8fw7qi_0
    <br/>
    <b>
     Name:
    </b>
    SAIT_semiconductors_ACS_2023_HfO_validation
    <br/>
    <b>
     Authors:
    </b>
    Geonu Kim, Byunggook Na, Gunhee Kim, Hyuntae Cho, Seung-Jin Kang, Hee Sun Lee, Saerom Choi, Heejae Kim, Seungwon Lee, Yongdeok Kim
    <br/>
    <b>
     Elements
    </b>
    : Hf, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,510
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 336,960
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/SAITPublic/MLFF-Framework" target="_blank">
     https://github.com/SAITPublic/MLFF-Framework
    </a>
    <br/>
    <a href="https://openreview.net/forum?id=hr9Bd1A9Un" target="_blank">
     https://openreview.net/forum?id=hr9Bd1A9Un
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_mo7yelvudt4k_0" target="_blank">
     SAIT_semiconductors_ACS_2023_SiN_out-of-domain
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Out-of-domain configurations from the SAIT_semiconductors_ACS_2023_SiN dataset. This dataset contains SiN, Si and N configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    SAIT_semiconductors_ACS_2023_SiN_out-of-domain__Kim-Na-Kim-Cho-Kang-Lee-Choi-Kim-Lee-Kim__DS_mo7yelvudt4k_0
    <br/>
    <b>
     Name:
    </b>
    SAIT_semiconductors_ACS_2023_SiN_out-of-domain
    <br/>
    <b>
     Authors:
    </b>
    Geonu Kim, Byunggook Na, Gunhee Kim, Hyuntae Cho, Seung-Jin Kang, Hee Sun Lee, Saerom Choi, Heejae Kim, Seungwon Lee, Yongdeok Kim
    <br/>
    <b>
     Elements
    </b>
    : N, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,235
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 129,675
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/SAITPublic/MLFF-Framework" target="_blank">
     https://github.com/SAITPublic/MLFF-Framework
    </a>
    <br/>
    <a href="https://openreview.net/forum?id=hr9Bd1A9Un" target="_blank">
     https://openreview.net/forum?id=hr9Bd1A9Un
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_piuigd7monq9_0" target="_blank">
     SAIT_semiconductors_ACS_2023_SiN_raw
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Structures from the SAIT_semiconductors_ACS_2023_SiN dataset, separated into N-only, Si-only, SiN, and out-of-domain melt, quench and relax configuration sets. This dataset contains SiN, Si and N configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    SAIT_semiconductors_ACS_2023_SiN_raw__Kim-Na-Kim-Cho-Kang-Lee-Choi-Kim-Lee-Kim__DS_piuigd7monq9_0
    <br/>
    <b>
     Name:
    </b>
    SAIT_semiconductors_ACS_2023_SiN_raw
    <br/>
    <b>
     Authors:
    </b>
    Geonu Kim, Byunggook Na, Gunhee Kim, Hyuntae Cho, Seung-Jin Kang, Hee Sun Lee, Saerom Choi, Heejae Kim, Seungwon Lee, Yongdeok Kim
    <br/>
    <b>
     Elements
    </b>
    : N, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 88,163
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 5,204,822
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/SAITPublic/MLFF-Framework" target="_blank">
     https://github.com/SAITPublic/MLFF-Framework
    </a>
    <br/>
    <a href="https://openreview.net/forum?id=hr9Bd1A9Un" target="_blank">
     https://openreview.net/forum?id=hr9Bd1A9Un
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_orfjpdomkekx_0" target="_blank">
     SAIT_semiconductors_ACS_2023_SiN_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test configurations from the SAIT_semiconductors_ACS_2023_SiN dataset. This dataset contains SiN, Si and N configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    SAIT_semiconductors_ACS_2023_SiN_test__Kim-Na-Kim-Cho-Kang-Lee-Choi-Kim-Lee-Kim__DS_orfjpdomkekx_0
    <br/>
    <b>
     Name:
    </b>
    SAIT_semiconductors_ACS_2023_SiN_test
    <br/>
    <b>
     Authors:
    </b>
    Geonu Kim, Byunggook Na, Gunhee Kim, Hyuntae Cho, Seung-Jin Kang, Hee Sun Lee, Saerom Choi, Heejae Kim, Seungwon Lee, Yongdeok Kim
    <br/>
    <b>
     Elements
    </b>
    : N, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,866
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 165,559
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/SAITPublic/MLFF-Framework" target="_blank">
     https://github.com/SAITPublic/MLFF-Framework
    </a>
    <br/>
    <a href="https://openreview.net/forum?id=hr9Bd1A9Un" target="_blank">
     https://openreview.net/forum?id=hr9Bd1A9Un
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_5yfdgzb5zhgm_0" target="_blank">
     SAIT_semiconductors_ACS_2023_SiN_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training configurations from the SAIT_semiconductors_ACS_2023_SiN dataset. This dataset contains SiN, Si and N configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    SAIT_semiconductors_ACS_2023_SiN_train__Kim-Na-Kim-Cho-Kang-Lee-Choi-Kim-Lee-Kim__DS_5yfdgzb5zhgm_0
    <br/>
    <b>
     Name:
    </b>
    SAIT_semiconductors_ACS_2023_SiN_train
    <br/>
    <b>
     Authors:
    </b>
    Geonu Kim, Byunggook Na, Gunhee Kim, Hyuntae Cho, Seung-Jin Kang, Hee Sun Lee, Saerom Choi, Heejae Kim, Seungwon Lee, Yongdeok Kim
    <br/>
    <b>
     Elements
    </b>
    : N, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 22,510
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,284,467
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/SAITPublic/MLFF-Framework" target="_blank">
     https://github.com/SAITPublic/MLFF-Framework
    </a>
    <br/>
    <a href="https://openreview.net/forum?id=hr9Bd1A9Un" target="_blank">
     https://openreview.net/forum?id=hr9Bd1A9Un
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_5piytwom0j25_0" target="_blank">
     SAIT_semiconductors_ACS_2023_SiN_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Validation configurations from the SAIT_semiconductors_ACS_2023_SiN dataset. This dataset contains SiN, Si and N configurations from the SAIT semiconductors datasets. SAIT semiconductors datasets comprise two rich datasets for the important semiconductor thin film materials silicon nitride (SiN) and hafnium oxide (HfO), gathered for the development of MLFFs. DFT simulations were conducted under various conditions that include differing initial structures, stoichiometry, temperature, strain, and defects.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    SAIT_semiconductors_ACS_2023_SiN_validation__Kim-Na-Kim-Cho-Kang-Lee-Choi-Kim-Lee-Kim__DS_5piytwom0j25_0
    <br/>
    <b>
     Name:
    </b>
    SAIT_semiconductors_ACS_2023_SiN_validation
    <br/>
    <b>
     Authors:
    </b>
    Geonu Kim, Byunggook Na, Gunhee Kim, Hyuntae Cho, Seung-Jin Kang, Hee Sun Lee, Saerom Choi, Heejae Kim, Seungwon Lee, Yongdeok Kim
    <br/>
    <b>
     Elements
    </b>
    : N, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,822
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 159,951
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/SAITPublic/MLFF-Framework" target="_blank">
     https://github.com/SAITPublic/MLFF-Framework
    </a>
    <br/>
    <a href="https://openreview.net/forum?id=hr9Bd1A9Un" target="_blank">
     https://openreview.net/forum?id=hr9Bd1A9Un
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_w2ngtxl0ep5d_0" target="_blank">
     SIMPLE_NN_SiO2
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    10,000 configurations of SiO2 used as an example for the SIMPLE-NN machine learning model. Dataset includes three types of crystals: quartz, cristobalite and tridymite; amorphous; and liquid phase SiO2. Structures with distortion from compression, monoaxial strain and shear strain were also included in the training set.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    SIMPLE_NN_SiO2__Lee-Yoo-Jeong-Han__DS_w2ngtxl0ep5d_0
    <br/>
    <b>
     Name:
    </b>
    SIMPLE_NN_SiO2
    <br/>
    <b>
     Authors:
    </b>
    Kyuhyun Lee, Dongsun Yoo, Wonseok Jeong, Seungwu Han
    <br/>
    <b>
     Elements
    </b>
    : O, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 10,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 600,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.17632/pjv2yr7pvr.1" target="_blank">
     https://doi.org/10.17632/pjv2yr7pvr.1
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.cpc.2019.04.014" target="_blank">
     https://doi.org/10.1016/j.cpc.2019.04.014
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_3tspv1150ejj_0" target="_blank">
     SN2_JCTC_2019
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The SN2 dataset was generated as a partner benchmark dataset, along with the 'solvated protein fragments' dataset, for measuring the performance of machine learning models, in particular PhysNet, at describing chemical reactions, long-range interactions, and condensed phase systems. SN2 probes chemical reactions of methyl halides with halide anions, i.e. X- + CH3Y -&gt; CH3X +  Y-, and contains structures, for all possible combinations of X,Y = F, Cl, Br, I. The dataset also includes various structures for several smaller molecules that can be formed in fragmentation reactions, such as CH3X, HX, CHX or CH2X- as well as geometries for H2, CH2, CH3+ and XY interhalogen compounds. In total, the dataset provides reference energies, forces, and dipole moments for 452709 structurescalculated at the DSD-BLYP-D3(BJ)/def2-TZVP level of theory using ORCA 4.0.1.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    SN2_JCTC_2019__Unke-Meuwly__DS_3tspv1150ejj_0
    <br/>
    <b>
     Name:
    </b>
    SN2_JCTC_2019
    <br/>
    <b>
     Authors:
    </b>
    Oliver T. Unke, Markus Meuwly
    <br/>
    <b>
     Elements
    </b>
    : Br, C, Cl, F, H, I
    <br/>
    <b>
     Number of Configurations
    </b>
    : 394,684
    <br/>
    <b>
     Number of Elements
    </b>
    : 6
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,194,246
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.2605341" target="_blank">
     https://doi.org/10.5281/zenodo.2605341
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.9b00181" target="_blank">
     https://doi.org/10.1021/acs.jctc.9b00181
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_kg0dv12aiq97_0" target="_blank">
     SPICE_2023
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    SPICE (Small-Molecule/Protein Interaction Chemical Energies) is a collection of quantum mechanical data for training potential functions. The emphasis is particularly on simulating drug-like small molecules interacting with proteins. Subsets of the dataset include the following: dipeptides: these provide comprehensive sampling of the covalent interactions found in proteins; solvated amino acids: these provide sampling of protein-water and water-water interactions; PubChem molecules: These sample a very wide variety of drug-like small molecules; monomer and dimer structures from DES370K: these provide sampling of a wide variety of non-covalent interactions; ion pairs: these provide further sampling of Coulomb interactions over a range of distances.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    SPICE_2023__Eastman-Behara-Dotson-Galvelis-Herr-Horton-Mao-Chodera-Pritchard-Wang-Fabritiis-Markland__DS_kg0dv12aiq97_0
    <br/>
    <b>
     Name:
    </b>
    SPICE_2023
    <br/>
    <b>
     Authors:
    </b>
    Peter Eastman, Pavan Kumar Behara, David L. Dotson, Raimondas Galvelis, John E. Herr, Josh T. Horton, Yuezhi Mao, John D. Chodera, Benjamin P. Pritchard, Yuanqing Wang, Gianni De Fabritiis, Thomas E. Markland
    <br/>
    <b>
     Elements
    </b>
    : Br, C, Ca, Cl, F, H, I, K, Li, N, Na, O, P, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 116,504
    <br/>
    <b>
     Number of Elements
    </b>
    : 14
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,382,829
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.8222043" target="_blank">
     https://doi.org/10.5281/zenodo.8222043
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-022-01882-6" target="_blank">
     https://doi.org/10.1038/s41597-022-01882-6
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_a3v4bu6mts6b_0" target="_blank">
     Si-H-GAP_reference
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A reference set of configurations of hydrogenated liquid and amorphous silicon from the datasets for Si-H-GAP. These configurations were used to evaluate training on a GAP model.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Si-H-GAP_reference__Unruh-Meidanshahi-Goodnick-Csányi-Zimányi__DS_a3v4bu6mts6b_0
    <br/>
    <b>
     Name:
    </b>
    Si-H-GAP_reference
    <br/>
    <b>
     Authors:
    </b>
    Davis Unruh, Reza Vatan Meidanshahi, Stephen M. Goodnick, Gábor Csányi, Gergely T. Zimányi
    <br/>
    <b>
     Elements
    </b>
    : H, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 114
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 24,895
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/dgunruh/Si-H-GAP" target="_blank">
     https://github.com/dgunruh/Si-H-GAP
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevMaterials.6.065603" target="_blank">
     https://doi.org/10.1103/PhysRevMaterials.6.065603
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_8arvjtldu4fw_0" target="_blank">
     Si-H-GAP_training
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A set of training configurations of hydrogenated liquid and amorphous silicon from the datasets for Si-H-GAP. Includes virial sigmas used for configurations used in the corresponding publication (virial-sigma-paper) as well as an alternate configuration defined by doubled virial sigma prefactors (from 0.025 to 0.05).
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Si-H-GAP_training__Unruh-Meidanshahi-Goodnick-Csányi-Zimányi__DS_8arvjtldu4fw_0
    <br/>
    <b>
     Name:
    </b>
    Si-H-GAP_training
    <br/>
    <b>
     Authors:
    </b>
    Davis Unruh, Reza Vatan Meidanshahi, Stephen M. Goodnick, Gábor Csányi, Gergely T. Zimányi
    <br/>
    <b>
     Elements
    </b>
    : H, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 392
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 65,909
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/dgunruh/Si-H-GAP" target="_blank">
     https://github.com/dgunruh/Si-H-GAP
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevMaterials.6.065603" target="_blank">
     https://doi.org/10.1103/PhysRevMaterials.6.065603
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_vywqeohskg2j_0" target="_blank">
     Si-H-GAP_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A set of validation configurations of hydrogenated liquid and amorphous silicon from the datasets for Si-H-GAP. These configurations served to augment the reference set as a final benchmark for NEP model performance.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Si-H-GAP_validation__Unruh-Meidanshahi-Goodnick-Csányi-Zimányi__DS_vywqeohskg2j_0
    <br/>
    <b>
     Name:
    </b>
    Si-H-GAP_validation
    <br/>
    <b>
     Authors:
    </b>
    Davis Unruh, Reza Vatan Meidanshahi, Stephen M. Goodnick, Gábor Csányi, Gergely T. Zimányi
    <br/>
    <b>
     Elements
    </b>
    : H, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 150
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 23,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/dgunruh/Si-H-GAP" target="_blank">
     https://github.com/dgunruh/Si-H-GAP
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevMaterials.6.065603" target="_blank">
     https://doi.org/10.1103/PhysRevMaterials.6.065603
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_dqascvm2baak_0" target="_blank">
     Si_Al_Ti_Seko_PRB_2019_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training sets from Si_Al_Ti_Seko_PRB_2019. This dataset is compiled of 10,000 selected structures from the ICSD, divided into training and test sets. The dataset was generated for the purpose of training a MLIP with introduced high-order linearly independent rotational invariants up to the sixth order based on spherical harmonics. DFT calculations were carried out with VASP using the PBE cross-correlation functional and an energy cutoff of 400 eV.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Si_Al_Ti_Seko_PRB_2019_test__Seko-Togo-Tanaka__DS_dqascvm2baak_0
    <br/>
    <b>
     Name:
    </b>
    Si_Al_Ti_Seko_PRB_2019_test
    <br/>
    <b>
     Authors:
    </b>
    Atsuto Seko, Atsushi Togo, Isao Tanaka
    <br/>
    <b>
     Elements
    </b>
    : Al, Si, Ti
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,989
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 197,628
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevB.99.214108" target="_blank">
     https://doi.org/10.1103/PhysRevB.99.214108
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0129045" target="_blank">
     https://doi.org/10.1063/5.0129045
    </a>
    <br/>
    <a href="https://sekocha.github.io/" target="_blank">
     https://sekocha.github.io/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_swqa99vqo249_0" target="_blank">
     Si_Al_Ti_Seko_PRB_2019_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test sets from Si_Al_Ti_Seko_PRB_2019. This dataset is compiled of 10,000 selected structures from the ICSD, divided into training and test sets. The dataset was generated for the purpose of training a MLIP with introduced high-order linearly independent rotational invariants up to the sixth order based on spherical harmonics. DFT calculations were carried out with VASP using the PBE cross-correlation functional and an energy cutoff of 400 eV.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Si_Al_Ti_Seko_PRB_2019_train__Seko-Togo-Tanaka__DS_swqa99vqo249_0
    <br/>
    <b>
     Name:
    </b>
    Si_Al_Ti_Seko_PRB_2019_train
    <br/>
    <b>
     Authors:
    </b>
    Atsuto Seko, Atsushi Togo, Isao Tanaka
    <br/>
    <b>
     Elements
    </b>
    : Al, Si, Ti
    <br/>
    <b>
     Number of Configurations
    </b>
    : 36,155
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,774,664
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevB.99.214108" target="_blank">
     https://doi.org/10.1103/PhysRevB.99.214108
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0129045" target="_blank">
     https://doi.org/10.1063/5.0129045
    </a>
    <br/>
    <a href="https://sekocha.github.io/" target="_blank">
     https://sekocha.github.io/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_paehju6qhaym_0" target="_blank">
     Si_JCP_2017
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A dataset of 64-atom silicon configurations in four phases: cubic-diamond, (beta)-tin, R8, and liquid. MD simulations are run at 300, 600 and 900 K for solid phases; up to 2500 K for the L phase. All relaxations performed at zero pressure. Additional configurations prepared by random distortion of crystal structures. VASP was used with a PAW pseudopotential and PBE exchange correlation. k-point mesh was optimized for energy convergence of 0.5 meV/atom and stress convergence of 0.1 kbar. The plane wave energy cutoff was set to 300 eV. To reduce the correlation between data points MD, data were thinned by using one of every 100 consecutive structures from the MD simulations at 300 K and one of every 20 structures from higher temperature MD simulations.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Si_JCP_2017__Cubuk-Malone-Onat-Waterland-Kaxiras__DS_paehju6qhaym_0
    <br/>
    <b>
     Name:
    </b>
    Si_JCP_2017
    <br/>
    <b>
     Authors:
    </b>
    Ekin D. Cubuk, Brad D. Malone, Berk Onat, Amos Waterland, Efthimios Kaxiras
    <br/>
    <b>
     Elements
    </b>
    : Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,117
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 71,424
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1063/1.4990503" target="_blank">
     https://doi.org/10.1063/1.4990503
    </a>
    <br/>
    <a href="https://doi.org/10.1063/1.4990503" target="_blank">
     https://doi.org/10.1063/1.4990503
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_u9wd92plbetq_0" target="_blank">
     Si_PRX_GAP
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The original DFT training data for the general-purpose silicon interatomic potential described in the associated publication. The kinds of configuration that we include are chosen using intuition and past experience to guide what needs to be included to obtain good coverage pertaining to a range of properties.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Si_PRX_GAP__Bartók-Kermode-Bernstein-Csányi__DS_u9wd92plbetq_0
    <br/>
    <b>
     Name:
    </b>
    Si_PRX_GAP
    <br/>
    <b>
     Authors:
    </b>
    Albert P. Bartók, James Kermode, Noam Bernstein, Gábor Csányi
    <br/>
    <b>
     Elements
    </b>
    : Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,472
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 171,164
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.17863/CAM.65004" target="_blank">
     https://doi.org/10.17863/CAM.65004
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevX.8.041048" target="_blank">
     https://doi.org/10.1103/PhysRevX.8.041048
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_14m394gnh3ae_0" target="_blank">
     Silica_NPJCM_2022
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset was created for the purpose of training an MLIP for silica (SiO2). For initial DFT computations, GPAW (in combination with ASE) was used with LDA, PBE and PBEsol functionals; and VASP with the SCAN functional. All calculations used the projector augmented-wave method. After comparison, it was found that SCAN performed best, and all values were recalculated using SCAN. An energy cut-off of 900 eV and a k-spacing of 0.23 Å-1 were used.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Silica_NPJCM_2022__Erhard-Rohrer-Albe-Deringer__DS_14m394gnh3ae_0
    <br/>
    <b>
     Name:
    </b>
    Silica_NPJCM_2022
    <br/>
    <b>
     Authors:
    </b>
    Linus C. Erhard, Jochen Rohrer, Karsten Albe, Volker L. Deringer
    <br/>
    <b>
     Elements
    </b>
    : O, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,074
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 268,118
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.6353683" target="_blank">
     https://doi.org/10.5281/zenodo.6353683
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-022-00768-w" target="_blank">
     https://doi.org/10.1038/s41524-022-00768-w
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_h1i3plesx7bc_0" target="_blank">
     Sn-SCAN_PRM_2023
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 6,500 configurations of Sn, including Sn8, Sn16 and Sn32, used in developing a deep potential that predicts the phase diagram of Sn.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Sn-SCAN_PRM_2023__Chen-Yuan-Liu-Geng-Zhang-Wang-Chen__DS_h1i3plesx7bc_0
    <br/>
    <b>
     Name:
    </b>
    Sn-SCAN_PRM_2023
    <br/>
    <b>
     Authors:
    </b>
    Tao Chen, Fengbo Yuan, Jianchuan Liu, Huayun Geng, Linfeng Zhang, Han Wang, Mohan Chen
    <br/>
    <b>
     Elements
    </b>
    : Sn
    <br/>
    <b>
     Number of Configurations
    </b>
    : 6,721
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 113,584
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=Sn-SCAN" target="_blank">
     https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=Sn-SCAN
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevMaterials.7.053603" target="_blank">
     https://doi.org/10.1103/PhysRevMaterials.7.053603
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_a0bxs66goqvv_0" target="_blank">
     TSFF_PLOS_2022
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    One configuration of an enzyme: training data for a quantum-guided molecular mechanics model.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    TSFF_PLOS_2022__Quinn-Patel-Koh-Haines-Norrby-Helquist-Wiest__DS_a0bxs66goqvv_0
    <br/>
    <b>
     Name:
    </b>
    TSFF_PLOS_2022
    <br/>
    <b>
     Authors:
    </b>
    Taylor R. Quinn, Himani N. Patel, Kevin H. Koh, Brandon E. Haines, Per-Ola Norrby, Paul Helquist, Olaf Wiest
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 117
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1371/journal.pone.0264960.s001" target="_blank">
     https://doi.org/10.1371/journal.pone.0264960.s001
    </a>
    <br/>
    <a href="https://doi.org/10.1371/journal.pone.0264960" target="_blank">
     https://doi.org/10.1371/journal.pone.0264960
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_rgtu2lkgv5rq_0" target="_blank">
     Ta_Linear_JCP2014
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This data set was originally used to generate a linear SNAP potential for solid and liquid tantalum as published in Thompson, A.P. et. al, J. Comp. Phys. 285 (2015) 316-330.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Ta_Linear_JCP2014__Thompson-Swiler-Trott-Foiles-Tucker__DS_rgtu2lkgv5rq_0
    <br/>
    <b>
     Name:
    </b>
    Ta_Linear_JCP2014
    <br/>
    <b>
     Authors:
    </b>
    Aidan P. Thompson, Laura P. Swiler, Christian R. Trott, Stephen M. Foiles, Garritt J. Tucker
    <br/>
    <b>
     Elements
    </b>
    : Ta
    <br/>
    <b>
     Number of Configurations
    </b>
    : 363
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 4,224
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/FitSNAP/FitSNAP/tree/master/examples/Ta_Linear_JCP2014" target="_blank">
     https://github.com/FitSNAP/FitSNAP/tree/master/examples/Ta_Linear_JCP2014
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.jcp.2014.12.018" target="_blank">
     https://doi.org/10.1016/j.jcp.2014.12.018
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_r6c6gt2s98xm_0" target="_blank">
     Ta_PINN_2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A dataset consisting of the energies of supercells containing from 1 to 250 atoms. The supercells represent energy-volume relations for 8 crystal structures of Ta, 5 uniform deformation paths between pairs of structures, vacancies, interstitials, surfaces with low-index orientations, 4 symmetrical tilt grain boundaries, γ-surfaces on the (110) and (211) fault planes, a [111] screw dislocation, liquid Ta, and several isolated clusters containing from 2 to 51 atoms. Some of the supercells contain static atomic configurations. However, most are snapshots of ab initio MD simulations at different densities, and temperatures ranging from 293 K to 3300 K. The BCC structure was sampled in the greatest detail, including a wide range of isotropic and uniaxial deformations.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Ta_PINN_2021__Lin-Pun-Mishin__DS_r6c6gt2s98xm_0
    <br/>
    <b>
     Name:
    </b>
    Ta_PINN_2021
    <br/>
    <b>
     Authors:
    </b>
    Yi-Shen Lin, Ganga P. Purja Pun, Yuri Mishin
    <br/>
    <b>
     Elements
    </b>
    : Ta
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,196
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 136,037
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1016/j.commatsci.2021.111180" target="_blank">
     https://doi.org/10.1016/j.commatsci.2021.111180
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.commatsci.2021.111180" target="_blank">
     https://doi.org/10.1016/j.commatsci.2021.111180
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_40zw467dnc6d_0" target="_blank">
     Ta_PRM2019
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset was designed to enable machine-learning of Ta elastic, thermal, and defect properties, as well as surface energetics, melting, and the structure of the liquid phase. The dataset was constructed by starting with the dataset from J. Byggmästar et al., Phys. Rev. B 100, 144105 (2019), then rescaling all of the configurations to the correct lattice spacing and adding in gamma surface configurations.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Ta_PRM2019__Byggmästar-Nordlund-Djurabekova__DS_40zw467dnc6d_0
    <br/>
    <b>
     Name:
    </b>
    Ta_PRM2019
    <br/>
    <b>
     Authors:
    </b>
    Jesper Byggmästar, Kai Nordlund, Flyura Djurabekova
    <br/>
    <b>
     Elements
    </b>
    : Ta
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,775
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 45,439
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://gitlab.com/acclab/gap-data/-/tree/master" target="_blank">
     https://gitlab.com/acclab/gap-data/-/tree/master
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevMaterials.4.093802" target="_blank">
     https://doi.org/10.1103/PhysRevMaterials.4.093802
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_yk3t004l8dpd_0" target="_blank">
     TdS-PdV_Atari5200
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 45,000 configurations of metal oxides of Mg, Ag, Pt, Cu and Zn, with initial training structures taken from the Materials Project database.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    TdS-PdV_Atari5200__Wisesa-Andolina-Saidi__DS_yk3t004l8dpd_0
    <br/>
    <b>
     Name:
    </b>
    TdS-PdV_Atari5200
    <br/>
    <b>
     Authors:
    </b>
    Pandu Wisesa, Christopher M. Andolina, Wissam A. Saidi
    <br/>
    <b>
     Elements
    </b>
    : Ag, Cu, Mg, O, Pt, Zn
    <br/>
    <b>
     Number of Configurations
    </b>
    : 44,404
    <br/>
    <b>
     Number of Elements
    </b>
    : 6
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,987,604
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7278341" target="_blank">
     https://doi.org/10.5281/zenodo.7278341
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpclett.2c03445" target="_blank">
     https://doi.org/10.1021/acs.jpclett.2c03445
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpclett.2c03445" target="_blank">
     https://doi.org/10.1021/acs.jpclett.2c03445
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_jn819esw58ah_0" target="_blank">
     TiMoS_alloys_CMS2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training set (DFT output) for CE models and MC simulation output for the manuscript 'Phase behaviour of (Ti:Mo)S2binary alloys arising from electron-lattice coupling'. The DFT calculations are performed using VASP 5.4.3, compiled with intel MPI and Intel MKL support.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    TiMoS_alloys_CMS2021__Silva-Polcar-Kramer__DS_jn819esw58ah_0
    <br/>
    <b>
     Name:
    </b>
    TiMoS_alloys_CMS2021
    <br/>
    <b>
     Authors:
    </b>
    Andrea Silva, Tomas Polcar, Denis Kramer
    <br/>
    <b>
     Elements
    </b>
    : Mo, S, Ti
    <br/>
    <b>
     Number of Configurations
    </b>
    : 259
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,996
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://eprints.soton.ac.uk/443461/" target="_blank">
     https://eprints.soton.ac.uk/443461/
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.commatsci.2020.110044" target="_blank">
     https://doi.org/10.1016/j.commatsci.2020.110044
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_kvjft3au55qb_0" target="_blank">
     TiO2_CMS2016
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    TiO2 dataset that was designed to build atom neural network potentials (ANN) by Artrith et al. using the AENET package. This dataset includes various crystalline phases of TiO2 and MD data that are extracted from ab inito calculations. The dataset includes 7815 structures with 165,229 atomic environments in the stochiometric ratio of 66% O to 34% Ti.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    TiO2_CMS2016__Artrith-Urban__DS_kvjft3au55qb_0
    <br/>
    <b>
     Name:
    </b>
    TiO2_CMS2016
    <br/>
    <b>
     Authors:
    </b>
    Nongnuch Artrith, Alexander Urban
    <br/>
    <b>
     Elements
    </b>
    : O, Ti
    <br/>
    <b>
     Number of Configurations
    </b>
    : 7,812
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 165,114
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/DescriptorZoo/sensitivity-dimensionality-results/tree/master/datasets/TiO2" target="_blank">
     https://github.com/DescriptorZoo/sensitivity-dimensionality-results/tree/master/datasets/TiO2
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.commatsci.2015.11.047" target="_blank">
     https://doi.org/10.1016/j.commatsci.2015.11.047
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_ngso7es93qnj_0" target="_blank">
     TiZrHfTa_APS2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A dataset used to train machine-learning interatomic potentials (moment tensor potentials) for multicomponent alloys to ab initio data in order to investigate the disordered body-centered cubic (bcc) TiZrHfTax system with varying Ta concentration.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    TiZrHfTa_APS2021__Gubaev-Ikeda-Tasnádi-Neugebauer-Shapeev-Grabowski-Körmann__DS_ngso7es93qnj_0
    <br/>
    <b>
     Name:
    </b>
    TiZrHfTa_APS2021
    <br/>
    <b>
     Authors:
    </b>
    Konstantin Gubaev, Yuji Ikeda, Ferenc Tasnádi, Jörg Neugebauer, Alexander V. Shapeev, Blazej Grabowski, Fritz Körmann
    <br/>
    <b>
     Elements
    </b>
    : Hf, Ta, Ti, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,623
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 223,984
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevMaterials.5.073801" target="_blank">
     https://doi.org/10.1103/PhysRevMaterials.5.073801
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_aaocsvp2x40m_0" target="_blank">
     Ti_NPJCM_2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Approximately 7,400 configurations of titanium used for training a deep potential using the DeePMD-kit molecular dynamics package and DP-GEN training scheme.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Ti_NPJCM_2021__Wen-Wang-Zhu-Zhang-Wang-Srolovitz-Wu__DS_aaocsvp2x40m_0
    <br/>
    <b>
     Name:
    </b>
    Ti_NPJCM_2021
    <br/>
    <b>
     Authors:
    </b>
    Tongqi Wen, Rui Wang, Lingyu Zhu, Linfeng Zhang, Han Wang, David J. Srolovitz, Zhaoxuan Wu
    <br/>
    <b>
     Elements
    </b>
    : Ti
    <br/>
    <b>
     Number of Configurations
    </b>
    : 7,378
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 143,856
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=Ti" target="_blank">
     https://www.aissquare.com/datasets/detail?pageType=datasets&amp;name=Ti
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-021-00661-y" target="_blank">
     https://doi.org/10.1038/s41524-021-00661-y
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_zzfaosyakwom_0" target="_blank">
     Transition1x-test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The test split of the Transition1x dataset. Transition1x is a benchmark dataset containing 9.6 million Density Functional Theory (DFT) calculations of forces and energies of molecular configurations on and around reaction pathways at the ωB97x/6-31 G(d) level of theory. The configurations contained in this dataset allow a better representation of features in transition state regions when compared to other benchmark datasets -- in particular QM9 and ANI1x.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Transition1x-test__Schreiner-Bhowmik-Vegge-Busk-Winther__DS_zzfaosyakwom_0
    <br/>
    <b>
     Name:
    </b>
    Transition1x-test
    <br/>
    <b>
     Authors:
    </b>
    Mathias Schreiner, Arghya Bhowmik, Tejs Vegge, Jonas Busk, Ole Winther
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 190,277
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,106,770
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.19614657.v4" target="_blank">
     https://doi.org/10.6084/m9.figshare.19614657.v4
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-022-01870-w" target="_blank">
     https://doi.org/10.1038/s41597-022-01870-w
    </a>
    <br/>
    <a href="https://gitlab.com/matschreiner/Transition1x" target="_blank">
     https://gitlab.com/matschreiner/Transition1x
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_ktku4cml3al7_0" target="_blank">
     Transition1x-validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The validation split of the Transition1x dataset. Transition1x is a benchmark dataset containing 9.6 million Density Functional Theory (DFT) calculations of forces and energies of molecular configurations on and around reaction pathways at the ωB97x/6-31 G(d) level of theory. The configurations contained in this dataset allow a better representation of features in transition state regions when compared to other benchmark datasets -- in particular QM9 and ANI1x.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Transition1x-validation__Schreiner-Bhowmik-Vegge-Busk-Winther__DS_ktku4cml3al7_0
    <br/>
    <b>
     Name:
    </b>
    Transition1x-validation
    <br/>
    <b>
     Authors:
    </b>
    Mathias Schreiner, Arghya Bhowmik, Tejs Vegge, Jonas Busk, Ole Winther
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 264,996
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,743,476
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.19614657.v4" target="_blank">
     https://doi.org/10.6084/m9.figshare.19614657.v4
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-022-01870-w" target="_blank">
     https://doi.org/10.1038/s41597-022-01870-w
    </a>
    <br/>
    <a href="https://gitlab.com/matschreiner/Transition1x" target="_blank">
     https://gitlab.com/matschreiner/Transition1x
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_sn0csplf32qj_0" target="_blank">
     Transition1x_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The training split of the Transition1x dataset. Transition1x is a benchmark dataset containing 9.6 million Density Functional Theory (DFT) calculations of forces and energies of molecular configurations on and around reaction pathways at the ωB97x/6-31 G(d) level of theory. The configurations contained in this dataset allow a better representation of features in transition state regions when compared to other benchmark datasets -- in particular QM9 and ANI1x.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Transition1x_train__Schreiner-Bhowmik-Vegge-Busk-Winther__DS_sn0csplf32qj_0
    <br/>
    <b>
     Name:
    </b>
    Transition1x_train
    <br/>
    <b>
     Authors:
    </b>
    Mathias Schreiner, Arghya Bhowmik, Tejs Vegge, Jonas Busk, Ole Winther
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 62,990
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 536,010
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.19614657.v4" target="_blank">
     https://doi.org/10.6084/m9.figshare.19614657.v4
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-022-01870-w" target="_blank">
     https://doi.org/10.1038/s41597-022-01870-w
    </a>
    <br/>
    <a href="https://gitlab.com/matschreiner/Transition1x" target="_blank">
     https://gitlab.com/matschreiner/Transition1x
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_jyklsju4h580_0" target="_blank">
     UNEP_v1_2023_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The test set for UNEP-v1 (version 1 of Unified NeuroEvolution Potential), a model implemented in GPUMD.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    UNEP-v1_2023__Song-Zhao-Liu-Wang-Lindgren-Wang-Chen-Xu-Liang-Ying-Xu-Zhao-Shi-Wang-Lyu-Zeng-Liang-Dong-Sun-Chen-Zhang-Guo-Qian-Sun-Erhart-Ala-Nissila-Su-Fan__DS_jyklsju4h580_0
    <br/>
    <b>
     Name:
    </b>
    UNEP_v1_2023_test
    <br/>
    <b>
     Authors:
    </b>
    Keke Song, Rui Zhao, Jiahui Liu, Yanzhou Wang, Eric Lindgren, Yong Wang, Shunda Chen, Ke Xu, Ting Liang, Penghua Ying, Nan Xu, Zhiqiang Zhao, Jiuyang Shi, Junjie Wang, Shuang Lyu, Zezhu Zeng, Shirong Liang, Haikuan Dong, Ligang Sun, Yue Chen, Zhuhua Zhang, Wanlin Guo, Ping Qian, Jian Sun, Paul Erhart, Tapio Ala-Nissila, Yanjing Su, Zheyong Fan
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, Au, Cr, Cu, Mg, Mo, Ni, Pb, Pd, Pt, Ta, Ti, V, W, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 4,411
    <br/>
    <b>
     Number of Elements
    </b>
    : 16
    <br/>
    <b>
     Number of Atoms
    </b>
    : 318,910
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://zenodo.org/doi/10.5281/zenodo.10081676" target="_blank">
     https://zenodo.org/doi/10.5281/zenodo.10081676
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2311.04732" target="_blank">
     https://doi.org/10.48550/arXiv.2311.04732
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_14h4rvviya0k_0" target="_blank">
     UNEP_v1_2023_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The training set for UNEP-v1 (version 1 of Unified NeuroEvolution Potential), a model implemented in GPUMD.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    UNEP-v1_2023__Song-Zhao-Liu-Wang-Lindgren-Wang-Chen-Xu-Liang-Ying-Xu-Zhao-Shi-Wang-Lyu-Zeng-Liang-Dong-Sun-Chen-Zhang-Guo-Qian-Sun-Erhart-Ala-Nissila-Su-Fan__DS_14h4rvviya0k_0
    <br/>
    <b>
     Name:
    </b>
    UNEP_v1_2023_train
    <br/>
    <b>
     Authors:
    </b>
    Keke Song, Rui Zhao, Jiahui Liu, Yanzhou Wang, Eric Lindgren, Yong Wang, Shunda Chen, Ke Xu, Ting Liang, Penghua Ying, Nan Xu, Zhiqiang Zhao, Jiuyang Shi, Junjie Wang, Shuang Lyu, Zezhu Zeng, Shirong Liang, Haikuan Dong, Ligang Sun, Yue Chen, Zhuhua Zhang, Wanlin Guo, Ping Qian, Jian Sun, Paul Erhart, Tapio Ala-Nissila, Yanjing Su, Zheyong Fan
    <br/>
    <b>
     Elements
    </b>
    : Ag, Al, Au, Cr, Cu, Mg, Mo, Ni, Pb, Pd, Pt, Ta, Ti, V, W, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 104,799
    <br/>
    <b>
     Number of Elements
    </b>
    : 16
    <br/>
    <b>
     Number of Atoms
    </b>
    : 6,840,534
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://zenodo.org/doi/10.5281/zenodo.10081676" target="_blank">
     https://zenodo.org/doi/10.5281/zenodo.10081676
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2311.04732" target="_blank">
     https://doi.org/10.48550/arXiv.2311.04732
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_ouwlietscprn_0" target="_blank">
     V_PRM2019
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset was designed to enable machine-learning of V elastic, thermal, and defect properties, as well as surface energetics, melting, and the structure of the liquid phase. The dataset was constructed by starting with the dataset from J. Byggmästar et al., Phys. Rev. B 100, 144105 (2019), then rescaling all of the configurations to the correct lattice spacing and adding in gamma surface configurations.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    V_PRM2019__Byggmästar-Nordlund-Djurabekova__DS_ouwlietscprn_0
    <br/>
    <b>
     Name:
    </b>
    V_PRM2019
    <br/>
    <b>
     Authors:
    </b>
    Jesper Byggmästar, Kai Nordlund, Flyura Djurabekova
    <br/>
    <b>
     Elements
    </b>
    : V
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,802
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 46,466
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://gitlab.com/acclab/gap-data/-/tree/master" target="_blank">
     https://gitlab.com/acclab/gap-data/-/tree/master
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevMaterials.4.093802" target="_blank">
     https://doi.org/10.1103/PhysRevMaterials.4.093802
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_kk4a8u1eo42m_0" target="_blank">
     W-14
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    158,000 diverse atomic environments of elemental tungsten.Includes DFT-PBE energies, forces and stresses for tungsten; periodic unit cells in the range of 1-135 atoms, including bcc primitive cell, 128-atom bcc cell, vacancies, low index surfaces, gamma-surfaces, and dislocation cores.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    W-14__Szlachta-Bartók-Csányi__DS_kk4a8u1eo42m_0
    <br/>
    <b>
     Name:
    </b>
    W-14
    <br/>
    <b>
     Authors:
    </b>
    Wojciech J. Szlachta, Albert P. Bartók, Gábor Csányi
    <br/>
    <b>
     Elements
    </b>
    : W
    <br/>
    <b>
     Number of Configurations
    </b>
    : 9,693
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 158,515
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://qmml.org/datasets.html" target="_blank">
     https://qmml.org/datasets.html
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevB.90.104108" target="_blank">
     https://doi.org/10.1103/PhysRevB.90.104108
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_yq2whjjndyq5_0" target="_blank">
     WBe_PRB2019
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This data set was originally used to generate a multi-component linear SNAP potential for tungsten and beryllium as published in Wood, M. A., et. al. Phys. Rev. B 99 (2019) 184305. This data set was developed for the purpose of studying plasma material interactions in fusion reactors.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    WBe_PRB2019__Wood-Cusentino-Wirth-Thompson__DS_yq2whjjndyq5_0
    <br/>
    <b>
     Name:
    </b>
    WBe_PRB2019
    <br/>
    <b>
     Authors:
    </b>
    Mitchell A. Wood, Mary Alice Cusentino, Brian D. Wirth, Aidan P. Thompson
    <br/>
    <b>
     Elements
    </b>
    : Be, W
    <br/>
    <b>
     Number of Configurations
    </b>
    : 25,120
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 525,915
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/FitSNAP/FitSNAP/tree/master/examples/WBe_PRB2019" target="_blank">
     https://github.com/FitSNAP/FitSNAP/tree/master/examples/WBe_PRB2019
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevB.99.184305" target="_blank">
     https://doi.org/10.1103/PhysRevB.99.184305
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_wqezpu0pw9io_0" target="_blank">
     WS22_acrolein
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of acrolein from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    WS22_acrolein__Jr-Zhang-Dral-Barbatti__DS_wqezpu0pw9io_0
    <br/>
    <b>
     Name:
    </b>
    WS22_acrolein
    <br/>
    <b>
     Authors:
    </b>
    Max Pinheiro Jr, Shuang Zhang, Pavlo O. Dral, Mario Barbatti
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 120,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 960,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7032333" target="_blank">
     https://doi.org/10.5281/zenodo.7032333
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-023-01998-3" target="_blank">
     https://doi.org/10.1038/s41597-023-01998-3
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_vxtsip8qgm7t_0" target="_blank">
     WS22_alanine
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of alanine from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    WS22_alanine__Jr-Zhang-Dral-Barbatti__DS_vxtsip8qgm7t_0
    <br/>
    <b>
     Name:
    </b>
    WS22_alanine
    <br/>
    <b>
     Authors:
    </b>
    Max Pinheiro Jr, Shuang Zhang, Pavlo O. Dral, Mario Barbatti
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 120,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,560,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7032333" target="_blank">
     https://doi.org/10.5281/zenodo.7032333
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-023-01998-3" target="_blank">
     https://doi.org/10.1038/s41597-023-01998-3
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_srva1ami36vs_0" target="_blank">
     WS22_dmabn
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of dmabn from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    WS22_dmabn__Jr-Zhang-Dral-Barbatti__DS_srva1ami36vs_0
    <br/>
    <b>
     Name:
    </b>
    WS22_dmabn
    <br/>
    <b>
     Authors:
    </b>
    Max Pinheiro Jr, Shuang Zhang, Pavlo O. Dral, Mario Barbatti
    <br/>
    <b>
     Elements
    </b>
    : C, H, N
    <br/>
    <b>
     Number of Configurations
    </b>
    : 120,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,520,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7032333" target="_blank">
     https://doi.org/10.5281/zenodo.7032333
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-023-01998-3" target="_blank">
     https://doi.org/10.1038/s41597-023-01998-3
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_s3qsg4c3hyk6_0" target="_blank">
     WS22_nitrophenol
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of nitrophenol from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    WS22_nitrophenol__Jr-Zhang-Dral-Barbatti__DS_s3qsg4c3hyk6_0
    <br/>
    <b>
     Name:
    </b>
    WS22_nitrophenol
    <br/>
    <b>
     Authors:
    </b>
    Max Pinheiro Jr, Shuang Zhang, Pavlo O. Dral, Mario Barbatti
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 120,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,800,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7032333" target="_blank">
     https://doi.org/10.5281/zenodo.7032333
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-023-01998-3" target="_blank">
     https://doi.org/10.1038/s41597-023-01998-3
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_ukw33o4tvaiy_0" target="_blank">
     WS22_o-hbdi
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of o-hbdi from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    WS22_o-hbdi__Jr-Zhang-Dral-Barbatti__DS_ukw33o4tvaiy_0
    <br/>
    <b>
     Name:
    </b>
    WS22_o-hbdi
    <br/>
    <b>
     Authors:
    </b>
    Max Pinheiro Jr, Shuang Zhang, Pavlo O. Dral, Mario Barbatti
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 120,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,640,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7032333" target="_blank">
     https://doi.org/10.5281/zenodo.7032333
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-023-01998-3" target="_blank">
     https://doi.org/10.1038/s41597-023-01998-3
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_jz8yny1m7xer_0" target="_blank">
     WS22_sma
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of sma from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    WS22_sma__Jr-Zhang-Dral-Barbatti__DS_jz8yny1m7xer_0
    <br/>
    <b>
     Name:
    </b>
    WS22_sma
    <br/>
    <b>
     Authors:
    </b>
    Max Pinheiro Jr, Shuang Zhang, Pavlo O. Dral, Mario Barbatti
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 120,040
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,280,760
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7032333" target="_blank">
     https://doi.org/10.5281/zenodo.7032333
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-023-01998-3" target="_blank">
     https://doi.org/10.1038/s41597-023-01998-3
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_ngd7bajt6nvx_0" target="_blank">
     WS22_thymine
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of o-hbdi from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    WS22_thymine__Jr-Zhang-Dral-Barbatti__DS_ngd7bajt6nvx_0
    <br/>
    <b>
     Name:
    </b>
    WS22_thymine
    <br/>
    <b>
     Authors:
    </b>
    Max Pinheiro Jr, Shuang Zhang, Pavlo O. Dral, Mario Barbatti
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 120,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,800,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7032333" target="_blank">
     https://doi.org/10.5281/zenodo.7032333
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-023-01998-3" target="_blank">
     https://doi.org/10.1038/s41597-023-01998-3
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_ztubsom32o4f_0" target="_blank">
     WS22_toluene
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of toluene from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    WS22_toluene__Jr-Zhang-Dral-Barbatti__DS_ztubsom32o4f_0
    <br/>
    <b>
     Name:
    </b>
    WS22_toluene
    <br/>
    <b>
     Authors:
    </b>
    Max Pinheiro Jr, Shuang Zhang, Pavlo O. Dral, Mario Barbatti
    <br/>
    <b>
     Elements
    </b>
    : C, H
    <br/>
    <b>
     Number of Configurations
    </b>
    : 100,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,500,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7032333" target="_blank">
     https://doi.org/10.5281/zenodo.7032333
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-023-01998-3" target="_blank">
     https://doi.org/10.1038/s41597-023-01998-3
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_0upwmrh3apql_0" target="_blank">
     WS22_urea
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of urea from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    WS22_urea__Jr-Zhang-Dral-Barbatti__DS_0upwmrh3apql_0
    <br/>
    <b>
     Name:
    </b>
    WS22_urea
    <br/>
    <b>
     Authors:
    </b>
    Max Pinheiro Jr, Shuang Zhang, Pavlo O. Dral, Mario Barbatti
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 120,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 960,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7032333" target="_blank">
     https://doi.org/10.5281/zenodo.7032333
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-023-01998-3" target="_blank">
     https://doi.org/10.1038/s41597-023-01998-3
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_w3sximoit8bj_0" target="_blank">
     WS22_urocanic
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of urocanic from WS22. The WS22 database combines Wigner sampling with geometry interpolation to generate 1.18 million molecular geometries equally distributed into 10 independent datasets of flexible organic molecules with varying sizes and chemical complexity. In addition to the potential energy and forces required to construct potential energy surfaces, the WS22 database provides several other quantum chemical properties, all obtained via single-point calculations for each molecular geometry. All quantum chemical calculations were performed with the Gaussian 09 program.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    WS22_urocanic__Jr-Zhang-Dral-Barbatti__DS_w3sximoit8bj_0
    <br/>
    <b>
     Name:
    </b>
    WS22_urocanic
    <br/>
    <b>
     Authors:
    </b>
    Max Pinheiro Jr, Shuang Zhang, Pavlo O. Dral, Mario Barbatti
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 120,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,920,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7032333" target="_blank">
     https://doi.org/10.5281/zenodo.7032333
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41597-023-01998-3" target="_blank">
     https://doi.org/10.1038/s41597-023-01998-3
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_0od3fns1ap8a_0" target="_blank">
     W_LML-retrain_bulk_MD_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test set from W_LML-retrain dataset, containing bulk tungsten calculations. The W_LML-retrain dataset contains DFT calculations used in testing a linear-in-descriptor machine learning potential that accounts for dislocation-defect interactions in tungsten. Density functional simulations were performed using VASP. The PBE generalised gradient approximation was used to describe effects of electron exchange and correlation together with a projector augmented wave (PAW) basis set with a cut-off energy of 550 eV. Occupancies were smeared with a Methfessel-Paxton scheme of order one with a 0.1 eV smearing width. The Brillouin zone was sampled with a Monkhorst-Pack k-point grid for the 2D cluster simulations periodic along the dislocation line and a single k-point was used for the calculations with 3D spherical QM regions. The values of these parameters were chosen after a series of convergence tests on forces with a tolerance of a few meV/Å.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    W_LML-retrain_bulk_MD_test__Onat-Ortner-Kermode__DS_0od3fns1ap8a_0
    <br/>
    <b>
     Name:
    </b>
    W_LML-retrain_bulk_MD_test
    <br/>
    <b>
     Authors:
    </b>
    Berk Onat, Christoph Ortner, James R. Kermode
    <br/>
    <b>
     Elements
    </b>
    : W
    <br/>
    <b>
     Number of Configurations
    </b>
    : 8
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,996
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/marseille-matmol/LML-retrain" target="_blank">
     https://github.com/marseille-matmol/LML-retrain
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.actamat.2023.118734" target="_blank">
     https://doi.org/10.1016/j.actamat.2023.118734
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_gh9s2sopu064_0" target="_blank">
     W_PRB2019
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset was originally designed to fit a GAP potential with a specific focus on properties relevant for simulations of radiation-induced collision cascades and the damage they produce, including a realistic repulsive potential for short-range many-body cascade dynamics and a good description of the liquid phase.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    W_PRB2019__Byggmästar-Hamedani-Nordlund-Djurabekova__DS_gh9s2sopu064_0
    <br/>
    <b>
     Name:
    </b>
    W_PRB2019
    <br/>
    <b>
     Authors:
    </b>
    Jesper Byggmästar, Ali Hamedani, Kai Nordlund, Flyura Djurabekova
    <br/>
    <b>
     Elements
    </b>
    : W
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,528
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 42,068
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://gitlab.com/acclab/gap-data/-/tree/master/W/2019-05-24" target="_blank">
     https://gitlab.com/acclab/gap-data/-/tree/master/W/2019-05-24
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevB.100.144105" target="_blank">
     https://doi.org/10.1103/PhysRevB.100.144105
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_xrfpxtaioi75_0" target="_blank">
     Yttrium-catalyzed_benzylic_C-H_alkylations_of_alkylpyridines_with_olefins
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This data was assembled to investigate rare-earth-catalyzed benzylic C(sp3)-H addition of pyridines to olefins. All calculations were performed with the Gaussian 09 software package. The B3PW91 functional was used for geometric optimization without any symmetric constraints. Each optimized structure was subsequently analyzed by harmonic vibrational frequencies at the same level of theory for characterization of a minimum (NImag = 0) or a transition state (NImag = 1) to obtain the thermodynamic data. The 6-31G(d) basis set was used for C, H, and N atoms, and Stuttgart/Dresden relativistic effective core potentials (RECPs) as well as the associated valence basis sets were used for the Y atom. To obtain more accurate energies, single-point energy calculations were performed with a larger basis set. In such single-point calculations, the M06-L functional, which often shows good performance in the treatment of transition-metal systems, was used together with the CPCM solvation model for consideration of the toluene solvation effect. The same basis set together with associated pseudopotentials as in geometry optimization was used for the Y atom, and the 6-311+G(d,p) basis set was used for the remaining atoms.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Yttrium-catalyzed_benzylic_C-H_alkylations_of_alkylpyridines_with_olefins__Zhou-Luo-Kang-Hou-Luo__DS_xrfpxtaioi75_0
    <br/>
    <b>
     Name:
    </b>
    Yttrium-catalyzed_benzylic_C-H_alkylations_of_alkylpyridines_with_olefins
    <br/>
    <b>
     Authors:
    </b>
    Guangli Zhou, Gen Luo, Xiaohui Kang, Zhaomin Hou, Yi Luo
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, Y
    <br/>
    <b>
     Number of Configurations
    </b>
    : 68
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 4,110
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1021/acs.organomet.8b00397.s002" target="_blank">
     https://doi.org/10.1021/acs.organomet.8b00397.s002
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.organomet.8b00397" target="_blank">
     https://doi.org/10.1021/acs.organomet.8b00397
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_sh7jt3ptmde4_0" target="_blank">
     ZIF-4_Amorphous_Zeolitic_Imidazolate_Frameworks_2023
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset contains four trajectories of amorphous zeolitic imidazolate frameworks (ZIF-4),  liquids calculated at four different volumes and at temperatures of 1500K and 1750K; and three trajectories of the ZIF-4 crystal: one at 300K and two at 1500K. Data was generated at the DFT-PBE-D3 level of theory.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    ZIF-4_Amorphous_Zeolitic_Imidazolate_Frameworks_2023__Castel-Andre-Edwards-Evans-Coudert__DS_sh7jt3ptmde4_0
    <br/>
    <b>
     Name:
    </b>
    ZIF-4_Amorphous_Zeolitic_Imidazolate_Frameworks_2023
    <br/>
    <b>
     Authors:
    </b>
    Nicolas Castel, Dune Andre, Connor Edwards, Jack D. Evans, Francois-Xavier Coudert
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, Zn
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,189,836
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 323,635,392
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.10015594" target="_blank">
     https://doi.org/10.5281/zenodo.10015594
    </a>
    <br/>
    <a href="https://doi.org/10.26434/chemrxiv-2023-8003d" target="_blank">
     https://doi.org/10.26434/chemrxiv-2023-8003d
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_9in0wrvt6qg2_0" target="_blank">
     Zeo-1_SD_2022
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    130,000 configurations of zeolite from the Database of Zeolite Structures. Calculations performed using Amsterdam Modeling Suite software.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Zeo-1_SD_2022__Komissarov-Verstraelen__DS_9in0wrvt6qg2_0
    <br/>
    <b>
     Name:
    </b>
    Zeo-1_SD_2022
    <br/>
    <b>
     Authors:
    </b>
    Leonid Komissarov, Toon Verstraelen
    <br/>
    <b>
     Elements
    </b>
    : Al, Ba, Be, C, Ca, Cs, F, Ge, H, K, Li, N, Na, O, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 12,930
    <br/>
    <b>
     Number of Elements
    </b>
    : 15
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,841,742
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1038/s41597-022-01160-5" target="_blank">
     https://doi.org/10.1038/s41597-022-01160-5
    </a>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:cv-zd" target="_blank">
     https://doi.org/10.24435/materialscloud:cv-zd
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_58y020ce6b6j_0" target="_blank">
     Zn_MTP_CMS2023
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A training dataset of diverse atomic configurations of Zn, varying in aggregation states, crystal structures, defect types, and sizes. The aim was to derive a potential capable of accurately describing a broad spectrum of local atomic configurations in Zn.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Zn_MTP_CMS2023__Mei-Cheng-Chen-Wang-Li-Kong__DS_58y020ce6b6j_0
    <br/>
    <b>
     Name:
    </b>
    Zn_MTP_CMS2023
    <br/>
    <b>
     Authors:
    </b>
    Haojie Mei, Luyao Cheng, Liang Chen, Feifei Wang, Jinfu Li, Lingti Kong
    <br/>
    <b>
     Elements
    </b>
    : Zn
    <br/>
    <b>
     Number of Configurations
    </b>
    : 13,552
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 278,996
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/meihaojie/Zn_system/tree/main" target="_blank">
     https://github.com/meihaojie/Zn_system/tree/main
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.commatsci.2023.112723" target="_blank">
     https://doi.org/10.1016/j.commatsci.2023.112723
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_6woak771jubv_0" target="_blank">
     Zr_Sn_JNM_2024
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset contains data from density functional theory calculations of various atomic configurations of pure Zr, pure Sn, and Zr-Sn alloys with different structures, defects, and compositions. Energies, forces, and stresses are calculated at the DFT level of theory. Includes 23,956 total configurations.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    Zr_Sn_JNM_2024__Mei-Chen-Wang-Liu-Hu-Lin-Shen-Li-Kong__DS_6woak771jubv_0
    <br/>
    <b>
     Name:
    </b>
    Zr_Sn_JNM_2024
    <br/>
    <b>
     Authors:
    </b>
    Haojie Mei, Liang Chen, Feifei Wang, Guisen Liu, Jing Hu, Weitong Lin, Yao Shen, Jinfu Li, Lingti Kong
    <br/>
    <b>
     Elements
    </b>
    : Sn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 23,611
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 688,087
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/meihaojie/Zr_Sn_system" target="_blank">
     https://github.com/meihaojie/Zr_Sn_system
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.jnucmat.2023.154794" target="_blank">
     https://doi.org/10.1016/j.jnucmat.2023.154794
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_70btumen3361_0" target="_blank">
     a-AlOx_JCP_2020
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset was used for the training of an MLIP for amorphous alumina (a-AlOx). Two configurations sets correspond to i) the actual training data and ii) additional reference data. Ab initio calculations were performedwith the Vienna Ab initio Simulation Package. The projector augmented wave method was used to treat the atomic core electrons,and the Perdew-Burke-Ernzerhof functional within the generalized gradient approximation was used to describe the electron-electron interactions. The cutoff energy for the plane-wave basis set was set to 550 eV during the ab initio calculation. The obtained reference database includes the DFT energies of 41,203 structures. The supercell size of the AlOx reference structures varied from 24 to 132 atoms. K-point values are given for structures with: Al0, Al12, Al24, Al48 and Al192.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    a-AlOx_JCP_2020__Li-Ando-Watanabe__DS_70btumen3361_0
    <br/>
    <b>
     Name:
    </b>
    a-AlOx_JCP_2020
    <br/>
    <b>
     Authors:
    </b>
    Wenwen Li, Yasunobu Ando, Satoshi Watanabe
    <br/>
    <b>
     Elements
    </b>
    : Al, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 123,586
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 4,541,918
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:y1-zd" target="_blank">
     https://doi.org/10.24435/materialscloud:y1-zd
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0026289" target="_blank">
     https://doi.org/10.1063/5.0026289
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_bmjfal3bj4ah_0" target="_blank">
     aC_JCP_2023
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The amorphous carbon dataset was generated using ab initio calculations with VASP software. We utilized the LDA exchange-correlation functional and the PAW potential for carbon. Melt-quench simulations were performed to create amorphous and liquid-state structures. A simple cubic lattice of 216 carbon atoms was chosen as the initial state. Simulations were conducted at densities of 1.5, 1.7, 2.0, 2.2, 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, and 3.5 g/cm3 to produce a variety of structures. The NVT ensemble was employed for all melt-quench simulations, and the density was adjusted by modifying the size of the simulation cell. A time step of 1 fs was used for the simulations. For all densities, only the Γ points were sampled in the k-space. To increase structural diversity, six independent simulations were performed.In the melt-quench simulations, the temperature was raised from 300 K to 9000 K over 2 ps to melt carbon. Equilibrium molecular dynamics (MD) was conducted at 9000 K for 3 ps to create a liquid state, followed by a decrease in temperature to 5000 K over 2 ps, with the system equilibrating at that temperature for 2 ps. Finally, the temperature was lowered from 5000 K to 300 K over 2 ps to generate an amorphous structure.During the melt-quench simulation, 30 snapshots were taken from the equilibrium MD trajectory at 9000 K, 100 from the cooling process between 9000 and 5000 K, 25 from the equilibrium MD trajectory at 5000 K, and 100 from the cooling process between 5000 and 300 K. This yielded a total of 16,830 data points.Data for diamond structures containing 216 atoms at densities of 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, and 3.5 g/cm3 were also prepared. Further data on the diamond structure were obtained from 80 snapshots taken from the 2 ps equilibrium MD trajectory at 300 K, resulting in 560 data points.To validate predictions for larger structures, we generated data for 512-atom systems using the same procedure as for the 216-atom systems. A single simulation was conducted for each density. The number of data points was 2,805 for amorphous and liquid states
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    aC_JCP_2023__Minamitani-Obayashi-Shimizu-Watanabe__DS_bmjfal3bj4ah_0
    <br/>
    <b>
     Name:
    </b>
    aC_JCP_2023
    <br/>
    <b>
     Authors:
    </b>
    Emi Minamitani, Ippei Obayashi, Koji Shimizu, Satoshi Watanabe
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 20,195
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 5,192,400
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7905585" target="_blank">
     https://doi.org/10.5281/zenodo.7905585
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0159349" target="_blank">
     https://doi.org/10.1063/5.0159349
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_sany2behzwv8_0" target="_blank">
     aC_JCP_2023_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test split from the 216-atom amorphous portion of the aC_JCP_2023 dataset. The amorphous carbon dataset was generated using ab initio calculations with VASP software. We utilized the LDA exchange-correlation functional and the PAW potential for carbon. Melt-quench simulations were performed to create amorphous and liquid-state structures. A simple cubic lattice of 216 carbon atoms was chosen as the initial state. Simulations were conducted at densities of 1.5, 1.7, 2.0, 2.2, 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, and 3.5 g/cm3 to produce a variety of structures. The NVT ensemble was employed for all melt-quench simulations, and the density was adjusted by modifying the size of the simulation cell. A time step of 1 fs was used for the simulations. For all densities, only the Γ points were sampled in the k-space. To increase structural diversity, six independent simulations were performed.In the melt-quench simulations, the temperature was raised from 300 K to 9000 K over 2 ps to melt carbon. Equilibrium molecular dynamics (MD) was conducted at 9000 K for 3 ps to create a liquid state, followed by a decrease in temperature to 5000 K over 2 ps, with the system equilibrating at that temperature for 2 ps. Finally, the temperature was lowered from 5000 K to 300 K over 2 ps to generate an amorphous structure.During the melt-quench simulation, 30 snapshots were taken from the equilibrium MD trajectory at 9000 K, 100 from the cooling process between 9000 and 5000 K, 25 from the equilibrium MD trajectory at 5000 K, and 100 from the cooling process between 5000 and 300 K. This yielded a total of 16,830 data points.Data for diamond structures containing 216 atoms at densities of 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, and 3.5 g/cm3 were also prepared. Further data on the diamond structure were obtained from 80 snapshots taken from the 2 ps equilibrium MD trajectory at 300 K, resulting in 560 data points.To validate predictions for larger structures, we generated data for 512-atom systems using the same procedure as for the 216-atom systems. A single simulation was conducted for each density. The number of data points was 2,805 for amorphous and liquid states
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    aC_JCP_2023_test__Minamitani-Obayashi-Shimizu-Watanabe__DS_sany2behzwv8_0
    <br/>
    <b>
     Name:
    </b>
    aC_JCP_2023_test
    <br/>
    <b>
     Authors:
    </b>
    Emi Minamitani, Ippei Obayashi, Koji Shimizu, Satoshi Watanabe
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,366
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 727,056
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7905585" target="_blank">
     https://doi.org/10.5281/zenodo.7905585
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0159349" target="_blank">
     https://doi.org/10.1063/5.0159349
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_20h8o5d0ltx1_0" target="_blank">
     aC_JCP_2023_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Train split from the 216-atom amorphous portion of the aC_JCP_2023 dataset. The amorphous carbon dataset was generated using ab initio calculations with VASP software. We utilized the LDA exchange-correlation functional and the PAW potential for carbon. Melt-quench simulations were performed to create amorphous and liquid-state structures. A simple cubic lattice of 216 carbon atoms was chosen as the initial state. Simulations were conducted at densities of 1.5, 1.7, 2.0, 2.2, 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, and 3.5 g/cm3 to produce a variety of structures. The NVT ensemble was employed for all melt-quench simulations, and the density was adjusted by modifying the size of the simulation cell. A time step of 1 fs was used for the simulations. For all densities, only the Γ points were sampled in the k-space. To increase structural diversity, six independent simulations were performed.In the melt-quench simulations, the temperature was raised from 300 K to 9000 K over 2 ps to melt carbon. Equilibrium molecular dynamics (MD) was conducted at 9000 K for 3 ps to create a liquid state, followed by a decrease in temperature to 5000 K over 2 ps, with the system equilibrating at that temperature for 2 ps. Finally, the temperature was lowered from 5000 K to 300 K over 2 ps to generate an amorphous structure.During the melt-quench simulation, 30 snapshots were taken from the equilibrium MD trajectory at 9000 K, 100 from the cooling process between 9000 and 5000 K, 25 from the equilibrium MD trajectory at 5000 K, and 100 from the cooling process between 5000 and 300 K. This yielded a total of 16,830 data points.Data for diamond structures containing 216 atoms at densities of 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, and 3.5 g/cm3 were also prepared. Further data on the diamond structure were obtained from 80 snapshots taken from the 2 ps equilibrium MD trajectory at 300 K, resulting in 560 data points.To validate predictions for larger structures, we generated data for 512-atom systems using the same procedure as for the 216-atom systems. A single simulation was conducted for each density. The number of data points was 2,805 for amorphous and liquid states
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    aC_JCP_2023_train__Minamitani-Obayashi-Shimizu-Watanabe__DS_20h8o5d0ltx1_0
    <br/>
    <b>
     Name:
    </b>
    aC_JCP_2023_train
    <br/>
    <b>
     Authors:
    </b>
    Emi Minamitani, Ippei Obayashi, Koji Shimizu, Satoshi Watanabe
    <br/>
    <b>
     Elements
    </b>
    : C
    <br/>
    <b>
     Number of Configurations
    </b>
    : 13,464
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,908,224
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7905585" target="_blank">
     https://doi.org/10.5281/zenodo.7905585
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0159349" target="_blank">
     https://doi.org/10.1063/5.0159349
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_08feepssbuq5_0" target="_blank">
     adatoms_on_single-layer_graphene_PRR2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset consists of graphene superlattices with tungsten adatoms with properties calculated at the DFT level of theory. The authors modeled the placement of tungsten adatoms on a graphene monolayer. The resulting superlattice structures were then used to calculate electronic band structure and phonon dispersion relations. The dataset was used to investigate the effect of adatom placement on electronic band structure and phonon dispersion relations of graphene superlattices. The creation of the dataset involved the following steps: 1. Selection of the graphene monolayer as the starting point for the superlattice construction. 2. Placement of tungsten adatoms in the center of the unit cell 3. Calculation of the electronic structure and other properties of the resulting superlattice using DFT. 4. Generation of a set of reduced Brillouin zones representing the symmetry of the superlattice. 5. Calculation of the electronic band structure and phonon dispersion relations for each superlattice structure in the dataset.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    adatoms_on_single-layer_graphene_PRR2021__Skurativska-Tsirkin-Natterer-Neupert-Fischer__DS_08feepssbuq5_0
    <br/>
    <b>
     Name:
    </b>
    adatoms_on_single-layer_graphene_PRR2021
    <br/>
    <b>
     Authors:
    </b>
    Anastasiia Skurativska, Stepan S. Tsirkin, Fabian D Natterer, Titus Neupert, Mark H Fischer
    <br/>
    <b>
     Elements
    </b>
    : C, Cr, Ir, Mo, Nb, Os, Re, Rh, Ru, Ta, W
    <br/>
    <b>
     Number of Configurations
    </b>
    : 18
    <br/>
    <b>
     Number of Elements
    </b>
    : 11
    <br/>
    <b>
     Number of Atoms
    </b>
    : 774
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:bj-bh" target="_blank">
     https://doi.org/10.24435/materialscloud:bj-bh
    </a>
    <br/>
    <a href="http://doi.org/10.1103/PhysRevResearch.3.L032003" target="_blank">
     http://doi.org/10.1103/PhysRevResearch.3.L032003
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_pms72aqfzc4v_0" target="_blank">
     aleatoric_epistemic_error_AIC2023
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset for H2CO, with and without added noise for testing the effects of noise on quality of fit. Configurations sets are included for clean energy values with different levels of gaussian noise added to atomic forces (including a set with no noise added), and energies perturbed at different levels (including a set with no perturbation). Configuration sets correspond to individual files found at the data link.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    aleatoric_epistemic_error_AIC2023__Goswami-Kaser-Bemish-Meuwly__DS_pms72aqfzc4v_0
    <br/>
    <b>
     Name:
    </b>
    aleatoric_epistemic_error_AIC2023
    <br/>
    <b>
     Authors:
    </b>
    Sugata Goswami, Silvan Käser, Raymond J. Bemish, Markus Meuwly
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 28,808
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 115,232
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/MMunibas/noise" target="_blank">
     https://github.com/MMunibas/noise
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.aichem.2023.100033" target="_blank">
     https://doi.org/10.1016/j.aichem.2023.100033
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_mnt5vb22pdze_0" target="_blank">
     alkali-metal_intercalation_in_disordered_carbon_anode_materials_JMCA2019
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A dataset created as part of a combination DFT-ML approach to study three alkali metals (K, Li, Na) in model carbon systems at a range of densities and degrees of disorder. The purpose of the study was to investigate the properties of alkali metals in hard (non-graphitising) and nanoporous carbons as potential anode materials for battery technology.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    alkali-metal_intercalation_in_disordered_carbon_anode_materials_JMCA2019__Huang-Csanyi-Zhao-Cheng-Deringer__DS_mnt5vb22pdze_0
    <br/>
    <b>
     Name:
    </b>
    alkali-metal_intercalation_in_disordered_carbon_anode_materials_JMCA2019
    <br/>
    <b>
     Authors:
    </b>
    Jian-Xing Huang, Gábor Csányi, Jin-Bao Zhao, Jun Cheng, Volker L. Deringer
    <br/>
    <b>
     Elements
    </b>
    : C, K, Li, Na
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,365
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 298,050
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.17863/CAM.42087" target="_blank">
     https://doi.org/10.17863/CAM.42087
    </a>
    <br/>
    <a href="https://doi.org/10.1039/C9TA05453G" target="_blank">
     https://doi.org/10.1039/C9TA05453G
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_agiti2oe5bqb_0" target="_blank">
     alpha_brass_nanoparticles
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    53,841 structures of alpha-brass (less than 40% Zinc). Includes atomic forces and total energy. Calculated using VASP at the DFT level of theory.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    alpha_brass_nanoparticles__Weinreich-Römer-Paleico-Behler__DS_agiti2oe5bqb_0
    <br/>
    <b>
     Name:
    </b>
    alpha_brass_nanoparticles
    <br/>
    <b>
     Authors:
    </b>
    Jan Weinreich, Anton Römer, Martín Leandro Paleico, Jörg Behler
    <br/>
    <b>
     Elements
    </b>
    : Cu, Zn
    <br/>
    <b>
     Number of Configurations
    </b>
    : 53,696
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,956,679
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:94-aq" target="_blank">
     https://doi.org/10.24435/materialscloud:94-aq
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpcc.1c02314" target="_blank">
     https://doi.org/10.1021/acs.jpcc.1c02314
    </a>
    <br/>
    <a href="http://doi.org/10.1021/acs.jpcc.0c00559" target="_blank">
     http://doi.org/10.1021/acs.jpcc.0c00559
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_xzaglubh0trq_0" target="_blank">
     cG-SchNet
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations from a cG-SchNet trained on a subset of the QM9dataset. Model was trained with the intention of providing molecules withspecified functional groups or motifs, relying on sampling of molecularfingerprint data. Relaxation data for the generated molecules is computedusing ORCA software. Configuration sets include raw data fromcG-SchNet-generated configurations, with models trained on several differenttypes of target data and DFT relaxation data as a separate configurationset. Includes approximately 80,000 configurations.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    cG-SchNet__Gebauer-Gastegger-Hessmann-Müller-Schütt__DS_xzaglubh0trq_0
    <br/>
    <b>
     Name:
    </b>
    cG-SchNet
    <br/>
    <b>
     Authors:
    </b>
    Niklas W.A. Gebauer, Michael Gastegger, Stefaan S.P. Hessmann, Klaus-Robert Müller, Kristof T. Schütt
    <br/>
    <b>
     Elements
    </b>
    : C, F, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 79,772
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,467,492
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/atomistic-machine-learning/cG-SchNet/" target="_blank">
     https://github.com/atomistic-machine-learning/cG-SchNet/
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-022-28526-y" target="_blank">
     https://doi.org/10.1038/s41467-022-28526-y
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_w59koijcczkz_0" target="_blank">
     calcium_ferrites_as_cathodes_ca4fe9o17
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset for "Appraisal of calcium ferrites as cathodes for calcium rechargeable batteries: DFT, synthesis, characterization and electrochemistry of Ca4Fe9O17" created to explore Fe-based cathode materials for Ca-ion batteries. Structures include CaFe(2+n)O(4+n), where 0 &lt; n &lt; 3.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    calcium_ferrites_as_cathodes_ca4fe9o17__Dompablo-Casals__DS_w59koijcczkz_0
    <br/>
    <b>
     Name:
    </b>
    calcium_ferrites_as_cathodes_ca4fe9o17
    <br/>
    <b>
     Authors:
    </b>
    M. Elena Arroyo-de Dompablo, José Luis Casals
    <br/>
    <b>
     Elements
    </b>
    : Ca, Fe, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 345
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 35,462
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:xk-sn" target="_blank">
     https://doi.org/10.24435/materialscloud:xk-sn
    </a>
    <br/>
    <a href="http://doi.org/10.1039/c9dt04688g" target="_blank">
     http://doi.org/10.1039/c9dt04688g
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_s00e64z80ujy_0" target="_blank">
     cathode_materials_for_rechargeable_Ca_batteries_CM2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Data from the publication "Enlisting Potential Cathode Materials for Rechargeable Ca Batteries". The development of rechargeable batteries based on a Ca metal anode demands the identification of suitable cathode materials. This work investigates the potential application of a variety of compounds, which are selected from the In-organic Crystal Structural Database (ICSD) considering 3d-transition metal oxysulphides, pyrophosphates, silicates, nitrides, and phosphates with a maximum of four different chemical elements in their composition. Cathode perfor-mance of CaFeSO, CaCoSO, CaNiN, Ca3MnN3, Ca2Fe(Si2O7), CaM(P2O7) (M = V, Cr, Mn, Fe, Co), CaV2(P2O7)2, Ca(VO)2(PO4)2 and α-VOPO4 is evaluated throughout the calculation of operation voltages, volume changes associated to the redox reaction and mobility of Ca2+ ions. Some materials exhibit attractive specific capacities and intercalation voltages combined with energy barriers for Ca migration around 1 eV (CaFeSO, Ca2FeSi2O7 and CaV2(P2O7)2). Based on the DFT results, αI-VOPO4 is identified as a potential Ca-cathode with a maximum theoretical specific capacity of 312 mAh/g, an average intercalation voltage of 2.8 V and calculated energy barriers for Ca migration below 0.65 eV (GGA functional).
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    cathode_materials_for_rechargeable_Ca_batteries_CM2021__Dompablo-Casals__DS_s00e64z80ujy_0
    <br/>
    <b>
     Name:
    </b>
    cathode_materials_for_rechargeable_Ca_batteries_CM2021
    <br/>
    <b>
     Authors:
    </b>
    M. Elena Arroyo-de Dompablo, Jose Luis Casals
    <br/>
    <b>
     Elements
    </b>
    : Ca, Co, Fe, Mn, N, Ni, O, P, S, Si, V
    <br/>
    <b>
     Number of Configurations
    </b>
    : 10,840
    <br/>
    <b>
     Number of Elements
    </b>
    : 11
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,034,770
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:3n-e8" target="_blank">
     https://doi.org/10.24435/materialscloud:3n-e8
    </a>
    <br/>
    <a href="http://doi.org/10.1038/s41598-019-46002-4" target="_blank">
     http://doi.org/10.1038/s41598-019-46002-4
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_mf8sn11cn6wa_0" target="_blank">
     datasets_for_magnetic_MTP_NatSR2024_training
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset comprises a training dataset for magnetic multi-component machine-learning potentials for Fe-Al systems, including different concentrations of Fe and Al (Al concentrations from 0%-50%), with fully equilibrated and perturbed atomic positions, lattice vectors and magnetic moments represented.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    datasets_for_magnetic_MTP_NatSR2024_training__Kotykhov-Gubaev-Hodapp-Tantardini-Shapeev-Novikov__DS_mf8sn11cn6wa_0
    <br/>
    <b>
     Name:
    </b>
    datasets_for_magnetic_MTP_NatSR2024_training
    <br/>
    <b>
     Authors:
    </b>
    Alexey S. Kotykhov, Konstantin Gubaev, Max Hodapp, Christian Tantardini, Alexander V. Shapeev, Ivan S. Novikov
    <br/>
    <b>
     Elements
    </b>
    : Al, Fe
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,012
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 11,440
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://gitlab.com/ivannovikov/datasets_for_magnetic_MTP" target="_blank">
     https://gitlab.com/ivannovikov/datasets_for_magnetic_MTP
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41598-023-46951-x" target="_blank">
     https://doi.org/10.1038/s41598-023-46951-x
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_wu6xd9i8cf7i_0" target="_blank">
     datasets_for_magnetic_MTP_NatSR2024_verification
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This is the verification dataset (see companion training dataset: datasets_for_magnetic_MTP_NatSR2024_training) used in training a magnetic multi-component machine-learning potential for Fe-Al systems. The configurations from the verification set include different levels of magnetic moment perturbation than configurations from the training set. For this reason, the authors refer to this dataset as a "verification set", rather than a "validation set".
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    datasets_for_magnetic_MTP_NatSR2024_verification__Kotykhov-Gubaev-Hodapp-Tantardini-Shapeev-Novikov__DS_wu6xd9i8cf7i_0
    <br/>
    <b>
     Name:
    </b>
    datasets_for_magnetic_MTP_NatSR2024_verification
    <br/>
    <b>
     Authors:
    </b>
    Alexey S. Kotykhov, Konstantin Gubaev, Max Hodapp, Christian Tantardini, Alexander V. Shapeev, Ivan S. Novikov
    <br/>
    <b>
     Elements
    </b>
    : Al, Fe
    <br/>
    <b>
     Number of Configurations
    </b>
    : 336
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,696
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://gitlab.com/ivannovikov/datasets_for_magnetic_MTP" target="_blank">
     https://gitlab.com/ivannovikov/datasets_for_magnetic_MTP
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41598-023-46951-x" target="_blank">
     https://doi.org/10.1038/s41598-023-46951-x
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_k059wtxqsksu_0" target="_blank">
     defected_phosphorene_ACS_2023
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset contains pristine monolayer phosphorene as well as structures with monovacancies which were used to train an artificial neural network (ANN) for use with a high-dimensional neural network potentials molecular dynamics (HDNNP-MD) simulation. The publication investigates the mechanism and rates of the processes of defect diffusion, as well as monovacancy-to-divacancy defect coalescence.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    defected_phosphorene_ACS_2023__Kývala-Angeletti-Franchini-Dellago__DS_k059wtxqsksu_0
    <br/>
    <b>
     Name:
    </b>
    defected_phosphorene_ACS_2023
    <br/>
    <b>
     Authors:
    </b>
    Lukáš Kývala, Andrea Angeletti, Cesare Franchini, Christoph Dellago
    <br/>
    <b>
     Elements
    </b>
    : P
    <br/>
    <b>
     Number of Configurations
    </b>
    : 5,091
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 722,311
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.8421094" target="_blank">
     https://doi.org/10.5281/zenodo.8421094
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpcc.3c05713" target="_blank">
     https://doi.org/10.1021/acs.jpcc.3c05713
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_q6e3bvq4y67a_0" target="_blank">
     discrepencies_and_error_metrics_NPJ_2023_enhanced_validation_set
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Structures from discrepencies_and_error_metrics_NPJ_2023 validation set, enhanced by inclusion of rare events. The full discrepencies_and_error_metrics_NPJ_2023 dataset includes the original mlearn_Si_train dataset, modified with the purpose of developing models with better diffusivity scores by replacing ~54% of the data with structures containing migrating interstitials. The enhanced validation set contains 50 total structures, consisting of 20 structures randomly selected from the 120 replaced structures of the original training dataset, 11 snapshots with vacancy rare events (RE) from AIMD simulations, and 19 snapshots with interstitial RE from AIMD simulations. We also construct interstitial-RE and vacancy-RE testing sets, each consisting of 100 snapshots of atomic configurations with a single migrating vacancy or interstitial, respectively, from AIMD simulations at 1230 K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    discrepencies_and_error_metrics_NPJ_2023_enhanced_validation_set__Liu-He-Mo__DS_q6e3bvq4y67a_0
    <br/>
    <b>
     Name:
    </b>
    discrepencies_and_error_metrics_NPJ_2023_enhanced_validation_set
    <br/>
    <b>
     Authors:
    </b>
    Yunsheng Liu, Xingfeng He, Yifei Mo
    <br/>
    <b>
     Elements
    </b>
    : Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 50
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,198
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/mogroupumd/Silicon_MLIP_datasets" target="_blank">
     https://github.com/mogroupumd/Silicon_MLIP_datasets
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-023-01123-3" target="_blank">
     https://doi.org/10.1038/s41524-023-01123-3
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_nublbp38wse0_0" target="_blank">
     discrepencies_and_error_metrics_NPJ_2023_interstitial_enhanced_training_set
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Structures from discrepencies_and_error_metrics_NPJ_2023 training set, enhanced by inclusion of interstitials. The full discrepencies_and_error_metrics_NPJ_2023 dataset includes the original mlearn_Si_train dataset, modified with the purpose of developing models with better diffusivity scores by replacing ~54% of the data with structures containing migrating interstitials. The enhanced validation set contains 50 total structures, consisting of 20 structures randomly selected from the 120 replaced structures of the original training dataset, 11 snapshots with vacancy rare events (RE) from AIMD simulations, and 19 snapshots with interstitial RE from AIMD simulations. We also construct interstitial-RE and vacancy-RE testing sets, each consisting of 100 snapshots of atomic configurations with a single migrating vacancy or interstitial, respectively, from AIMD simulations at 1230 K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    discrepencies_and_error_metrics_NPJ_2023_interstitial_enhanced_training_set__Liu-He-Mo__DS_nublbp38wse0_0
    <br/>
    <b>
     Name:
    </b>
    discrepencies_and_error_metrics_NPJ_2023_interstitial_enhanced_training_set
    <br/>
    <b>
     Authors:
    </b>
    Yunsheng Liu, Xingfeng He, Yifei Mo
    <br/>
    <b>
     Elements
    </b>
    : Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 218
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 13,629
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/mogroupumd/Silicon_MLIP_datasets" target="_blank">
     https://github.com/mogroupumd/Silicon_MLIP_datasets
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-023-01123-3" target="_blank">
     https://doi.org/10.1038/s41524-023-01123-3
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_dhe9aqs9q1wf_0" target="_blank">
     discrepencies_and_error_metrics_NPJ_2023_interstitial_re_testing_set
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Structures from discrepencies_and_error_metrics_NPJ_2023 test set; these include an interstitial. The full discrepencies_and_error_metrics_NPJ_2023 dataset includes the original mlearn_Si_train dataset, modified with the purpose of developing models with better diffusivity scores by replacing ~54% of the data with structures containing migrating interstitials. The enhanced validation set contains 50 total structures, consisting of 20 structures randomly selected from the 120 replaced structures of the original training dataset, 11 snapshots with vacancy rare events (RE) from AIMD simulations, and 19 snapshots with interstitial RE from AIMD simulations. We also construct interstitial-RE and vacancy-RE testing sets, each consisting of 100 snapshots of atomic configurations with a single migrating vacancy or interstitial, respectively, from AIMD simulations at 1230 K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    discrepencies_and_error_metrics_NPJ_2023_interstitial_re_testing_set__Liu-He-Mo__DS_dhe9aqs9q1wf_0
    <br/>
    <b>
     Name:
    </b>
    discrepencies_and_error_metrics_NPJ_2023_interstitial_re_testing_set
    <br/>
    <b>
     Authors:
    </b>
    Yunsheng Liu, Xingfeng He, Yifei Mo
    <br/>
    <b>
     Elements
    </b>
    : Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 100
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 6,500
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/mogroupumd/Silicon_MLIP_datasets" target="_blank">
     https://github.com/mogroupumd/Silicon_MLIP_datasets
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-023-01123-3" target="_blank">
     https://doi.org/10.1038/s41524-023-01123-3
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_qxd7wv9yabtp_0" target="_blank">
     discrepencies_and_error_metrics_NPJ_2023_vacancy_enhanced_training_set
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Structures from discrepencies_and_error_metrics_NPJ_2023 training set; includes some structures with vacancies. The full discrepencies_and_error_metrics_NPJ_2023 dataset includes the original mlearn_Si_train dataset, modified with the purpose of developing models with better diffusivity scores by replacing ~54% of the data with structures containing migrating interstitials. The enhanced validation set contains 50 total structures, consisting of 20 structures randomly selected from the 120 replaced structures of the original training dataset, 11 snapshots with vacancy rare events (RE) from AIMD simulations, and 19 snapshots with interstitial RE from AIMD simulations. We also construct interstitial-RE and vacancy-RE testing sets, each consisting of 100 snapshots of atomic configurations with a single migrating vacancy or interstitial, respectively, from AIMD simulations at 1230 K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    discrepencies_and_error_metrics_NPJ_2023_vacancy_enhanced_training_set__Liu-He-Mo__DS_qxd7wv9yabtp_0
    <br/>
    <b>
     Name:
    </b>
    discrepencies_and_error_metrics_NPJ_2023_vacancy_enhanced_training_set
    <br/>
    <b>
     Authors:
    </b>
    Yunsheng Liu, Xingfeng He, Yifei Mo
    <br/>
    <b>
     Elements
    </b>
    : Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 218
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 13,389
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/mogroupumd/Silicon_MLIP_datasets" target="_blank">
     https://github.com/mogroupumd/Silicon_MLIP_datasets
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-023-01123-3" target="_blank">
     https://doi.org/10.1038/s41524-023-01123-3
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_la08goe2lz0g_0" target="_blank">
     discrepencies_and_error_metrics_NPJ_2023_vacancy_re_testing_set
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Structures from discrepencies_and_error_metrics_NPJ_2023 test set; these include a single migrating vacancy. The full discrepencies_and_error_metrics_NPJ_2023 dataset includes the original mlearn_Si_train dataset, modified with the purpose of developing models with better diffusivity scores by replacing ~54% of the data with structures containing migrating interstitials. The enhanced validation set contains 50 total structures, consisting of 20 structures randomly selected from the 120 replaced structures of the original training dataset, 11 snapshots with vacancy rare events (RE) from AIMD simulations, and 19 snapshots with interstitial RE from AIMD simulations. We also construct interstitial-RE and vacancy-RE testing sets, each consisting of 100 snapshots of atomic configurations with a single migrating vacancy or interstitial, respectively, from AIMD simulations at 1230 K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    discrepencies_and_error_metrics_NPJ_2023_vacancy_re_testing_set__Liu-He-Mo__DS_la08goe2lz0g_0
    <br/>
    <b>
     Name:
    </b>
    discrepencies_and_error_metrics_NPJ_2023_vacancy_re_testing_set
    <br/>
    <b>
     Authors:
    </b>
    Yunsheng Liu, Xingfeng He, Yifei Mo
    <br/>
    <b>
     Elements
    </b>
    : Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 100
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 6,300
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/mogroupumd/Silicon_MLIP_datasets" target="_blank">
     https://github.com/mogroupumd/Silicon_MLIP_datasets
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41524-023-01123-3" target="_blank">
     https://doi.org/10.1038/s41524-023-01123-3
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_k1k8iul6kgm2_0" target="_blank">
     disordered_transition_metal_oxyfluorides_EA2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Data from "On-the-fly assessment of diffusion barriers of disordered transition metal oxyfluorides using local descriptors". The dataset contains the result of 48 Nudged Elastic Band calculations of Li(2-x)VO2F diffusion barriers. The NEB was performed with VASP, using projector augmented-wave (PAW) method to describe electron-ion interaction. The disordered rock salt cells were created using a 3 x 4 x 4 supercell containing 96 atoms (in case of no vacancies). PBE is used as XC functional while a rotationally invariant Hubbard U correction was applied to the d orbital of V with a U value of 3.25 eV.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    disordered_transition_metal_oxyfluorides_EA2021__Chang-Jørgensen-Loftager-Bhowmik-Lastra-Vegge__DS_k1k8iul6kgm2_0
    <br/>
    <b>
     Name:
    </b>
    disordered_transition_metal_oxyfluorides_EA2021
    <br/>
    <b>
     Authors:
    </b>
    Jin Hyun Chang, Peter Bjørn Jørgensen, Simon Loftager, Arghya Bhowmik, Juan María García Lastra, Tejs Vegge
    <br/>
    <b>
     Elements
    </b>
    : F, Li, O, V
    <br/>
    <b>
     Number of Configurations
    </b>
    : 233
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 20,670
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:9v-3q" target="_blank">
     https://doi.org/10.24435/materialscloud:9v-3q
    </a>
    <br/>
    <a href="http://doi.org/10.1016/j.electacta.2021.138551" target="_blank">
     http://doi.org/10.1016/j.electacta.2021.138551
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_a2ftwj9b873a_0" target="_blank">
     doped_CsPbI3_energetics_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The test set from the doped CsPbI3 energetics dataset. This dataset was created to explore the effect of Cd and Pb substitutions on the structural stability of inorganic lead halide perovskite CsPbI3. CsPbI3 undergoes a direct to indirect band-gap phase transition at room temperature. The dataset contains configurations of CsPbI3 with low levels of Cd and Zn, which were used to train a GNN model to predict the energetics of structures with higher levels of substitutions.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    doped_CsPbI3_energetics_test__Eremin-Humonen-Lazarev-Pushkarev-Budennyy__DS_a2ftwj9b873a_0
    <br/>
    <b>
     Name:
    </b>
    doped_CsPbI3_energetics_test
    <br/>
    <b>
     Authors:
    </b>
    Roman A. Eremin, Innokentiy S. Humonen, Alexey A. Kazakov, Vladimir D. Lazarev, Anatoly P. Pushkarev, Semen A. Budennyy
    <br/>
    <b>
     Elements
    </b>
    : Cd, Cs, I, Pb, Zn
    <br/>
    <b>
     Number of Configurations
    </b>
    : 60
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 9,600
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/AIRI-Institute/doped_CsPbI3_energetics" target="_blank">
     https://github.com/AIRI-Institute/doped_CsPbI3_energetics
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.commatsci.2023.112672" target="_blank">
     https://doi.org/10.1016/j.commatsci.2023.112672
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_doj9b688juif_0" target="_blank">
     doped_CsPbI3_energetics_train_validate
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The training + validation set from the doped CsPbI3 energetics dataset. This dataset was created to explore the effect of Cd and Pb substitutions on the structural stability of inorganic lead halide perovskite CsPbI3. CsPbI3 undergoes a direct to indirect band-gap phase transition at room temperature. The dataset contains configurations of CsPbI3 with low levels of Cd and Zn, which were used to train a GNN model to predict the energetics of structures with higher levels of substitutions.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    doped_CsPbI3_energetics_train_validate__Eremin-Humonen-Lazarev-Pushkarev-Budennyy__DS_doj9b688juif_0
    <br/>
    <b>
     Name:
    </b>
    doped_CsPbI3_energetics_train_validate
    <br/>
    <b>
     Authors:
    </b>
    Roman A. Eremin, Innokentiy S. Humonen, Alexey A. Kazakov, Vladimir D. Lazarev, Anatoly P. Pushkarev, Semen A. Budennyy
    <br/>
    <b>
     Elements
    </b>
    : Cd, Cs, I, Pb, Zn
    <br/>
    <b>
     Number of Configurations
    </b>
    : 142
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 22,720
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/AIRI-Institute/doped_CsPbI3_energetics" target="_blank">
     https://github.com/AIRI-Institute/doped_CsPbI3_energetics
    </a>
    <br/>
    <a href="https://doi.org/10.1016/j.commatsci.2023.112672" target="_blank">
     https://doi.org/10.1016/j.commatsci.2023.112672
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_swl40hdnt479_0" target="_blank">
     electrode_materials_for_ca-based_rechargeable_batteries
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset for "Analysis of minerals as electrode materials for Ca-based rechargeable batteries". Includes DFT structures of pyroxenes, garnet and carbonates. Dataset was produced to pursue identification of Ca-based high specific energy cathode materials.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    electrode_materials_for_ca-based_rechargeable_batteries__Dompablo-Casals__DS_swl40hdnt479_0
    <br/>
    <b>
     Name:
    </b>
    electrode_materials_for_ca-based_rechargeable_batteries
    <br/>
    <b>
     Authors:
    </b>
    M. Elena Arroyo-de Dompablo, Jose Luis Casals
    <br/>
    <b>
     Elements
    </b>
    : C, Ca, Cr, Mn, O, Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 4,726
    <br/>
    <b>
     Number of Elements
    </b>
    : 6
    <br/>
    <b>
     Number of Atoms
    </b>
    : 550,074
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:3n-e8" target="_blank">
     https://doi.org/10.24435/materialscloud:3n-e8
    </a>
    <br/>
    <a href="http://doi.org/10.1038/s41598-019-46002-4" target="_blank">
     http://doi.org/10.1038/s41598-019-46002-4
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_1t2xs8bzygtp_0" target="_blank">
     ferroelectricity_and_metallicity_in_BaTiO3_JMCC2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset for "Interplay between ferroelectricity and metallicity in BaTiO3", exploring properties of ferroelectric barium titanate (BaTiO3), including the effects of electron and hole doping. Includes configuration sets for unit cells and supercells of BaTiO3.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    ferroelectricity_and_metallicity_in_BaTiO3_JMCC2021__Michel-Esswein-Spaldin__DS_1t2xs8bzygtp_0
    <br/>
    <b>
     Name:
    </b>
    ferroelectricity_and_metallicity_in_BaTiO3_JMCC2021
    <br/>
    <b>
     Authors:
    </b>
    Veronica F. Michel, Tobias Esswein, Nicola A. Spaldin
    <br/>
    <b>
     Elements
    </b>
    : Al, Ba, K, La, Nb, O, Sc, Ti, V
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,127
    <br/>
    <b>
     Number of Elements
    </b>
    : 9
    <br/>
    <b>
     Number of Atoms
    </b>
    : 19,125
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:f4-94" target="_blank">
     https://doi.org/10.24435/materialscloud:f4-94
    </a>
    <br/>
    <a href="http://doi.org/10.1039/D1TC01868J" target="_blank">
     http://doi.org/10.1039/D1TC01868J
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_i23sbm1o45sj_0" target="_blank">
     flexible_molecules_JCP2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of azobenzene featuring a cis to trans thermal inversion through three channels: inversion, rotation, and rotation assisted by inversion; and configurations of glycine as a simpler comparison molecule. All calculations were performed in FHI-aims software using the Perdew-Burke-Ernzerhof (PBE) exchange-correlation functional with the Tkatchenko-Scheffler (TS) method to account for van der Waals (vdW) interactions. The azobenzene sets contain calculations from several different MD simulations, including two long simulations initialized at 300 K; short simulations (300 steps) initialized at 300 K and shorter (.5fs) timestep; four simulations, two starting from each of cis and trans isomer, at 750 K (initialized at 3000 K); and simulations at 50 K (initialized at 300 K). The glycine isomerization set was built using one MD simulation starting from each of two different minima. Initializatin and simulation temperature were 500 K.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    flexible_molecules_JCP2021__Vassilev-Galindo-Fonseca-Poltavsky-Tkatchenko__DS_i23sbm1o45sj_0
    <br/>
    <b>
     Name:
    </b>
    flexible_molecules_JCP2021
    <br/>
    <b>
     Authors:
    </b>
    Valentin Vassilev-Galindo, Gregory Fonseca, Igor Poltavsky, Alexandre Tkatchenko
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 69,182
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,520,340
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.1063/5.0038516" target="_blank">
     https://doi.org/10.1063/5.0038516
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0038516" target="_blank">
     https://doi.org/10.1063/5.0038516
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_cpznjcu51bvg_0" target="_blank">
     glass-ceramic_lithium_thiophosphate_electrolytes_
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This database contains computationally generated atomic structures of glass-ceramics lithium thiophosphates (gc-LPS) with the general composition (Li2S)x(P2S5)1-x. Total energies and interatomic forces from density-functional theory (DFT) calculations are included. The DFT calculations used projector-augmented-wave (PAW) pseudopotentials and the Perdew-Burke-Ernzerhof (PBE) exchange-correlation functional as implemented in the Vienna Ab Initio Simulation Package (VASP) and a kinetic energy cutoff of 520 eV. The first Brillouin zone was sampled using VASP's fully automatic k-point scheme with a length parameter Rk = 25Å. The gc-LPS structures were generated using a combination of different sampling methods. Initial amorphous structure models were generated with ab initio molecular dynamics (AIMD) simulations of supercells at 1200 K using a Nose-Hoover thermostat with a time step of 1 fs. To obtain near-ground-state structures as reference for the machine-learning potential, 150 evenly spaced snapshots were extracted from the AIMD trajectories that were reoptimized with DFT geometry optimizations at zero Kelvin. Additional structures were generated by scaling the lattice parameters of the crystalline LPS structures (see below) by ±15% and perturbing atomic positions in AIMD simulations as described above.The resulting database was used to train a specialized ANN potential for the sampling of structures along the Li2S-P2S5 composition line with a genetic-algorithm (GA) as implemented in the atomistic evolution (ævo) package, following a previously reported protocol. Starting from supercells of the ideal crystal structures, either Li and S atoms were removed with a ratio of 2:1, or P and S atoms were removed with a ratio of 2:5, and low-energy configurations were determined with GA sampling. A population size of 32 trials and a mutation rate of 10% were employed. The ANN potential was iteratively refined by including additional sampled structures in the training. For each composition, at least 10 lowest energy structure models identified with the ANN-GA approach were selected and fully relaxed with DFT.Also included in the present database are the XSF files of the previously reported crystalline phases LiPS3, Li2PS3, Li4P2S7, Li7P3S11, α-Li3PS4, β-Li3PS4, γ-Li3PS4, and Li48P16S61. The crystal structures were obtained from the Inorganic Crystal Structure Database (ICSD). the Materials Project (MP) database, the Open Quantum Materials Database (OQMD), and the AFLOW database. The configuration names indicate the journal reference and the database.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    glass-ceramic_lithium_thiophosphate_electrolytes___Guo-Artrith__DS_cpznjcu51bvg_0
    <br/>
    <b>
     Name:
    </b>
    glass-ceramic_lithium_thiophosphate_electrolytes_
    <br/>
    <b>
     Authors:
    </b>
    Haoyue Guo, Nongnuch Artrith
    <br/>
    <b>
     Elements
    </b>
    : Li, P, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 6,055
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 264,604
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:j5-tz" target="_blank">
     https://doi.org/10.24435/materialscloud:j5-tz
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.chemmater.2c00267" target="_blank">
     https://doi.org/10.1021/acs.chemmater.2c00267
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_82x5bfiiyaij_0" target="_blank">
     linear_magnetic_coefficient_in_Cr2O3_JPCM2024
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    We establish the sign of the linear magnetoelectric (ME) coefficient, α, in chromia, Cr₂O₃. Cr₂O₃ is the prototypical linear ME material, in which an electric (magnetic) field induces a linearly proportional magnetization (polarization), and a single magnetic domain can be selected by annealing in combined magnetic (H) and electric (E) fields. Opposite antiferromagnetic domains have opposite ME responses, and which antiferromagnetic domain corresponds to which sign of response has previously been unclear. We use density functional theory (DFT) to calculate the magnetic response of a single antiferromagnetic domain of Cr₂O₃ to an applied in-plane electric field at 0 K. We find that the domain with nearest neighbor magnetic moments oriented away from (towards) each other has a negative (positive) in-plane ME coefficient, α⊥, at 0 K. We show that this sign is consistent with all other DFT calculations in the literature that specified the domain orientation, independent of the choice of DFT code or functional, the method used to apply the field, and whether the direct (magnetic field) or inverse (electric field) ME response was calculated. Next, we reanalyze our previously published spherical neutron polarimetry data to determine the antiferromagnetic domain produced by annealing in combined E and H fields oriented along the crystallographic symmetry axis at room temperature. We find that the antiferromagnetic domain with nearest-neighbor magnetic moments oriented away from (towards) each other is produced by annealing in (anti-)parallel E and H fields, corresponding to a positive (negative) axial ME coefficient, α∥, at room temperature. Since α⊥ at 0 K and α∥ at room temperature are known to be of opposite sign, our computational and experimental results are consistent. This dataset contains the input data to reproduce the calculation of the magnetoelectric effect as plotted in Fig. 3 of the manuscript, for Elk, Vasp, and Quantum Espresso.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    linear_magnetic_coefficient_in_Cr2O3_JPCM2024__Bousquet-Lelièvre-Berna-Qureshi-Soh-Spaldin-Urru-Verbeek-Weber__DS_82x5bfiiyaij_0
    <br/>
    <b>
     Name:
    </b>
    linear_magnetic_coefficient_in_Cr2O3_JPCM2024
    <br/>
    <b>
     Authors:
    </b>
    Eric Bousquet, Eddy Lelièvre-Berna, Navid Qureshi, Jian-Rui Soh, Nicola Ann Spaldin, Andrea Urru, Xanthe Henderike Verbeek, Sophie Francis Weber
    <br/>
    <b>
     Elements
    </b>
    : Cr, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 165
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,650
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:ek-fp" target="_blank">
     https://doi.org/10.24435/materialscloud:ek-fp
    </a>
    <br/>
    <a href="http://doi.org/10.1088/1361-648X/ad1a59" target="_blank">
     http://doi.org/10.1088/1361-648X/ad1a59
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_t8goqf2uglhj_0" target="_blank">
     local_polarization_in_oxygen-deficient_LaMnO3_PRR2020
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset contains structural calculations of LaMnO3 carried out in Quantum ESPRESSO at the DFT-PBEsol+U level of theory. The dataset was built to explore strained and stoichiometric and oxygen-deficient LaMnO3.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    local_polarization_in_oxygen-deficient_LaMnO3_PRR2020__Ricca-Niederhauser-Aschauer__DS_t8goqf2uglhj_0
    <br/>
    <b>
     Name:
    </b>
    local_polarization_in_oxygen-deficient_LaMnO3_PRR2020
    <br/>
    <b>
     Authors:
    </b>
    Chiara Ricca, Nicolas Niederhauser, Ulrich Aschauer
    <br/>
    <b>
     Elements
    </b>
    : Ba, La, Mn, O, Ti
    <br/>
    <b>
     Number of Configurations
    </b>
    : 4,514
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 174,337
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:m9-9d" target="_blank">
     https://doi.org/10.24435/materialscloud:m9-9d
    </a>
    <br/>
    <a href="http://doi.org/10.1103/PhysRevResearch.2.042040" target="_blank">
     http://doi.org/10.1103/PhysRevResearch.2.042040
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_e94my2wrh074_0" target="_blank">
     mbGDML_maldonado_2023
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Configurations of water, acetonitrile and methanol, simulated with ASE and modeled using a variety of software and methods: GAP, SchNet, GDML, ORCA and mbGDML. Forces and potential energy included; metadata includes kinetic energy and velocities.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    mbGDML_maldonado_2023__Maldonado__DS_e94my2wrh074_0
    <br/>
    <b>
     Name:
    </b>
    mbGDML_maldonado_2023
    <br/>
    <b>
     Authors:
    </b>
    Alex M. Maldonado
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 24,543
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 712,134
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.7112197" target="_blank">
     https://doi.org/10.5281/zenodo.7112197
    </a>
    <br/>
    <a href="https://doi.org/10.26434/chemrxiv-2023-wdd1r" target="_blank">
     https://doi.org/10.26434/chemrxiv-2023-wdd1r
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_xmgn3ofqzon9_0" target="_blank">
     mlearn_Cu_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Cu configurations
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    mlearn_Cu_test__Zuo-Chen-Li-Deng-Chen-Behler-Csányi-Shapeev-Thompson-Wood-Ong__DS_xmgn3ofqzon9_0
    <br/>
    <b>
     Name:
    </b>
    mlearn_Cu_test
    <br/>
    <b>
     Authors:
    </b>
    Yunxing Zuo, Chi Chen, Xiangguo Li, Zhi Deng, Yiming Chen, Jörg Behler, Gábor Csányi, Alexander V. Shapeev, Aidan P. Thompson, Mitchell A. Wood, Shyue Ping Ong
    <br/>
    <b>
     Elements
    </b>
    : Cu
    <br/>
    <b>
     Number of Configurations
    </b>
    : 31
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,178
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/materialsvirtuallab/mlearn" target="_blank">
     https://github.com/materialsvirtuallab/mlearn
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpca.9b08723" target="_blank">
     https://doi.org/10.1021/acs.jpca.9b08723
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_3pv3hck35iy6_0" target="_blank">
     mlearn_Cu_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Cu configurations
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    mlearn_Cu_train__Zuo-Chen-Li-Deng-Chen-Behler-Csányi-Shapeev-Thompson-Wood-Ong__DS_3pv3hck35iy6_0
    <br/>
    <b>
     Name:
    </b>
    mlearn_Cu_train
    <br/>
    <b>
     Authors:
    </b>
    Yunxing Zuo, Chi Chen, Xiangguo Li, Zhi Deng, Yiming Chen, Jörg Behler, Gábor Csányi, Alexander V. Shapeev, Aidan P. Thompson, Mitchell A. Wood, Shyue Ping Ong
    <br/>
    <b>
     Elements
    </b>
    : Cu
    <br/>
    <b>
     Number of Configurations
    </b>
    : 262
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 27,416
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/materialsvirtuallab/mlearn/tree/master/data" target="_blank">
     https://github.com/materialsvirtuallab/mlearn/tree/master/data
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpca.9b08723" target="_blank">
     https://doi.org/10.1021/acs.jpca.9b08723
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_pyrk84w3auvb_0" target="_blank">
     mlearn_Ge_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Ge configurations
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    mlearn_Ge_test__Zuo-Chen-Li-Deng-Chen-Behler-Csányi-Shapeev-Thompson-Wood-Ong__DS_pyrk84w3auvb_0
    <br/>
    <b>
     Name:
    </b>
    mlearn_Ge_test
    <br/>
    <b>
     Authors:
    </b>
    Yunxing Zuo, Chi Chen, Xiangguo Li, Zhi Deng, Yiming Chen, Jörg Behler, Gábor Csányi, Alexander V. Shapeev, Aidan P. Thompson, Mitchell A. Wood, Shyue Ping Ong
    <br/>
    <b>
     Elements
    </b>
    : Ge
    <br/>
    <b>
     Number of Configurations
    </b>
    : 25
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,568
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/materialsvirtuallab/mlearn" target="_blank">
     https://github.com/materialsvirtuallab/mlearn
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpca.9b08723" target="_blank">
     https://doi.org/10.1021/acs.jpca.9b08723
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_ot3m0rxle8fs_0" target="_blank">
     mlearn_Ge_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Ge configurations
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    mlearn_Ge_train__Zuo-Chen-Li-Deng-Chen-Behler-Csányi-Shapeev-Thompson-Wood-Ong__DS_ot3m0rxle8fs_0
    <br/>
    <b>
     Name:
    </b>
    mlearn_Ge_train
    <br/>
    <b>
     Authors:
    </b>
    Yunxing Zuo, Chi Chen, Xiangguo Li, Zhi Deng, Yiming Chen, Jörg Behler, Gábor Csányi, Alexander V. Shapeev, Aidan P. Thompson, Mitchell A. Wood, Shyue Ping Ong
    <br/>
    <b>
     Elements
    </b>
    : Ge
    <br/>
    <b>
     Number of Configurations
    </b>
    : 228
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 14,072
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/materialsvirtuallab/mlearn/tree/master/data" target="_blank">
     https://github.com/materialsvirtuallab/mlearn/tree/master/data
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpca.9b08723" target="_blank">
     https://doi.org/10.1021/acs.jpca.9b08723
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_jf20mvwao5xi_0" target="_blank">
     mlearn_Li_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Li configurations
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    mlearn_Li_test__Zuo-Chen-Li-Deng-Chen-Behler-Csányi-Shapeev-Thompson-Wood-Ong__DS_jf20mvwao5xi_0
    <br/>
    <b>
     Name:
    </b>
    mlearn_Li_test
    <br/>
    <b>
     Authors:
    </b>
    Yunxing Zuo, Chi Chen, Xiangguo Li, Zhi Deng, Yiming Chen, Jörg Behler, Gábor Csányi, Alexander V. Shapeev, Aidan P. Thompson, Mitchell A. Wood, Shyue Ping Ong
    <br/>
    <b>
     Elements
    </b>
    : Li
    <br/>
    <b>
     Number of Configurations
    </b>
    : 29
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,320
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/materialsvirtuallab/mlearn" target="_blank">
     https://github.com/materialsvirtuallab/mlearn
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpca.9b08723" target="_blank">
     https://doi.org/10.1021/acs.jpca.9b08723
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_h7zao9ya9sd8_0" target="_blank">
     mlearn_Li_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Li configurations
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    mlearn_Li_train__Zuo-Chen-Li-Deng-Chen-Behler-Csányi-Shapeev-Thompson-Wood-Ong__DS_h7zao9ya9sd8_0
    <br/>
    <b>
     Name:
    </b>
    mlearn_Li_train
    <br/>
    <b>
     Authors:
    </b>
    Yunxing Zuo, Chi Chen, Xiangguo Li, Zhi Deng, Yiming Chen, Jörg Behler, Gábor Csányi, Alexander V. Shapeev, Aidan P. Thompson, Mitchell A. Wood, Shyue Ping Ong
    <br/>
    <b>
     Elements
    </b>
    : Li
    <br/>
    <b>
     Number of Configurations
    </b>
    : 241
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 11,576
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/materialsvirtuallab/mlearn/tree/master/data" target="_blank">
     https://github.com/materialsvirtuallab/mlearn/tree/master/data
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpca.9b08723" target="_blank">
     https://doi.org/10.1021/acs.jpca.9b08723
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_l0b6iq3no012_0" target="_blank">
     mlearn_Mo_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Mo configurations
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    mlearn_Mo_test__Zuo-Chen-Li-Deng-Chen-Behler-Csányi-Shapeev-Thompson-Wood-Ong__DS_l0b6iq3no012_0
    <br/>
    <b>
     Name:
    </b>
    mlearn_Mo_test
    <br/>
    <b>
     Authors:
    </b>
    Yunxing Zuo, Chi Chen, Xiangguo Li, Zhi Deng, Yiming Chen, Jörg Behler, Gábor Csányi, Alexander V. Shapeev, Aidan P. Thompson, Mitchell A. Wood, Shyue Ping Ong
    <br/>
    <b>
     Elements
    </b>
    : Mo
    <br/>
    <b>
     Number of Configurations
    </b>
    : 23
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,189
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/materialsvirtuallab/mlearn" target="_blank">
     https://github.com/materialsvirtuallab/mlearn
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpca.9b08723" target="_blank">
     https://doi.org/10.1021/acs.jpca.9b08723
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_ytoet4uyc32k_0" target="_blank">
     mlearn_Mo_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Mo configurations
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    mlearn_Mo_train__Zuo-Chen-Li-Deng-Chen-Behler-Csányi-Shapeev-Thompson-Wood-Ong__DS_ytoet4uyc32k_0
    <br/>
    <b>
     Name:
    </b>
    mlearn_Mo_train
    <br/>
    <b>
     Authors:
    </b>
    Yunxing Zuo, Chi Chen, Xiangguo Li, Zhi Deng, Yiming Chen, Jörg Behler, Gábor Csányi, Alexander V. Shapeev, Aidan P. Thompson, Mitchell A. Wood, Shyue Ping Ong
    <br/>
    <b>
     Elements
    </b>
    : Mo
    <br/>
    <b>
     Number of Configurations
    </b>
    : 194
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 10,087
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/materialsvirtuallab/mlearn/tree/master/data" target="_blank">
     https://github.com/materialsvirtuallab/mlearn/tree/master/data
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpca.9b08723" target="_blank">
     https://doi.org/10.1021/acs.jpca.9b08723
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_zjkz9664bapl_0" target="_blank">
     mlearn_Ni_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Ni configurations
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    mlearn_Ni_test__Zuo-Chen-Li-Deng-Chen-Behler-Csányi-Shapeev-Thompson-Wood-Ong__DS_zjkz9664bapl_0
    <br/>
    <b>
     Name:
    </b>
    mlearn_Ni_test
    <br/>
    <b>
     Authors:
    </b>
    Yunxing Zuo, Chi Chen, Xiangguo Li, Zhi Deng, Yiming Chen, Jörg Behler, Gábor Csányi, Alexander V. Shapeev, Aidan P. Thompson, Mitchell A. Wood, Shyue Ping Ong
    <br/>
    <b>
     Elements
    </b>
    : Ni
    <br/>
    <b>
     Number of Configurations
    </b>
    : 31
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 3,158
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/materialsvirtuallab/mlearn" target="_blank">
     https://github.com/materialsvirtuallab/mlearn
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpca.9b08723" target="_blank">
     https://doi.org/10.1021/acs.jpca.9b08723
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_lfyd4jv627cr_0" target="_blank">
     mlearn_Ni_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Ni configurations
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    mlearn_Ni_train__Zuo-Chen-Li-Deng-Chen-Behler-Csányi-Shapeev-Thompson-Wood-Ong__DS_lfyd4jv627cr_0
    <br/>
    <b>
     Name:
    </b>
    mlearn_Ni_train
    <br/>
    <b>
     Authors:
    </b>
    Yunxing Zuo, Chi Chen, Xiangguo Li, Zhi Deng, Yiming Chen, Jörg Behler, Gábor Csányi, Alexander V. Shapeev, Aidan P. Thompson, Mitchell A. Wood, Shyue Ping Ong
    <br/>
    <b>
     Elements
    </b>
    : Ni
    <br/>
    <b>
     Number of Configurations
    </b>
    : 263
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 27,420
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/materialsvirtuallab/mlearn/tree/master/data" target="_blank">
     https://github.com/materialsvirtuallab/mlearn/tree/master/data
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpca.9b08723" target="_blank">
     https://doi.org/10.1021/acs.jpca.9b08723
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_lvjng6g41bwy_0" target="_blank">
     mlearn_Si_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Si configurations
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    mlearn_Si_test__Zuo-Chen-Li-Deng-Chen-Behler-Csányi-Shapeev-Thompson-Wood-Ong__DS_lvjng6g41bwy_0
    <br/>
    <b>
     Name:
    </b>
    mlearn_Si_test
    <br/>
    <b>
     Authors:
    </b>
    Yunxing Zuo, Chi Chen, Xiangguo Li, Zhi Deng, Yiming Chen, Jörg Behler, Gábor Csányi, Alexander V. Shapeev, Aidan P. Thompson, Mitchell A. Wood, Shyue Ping Ong
    <br/>
    <b>
     Elements
    </b>
    : Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 25
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,525
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/materialsvirtuallab/mlearn" target="_blank">
     https://github.com/materialsvirtuallab/mlearn
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpca.9b08723" target="_blank">
     https://doi.org/10.1021/acs.jpca.9b08723
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_eltotrjoqonr_0" target="_blank">
     mlearn_Si_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A comprehensive DFT data set was generated for six elements - Li, Mo, Ni, Cu, Si, and Ge. These elements were chosen to span a variety of chemistries (main group metal, transition metal, and semiconductor), crystal structures (bcc, fcc, and diamond) and bonding types (metallic and covalent). This dataset comprises only the Si configurations
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    mlearn_Si_train__Zuo-Chen-Li-Deng-Chen-Behler-Csányi-Shapeev-Thompson-Wood-Ong__DS_eltotrjoqonr_0
    <br/>
    <b>
     Name:
    </b>
    mlearn_Si_train
    <br/>
    <b>
     Authors:
    </b>
    Yunxing Zuo, Chi Chen, Xiangguo Li, Zhi Deng, Yiming Chen, Jörg Behler, Gábor Csányi, Alexander V. Shapeev, Aidan P. Thompson, Mitchell A. Wood, Shyue Ping Ong
    <br/>
    <b>
     Elements
    </b>
    : Si
    <br/>
    <b>
     Number of Configurations
    </b>
    : 214
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 13,233
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/materialsvirtuallab/mlearn/tree/master/data" target="_blank">
     https://github.com/materialsvirtuallab/mlearn/tree/master/data
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpca.9b08723" target="_blank">
     https://doi.org/10.1021/acs.jpca.9b08723
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_ots0ul69q842_0" target="_blank">
     oxygen-vacancy_defects_in_Cu2O(111)
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset investigates the effect of defects, such as copper and oxygen vacancies, in cuprous oxide films. Structures include oxygen vacancies formed in proximity of a reconstructed Cu2O(111) surface, where the outermost unsaturated copper atoms are removed, thus forming non-stoichiometric surface layers with copper vacancies. Surface and bulk properties are addressed by modelling a thick and symmetric slab consisting of 8 atomic layers and 736 atoms. Configuration sets include bulk, slab, vacancy and oxygen gas. Version v1
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    oxygen-vacancy_defects_in_Cu2O(111)__Dongfang-Iannuzzi-Al-Hamdani__DS_ots0ul69q842_0
    <br/>
    <b>
     Name:
    </b>
    oxygen-vacancy_defects_in_Cu2O(111)
    <br/>
    <b>
     Authors:
    </b>
    Nanchen Dongfang, Marcella Iannuzzi, Yasmine Al-Hamdani
    <br/>
    <b>
     Elements
    </b>
    : Cu, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 864
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 606,270
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:3z-bk" target="_blank">
     https://doi.org/10.24435/materialscloud:3z-bk
    </a>
    <br/>
    <a href="http://doi.org/10.1088/2516-1075/ace0aa" target="_blank">
     http://doi.org/10.1088/2516-1075/ace0aa
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_48yn17885mdi_0" target="_blank">
     pure_magnesium_DFT_PRM2020
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset provides DFT (as implemented in VASP) calculations for pure magnesium. Configuration sets include bulk, generalized stacking fault energies, stable stacking fault, decohesion, relaxed surfaces, dimer, corner and rod, and vacancy configurations of Mg.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    pure_magnesium_DFT_PRM2020__Yin-Stricker-Curtin__DS_48yn17885mdi_0
    <br/>
    <b>
     Name:
    </b>
    pure_magnesium_DFT_PRM2020
    <br/>
    <b>
     Authors:
    </b>
    Binglun Yin, Markus Stricker, W. A. Curtin
    <br/>
    <b>
     Elements
    </b>
    : Mg
    <br/>
    <b>
     Number of Configurations
    </b>
    : 405
    <br/>
    <b>
     Number of Elements
    </b>
    : 1
    <br/>
    <b>
     Number of Atoms
    </b>
    : 10,730
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:8f-1s" target="_blank">
     https://doi.org/10.24435/materialscloud:8f-1s
    </a>
    <br/>
    <a href="https://doi.org/10.1103/PhysRevMaterials.4.103602" target="_blank">
     https://doi.org/10.1103/PhysRevMaterials.4.103602
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_p9qjlc9l8scx_0" target="_blank">
     q-AQUA
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The a-AQUA dataset was generated to address the need for a training set for a water PES that includes 2-body, 3-body and 4-body interactions calculated at the CCSD(T) level of theory. Structures were selected from the existing HBB2-pol and MB-pol datasets. For each water dimer structure, CCSD(T)/aug-cc-pVTZ calculations were performed with an additional 3s3p2d1f basis set; exponents equal to (0.9, 0.3, 0.1) for sp, (0.6, 0.2) for d, and 0.3 for f. This additional basis is placed at the center of mass (COM) of each dimer configuration. The basis set superposition error (BSSE) correction was determined with the counterpoise scheme. CCSD(T)/aug-cc-pVQZ calculations were then performed with the same additional basis set and BSSE correction. Final CCSD(T)/CBS energies were obtained by extrapolation over the CCSD(T)/aug-cc-pVTZ and CCSD(T)/aug-cc-pVQZ 2-b energies. All ab initio calculations were performed using Molpro package.Trimer structures were calculated at CCSD(T)-F12a/aug-cc-pVTZ with BSSE correction. Four-body structure calculations were performed at CCSD(T)-F12 level.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    q-AQUA__Yu-Qu-Houston-Conte-Nandi-Bowman__DS_p9qjlc9l8scx_0
    <br/>
    <b>
     Name:
    </b>
    q-AQUA
    <br/>
    <b>
     Authors:
    </b>
    Qi Yu, Chen Qu, Paul L. Houston, Riccardo Conte, Apurba Nandi, Joel M. Bowman
    <br/>
    <b>
     Elements
    </b>
    : H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 120,372
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 878,157
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/jmbowma/q-AQUA" target="_blank">
     https://github.com/jmbowma/q-AQUA
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jpclett.2c00966" target="_blank">
     https://doi.org/10.1021/acs.jpclett.2c00966
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_8rafgy0ly6bt_0" target="_blank">
     rMD17
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    A dataset of 10 molecules (aspirin, azobenzene, benzene, ethanol, malonaldehyde, naphthalene, paracetamol, salicylic, toluene, uracil) with 100,000 structures calculated for each at the PBE/def2-SVP level of theory using ORCA. Based on the MD17 dataset, but with refined measurements.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    rMD17__Christensen-Lilienfeld__DS_8rafgy0ly6bt_0
    <br/>
    <b>
     Name:
    </b>
    rMD17
    <br/>
    <b>
     Authors:
    </b>
    Anders S. Christensen, O. Anatole von Lilienfeld
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 999,988
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 15,599,712
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.6084/m9.figshare.12672038.v3" target="_blank">
     https://doi.org/10.6084/m9.figshare.12672038.v3
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2007.09593" target="_blank">
     https://doi.org/10.48550/arXiv.2007.09593
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_apdpxdjx082p_0" target="_blank">
     reactive_hydrogen_ACS_2023
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset contains structures of Cu, including Cu(111), Cu(100), Cu(110), and Cu(211). Slab settings are as follows: 3 x 3, 6-layered slabs for Cu(111), (100), and (110) surfaces; 1 x 3, 6-layered slabs for Cu(211) surface. Includes some structures representing interation of H2 with one of the Cu surfaces and some structures of Cu sampled at different temperatures.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    reactive_hydrogen_ACS_2023__Stark-Westermayr-Douglas-Gallardo-Gardner-Habershon-Maurer__DS_apdpxdjx082p_0
    <br/>
    <b>
     Name:
    </b>
    reactive_hydrogen_ACS_2023
    <br/>
    <b>
     Authors:
    </b>
    Wojciech G. Stark, Julia Westermayr, Oscar A. Douglas-Gallardo, James Gardner, Scott Habershon, Reinhard J. Maurer
    <br/>
    <b>
     Elements
    </b>
    : Cu, H
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,413
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 191,104
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://dx.doi.org/10.17172/NOMAD/2023.05.03-2" target="_blank">
     https://dx.doi.org/10.17172/NOMAD/2023.05.03-2
    </a>
    <br/>
    <a href="https://pubs.acs.org/doi/full/10.1021/acs.jpcc.3c06648" target="_blank">
     https://pubs.acs.org/doi/full/10.1021/acs.jpcc.3c06648
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_u7fu1meyxvpc_0" target="_blank">
     reduced-perovskite_and_oxidized-marokite_oxides
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset contains DFT calculations of oxygen-deficient perovskites from the Ca2Fe2O5-brownmillerite and Ca2Mn2O5 structures; and tunnel CaMn4O8, a derivative of the CaMn2O4-marokite with Ca vacancies. The dataset was produced to investigate the effects of oxygen introduction or Ca vacancy introduction in ternary transition metal oxides, as a means to assess potential new Ca-ion battery materials.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    reduced-perovskite_and_oxidized-marokite_oxides__Dompablo-Casals__DS_u7fu1meyxvpc_0
    <br/>
    <b>
     Name:
    </b>
    reduced-perovskite_and_oxidized-marokite_oxides
    <br/>
    <b>
     Authors:
    </b>
    M. Elena Arroyo-de Dompablo, José Luis Casals
    <br/>
    <b>
     Elements
    </b>
    : Ca, Fe, Mn, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,919
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 387,258
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:x9-qr" target="_blank">
     https://doi.org/10.24435/materialscloud:x9-qr
    </a>
    <br/>
    <a href="http://doi.org/10.1016/j.ensm.2019.06.002" target="_blank">
     http://doi.org/10.1016/j.ensm.2019.06.002
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_2oy3ipc8qxa7_0" target="_blank">
     sAlex_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The validation split of sAlex. sAlex is a subsample of the Alexandria dataset that was used to fine tune the OMat24 (Open Materials 2024) models. From the site: sAlex was created by removing structures matched in WBM and only sampling structure along a trajectory with an energy difference greater than 10 meV/atom.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    sAlex_validation__Barroso-Luque-Shuaibi-Fu-Wood-Dzamba-Gao-Rizvi-Zitnick-Ulissi__DS_2oy3ipc8qxa7_0
    <br/>
    <b>
     Name:
    </b>
    sAlex_validation
    <br/>
    <b>
     Authors:
    </b>
    Luis Barroso-Luque, Muhammed Shuaibi, Xiang Fu, Brandon M. Wood, Misko Dzamba, Meng Gao, Ammar Rizvi, C. Lawrence Zitnick, Zachary W. Ulissi
    <br/>
    <b>
     Elements
    </b>
    : Ac, Ag, Al, As, Au, B, Ba, Be, Bi, Br, C, Ca, Cd, Ce, Cl, Co, Cr, Cs, Cu, Dy, Er, Eu, F, Fe, Ga, Gd, Ge, H, He, Hf, Hg, Ho, I, In, Ir, K, La, Li, Lu, Mg, Mn, Mo, N, Na, Nb, Nd, Ni, Np, O, Os, P, Pa, Pb, Pd, Pm, Pr, Pt, Pu, Rb, Re, Rh, Ru, S, Sb, Sc, Se, Si, Sm, Sn, Sr, Ta, Tb, Tc, Te, Th, Ti, Tl, Tm, U, V, W, Xe, Y, Yb, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 547,885
    <br/>
    <b>
     Number of Elements
    </b>
    : 86
    <br/>
    <b>
     Number of Atoms
    </b>
    : 5,670,890
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://fair-chem.github.io/core/datasets/omat24.html" target="_blank">
     https://fair-chem.github.io/core/datasets/omat24.html
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2410.12771" target="_blank">
     https://doi.org/10.48550/arXiv.2410.12771
    </a>
    <br/>
    <a href="https://doi.org/10.1002/adma.202210788" target="_blank">
     https://doi.org/10.1002/adma.202210788
    </a>
    <br/>
    <a href="https://huggingface.co/datasets/fairchem/OMAT24" target="_blank">
     https://huggingface.co/datasets/fairchem/OMAT24
    </a>
    <br/>
    <a href="https://alexandria.icams.rub.de/" target="_blank">
     https://alexandria.icams.rub.de/
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_4e7a9g2kav0a_0" target="_blank">
     sGDML_Aspirin_ccsd_NC2018_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The test set of a train/test pair from the aspirin dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running abinitio MD in the NVT ensemble using the Nosé-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated by all-electron coupled cluster with single, double and perturbative triple excitations (CCSD(T)). The Dunning correlation-consistent basis set CCSD/cc-pVDZ was used for aspirin. All calculations were performed with the Psi4 software suite.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    sGDML_Aspirin_ccsd_NC2018_test__Chmiela-Sauceda-Müller-Tkatchenko__DS_4e7a9g2kav0a_0
    <br/>
    <b>
     Name:
    </b>
    sGDML_Aspirin_ccsd_NC2018_test
    <br/>
    <b>
     Authors:
    </b>
    Stefan Chmiela, Huziel E. Sauceda, Klaus-Robert Müller, Alexandre Tkatchenko
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 500
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 10,500
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://sgdml.org/" target="_blank">
     http://sgdml.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-018-06169-2" target="_blank">
     https://doi.org/10.1038/s41467-018-06169-2
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_xy48avqcknnk_0" target="_blank">
     sGDML_Aspirin_ccsd_NC2018_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The train set of a train/test pair from the aspirin dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running abinitio MD in the NVT ensemble using the Nosé-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated by all-electron coupled cluster with single, double and perturbative triple excitations (CCSD(T)). The Dunning correlation-consistent basis set CCSD/cc-pVDZ was used for aspirin. All calculations were performed with the Psi4 software suite.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    sGDML_Aspirin_ccsd_NC2018_train__Chmiela-Sauceda-Müller-Tkatchenko__DS_xy48avqcknnk_0
    <br/>
    <b>
     Name:
    </b>
    sGDML_Aspirin_ccsd_NC2018_train
    <br/>
    <b>
     Authors:
    </b>
    Stefan Chmiela, Huziel E. Sauceda, Klaus-Robert Müller, Alexandre Tkatchenko
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 21,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://sgdml.org/" target="_blank">
     http://sgdml.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-018-06169-2" target="_blank">
     https://doi.org/10.1038/s41467-018-06169-2
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_q9y1aat05u42_0" target="_blank">
     sGDML_Benzene_DFT_NC2018
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The data used for training the DFT models were created running ab initio MD in the NVT ensemble using the Nosé-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Forces and energies were computed using all-electrons at the generalized gradient approximation level of theory with the Perdew-Burke-Ernzerhof (PBE) exchange-correlation functional, treating van der Waals interactions with the Tkatchenko-Scheffler (TS) method. All calculations were performed with FHI-aims. The final training data was generated by subsampling the full trajectory under preservation of the Maxwell-Boltzmann distribution for the energies.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    sGDML_Benzene_DFT_NC2018__Chmiela-Sauceda-Müller-Tkatchenko__DS_q9y1aat05u42_0
    <br/>
    <b>
     Name:
    </b>
    sGDML_Benzene_DFT_NC2018
    <br/>
    <b>
     Authors:
    </b>
    Stefan Chmiela, Huziel E. Sauceda, Klaus-Robert Müller, Alexandre Tkatchenko
    <br/>
    <b>
     Elements
    </b>
    : C, H
    <br/>
    <b>
     Number of Configurations
    </b>
    : 49,863
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 598,356
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://sgdml.org/" target="_blank">
     http://sgdml.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1126/sciadv.1603015" target="_blank">
     https://doi.org/10.1126/sciadv.1603015
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-018-06169-2" target="_blank">
     https://doi.org/10.1038/s41467-018-06169-2
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_jol8dvjej92n_0" target="_blank">
     sGDML_Benzene_ccsdt_NC2018_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The test set of a train/test pair from the benzene dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running ab initio MD in the NVT ensemble using the Nosé-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated using all-electron coupled cluster with single , double and perturbative triple excitations (CCSD(T)). The Dunning correlation-consistent basis set cc-pVDZ was used for benzene. All calculations were performed with the Psi4 software suite.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    sGDML_Benzene_ccsdt_NC2018_test__Chmiela-Sauceda-Müller-Tkatchenko__DS_jol8dvjej92n_0
    <br/>
    <b>
     Name:
    </b>
    sGDML_Benzene_ccsdt_NC2018_test
    <br/>
    <b>
     Authors:
    </b>
    Stefan Chmiela, Huziel E. Sauceda, Klaus-Robert Müller, Alexandre Tkatchenko
    <br/>
    <b>
     Elements
    </b>
    : C, H
    <br/>
    <b>
     Number of Configurations
    </b>
    : 500
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 6,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://sgdml.org/" target="_blank">
     http://sgdml.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-018-06169-2" target="_blank">
     https://doi.org/10.1038/s41467-018-06169-2
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_8xk0v5v9dbx0_0" target="_blank">
     sGDML_Benzene_ccsdt_NC2018_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The train set of a train/test pair from the benzene dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running ab initio MD in the NVT ensemble using the Nosé-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated using all-electron coupled cluster with single , double and perturbative triple excitations (CCSD(T)). The Dunning correlation-consistent basis set cc-pVDZ was used for benzene. All calculations were performed with the Psi4 software suite.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    sGDML_Benzene_ccsdt_NC2018_train__Chmiela-Sauceda-Müller-Tkatchenko__DS_8xk0v5v9dbx0_0
    <br/>
    <b>
     Name:
    </b>
    sGDML_Benzene_ccsdt_NC2018_train
    <br/>
    <b>
     Authors:
    </b>
    Stefan Chmiela, Huziel E. Sauceda, Klaus-Robert Müller, Alexandre Tkatchenko
    <br/>
    <b>
     Elements
    </b>
    : C, H
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 12,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://sgdml.org/" target="_blank">
     http://sgdml.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-018-06169-2" target="_blank">
     https://doi.org/10.1038/s41467-018-06169-2
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_iavvqpb14zqv_0" target="_blank">
     sGDML_Ethanol_ccsdt_NC2018_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The test set of a train/test pair from the ethanol dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running ab initio MD in the NVT ensemble using the Nosé-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated using all-electron coupled cluster with single, double and perturbative triple excitations (CCSD(T)).The Dunning correlation-consistent basis set cc-pVTZ was used for ethanol. All calculations were performed with the Psi4 software suite.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    sGDML_Ethanol_ccsdt_NC2018_test__Chmiela-Sauceda-Müller-Tkatchenko__DS_iavvqpb14zqv_0
    <br/>
    <b>
     Name:
    </b>
    sGDML_Ethanol_ccsdt_NC2018_test
    <br/>
    <b>
     Authors:
    </b>
    Stefan Chmiela, Huziel E. Sauceda, Klaus-Robert Müller, Alexandre Tkatchenko
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 9,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://sgdml.org/" target="_blank">
     http://sgdml.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-018-06169-2" target="_blank">
     https://doi.org/10.1038/s41467-018-06169-2
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_dalgmbg32lwz_0" target="_blank">
     sGDML_Ethanol_ccsdt_NC2018_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The train set of a train/test pair from the ethanol dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running ab initio MD in the NVT ensemble using the Nosé-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated using all-electron coupled cluster with single, double and perturbative triple excitations (CCSD(T)).The Dunning correlation-consistent basis set cc-pVTZ was used for ethanol. All calculations were performed with the Psi4 software suite.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    sGDML_Ethanol_ccsdt_NC2018_train__Chmiela-Sauceda-Müller-Tkatchenko__DS_dalgmbg32lwz_0
    <br/>
    <b>
     Name:
    </b>
    sGDML_Ethanol_ccsdt_NC2018_train
    <br/>
    <b>
     Authors:
    </b>
    Stefan Chmiela, Huziel E. Sauceda, Klaus-Robert Müller, Alexandre Tkatchenko
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 998
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 8,982
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://sgdml.org/" target="_blank">
     http://sgdml.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-018-06169-2" target="_blank">
     https://doi.org/10.1038/s41467-018-06169-2
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_yw3sqtaq40ll_0" target="_blank">
     sGDML_Malonaldehyde_ccsdt_NC2018_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The test set of a train/test pair from the malonaldehyde dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running ab initio MD in the NVT ensemble using the Nosé-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated using all-electron coupled cluster with single, double and perturbative triple excitations (CCSD(T)). The Dunning correlation-consistent basis set cc-pVDZ was used for malonaldehyde. All calculations were performed with the Psi4 software suite.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    sGDML_Malonaldehyde_ccsdt_NC2018_test__Chmiela-Sauceda-Müller-Tkatchenko__DS_yw3sqtaq40ll_0
    <br/>
    <b>
     Name:
    </b>
    sGDML_Malonaldehyde_ccsdt_NC2018_test
    <br/>
    <b>
     Authors:
    </b>
    Stefan Chmiela, Huziel E. Sauceda, Klaus-Robert Müller, Alexandre Tkatchenko
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 500
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 4,500
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://sgdml.org/" target="_blank">
     http://sgdml.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-018-06169-2" target="_blank">
     https://doi.org/10.1038/s41467-018-06169-2
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_201vhvywll83_0" target="_blank">
     sGDML_Malonaldehyde_ccsdt_NC2018_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The train set of a train/test pair from the malonaldehyde dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running ab initio MD in the NVT ensemble using the Nosé-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated using all-electron coupled cluster with single, double and perturbative triple excitations (CCSD(T)). The Dunning correlation-consistent basis set cc-pVDZ was used for malonaldehyde. All calculations were performed with the Psi4 software suite.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    sGDML_Malonaldehyde_ccsdt_NC2018_train__Chmiela-Sauceda-Müller-Tkatchenko__DS_201vhvywll83_0
    <br/>
    <b>
     Name:
    </b>
    sGDML_Malonaldehyde_ccsdt_NC2018_train
    <br/>
    <b>
     Authors:
    </b>
    Stefan Chmiela, Huziel E. Sauceda, Klaus-Robert Müller, Alexandre Tkatchenko
    <br/>
    <b>
     Elements
    </b>
    : C, H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 3
    <br/>
    <b>
     Number of Atoms
    </b>
    : 9,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://sgdml.org/" target="_blank">
     http://sgdml.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-018-06169-2" target="_blank">
     https://doi.org/10.1038/s41467-018-06169-2
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_ega58h2u9rwr_0" target="_blank">
     sGDML_Toluene_ccsdt_NC2018_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The test set of a train/test pair from the toluene dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running ab initio MD in the NVT ensemble using the Nosé-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated using all-electron coupled cluster with single, double and perturbative triple excitations (CCSD(T)). The Dunning correlation-consistent basis set cc-pVDZ was used for malonaldehyde. All calculations were performed with the Psi4 software suite.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    sGDML_Toluene_ccsdt_NC2018_test__Chmiela-Sauceda-Müller-Tkatchenko__DS_ega58h2u9rwr_0
    <br/>
    <b>
     Name:
    </b>
    sGDML_Toluene_ccsdt_NC2018_test
    <br/>
    <b>
     Authors:
    </b>
    Stefan Chmiela, Huziel E. Sauceda, Klaus-Robert Müller, Alexandre Tkatchenko
    <br/>
    <b>
     Elements
    </b>
    : C, H
    <br/>
    <b>
     Number of Configurations
    </b>
    : 501
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 7,515
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://sgdml.org/" target="_blank">
     http://sgdml.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-018-06169-2" target="_blank">
     https://doi.org/10.1038/s41467-018-06169-2
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_o8ssxj2re4kx_0" target="_blank">
     sGDML_Toluene_ccsdt_NC2018_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The train set of a train/test pair from the toluene dataset from sGDML. To create the coupled cluster datasets, the data used for training the models were created by running ab initio MD in the NVT ensemble using the Nosé-Hoover thermostat at 500 K during a 200 ps simulation with a resolution of 0.5 fs. Energies and forces were recalculated using all-electron coupled cluster with single, double and perturbative triple excitations (CCSD(T)). The Dunning correlation-consistent basis set cc-pVDZ was used for malonaldehyde. All calculations were performed with the Psi4 software suite.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    sGDML_Toluene_ccsdt_NC2018_train__Chmiela-Sauceda-Müller-Tkatchenko__DS_o8ssxj2re4kx_0
    <br/>
    <b>
     Name:
    </b>
    sGDML_Toluene_ccsdt_NC2018_train
    <br/>
    <b>
     Authors:
    </b>
    Stefan Chmiela, Huziel E. Sauceda, Klaus-Robert Müller, Alexandre Tkatchenko
    <br/>
    <b>
     Elements
    </b>
    : C, H
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 15,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="http://sgdml.org/" target="_blank">
     http://sgdml.org/
    </a>
    <br/>
    <a href="https://doi.org/10.1038/s41467-018-06169-2" target="_blank">
     https://doi.org/10.1038/s41467-018-06169-2
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_b4o9n4n8b7sp_0" target="_blank">
     solute_strengthening_of_prism_edge_locations_in_Mg_alloys
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset includes Mg and Mg-Zn alloy structures with solute atoms at the prism edge locations. The dataset was created to study the strengthening effect of solute atoms at the prism edge locations in Mg alloys.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    solute_strengthening_of_prism_edge_locations_in_Mg_alloys__Niazi-Curtin__DS_b4o9n4n8b7sp_0
    <br/>
    <b>
     Name:
    </b>
    solute_strengthening_of_prism_edge_locations_in_Mg_alloys
    <br/>
    <b>
     Authors:
    </b>
    Masoud Rahbar Niazi, W. A Curtin
    <br/>
    <b>
     Elements
    </b>
    : Mg, Zn
    <br/>
    <b>
     Number of Configurations
    </b>
    : 94
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 28,615
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:1e-c7" target="_blank">
     https://doi.org/10.24435/materialscloud:1e-c7
    </a>
    <br/>
    <a href="http://doi.org/10.1016/j.euromechsol.2023.105128" target="_blank">
     http://doi.org/10.1016/j.euromechsol.2023.105128
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_ctjgc03xdauc_0" target="_blank">
     solvated_protein_fragments_JCTC_2019
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The solvated protein fragments dataset was generated as a partner benchmark dataset, along with SN2, for measuring the performance of machine learning models, in particular PhysNet, at describing chemical reactions, long-range interactions, and condensed phase systems. The dataset contains structures for all possible "amons" (hydrogen-saturated covalently bonded fragments) of up to eight heavy atoms (C, N, O, S) that can be derived from chemical graphs of proteins containing the 20 natural amino acids connected via peptide bonds or disulfide bridges. For amino acids that can occur in different charge states due to (de)protonation (i.e., carboxylic acids that can be negatively charged or amines that can be positively charged), all possible structures with up to a total charge of +-2e are included. In total, the dataset provides reference energies, forces, and dipole moments for 2,731,180 structures calculated at the revPBE-D3(BJ)/def2-TZVP level of theory using ORCA 4.0.1.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    solvated_protein_fragments_JCTC_2019__Unke-Meuwly__DS_ctjgc03xdauc_0
    <br/>
    <b>
     Name:
    </b>
    solvated_protein_fragments_JCTC_2019
    <br/>
    <b>
     Authors:
    </b>
    Oliver T. Unke, Markus Meuwly
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 2,731,180
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 58,395,272
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.2605372" target="_blank">
     https://doi.org/10.5281/zenodo.2605372
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jctc.9b00181" target="_blank">
     https://doi.org/10.1021/acs.jctc.9b00181
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_qk4qfeka1e3m_0" target="_blank">
     stable_and_metastable_phases_in_sputtered_CuInS2
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The chalcopyrite Cu(In,Ga)S2 has gained renewed interest in recent years due to its potential application in tandem solar cells. In this contribution, a combined theoretical and experimental approach is applied to investigate stable and metastable phases forming in sputtered CuInS2 (CIS) thin films. Ab initio calculations are performed to obtain formation energies, X-ray diffraction patterns, and Raman spectra of various CIS polytypes and related compounds. Multiple low-energy CIS structures with zinc-blende and wurtzite-derived lattices are identified and their XRD/Raman patterns are shown to contain many overlapping features, which could lead to misidentification unless the techniques are duly combined and analyzed. The results are verified against experimental XRD/Raman spectra measured on a series of CIS films with different compositions and treated at different temperatures, revealing the formation of several CIS polymorphs and secondary phases. The characteristic features and the mechanisms behind the formation of different phases are discussed with the focus on the thin-film photovoltaic application of CIS. The dataset contains structures and VASP output files used to derive the discussed trends. version 2
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    stable_and_metastable_phases_in_sputtered_CuInS2__Larsen-Sopiha-Persson-Platzer-Björkman-Edoff__DS_qk4qfeka1e3m_0
    <br/>
    <b>
     Name:
    </b>
    stable_and_metastable_phases_in_sputtered_CuInS2
    <br/>
    <b>
     Authors:
    </b>
    Jes Larsen, Kostiantyn Sopiha, Clas Persson, Charlotte Platzer-Björkman, Marika Edoff
    <br/>
    <b>
     Elements
    </b>
    : Cu, In, Na, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 3,105
    <br/>
    <b>
     Number of Elements
    </b>
    : 4
    <br/>
    <b>
     Number of Atoms
    </b>
    : 117,948
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:5n-1e" target="_blank">
     https://doi.org/10.24435/materialscloud:5n-1e
    </a>
    <br/>
    <a href="http://doi.org/https://doi.org/10.1002/advs.202200848" target="_blank">
     http://doi.org/https://doi.org/10.1002/advs.202200848
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_qde4v70hhmmj_0" target="_blank">
     tmQM_wB97MV
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    tmQM_wB97MV contains configurations from the tmQM dataset, with several structures from tmQM that were found to be missing hydrogens filtered out, and energies of all other structures recomputed at the wB97M-V/def2-SVPD level of DFT.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    tmQM_wB97MV__Garrison-Heras-Domingo-Kitchin-Gomes-Ulissi-Blau__DS_qde4v70hhmmj_0
    <br/>
    <b>
     Name:
    </b>
    tmQM_wB97MV
    <br/>
    <b>
     Authors:
    </b>
    Aaron G. Garrison, Javier Heras-Domingo, John R. Kitchin, Gabriel dos Passos Gomes, Zachary W. Ulissi, Samuel M. Blau
    <br/>
    <b>
     Elements
    </b>
    : Ag, As, Au, B, Br, C, Cd, Cl, Co, Cr, Cu, F, Fe, H, Hf, Hg, I, Ir, La, Mn, Mo, N, Nb, Ni, O, Os, P, Pd, Pt, Re, Rh, Ru, S, Sc, Se, Si, Ta, Tc, Ti, V, W, Y, Zn, Zr
    <br/>
    <b>
     Number of Configurations
    </b>
    : 86,507
    <br/>
    <b>
     Number of Elements
    </b>
    : 44
    <br/>
    <b>
     Number of Atoms
    </b>
    : 5,710,877
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/ulissigroup/tmQM_wB97MV" target="_blank">
     https://github.com/ulissigroup/tmQM_wB97MV
    </a>
    <br/>
    <a href="https://doi.org/10.1021/acs.jcim.3c01226" target="_blank">
     https://doi.org/10.1021/acs.jcim.3c01226
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_4mjnowmrcqib_0" target="_blank">
     vanadium_in_high_entropy_alloys_AM2020
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset created for "Vanadium is an optimal element for strengthening in both fcc and bcc high-entropy alloys", to explore the effect of V in the high-entropy systems fcc Co-Cr-Fe-Mn-Ni-V and bcc Cr-Mo-Nb-Ta-V-W-Hf-Ti-Zr. Structures include pure V, misfit volumes of V in Ni, and misfit volumes of Ni2V random alloys
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    vanadium_in_high_entropy_alloys_AM2020__Yin-Maresca-Curtin__DS_4mjnowmrcqib_0
    <br/>
    <b>
     Name:
    </b>
    vanadium_in_high_entropy_alloys_AM2020
    <br/>
    <b>
     Authors:
    </b>
    Binglun Yin, Francesco Maresca, W. A. Curtin
    <br/>
    <b>
     Elements
    </b>
    : Ni, V
    <br/>
    <b>
     Number of Configurations
    </b>
    : 261
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 22,196
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:2020.0020/v1" target="_blank">
     https://doi.org/10.24435/materialscloud:2020.0020/v1
    </a>
    <br/>
    <a href="http://doi.org/10.1016/j.actamat.2020.01.062" target="_blank">
     http://doi.org/10.1016/j.actamat.2020.01.062
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_kl12pfupgv5e_0" target="_blank">
     water_and_Cu+_synergy_in_selective_CO2_hydrogenation_to_methanol_over_Cu/MgO_catalysts
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    This dataset was created to investigate the role of surface water and hydroxyl groups in facilitating spontaneous CO₂ activation at Cu⁺ sites and the formation of monodentate formate species in the context of using CO2 hydrogenation to produce methanol.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    water_and_Cu+_synergy_in_selective_CO2_hydrogenation_to_methanol_over_Cu/MgO_catalysts__Villanueva-Lustemberg-Zhao-Soriano-Concepción-Pirovano__DS_kl12pfupgv5e_0
    <br/>
    <b>
     Name:
    </b>
    water_and_Cu+_synergy_in_selective_CO2_hydrogenation_to_methanol_over_Cu/MgO_catalysts
    <br/>
    <b>
     Authors:
    </b>
    Estefanía Fernández Villanueva, Pablo Germán Lustemberg, Minjie Zhao, Jose Soriano, Patricia Concepción, María Verónica Ganduglia Pirovano
    <br/>
    <b>
     Elements
    </b>
    : C, Cu, H, Mg, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 14,962
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,043,709
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.24435/materialscloud:tz-pn" target="_blank">
     https://doi.org/10.24435/materialscloud:tz-pn
    </a>
    <br/>
    <a href="https://doi.org/10.1021/jacs.3c10685" target="_blank">
     https://doi.org/10.1021/jacs.3c10685
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_p2aaxa2vfnr6_0" target="_blank">
     water_ice_JCP_2020
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Starting from a single reference ab initio simulation, we use active learning to expand into new state points and to describe the quantum nature of the nuclei. The final model, trained on 814 reference calculations, yields excellent results under a range of conditions, from liquid water at ambient and elevated temperatures and pressures to different phases of ice, and the air-water interface — all including nuclear quantum effects.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    water_ice_JCP_2020__Schran-Brezina-Marsalek__DS_p2aaxa2vfnr6_0
    <br/>
    <b>
     Name:
    </b>
    water_ice_JCP_2020
    <br/>
    <b>
     Authors:
    </b>
    Christoph Schran, Kyrstof Brezina, Ondrej Marsalek
    <br/>
    <b>
     Elements
    </b>
    : H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 8,000
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 2,088,000
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.4004590" target="_blank">
     https://doi.org/10.5281/zenodo.4004590
    </a>
    <br/>
    <a href="https://doi.org/10.1063/5.0016004" target="_blank">
     https://doi.org/10.1063/5.0016004
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_wm2xs5bhyzcx_0" target="_blank">
     water_ice_NEP_2023
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    The main part of the dataset consists of structures of liquid water at 300 K from first-principles molecular dynamics (FPMD) simulations using a hybrid density functional with dispersion corrections. The dataset is expanded to include nuclear quantum effects by adding structures from path-integral molecular dynamics (PIMD) simulations. The final dataset contains 814 structures of liquid water at different temperatures and pressures, water slab, and ice Ih and ice VIII. These systems cover a wide range of structural and dynamical properties of water and ice. This dataset builds on the dataset from Schran, et al (2020) https://doi.org/10.1063/5.0016004
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    water_ice_NEP_2023__Chen-Berrens-Chan-Fan-Donadio__DS_wm2xs5bhyzcx_0
    <br/>
    <b>
     Name:
    </b>
    water_ice_NEP_2023
    <br/>
    <b>
     Authors:
    </b>
    Zekun Chen, Margaret L. Berrens, Kam-Tung Chan, Zheyong Fan, Davide Donadio
    <br/>
    <b>
     Elements
    </b>
    : H, O
    <br/>
    <b>
     Number of Configurations
    </b>
    : 814
    <br/>
    <b>
     Number of Elements
    </b>
    : 2
    <br/>
    <b>
     Number of Atoms
    </b>
    : 216,144
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/ZKC19940412/water_ice_nep" target="_blank">
     https://github.com/ZKC19940412/water_ice_nep
    </a>
    <br/>
    <a href="https://doi.org/10.26434/chemrxiv-2023-sr496" target="_blank">
     https://doi.org/10.26434/chemrxiv-2023-sr496
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_mz0vl4zs7bex_0" target="_blank">
     water_ice_PNAS_2021
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Dataset generated using a committee-based active learning strategy to build a training dataset for modeling complex aqueous systems.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    water_ice_PNAS_2021__Schran-Thiemann-Rowe-Müller-Marsalek-Michaelides__DS_mz0vl4zs7bex_0
    <br/>
    <b>
     Name:
    </b>
    water_ice_PNAS_2021
    <br/>
    <b>
     Authors:
    </b>
    Christoph Schran, Fabian L. Thiemann, Patrick Rowe, Erich A. Müller, Ondrej Marsalek, Angelos Michaelides
    <br/>
    <b>
     Elements
    </b>
    : B, C, F, H, Mo, N, O, S, Ti
    <br/>
    <b>
     Number of Configurations
    </b>
    : 1,786
    <br/>
    <b>
     Number of Elements
    </b>
    : 9
    <br/>
    <b>
     Number of Atoms
    </b>
    : 681,912
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://doi.org/10.5281/zenodo.5235246" target="_blank">
     https://doi.org/10.5281/zenodo.5235246
    </a>
    <br/>
    <a href="https://doi.org/10.1073/pnas.2110077118" target="_blank">
     https://doi.org/10.1073/pnas.2110077118
    </a>
    <br/>
    <a href="https://github.com/water-ice-group/simple-MLP" target="_blank">
     https://github.com/water-ice-group/simple-MLP
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_sytg0385f79a_0" target="_blank">
     xxMD-CASSCF_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test dataset from xxMD-CASSCF. The xxMD (Extended Excited-state Molecular Dynamics) dataset is a comprehensive collection of non-adiabatic trajectories encompassing several photo-sensitive molecules. This dataset challenges existing Neural Force Field (NFF) models with broader nuclear configuration spaces that span reactant, transition state, product, and conical intersection regions, making it more chemically representative than its contemporaries. xxMD is divided into two datasets, each with corresponding train, test and validation splits. xxMD-CASSCF contains calculations generated using state-averaged complete active state self-consistent field (SA-CASSCF) electronic theory. xxMD-DFT contains recalculated single-point spin-polarized (unrestricted) DFT values.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    xxMD-CASSCF_test__Pengmei-Shu-Liu__DS_sytg0385f79a_0
    <br/>
    <b>
     Name:
    </b>
    xxMD-CASSCF_test
    <br/>
    <b>
     Authors:
    </b>
    Zihan Pengmei, Yinan Shu, Junyu Liu
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 65,100
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 707,552
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/zpengmei/xxMD" target="_blank">
     https://github.com/zpengmei/xxMD
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2308.11155" target="_blank">
     https://doi.org/10.48550/arXiv.2308.11155
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_qvljl3vtdp0f_0" target="_blank">
     xxMD-CASSCF_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training dataset from xxMD-CASSCF. The xxMD (Extended Excited-state Molecular Dynamics) dataset is a comprehensive collection of non-adiabatic trajectories encompassing several photo-sensitive molecules. This dataset challenges existing Neural Force Field (NFF) models with broader nuclear configuration spaces that span reactant, transition state, product, and conical intersection regions, making it more chemically representative than its contemporaries. xxMD is divided into two datasets, each with corresponding train, test and validation splits. xxMD-CASSCF contains calculations generated using state-averaged complete active state self-consistent field (SA-CASSCF) electronic theory. xxMD-DFT contains recalculated single-point spin-polarized (unrestricted) DFT values.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    xxMD-CASSCF_train__Pengmei-Shu-Liu__DS_qvljl3vtdp0f_0
    <br/>
    <b>
     Name:
    </b>
    xxMD-CASSCF_train
    <br/>
    <b>
     Authors:
    </b>
    Zihan Pengmei, Yinan Shu, Junyu Liu
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 130,179
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 1,411,656
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/zpengmei/xxMD" target="_blank">
     https://github.com/zpengmei/xxMD
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2308.11155" target="_blank">
     https://doi.org/10.48550/arXiv.2308.11155
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_e0nl35x6dl85_0" target="_blank">
     xxMD-CASSCF_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Validation dataset from xxMD-CASSCF. The xxMD (Extended Excited-state Molecular Dynamics) dataset is a comprehensive collection of non-adiabatic trajectories encompassing several photo-sensitive molecules. This dataset challenges existing Neural Force Field (NFF) models with broader nuclear configuration spaces that span reactant, transition state, product, and conical intersection regions, making it more chemically representative than its contemporaries. xxMD is divided into two datasets, each with corresponding train, test and validation splits. xxMD-CASSCF contains calculations generated using state-averaged complete active state self-consistent field (SA-CASSCF) electronic theory. xxMD-DFT contains recalculated single-point spin-polarized (unrestricted) DFT values.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    xxMD-CASSCF_validation__Pengmei-Shu-Liu__DS_e0nl35x6dl85_0
    <br/>
    <b>
     Name:
    </b>
    xxMD-CASSCF_validation
    <br/>
    <b>
     Authors:
    </b>
    Zihan Pengmei, Yinan Shu, Junyu Liu
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 64,848
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 701,516
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/zpengmei/xxMD" target="_blank">
     https://github.com/zpengmei/xxMD
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2308.11155" target="_blank">
     https://doi.org/10.48550/arXiv.2308.11155
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_q007ninsdjvf_0" target="_blank">
     xxMD-DFT_test
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Test dataset from xxMD-DFT. The xxMD (Extended Excited-state Molecular Dynamics) dataset is a comprehensive collection of non-adiabatic trajectories encompassing several photo-sensitive molecules. This dataset challenges existing Neural Force Field (NFF) models with broader nuclear configuration spaces that span reactant, transition state, product, and conical intersection regions, making it more chemically representative than its contemporaries. xxMD is divided into two datasets, each with corresponding train, test and validation splits. xxMD-CASSCF contains calculations generated using state-averaged complete active state self-consistent field (SA-CASSCF) electronic theory. xxMD-DFT contains recalculated single-point spin-polarized (unrestricted) DFT values.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    xxMD-DFT_test__Pengmei-Shu-Liu__DS_q007ninsdjvf_0
    <br/>
    <b>
     Name:
    </b>
    xxMD-DFT_test
    <br/>
    <b>
     Authors:
    </b>
    Zihan Pengmei, Yinan Shu, Junyu Liu
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 21,661
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 402,856
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/zpengmei/xxMD" target="_blank">
     https://github.com/zpengmei/xxMD
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2308.11155" target="_blank">
     https://doi.org/10.48550/arXiv.2308.11155
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="">
    <a href="https://materials.colabfit.org/id/DS_fblez5twbr07_0" target="_blank">
     xxMD-DFT_train
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Training dataset from xxMD-DFT. The xxMD (Extended Excited-state Molecular Dynamics) dataset is a comprehensive collection of non-adiabatic trajectories encompassing several photo-sensitive molecules. This dataset challenges existing Neural Force Field (NFF) models with broader nuclear configuration spaces that span reactant, transition state, product, and conical intersection regions, making it more chemically representative than its contemporaries. xxMD is divided into two datasets, each with corresponding train, test and validation splits. xxMD-CASSCF contains calculations generated using state-averaged complete active state self-consistent field (SA-CASSCF) electronic theory. xxMD-DFT contains recalculated single-point spin-polarized (unrestricted) DFT values.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    xxMD-DFT_train__Pengmei-Shu-Liu__DS_fblez5twbr07_0
    <br/>
    <b>
     Name:
    </b>
    xxMD-DFT_train
    <br/>
    <b>
     Authors:
    </b>
    Zihan Pengmei, Yinan Shu, Junyu Liu
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 43,395
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 807,416
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/zpengmei/xxMD" target="_blank">
     https://github.com/zpengmei/xxMD
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2308.11155" target="_blank">
     https://doi.org/10.48550/arXiv.2308.11155
    </a>
    <br/>
   </div>
  </div>
  <div class="card px-0" style="margin-bottom: 2em;">
   <h5 class="card-header" style="background-color: #eef;">
    <a href="https://materials.colabfit.org/id/DS_d1nmrzy4csx1_0" target="_blank">
     xxMD-DFT_validation
    </a>
   </h5>
   <div class="card-body">
    <b>
     Description:
    </b>
    Validation dataset from xxMD-DFT. The xxMD (Extended Excited-state Molecular Dynamics) dataset is a comprehensive collection of non-adiabatic trajectories encompassing several photo-sensitive molecules. This dataset challenges existing Neural Force Field (NFF) models with broader nuclear configuration spaces that span reactant, transition state, product, and conical intersection regions, making it more chemically representative than its contemporaries. xxMD is divided into two datasets, each with corresponding train, test and validation splits. xxMD-CASSCF contains calculations generated using state-averaged complete active state self-consistent field (SA-CASSCF) electronic theory. xxMD-DFT contains recalculated single-point spin-polarized (unrestricted) DFT values.
    <br/>
    <br/>
    <b>
     ColabFit ID:
    </b>
    xxMD-DFT_validation__Pengmei-Shu-Liu__DS_d1nmrzy4csx1_0
    <br/>
    <b>
     Name:
    </b>
    xxMD-DFT_validation
    <br/>
    <b>
     Authors:
    </b>
    Zihan Pengmei, Yinan Shu, Junyu Liu
    <br/>
    <b>
     Elements
    </b>
    : C, H, N, O, S
    <br/>
    <b>
     Number of Configurations
    </b>
    : 21,606
    <br/>
    <b>
     Number of Elements
    </b>
    : 5
    <br/>
    <b>
     Number of Atoms
    </b>
    : 402,151
    <br/>
    <!--<b>Property Types</b>: <br>-->
    <br/>
    <b>
     Links:
    </b>
    <br/>
    <a href="https://github.com/zpengmei/xxMD" target="_blank">
     https://github.com/zpengmei/xxMD
    </a>
    <br/>
    <a href="https://doi.org/10.48550/arXiv.2308.11155" target="_blank">
     https://doi.org/10.48550/arXiv.2308.11155
    </a>
    <br/>
   </div>
  </div>
 </body>
</html>
